require_relative 'test_case'

class RI::TestResouce < RI::TestCase
  def setup
    @client = ResourceIndex::Base.new(adapter: "amalgalite", username: "test", password: "test", database: "")
    @client.db.create_table :obr_resource do
      primary_key :id
      String :name
      String :resource_id
      String :structure
      String :main_context
      String :url
      String :element_url
      String :description
      String :logo
      Integer :dictionary_id
      Integer :total_element
      Time :last_update_date
      Time :workflow_completed_date
    end
    @client.db.run(TEST_DATA)
  end

  def test_resource
    res = @client.resource("PGDR")
    assert_equal "PGDR", res.acronym
    assert_equal "PharmGKB [Drug]", res.name
    res = @client.resource("AE")
    assert_equal "AE", res.acronym
    assert_equal "ArrayExpress", res.name
  end

  end

  def test_resources
    res = @client.resources
    assert_equal 10, res.length
    known_resource_ids = ["AE", "CT", "GM", "OMIM", "CDD", "PGDI", "PGDR", "PGGE", "REAC", "UPKB"].sort
    known_names = [
      "ArrayExpress",
      "ClinicalTrials.gov",
      "ARRS GoldMiner",
      "Online Mendelian Inheritance in Man",
      "Conserved Domain Database (CDD)",
      "PharmGKB [Disease]",
      "PharmGKB [Drug]",
      "PharmGKB [Gene]",
      "Reactome",
      "UniProt KB"
    ].sort
    resource_ids = res.map {|r| r.acronym}.sort
    names = res.map {|r| r.name}.sort
    assert_equal known_resource_ids, resource_ids
    assert_equal known_names, names
  end

TEST_DATA = <<-EOS
INSERT INTO `obr_resource` (`id`, `name`, `resource_id`, `structure`, `main_context`, `url`, `element_url`, `description`, `logo`, `dictionary_id`, `total_element`, `last_update_date`, `workflow_completed_date`)
VALUES
  (1, 'ArrayExpress', 'AE', '<obs.obr.populate.Structure>\n  <resourceId>AE</resourceId>\n  <contexts>\n    <entry>\n      <string>AE_name</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>AE_description</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>AE_species</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>AE_experiment_type</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>name</string>\n    <string>description</string>\n    <string>species</string>\n    <string>experiment_type</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>AE_name</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>AE_description</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>AE_species</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>AE_experiment_type</string>\n      <double>0.9</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>AE_name</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>AE_description</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>AE_species</string>\n      <string>1132</string>\n    </entry>\n    <entry>\n      <string>AE_experiment_type</string>\n      <string>1136</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'AE_name', 'http://www.ebi.ac.uk/arrayexpress/', 'http://www.ebi.ac.uk/arrayexpress/experiments/', 'ArrayExpress is a public repository for microarray data, which is aimed at storing MIAME-compliant data in accordance with MGED recommendations. The ArrayExpress Data Warehouse stores gene-indexed expression profiles from a curated subset of experiments in the repository.', 'http://www.ebi.ac.uk/microarray-as/aer/include/aelogo.png', 23, 35877, '2013-02-05 21:35:46', '2013-09-26 05:13:05'),
  (2, 'ClinicalTrials.gov', 'CT', '<obs.obr.populate.Structure>\n  <resourceId>CT</resourceId>\n  <contexts>\n    <entry>\n      <string>CT_intervention</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_official_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_brief_summary</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_condition</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_brief_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_detailed_description</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>brief_title</string>\n    <string>official_title</string>\n    <string>brief_summary</string>\n    <string>detailed_description</string>\n    <string>condition</string>\n    <string>intervention</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>CT_intervention</string>\n      <double>0.6</double>\n    </entry>\n    <entry>\n      <string>CT_official_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>CT_brief_summary</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>CT_condition</string>\n      <double>0.6</double>\n    </entry>\n    <entry>\n      <string>CT_brief_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>CT_detailed_description</string>\n      <double>0.7</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>CT_intervention</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_official_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_brief_summary</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_condition</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_brief_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_detailed_description</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'CT_brief_title', 'http://clinicaltrials.gov/', 'http://clinicaltrials.gov/ct/show/', 'ClinicalTrials.gov provides regularly updated information about federally and privately supported clinical research in human volunteers. ClinicalTrials.gov gives you information about a trial''s purpose, who may participate, locations, and phone numbers for more details. The information provided on ClinicalTrials.gov should be used in conjunction with advice from health care professionals. Before searching, you may want to learn more about clinical trials.', 'http://clinicaltrials.gov/ct2/html/images/frame/title.gif', 23, 122591, '2012-03-08 21:00:13', '2013-10-02 07:18:39'),
  (3, 'ARRS GoldMiner', 'GM', '<obs.obr.populate.Structure>\n  <resourceId>GM</resourceId>\n  <contexts>\n    <entry>\n      <string>GM_caption</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>GM_title</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>title</string>\n    <string>caption</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>GM_caption</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>GM_title</string>\n      <double>1.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>GM_caption</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>GM_title</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'GM_title', 'http://goldminer.arrs.org/', 'http://goldminer.arrs.org/link.php?id=', 'ARRS GoldMiner provides instant access to images published in selected peer-reviewed radiology journals. This new, web-based system allows viewers to search for images by findings, anatomy, imaging technique, and patient age and sex.', 'http://goldminer.arrs.org/image/goldminer.gif', 23, 1155, '2012-03-06 05:56:19', '2013-09-30 22:08:42'),
  (4, 'Online Mendelian Inheritance in Man', 'OMIM', '<obs.obr.populate.Structure>\n  <resourceId>OMIM</resourceId>\n  <contexts>\n    <entry>\n      <string>OMIM_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>OMIM_locus</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>OMIM_uid</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>OMIM_altTitles</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>uid</string>\n    <string>title</string>\n    <string>altTitles</string>\n    <string>locus</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>OMIM_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>OMIM_locus</string>\n      <double>0.0</double>\n    </entry>\n    <entry>\n      <string>OMIM_uid</string>\n      <double>0.0</double>\n    </entry>\n    <entry>\n      <string>OMIM_altTitles</string>\n      <double>0.7</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>OMIM_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>OMIM_locus</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>OMIM_uid</string>\n      <string>-1</string>\n    </entry>\n    <entry>\n      <string>OMIM_altTitles</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'OMIM_title', 'http://www.ncbi.nlm.nih.gov/omim', 'http://www.ncbi.nlm.nih.gov/omim/', 'OMIM is a comprehensive, authoritative, and timely compendium of human genes and genetic phenotypes. The full-text, referenced overviews in OMIM contain information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype.', 'http://www.ncbi.nlm.nih.gov/entrez/query/static/gifs/entrez_omim.gif', 23, 22186, '2012-03-09 03:49:31', '2013-10-23 05:30:26'),
  (5, 'Conserved Domain Database (CDD)', 'CDD', '<obs.obr.populate.Structure>\n  <resourceId>CDD</resourceId>\n  <contexts>\n    <entry>\n      <string>CDD_abstract</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CDD_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CDD_uid</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>uid</string>\n    <string>title</string>\n    <string>abstract</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>CDD_abstract</string>\n      <double>0.6</double>\n    </entry>\n    <entry>\n      <string>CDD_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>CDD_uid</string>\n      <double>0.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>CDD_abstract</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CDD_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CDD_uid</string>\n      <string>-1</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'CDD_title', 'http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml', 'http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=', 'The Conserved Domain Database (CDD) contains protein domain models imported from outside sources, such as Pfam and SMART, and curated at NCBI. CDD contains over 12,000 such models and is linked to other NCBI databases, including protein sequences, bibliographic citations, and taxonomy.', 'http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd2.gif', 23, 46346, '2012-03-09 03:25:52', '2013-10-23 02:03:22'),
  (6, 'PharmGKB [Disease]', 'PGDI', '<obs.obr.populate.Structure>\n  <resourceId>PGDI</resourceId>\n  <contexts>\n    <entry>\n      <string>PGDI_diseaseRelatedGenes</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPathways</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPhenotypeDatasets</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedDrugs</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseAlternateNames</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>diseaseName</string>\n    <string>diseaseAlternateNames</string>\n    <string>diseaseRelatedGenes</string>\n    <string>diseaseRelatedPathways</string>\n    <string>diseaseRelatedDrugs</string>\n    <string>diseaseRelatedPhenotypeDatasets</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>PGDI_diseaseRelatedGenes</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPathways</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPhenotypeDatasets</string>\n      <double>0.4</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseName</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedDrugs</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseAlternateNames</string>\n      <double>0.8</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>PGDI_diseaseRelatedGenes</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPathways</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPhenotypeDatasets</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedDrugs</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseAlternateNames</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'PGDI_diseaseName', 'http://www.pharmgkb.org/', 'https://www.pharmgkb.org/do/serve?objId=', 'PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.', 'https://www.pharmgkb.org/images/header/title.png', 23, 837, '2012-03-05 22:28:11', '2013-09-25 23:02:14'),
  (7, 'PharmGKB [Drug]', 'PGDR', '<obs.obr.populate.Structure>\n  <resourceId>PGDR</resourceId>\n  <contexts>\n    <entry>\n      <string>PGDR_drugRelatedPhenotypeDatasets</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugTradeNames</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedDiseases</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedGenes</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedPathways</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugGenericNames</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>drugName</string>\n    <string>drugTradeNames</string>\n    <string>drugGenericNames</string>\n    <string>drugRelatedGenes</string>\n    <string>drugRelatedPathways</string>\n    <string>drugRelatedDiseases</string>\n    <string>drugRelatedPhenotypeDatasets</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>PGDR_drugRelatedPhenotypeDatasets</string>\n      <double>0.4</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugTradeNames</string>\n      <double>0.9</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedDiseases</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugName</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedGenes</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedPathways</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugGenericNames</string>\n      <double>0.9</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>PGDR_drugRelatedPhenotypeDatasets</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugTradeNames</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedDiseases</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedGenes</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedPathways</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugGenericNames</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'PGDR_drugName', 'http://www.pharmgkb.org/', 'https://www.pharmgkb.org/do/serve?objId=', 'PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.', 'https://www.pharmgkb.org/images/header/title.png', 23, 1643, '2012-03-05 22:28:10', '2013-09-25 22:55:30'),
  (8, 'PharmGKB [Gene]', 'PGGE', '<obs.obr.populate.Structure>\n  <resourceId>PGGE</resourceId>\n  <contexts>\n    <entry>\n      <string>PGGE_geneName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneSymbol</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneAlternateSymbols</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDiseases</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPathways</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDrugs</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPhenotypeDatasets</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>geneName</string>\n    <string>geneSymbol</string>\n    <string>geneAlternateSymbols</string>\n    <string>geneRelatedDrugs</string>\n    <string>geneRelatedPathways</string>\n    <string>geneRelatedDiseases</string>\n    <string>geneRelatedPhenotypeDatasets</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>PGGE_geneName</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneSymbol</string>\n      <double>0.9</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneAlternateSymbols</string>\n      <double>0.9</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDiseases</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPathways</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDrugs</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPhenotypeDatasets</string>\n      <double>0.4</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>PGGE_geneName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneSymbol</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneAlternateSymbols</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDiseases</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPathways</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDrugs</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPhenotypeDatasets</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'PGGE_geneName', 'http://www.pharmgkb.org/', 'https://www.pharmgkb.org/do/serve?objId=', 'PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.', 'https://www.pharmgkb.org/images/header/title.png', 23, 1242, '2012-01-24 01:50:10', '2013-09-25 22:59:06'),
  (9, 'Reactome', 'REAC', '<obs.obr.populate.Structure>\n  <resourceId>REAC</resourceId>\n  <contexts>\n    <entry>\n      <string>REAC_goCellCompartiment</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>REAC_participants</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>REAC_goBiologicalProcess</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>REAC_name</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>name</string>\n    <string>participants</string>\n    <string>goBiologicalProcess</string>\n    <string>goCellCompartiment</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>REAC_goCellCompartiment</string>\n      <double>0.2</double>\n    </entry>\n    <entry>\n      <string>REAC_participants</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>REAC_goBiologicalProcess</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>REAC_name</string>\n      <double>1.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>REAC_goCellCompartiment</string>\n      <string>1070</string>\n    </entry>\n    <entry>\n      <string>REAC_participants</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>REAC_goBiologicalProcess</string>\n      <string>1070</string>\n    </entry>\n    <entry>\n      <string>REAC_name</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'REAC_name', 'http://www.reactome.org/', 'http://www.reactome.org/cgi-bin/eventbrowser?DB=gk_current&ID=', 'A curated knowledgebase of biological pathways.', 'http://www.reactome.org/icons/R-purple.png', 23, 208093, '2012-03-08 00:23:55', '2013-10-03 21:30:23'),
  (10, 'UniProt KB', 'UPKB', '<obs.obr.populate.Structure>\n  <resourceId>UPKB</resourceId>\n  <contexts>\n    <entry>\n      <string>UPKB_naturalVariant</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_goAnnotationList</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_geneSymbol</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_cellularComponent</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_proteinName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_molecularFunction</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_mutagenesis</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_biologicalProcess</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_organism</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>geneSymbol</string>\n    <string>goAnnotationList</string>\n    <string>proteinName</string>\n    <string>organism</string>\n    <string>naturalVariant</string>\n    <string>mutagenesis</string>\n    <string>biologicalProcess</string>\n    <string>cellularComponent</string>\n    <string>molecularFunction</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>UPKB_naturalVariant</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_goAnnotationList</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>UPKB_geneSymbol</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>UPKB_cellularComponent</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_proteinName</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_molecularFunction</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_mutagenesis</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_biologicalProcess</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_organism</string>\n      <double>0.8</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>UPKB_naturalVariant</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_goAnnotationList</string>\n      <string>1070</string>\n    </entry>\n    <entry>\n      <string>UPKB_geneSymbol</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_cellularComponent</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_proteinName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_molecularFunction</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_mutagenesis</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_biologicalProcess</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_organism</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'UPKB_geneSymbol', 'http://www.uniprot.org/', 'http://www.uniprot.org/uniprot/', 'The mission of UniProt is to provide the scientific community with a comprehensive, high-quality and freely accessible resource of protein sequence and functional information.', 'http://www.uniprot.org/images/logo.gif', 23, 533515, '2012-02-27 02:41:23', '2013-11-03 16:12:14');
EOS
end