##
# Test data for use in populations so we don't need redis, mgrep, 4store, etc

# This is the dictionary used to mock MGREP responses (label => string_id)
DICTIONARY = {"WITCH" => 1659609209, "WEST" => 4234099798, "EAST" => 21367710, "SORCERER" => 2571402480}

# This is a hash used to mock the annotator cache (string id => array of ontology acronym/class id pairs)
LABEL_ID_TO_CLASS_MAP = {
  1659609209 => [
    ["NCBITAXON", "http://purl.obolibrary.org/obo/NCBITaxon_34819"],
    ["VTO", "http://purl.obolibrary.org/obo/VTO_0055684"],
    ["ATMO", "http://purl.obolibrary.org/obo/ATM_00010"]
  ],
  4234099798 => [
    ["NCIT", "http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C45852"]
  ],
  21367710 => [
    ["CCO", "http://purl.obolibrary.org/obo/NCBIGene_46006"],
    ["NCIT", "http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C45851"]
  ],
  2571402480 => [
    ["ATMO", "http://purl.obolibrary.org/obo/ATM_00010"]
  ]
}

# A list of all of the XXHASH ids produced by the classes from above
# For the purposes of the tests, these classes and their ancestors are the only classes in BioPortal
CLASS_XXHASH = [2135716011, 2957120221, 921784164, 3305416963, 560039333, 1829708204]

# The ancestors for the above classes
XXHASH_TO_ANCESTOR_XXHASH = {
  2135716011 =>
    [232061624, 1230672067, 1163287331, 3859148251, 4203259731,
     693853866, 2563377719, 607172277, 1669326367, 1998261360, 1373969471,
     2438883063, 3961879054, 3799322241, 1340533539, 1548750669, 1030780602,
     3271036144, 2927342877, 2147605913, 3832956476, 3836248483, 1366574731,
     1501807657, 1846285280, 3343464566, 43277846, 821166383, 675582466,
     3917043077, 1872022469, 2730638237, 784370039, 3545131324,
     1154873594, 1937902181, 3834913715],
  2957120221 =>
    [3236881980, 2366864211, 2343622782, 587609268, 127928309, 1188046131,
     2416684808, 988732284, 228868388, 31435392, 3768856267, 3647089701,
     3224761600, 4208738311, 1965813958, 1475106836, 1777292012, 2511237278],
  921784164 =>
    [3093627464],
  3305416963 =>
    [2712116084, 3483783616, 3550205860, 418441167, 823727635],
  560039333 =>
    [734407967, 1197614048, 1777281735, 3102301245, 1090975899, 869965023],
  1829708204 =>
    [2712116084, 3483783616, 3550205860, 418441167, 823727635]
}

DIRECT_ANNOTATION_COUNTS = {2135716011=>3, 2957120221=>3, 921784164=>4, 3305416963=>3, 560039333=>1, 1829708204=>1}
ANCESTOR_ANNOTATION_COUNTS = {232061624=>3,  1230672067=>3,  1163287331=>3,  3859148251=>3,  4203259731=>3,  693853866=>3,  2563377719=>3,  607172277=>3,  1669326367=>3,  1998261360=>3,  1373969471=>3,  2438883063=>3,  3961879054=>3,  3799322241=>3,  1340533539=>3,  1548750669=>3,  1030780602=>3,  3271036144=>3,  2927342877=>3,  2147605913=>3,  3832956476=>3,  3836248483=>3,  1366574731=>3,  1501807657=>3,  1846285280=>3,  3343464566=>3,  43277846=>3,  821166383=>3,  675582466=>3,  3917043077=>3,  1872022469=>3,  2730638237=>3,  784370039=>3,  3545131324=>3,  1154873594=>3,  1937902181=>3,  3834913715=>3,  3236881980=>3,  2366864211=>3,  2343622782=>3,  587609268=>3,  127928309=>3,  1188046131=>3,  2416684808=>3,  988732284=>3,  228868388=>3,  31435392=>3,  3768856267=>3,  3647089701=>3,  3224761600=>3,  4208738311=>3,  1965813958=>3,  1475106836=>3,  1777292012=>3,  2511237278=>3,  3093627464=>4,  2712116084=>4,  3483783616=>4,  3550205860=>4,  418441167=>4,  823727635=>4,  734407967=>1,  1197614048=>1,  1777281735=>1,  3102301245=>1,  1090975899=>1,  869965023=>1}
MANUAL_ANNOTATION_XXHASH = [2471850823, 4174731843]

# A list of the class XXHASHes and the documents that contain them
XXHASH_TO_DOCS = {
  2135716011=>["1", "2", "3"],
  2957120221=>["1", "2", "3"],
  921784164=>["1", "2", "3", "4"],
  3305416963=>["1", "3", "4"],
  560039333=>["2"],
  1829708204=>["2"]
}

# A list of the ancestors for the XXHASH classes and the documents that contain them
ANCESTOR_XXHASH_TO_DOCS = {232061624=>["1", "2", "3"], 1230672067=>["1", "2", "3"], 1163287331=>["1", "2", "3"], 3859148251=>["1", "2", "3"], 4203259731=>["1", "2", "3"], 693853866=>["1", "2", "3"], 2563377719=>["1", "2", "3"], 607172277=>["1", "2", "3"], 1669326367=>["1", "2", "3"], 1998261360=>["1", "2", "3"], 1373969471=>["1", "2", "3"], 2438883063=>["1", "2", "3"], 3961879054=>["1", "2", "3"], 3799322241=>["1", "2", "3"], 1340533539=>["1", "2", "3"], 1548750669=>["1", "2", "3"], 1030780602=>["1", "2", "3"], 3271036144=>["1", "2", "3"], 2927342877=>["1", "2", "3"], 2147605913=>["1", "2", "3"], 3832956476=>["1", "2", "3"], 3836248483=>["1", "2", "3"], 1366574731=>["1", "2", "3"], 1501807657=>["1", "2", "3"], 1846285280=>["1", "2", "3"], 3343464566=>["1", "2", "3"], 43277846=>["1", "2", "3"], 821166383=>["1", "2", "3"], 675582466=>["1", "2", "3"], 3917043077=>["1", "2", "3"], 1872022469=>["1", "2", "3"], 2730638237=>["1", "2", "3"], 784370039=>["1", "2", "3"], 3545131324=>["1", "2", "3"], 1154873594=>["1", "2", "3"], 1937902181=>["1", "2", "3"], 3834913715=>["1", "2", "3"], 3236881980=>["1", "2", "3"], 2366864211=>["1", "2", "3"], 2343622782=>["1", "2", "3"], 587609268=>["1", "2", "3"], 127928309=>["1", "2", "3"], 1188046131=>["1", "2", "3"], 2416684808=>["1", "2", "3"], 988732284=>["1", "2", "3"], 228868388=>["1", "2", "3"], 31435392=>["1", "2", "3"], 3768856267=>["1", "2", "3"], 3647089701=>["1", "2", "3"], 3224761600=>["1", "2", "3"], 4208738311=>["1", "2", "3"], 1965813958=>["1", "2", "3"], 1475106836=>["1", "2", "3"], 1777292012=>["1", "2", "3"], 2511237278=>["1", "2", "3"], 3093627464=>["1", "2", "3", "4"], 2712116084=>["1", "2", "3", "4"], 3483783616=>["1", "2", "3", "4"], 3550205860=>["1", "2", "3", "4"], 418441167=>["1", "2", "3", "4"], 823727635=>["1", "2", "3", "4"], 734407967=>["2"], 1197614048=>["2"], 1777281735=>["2"], 3102301245=>["2"], 1090975899=>["2"], 869965023=>["2"]}


# The inserts in this file use UNION SELECT because earlier versions of SQLite don't support VALUE inserts
# Don't revert this change unless you are sure your test env supports newer SQLite versions (Jruby 1.7.12 does not, for example)

ONTOLOGIES_TABLE = <<-EOS
INSERT INTO 'obs_ontology'
  SELECT 1318 AS 'id', '47845' AS 'local_ontology_id', 'NCBI organismal classification' AS 'name', '1.2' AS 'version', '' AS 'description', 28 AS 'status', '1132' AS 'virtual_ontology_id', 'OWL' AS 'format', 21 AS 'dictionary_id'
EOS

CONCEPTS_TABLE = <<-EOS
INSERT INTO 'obs_concept'
  SELECT 38776185 AS 'id', '47845/obo:NCBITaxon_10090' AS 'local_concept_id', 1318 AS 'ontology_id', 0 AS 'is_toplevel', 'http://purl.obolibrary.org/obo/NCBITaxon_10090' AS 'full_id'
  UNION SELECT 38778061, '47845/obo:NCBITaxon_10116', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_10116'
  UNION SELECT 38869745, '47845/obo:NCBITaxon_119951', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_119951'
  UNION SELECT 38878148, '47845/obo:NCBITaxon_128735', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_128735'
  UNION SELECT 38906355, '47845/obo:NCBITaxon_1590', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_1590'
EOS

RESOURCES_TEST_DATA = <<-EOS
INSERT INTO 'obr_resource'
  SELECT 1 AS 'id', 'ArrayExpress' AS 'name', 'AE' AS 'resource_id', '<obs.obr.populate.Structure>\n  <resourceId>AE</resourceId>\n  <contexts>\n    <entry>\n      <string>AE_name</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>AE_description</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>AE_species</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>AE_experiment_type</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>name</string>\n    <string>description</string>\n    <string>species</string>\n    <string>experiment_type</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>AE_name</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>AE_description</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>AE_species</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>AE_experiment_type</string>\n      <double>0.9</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>AE_name</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>AE_description</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>AE_species</string>\n      <string>1132</string>\n    </entry>\n    <entry>\n      <string>AE_experiment_type</string>\n      <string>1136</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>' AS 'structure', 'AE_name' AS 'main_context', 'http://www.ebi.ac.uk/arrayexpress/' AS 'url', 'http://www.ebi.ac.uk/arrayexpress/experiments/' AS 'element_url', 'ArrayExpress is a public repository for microarray data, which is aimed at storing MIAME-compliant data in accordance with MGED recommendations. The ArrayExpress Data Warehouse stores gene-indexed expression profiles from a curated subset of experiments in the repository.' AS 'description', 'http://www.ebi.ac.uk/microarray-as/aer/include/aelogo.png' AS 'logo', 23 AS 'dictionary_id', 35877 AS 'total_element', '2013-02-05 21:35:46' AS 'last_update_date', '2013-09-26 05:13:05' AS 'workflow_completed_date'
  UNION SELECT 2, 'ClinicalTrials.gov', 'CT', '<obs.obr.populate.Structure>\n  <resourceId>CT</resourceId>\n  <contexts>\n    <entry>\n      <string>CT_intervention</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_official_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_brief_summary</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_condition</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_brief_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_detailed_description</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>brief_title</string>\n    <string>official_title</string>\n    <string>brief_summary</string>\n    <string>detailed_description</string>\n    <string>condition</string>\n    <string>intervention</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>CT_intervention</string>\n      <double>0.6</double>\n    </entry>\n    <entry>\n      <string>CT_official_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>CT_brief_summary</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>CT_condition</string>\n      <double>0.6</double>\n    </entry>\n    <entry>\n      <string>CT_brief_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>CT_detailed_description</string>\n      <double>0.7</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>CT_intervention</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_official_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_brief_summary</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_condition</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_brief_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_detailed_description</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'CT_brief_title', 'http://clinicaltrials.gov/', 'http://clinicaltrials.gov/ct/show/', 'ClinicalTrials.gov provides regularly updated information about federally and privately supported clinical research in human volunteers. ClinicalTrials.gov gives you information about a trial''s purpose, who may participate, locations, and phone numbers for more details. The information provided on ClinicalTrials.gov should be used in conjunction with advice from health care professionals. Before searching, you may want to learn more about clinical trials.', 'http://clinicaltrials.gov/ct2/html/images/frame/title.gif', 23, 122591, '2012-03-08 21:00:13', '2013-10-02 07:18:39'
  UNION SELECT 3, 'ARRS GoldMiner', 'GM', '<obs.obr.populate.Structure>\n  <resourceId>GM</resourceId>\n  <contexts>\n    <entry>\n      <string>GM_caption</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>GM_title</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>title</string>\n    <string>caption</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>GM_caption</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>GM_title</string>\n      <double>1.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>GM_caption</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>GM_title</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'GM_title', 'http://goldminer.arrs.org/', 'http://goldminer.arrs.org/link.php?id=', 'ARRS GoldMiner provides instant access to images published in selected peer-reviewed radiology journals. This new, web-based system allows viewers to search for images by findings, anatomy, imaging technique, and patient age and sex.', 'http://goldminer.arrs.org/image/goldminer.gif', 23, 1155, '2012-03-06 05:56:19', '2013-09-30 22:08:42'
  UNION SELECT 4, 'Online Mendelian Inheritance in Man', 'OMIM', '<obs.obr.populate.Structure>\n  <resourceId>OMIM</resourceId>\n  <contexts>\n    <entry>\n      <string>OMIM_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>OMIM_locus</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>OMIM_uid</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>OMIM_altTitles</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>uid</string>\n    <string>title</string>\n    <string>altTitles</string>\n    <string>locus</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>OMIM_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>OMIM_locus</string>\n      <double>0.0</double>\n    </entry>\n    <entry>\n      <string>OMIM_uid</string>\n      <double>0.0</double>\n    </entry>\n    <entry>\n      <string>OMIM_altTitles</string>\n      <double>0.7</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>OMIM_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>OMIM_locus</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>OMIM_uid</string>\n      <string>-1</string>\n    </entry>\n    <entry>\n      <string>OMIM_altTitles</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'OMIM_title', 'http://www.ncbi.nlm.nih.gov/omim', 'http://www.ncbi.nlm.nih.gov/omim/', 'OMIM is a comprehensive, authoritative, and timely compendium of human genes and genetic phenotypes. The full-text, referenced overviews in OMIM contain information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype.', 'http://www.ncbi.nlm.nih.gov/entrez/query/static/gifs/entrez_omim.gif', 23, 22186, '2012-03-09 03:49:31', '2013-10-23 05:30:26'
  UNION SELECT 5, 'Conserved Domain Database (CDD)', 'CDD', '<obs.obr.populate.Structure>\n  <resourceId>CDD</resourceId>\n  <contexts>\n    <entry>\n      <string>CDD_abstract</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CDD_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CDD_uid</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>uid</string>\n    <string>title</string>\n    <string>abstract</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>CDD_abstract</string>\n      <double>0.6</double>\n    </entry>\n    <entry>\n      <string>CDD_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>CDD_uid</string>\n      <double>0.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>CDD_abstract</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CDD_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CDD_uid</string>\n      <string>-1</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'CDD_title', 'http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml', 'http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=', 'The Conserved Domain Database (CDD) contains protein domain models imported from outside sources, such as Pfam and SMART, and curated at NCBI. CDD contains over 12,000 such models and is linked to other NCBI databases, including protein sequences, bibliographic citations, and taxonomy.', 'http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd2.gif', 23, 46346, '2012-03-09 03:25:52', '2013-10-23 02:03:22'
  UNION SELECT 6, 'PharmGKB [Disease]', 'PGDI', '<obs.obr.populate.Structure>\n  <resourceId>PGDI</resourceId>\n  <contexts>\n    <entry>\n      <string>PGDI_diseaseRelatedGenes</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPathways</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPhenotypeDatasets</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedDrugs</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseAlternateNames</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>diseaseName</string>\n    <string>diseaseAlternateNames</string>\n    <string>diseaseRelatedGenes</string>\n    <string>diseaseRelatedPathways</string>\n    <string>diseaseRelatedDrugs</string>\n    <string>diseaseRelatedPhenotypeDatasets</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>PGDI_diseaseRelatedGenes</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPathways</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPhenotypeDatasets</string>\n      <double>0.4</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseName</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedDrugs</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseAlternateNames</string>\n      <double>0.8</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>PGDI_diseaseRelatedGenes</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPathways</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPhenotypeDatasets</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedDrugs</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseAlternateNames</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'PGDI_diseaseName', 'http://www.pharmgkb.org/', 'https://www.pharmgkb.org/do/serve?objId=', 'PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.', 'https://www.pharmgkb.org/images/header/title.png', 23, 837, '2012-03-05 22:28:11', '2013-09-25 23:02:14'
  UNION SELECT 7, 'PharmGKB [Drug]', 'PGDR', '<obs.obr.populate.Structure>\n  <resourceId>PGDR</resourceId>\n  <contexts>\n    <entry>\n      <string>PGDR_drugRelatedPhenotypeDatasets</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugTradeNames</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedDiseases</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedGenes</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedPathways</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugGenericNames</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>drugName</string>\n    <string>drugTradeNames</string>\n    <string>drugGenericNames</string>\n    <string>drugRelatedGenes</string>\n    <string>drugRelatedPathways</string>\n    <string>drugRelatedDiseases</string>\n    <string>drugRelatedPhenotypeDatasets</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>PGDR_drugRelatedPhenotypeDatasets</string>\n      <double>0.4</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugTradeNames</string>\n      <double>0.9</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedDiseases</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugName</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedGenes</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedPathways</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugGenericNames</string>\n      <double>0.9</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>PGDR_drugRelatedPhenotypeDatasets</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugTradeNames</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedDiseases</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedGenes</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedPathways</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugGenericNames</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'PGDR_drugName', 'http://www.pharmgkb.org/', 'https://www.pharmgkb.org/do/serve?objId=', 'PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.', 'https://www.pharmgkb.org/images/header/title.png', 23, 1643, '2012-03-05 22:28:10', '2013-09-25 22:55:30'
  UNION SELECT 8, 'PharmGKB [Gene]', 'PGGE', '<obs.obr.populate.Structure>\n  <resourceId>PGGE</resourceId>\n  <contexts>\n    <entry>\n      <string>PGGE_geneName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneSymbol</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneAlternateSymbols</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDiseases</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPathways</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDrugs</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPhenotypeDatasets</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>geneName</string>\n    <string>geneSymbol</string>\n    <string>geneAlternateSymbols</string>\n    <string>geneRelatedDrugs</string>\n    <string>geneRelatedPathways</string>\n    <string>geneRelatedDiseases</string>\n    <string>geneRelatedPhenotypeDatasets</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>PGGE_geneName</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneSymbol</string>\n      <double>0.9</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneAlternateSymbols</string>\n      <double>0.9</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDiseases</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPathways</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDrugs</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPhenotypeDatasets</string>\n      <double>0.4</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>PGGE_geneName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneSymbol</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneAlternateSymbols</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDiseases</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPathways</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDrugs</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPhenotypeDatasets</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'PGGE_geneName', 'http://www.pharmgkb.org/', 'https://www.pharmgkb.org/do/serve?objId=', 'PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.', 'https://www.pharmgkb.org/images/header/title.png', 23, 1242, '2012-01-24 01:50:10', '2013-09-25 22:59:06'
  UNION SELECT 9, 'Reactome', 'REAC', '<obs.obr.populate.Structure>\n  <resourceId>REAC</resourceId>\n  <contexts>\n    <entry>\n      <string>REAC_goCellCompartiment</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>REAC_participants</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>REAC_goBiologicalProcess</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>REAC_name</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>name</string>\n    <string>participants</string>\n    <string>goBiologicalProcess</string>\n    <string>goCellCompartiment</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>REAC_goCellCompartiment</string>\n      <double>0.2</double>\n    </entry>\n    <entry>\n      <string>REAC_participants</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>REAC_goBiologicalProcess</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>REAC_name</string>\n      <double>1.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>REAC_goCellCompartiment</string>\n      <string>1070</string>\n    </entry>\n    <entry>\n      <string>REAC_participants</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>REAC_goBiologicalProcess</string>\n      <string>1070</string>\n    </entry>\n    <entry>\n      <string>REAC_name</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'REAC_name', 'http://www.reactome.org/', 'http://www.reactome.org/cgi-bin/eventbrowser?DB=gk_current&ID=', 'A curated knowledgebase of biological pathways.', 'http://www.reactome.org/icons/R-purple.png', 23, 208093, '2012-03-08 00:23:55', '2013-10-03 21:30:23'
  UNION SELECT 10, 'UniProt KB', 'UPKB', '<obs.obr.populate.Structure>\n  <resourceId>UPKB</resourceId>\n  <contexts>\n    <entry>\n      <string>UPKB_naturalVariant</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_goAnnotationList</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_geneSymbol</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_cellularComponent</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_proteinName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_molecularFunction</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_mutagenesis</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_biologicalProcess</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_organism</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>geneSymbol</string>\n    <string>goAnnotationList</string>\n    <string>proteinName</string>\n    <string>organism</string>\n    <string>naturalVariant</string>\n    <string>mutagenesis</string>\n    <string>biologicalProcess</string>\n    <string>cellularComponent</string>\n    <string>molecularFunction</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>UPKB_naturalVariant</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_goAnnotationList</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>UPKB_geneSymbol</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>UPKB_cellularComponent</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_proteinName</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_molecularFunction</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_mutagenesis</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_biologicalProcess</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_organism</string>\n      <double>0.8</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>UPKB_naturalVariant</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_goAnnotationList</string>\n      <string>1070</string>\n    </entry>\n    <entry>\n      <string>UPKB_geneSymbol</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_cellularComponent</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_proteinName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_molecularFunction</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_mutagenesis</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_biologicalProcess</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_organism</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'UPKB_geneSymbol', 'http://www.uniprot.org/', 'http://www.uniprot.org/uniprot/', 'The mission of UniProt is to provide the scientific community with a comprehensive, high-quality and freely accessible resource of protein sequence and functional information.', 'http://www.uniprot.org/images/logo.gif', 23, 533515, '2012-02-27 02:41:23', '2013-11-03 16:12:14'
  UNION SELECT 99, 'Wicked Witch', 'WITCH', '<obs.obr.populate.Structure>\n  <resourceId>WITCH</resourceId>\n  <contexts>\n    <entry>\n      <string>WITCH_sentence</string>\n    <null/>\n    </entry>\n    <entry>\n      <string>WITCH_class</string>\n    <null/>\n    </entry>\n    </contexts>\n  <itemKeys>\n    <string>sentence</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>WITCH_sentence</string>\n      <double>1.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>WITCH_sentence</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>WITCH_class</string>\n      <string>1132</string>\n    </entry>\n    </ontoIds>\n</obs.obr.populate.Structure>', 'WITCH_title', 'http://example.org', 'http://example.org', 'The wicked witch of the west is a mean lady', '', 23, 4, '2014-05-12 21:18:26', '2014-05-12 21:18:26'
EOS

DOCUMENTS_TEST_DATA_WITCH = <<-EOS
INSERT INTO `obr_witch_element`
  SELECT 1 AS 'id', '1' AS 'local_element_id', 'wicked witch of the west' AS 'WITCH_sentence', '1132/obo:NCBITaxon_10090' AS 'WITCH_class'
  UNION SELECT 2, '2', 'wicked witch of the east', '1132/obo:NCBITaxon_10116'
  UNION SELECT 3, '3', 'wicked witch sorcerer of the west', ''
  UNION SELECT 4, '4', 'wicked sorcerer of the west', ''
EOS