# The inserts in this file use UNION SELECT because earlier versions of SQLite don't support VALUE inserts
# Don't revert this change unless you are sure your test env supports newer SQLite versions (Jruby 1.7.12 does not, for example)

ONTOLOGIES_TABLE = <<-EOS
INSERT INTO 'obs_ontology'
  SELECT 1318 AS 'id', '47845' AS 'local_ontology_id', 'NCBI organismal classification' AS 'name', '1.2' AS 'version', '' AS 'description', 28 AS 'status', '1132' AS 'virtual_ontology_id', 'OWL' AS 'format', 21 AS 'dictionary_id'
  UNION SELECT 1889, '50608', 'Experimental Factor Ontology', '2.39.1', '', 28, '1136', 'OWL', 23
EOS

CONCEPTS_TABLE = <<-EOS
INSERT INTO 'obs_concept'
  SELECT 38776185 AS 'id', '47845/obo:NCBITaxon_10090' AS 'local_concept_id', 1318 AS 'ontology_id', 0 AS 'is_toplevel', 'http://purl.obolibrary.org/obo/NCBITaxon_10090' AS 'full_id'
  UNION SELECT 38778061, '47845/obo:NCBITaxon_10116', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_10116'
  UNION SELECT 38869745, '47845/obo:NCBITaxon_119951', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_119951'
  UNION SELECT 38878148, '47845/obo:NCBITaxon_128735', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_128735'
  UNION SELECT 38906355, '47845/obo:NCBITaxon_1590', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_1590'
  UNION SELECT 38941396, '47845/obo:NCBITaxon_196627', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_196627'
  UNION SELECT 38950313, '47845/obo:NCBITaxon_205918', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_205918'
  UNION SELECT 38967263, '47845/obo:NCBITaxon_224308', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_224308'
  UNION SELECT 38977399, '47845/obo:NCBITaxon_235443', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_235443'
  UNION SELECT 38985226, '47845/obo:NCBITaxon_244320', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_244320'
  UNION SELECT 38987111, '47845/obo:NCBITaxon_246437', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_246437'
  UNION SELECT 38992982, '47845/obo:NCBITaxon_252598', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_252598'
  UNION SELECT 39001776, '47845/obo:NCBITaxon_262316', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_262316'
  UNION SELECT 39022429, '47845/obo:NCBITaxon_287', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_287'
  UNION SELECT 39025667, '47845/obo:NCBITaxon_2903', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_2903'
  UNION SELECT 39040690, '47845/obo:NCBITaxon_3055', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_3055'
  UNION SELECT 39080035, '47845/obo:NCBITaxon_34506', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_34506'
  UNION SELECT 39089796, '47845/obo:NCBITaxon_35525', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_35525'
  UNION SELECT 39104845, '47845/obo:NCBITaxon_3702', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_3702'
  UNION SELECT 39111319, '47845/obo:NCBITaxon_3767', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_3767'
  UNION SELECT 39116621, '47845/obo:NCBITaxon_382', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_382'
  UNION SELECT 39122680, '47845/obo:NCBITaxon_3880', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_3880'
  UNION SELECT 39123480, '47845/obo:NCBITaxon_3888', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_3888'
  UNION SELECT 39132973, '47845/obo:NCBITaxon_3983', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_3983'
  UNION SELECT 39134149, '47845/obo:NCBITaxon_39947', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_39947'
  UNION SELECT 39140274, '47845/obo:NCBITaxon_40559', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_40559'
  UNION SELECT 39142817, '47845/obo:NCBITaxon_4081', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_4081'
  UNION SELECT 39144655, '47845/obo:NCBITaxon_4100', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_4100'
  UNION SELECT 39190748, '47845/obo:NCBITaxon_4565', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_4565'
  UNION SELECT 39191939, '47845/obo:NCBITaxon_4577', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_4577'
  UNION SELECT 39223416, '47845/obo:NCBITaxon_4896', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_4896'
  UNION SELECT 39227000, '47845/obo:NCBITaxon_4932', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_4932'
  UNION SELECT 39249073, '47845/obo:NCBITaxon_51589', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_51589'
  UNION SELECT 39250688, '47845/obo:NCBITaxon_51751', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_51751'
  UNION SELECT 39265689, '47845/obo:NCBITaxon_5346', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_5346'
  UNION SELECT 39277401, '47845/obo:NCBITaxon_5476', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_5476'
  UNION SELECT 39291105, '47845/obo:NCBITaxon_562', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_562'
  UNION SELECT 39300238, '47845/obo:NCBITaxon_573', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_573'
  UNION SELECT 39307459, '47845/obo:NCBITaxon_5811', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_5811'
  UNION SELECT 39325337, '47845/obo:NCBITaxon_6161', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_6161'
  UNION SELECT 39328296, '47845/obo:NCBITaxon_6239', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_6239'
  UNION SELECT 39331060, '47845/obo:NCBITaxon_6280', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_6280'
  UNION SELECT 39331087, '47845/obo:NCBITaxon_6282', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_6282'
  UNION SELECT 39362570, '47845/obo:NCBITaxon_666', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_666'
  UNION SELECT 39412250, '47845/obo:NCBITaxon_7227', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_7227'
  UNION SELECT 39412568, '47845/obo:NCBITaxon_7230', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_7230'
  UNION SELECT 39420840, '47845/obo:NCBITaxon_7460', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_7460'
  UNION SELECT 39445054, '47845/obo:NCBITaxon_7955', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_7955'
  UNION SELECT 39448286, '47845/obo:NCBITaxon_8030', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_8030'
  UNION SELECT 39448484, '47845/obo:NCBITaxon_8049', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_8049'
  UNION SELECT 39458543, '47845/obo:NCBITaxon_82328', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_82328'
  UNION SELECT 39464973, '47845/obo:NCBITaxon_82927', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_82927'
  UNION SELECT 39469436, '47845/obo:NCBITaxon_83332', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_83332'
  UNION SELECT 39471826, '47845/obo:NCBITaxon_8364', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_8364'
  UNION SELECT 39490560, '47845/obo:NCBITaxon_85433', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_85433'
  UNION SELECT 39536578, '47845/obo:NCBITaxon_9031', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_9031'
  UNION SELECT 39581772, '47845/obo:NCBITaxon_9598', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_9598'
  UNION SELECT 39582213, '47845/obo:NCBITaxon_9606', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_9606'
  UNION SELECT 39583207, '47845/obo:NCBITaxon_9615', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_9615'
  UNION SELECT 39603792, '47845/obo:NCBITaxon_9823', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_9823'
  UNION SELECT 39610688, '47845/obo:NCBITaxon_9913', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_9913'
  UNION SELECT 39616001, '47845/obo:NCBITaxon_9986', 1318, 0, 'http://purl.obolibrary.org/obo/NCBITaxon_9986'
  UNION SELECT 46054849, '50608/efo:EFO_0000749', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0000749'
  UNION SELECT 46054853, '50608/efo:EFO_0000753', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0000753'
  UNION SELECT 46055110, '50608/efo:EFO_0001030', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0001030'
  UNION SELECT 46055113, '50608/efo:EFO_0001033', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0001033'
  UNION SELECT 46056251, '50608/efo:EFO_0002692', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0002692'
  UNION SELECT 46056298, '50608/efo:EFO_0002759', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0002759'
  UNION SELECT 46056299, '50608/efo:EFO_0002760', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0002760'
  UNION SELECT 46056300, '50608/efo:EFO_0002761', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0002761'
  UNION SELECT 46056301, '50608/efo:EFO_0002762', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0002762'
  UNION SELECT 46056304, '50608/efo:EFO_0002765', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0002765'
  UNION SELECT 46056307, '50608/efo:EFO_0002768', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0002768'
  UNION SELECT 46056308, '50608/efo:EFO_0002769', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0002769'
  UNION SELECT 46056310, '50608/efo:EFO_0002771', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0002771'
  UNION SELECT 46056666, '50608/efo:EFO_0003143', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0003143'
  UNION SELECT 46057121, '50608/efo:EFO_0003737', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0003737'
  UNION SELECT 46057122, '50608/efo:EFO_0003738', 1889, 0, 'http://www.ebi.ac.uk/efo/EFO_0003738'
EOS

MANUAL_ANNOTATION_XXHASH = [1041899727, 1116757309, 1118768622, 1132799958, 1276288551, 1278301126, 1294955310, 1367038899, 1397464604, 1429265218, 1436188383, 1457559688, 1564120855, 1648354265, 1737724267, 1775001483, 1815619048, 1845933306, 1855054362, 1891098225, 1897602846, 1900980300, 1911650997, 1941539371, 1960287610, 1968696234, 2015319689, 2158142354, 2169457277, 2172433442, 2223595865, 2265630877, 2277609132, 2342909499, 2471850823, 2492339354, 2561651131, 2588010934, 2634609409, 2659771975, 2719637631, 2979941855, 301104715, 3061354475, 3111427210, 3131668133, 3174080258, 3186034290, 3189822825, 3224726537, 33311642, 3382211325, 3478116066, 3479363986, 3578505328, 3585924929, 3605434666, 3616942866, 3666208680, 370498697, 3770678017, 3887744824, 3927646680, 3981139458, 3984609496, 4052308340, 4150125671, 4174731843, 499876719, 528574762, 638074822, 66776817, 689454552, 69952072, 744587962, 868280579, 93341366, 934852592]
MANUAL_ANNOTATION_COUNTS = {1041899727=>1, 1116757309=>1, 1118768622=>1, 1132799958=>1, 1276288551=>1, 1278301126=>2, 1294955310=>1, 1367038899=>1, 1397464604=>21, 1429265218=>1, 1436188383=>19, 1457559688=>1, 1564120855=>6, 1648354265=>2, 1737724267=>1, 1775001483=>18, 1815619048=>1, 1845933306=>1, 1855054362=>1, 1891098225=>1, 1897602846=>7, 1900980300=>3, 1911650997=>14, 1941539371=>1, 1960287610=>1, 1968696234=>327, 2015319689=>1, 2158142354=>1, 2169457277=>1, 2172433442=>1, 2223595865=>4, 2265630877=>1, 2277609132=>1, 2342909499=>1, 2471850823=>143, 2492339354=>3, 2561651131=>1, 2588010934=>1, 2634609409=>1, 2659771975=>1, 2719637631=>2, 2979941855=>4, 301104715=>1, 3061354475=>1, 3111427210=>1, 3131668133=>47, 3174080258=>189, 3186034290=>3, 3189822825=>1, 3224726537=>1, 33311642=>1, 3382211325=>1, 3478116066=>4, 3479363986=>3, 3578505328=>1, 3585924929=>2, 3605434666=>3, 3616942866=>1, 3666208680=>1, 370498697=>1, 3770678017=>8, 3887744824=>1, 3927646680=>1, 3981139458=>1, 3984609496=>1, 4052308340=>1, 4150125671=>1, 4174731843=>19, 499876719=>1, 528574762=>1, 638074822=>2, 66776817=>3, 689454552=>1, 69952072=>1, 744587962=>1, 868280579=>1, 93341366=>2, 934852592=>1}

RESOURCES_TEST_DATA = <<-EOS
INSERT INTO 'obr_resource'
  SELECT 1 AS 'id', 'ArrayExpress' AS 'name', 'AE_test' AS 'resource_id', '<obs.obr.populate.Structure>\n  <resourceId>AE</resourceId>\n  <contexts>\n    <entry>\n      <string>AE_name</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>AE_description</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>AE_species</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>AE_experiment_type</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>name</string>\n    <string>description</string>\n    <string>species</string>\n    <string>experiment_type</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>AE_name</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>AE_description</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>AE_species</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>AE_experiment_type</string>\n      <double>0.9</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>AE_name</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>AE_description</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>AE_species</string>\n      <string>1132</string>\n    </entry>\n    <entry>\n      <string>AE_experiment_type</string>\n      <string>1136</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>' AS 'structure', 'AE_name' AS 'main_context', 'http://www.ebi.ac.uk/arrayexpress/' AS 'url', 'http://www.ebi.ac.uk/arrayexpress/experiments/' AS 'element_url', 'ArrayExpress is a public repository for microarray data, which is aimed at storing MIAME-compliant data in accordance with MGED recommendations. The ArrayExpress Data Warehouse stores gene-indexed expression profiles from a curated subset of experiments in the repository.' AS 'description', 'http://www.ebi.ac.uk/microarray-as/aer/include/aelogo.png' AS 'logo', 23 AS 'dictionary_id', 35877 AS 'total_element', '2013-02-05 21:35:46' AS 'last_update_date', '2013-09-26 05:13:05' AS 'workflow_completed_date'
  UNION SELECT 2, 'ClinicalTrials.gov', 'CT', '<obs.obr.populate.Structure>\n  <resourceId>CT</resourceId>\n  <contexts>\n    <entry>\n      <string>CT_intervention</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_official_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_brief_summary</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_condition</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_brief_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CT_detailed_description</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>brief_title</string>\n    <string>official_title</string>\n    <string>brief_summary</string>\n    <string>detailed_description</string>\n    <string>condition</string>\n    <string>intervention</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>CT_intervention</string>\n      <double>0.6</double>\n    </entry>\n    <entry>\n      <string>CT_official_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>CT_brief_summary</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>CT_condition</string>\n      <double>0.6</double>\n    </entry>\n    <entry>\n      <string>CT_brief_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>CT_detailed_description</string>\n      <double>0.7</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>CT_intervention</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_official_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_brief_summary</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_condition</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_brief_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CT_detailed_description</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'CT_brief_title', 'http://clinicaltrials.gov/', 'http://clinicaltrials.gov/ct/show/', 'ClinicalTrials.gov provides regularly updated information about federally and privately supported clinical research in human volunteers. ClinicalTrials.gov gives you information about a trial''s purpose, who may participate, locations, and phone numbers for more details. The information provided on ClinicalTrials.gov should be used in conjunction with advice from health care professionals. Before searching, you may want to learn more about clinical trials.', 'http://clinicaltrials.gov/ct2/html/images/frame/title.gif', 23, 122591, '2012-03-08 21:00:13', '2013-10-02 07:18:39'
  UNION SELECT 3, 'ARRS GoldMiner', 'GM', '<obs.obr.populate.Structure>\n  <resourceId>GM</resourceId>\n  <contexts>\n    <entry>\n      <string>GM_caption</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>GM_title</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>title</string>\n    <string>caption</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>GM_caption</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>GM_title</string>\n      <double>1.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>GM_caption</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>GM_title</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'GM_title', 'http://goldminer.arrs.org/', 'http://goldminer.arrs.org/link.php?id=', 'ARRS GoldMiner provides instant access to images published in selected peer-reviewed radiology journals. This new, web-based system allows viewers to search for images by findings, anatomy, imaging technique, and patient age and sex.', 'http://goldminer.arrs.org/image/goldminer.gif', 23, 1155, '2012-03-06 05:56:19', '2013-09-30 22:08:42'
  UNION SELECT 4, 'Online Mendelian Inheritance in Man', 'OMIM', '<obs.obr.populate.Structure>\n  <resourceId>OMIM</resourceId>\n  <contexts>\n    <entry>\n      <string>OMIM_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>OMIM_locus</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>OMIM_uid</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>OMIM_altTitles</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>uid</string>\n    <string>title</string>\n    <string>altTitles</string>\n    <string>locus</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>OMIM_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>OMIM_locus</string>\n      <double>0.0</double>\n    </entry>\n    <entry>\n      <string>OMIM_uid</string>\n      <double>0.0</double>\n    </entry>\n    <entry>\n      <string>OMIM_altTitles</string>\n      <double>0.7</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>OMIM_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>OMIM_locus</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>OMIM_uid</string>\n      <string>-1</string>\n    </entry>\n    <entry>\n      <string>OMIM_altTitles</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'OMIM_title', 'http://www.ncbi.nlm.nih.gov/omim', 'http://www.ncbi.nlm.nih.gov/omim/', 'OMIM is a comprehensive, authoritative, and timely compendium of human genes and genetic phenotypes. The full-text, referenced overviews in OMIM contain information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype.', 'http://www.ncbi.nlm.nih.gov/entrez/query/static/gifs/entrez_omim.gif', 23, 22186, '2012-03-09 03:49:31', '2013-10-23 05:30:26'
  UNION SELECT 5, 'Conserved Domain Database (CDD)', 'CDD', '<obs.obr.populate.Structure>\n  <resourceId>CDD</resourceId>\n  <contexts>\n    <entry>\n      <string>CDD_abstract</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CDD_title</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>CDD_uid</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>uid</string>\n    <string>title</string>\n    <string>abstract</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>CDD_abstract</string>\n      <double>0.6</double>\n    </entry>\n    <entry>\n      <string>CDD_title</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>CDD_uid</string>\n      <double>0.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>CDD_abstract</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CDD_title</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>CDD_uid</string>\n      <string>-1</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'CDD_title', 'http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml', 'http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=', 'The Conserved Domain Database (CDD) contains protein domain models imported from outside sources, such as Pfam and SMART, and curated at NCBI. CDD contains over 12,000 such models and is linked to other NCBI databases, including protein sequences, bibliographic citations, and taxonomy.', 'http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd2.gif', 23, 46346, '2012-03-09 03:25:52', '2013-10-23 02:03:22'
  UNION SELECT 6, 'PharmGKB [Disease]', 'PGDI', '<obs.obr.populate.Structure>\n  <resourceId>PGDI</resourceId>\n  <contexts>\n    <entry>\n      <string>PGDI_diseaseRelatedGenes</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPathways</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPhenotypeDatasets</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedDrugs</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseAlternateNames</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>diseaseName</string>\n    <string>diseaseAlternateNames</string>\n    <string>diseaseRelatedGenes</string>\n    <string>diseaseRelatedPathways</string>\n    <string>diseaseRelatedDrugs</string>\n    <string>diseaseRelatedPhenotypeDatasets</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>PGDI_diseaseRelatedGenes</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPathways</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPhenotypeDatasets</string>\n      <double>0.4</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseName</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedDrugs</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseAlternateNames</string>\n      <double>0.8</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>PGDI_diseaseRelatedGenes</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPathways</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedPhenotypeDatasets</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseRelatedDrugs</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDI_diseaseAlternateNames</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'PGDI_diseaseName', 'http://www.pharmgkb.org/', 'https://www.pharmgkb.org/do/serve?objId=', 'PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.', 'https://www.pharmgkb.org/images/header/title.png', 23, 837, '2012-03-05 22:28:11', '2013-09-25 23:02:14'
  UNION SELECT 7, 'PharmGKB [Drug]', 'PGDR', '<obs.obr.populate.Structure>\n  <resourceId>PGDR</resourceId>\n  <contexts>\n    <entry>\n      <string>PGDR_drugRelatedPhenotypeDatasets</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugTradeNames</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedDiseases</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedGenes</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedPathways</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGDR_drugGenericNames</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>drugName</string>\n    <string>drugTradeNames</string>\n    <string>drugGenericNames</string>\n    <string>drugRelatedGenes</string>\n    <string>drugRelatedPathways</string>\n    <string>drugRelatedDiseases</string>\n    <string>drugRelatedPhenotypeDatasets</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>PGDR_drugRelatedPhenotypeDatasets</string>\n      <double>0.4</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugTradeNames</string>\n      <double>0.9</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedDiseases</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugName</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedGenes</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedPathways</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGDR_drugGenericNames</string>\n      <double>0.9</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>PGDR_drugRelatedPhenotypeDatasets</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugTradeNames</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedDiseases</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedGenes</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugRelatedPathways</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGDR_drugGenericNames</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'PGDR_drugName', 'http://www.pharmgkb.org/', 'https://www.pharmgkb.org/do/serve?objId=', 'PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.', 'https://www.pharmgkb.org/images/header/title.png', 23, 1643, '2012-03-05 22:28:10', '2013-09-25 22:55:30'
  UNION SELECT 8, 'PharmGKB [Gene]', 'PGGE', '<obs.obr.populate.Structure>\n  <resourceId>PGGE</resourceId>\n  <contexts>\n    <entry>\n      <string>PGGE_geneName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneSymbol</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneAlternateSymbols</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDiseases</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPathways</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDrugs</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPhenotypeDatasets</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>geneName</string>\n    <string>geneSymbol</string>\n    <string>geneAlternateSymbols</string>\n    <string>geneRelatedDrugs</string>\n    <string>geneRelatedPathways</string>\n    <string>geneRelatedDiseases</string>\n    <string>geneRelatedPhenotypeDatasets</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>PGGE_geneName</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneSymbol</string>\n      <double>0.9</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneAlternateSymbols</string>\n      <double>0.9</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDiseases</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPathways</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDrugs</string>\n      <double>0.5</double>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPhenotypeDatasets</string>\n      <double>0.4</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>PGGE_geneName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneSymbol</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneAlternateSymbols</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDiseases</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPathways</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedDrugs</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>PGGE_geneRelatedPhenotypeDatasets</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'PGGE_geneName', 'http://www.pharmgkb.org/', 'https://www.pharmgkb.org/do/serve?objId=', 'PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.', 'https://www.pharmgkb.org/images/header/title.png', 23, 1242, '2012-01-24 01:50:10', '2013-09-25 22:59:06'
  UNION SELECT 9, 'Reactome', 'REAC', '<obs.obr.populate.Structure>\n  <resourceId>REAC</resourceId>\n  <contexts>\n    <entry>\n      <string>REAC_goCellCompartiment</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>REAC_participants</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>REAC_goBiologicalProcess</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>REAC_name</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>name</string>\n    <string>participants</string>\n    <string>goBiologicalProcess</string>\n    <string>goCellCompartiment</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>REAC_goCellCompartiment</string>\n      <double>0.2</double>\n    </entry>\n    <entry>\n      <string>REAC_participants</string>\n      <double>0.8</double>\n    </entry>\n    <entry>\n      <string>REAC_goBiologicalProcess</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>REAC_name</string>\n      <double>1.0</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>REAC_goCellCompartiment</string>\n      <string>1070</string>\n    </entry>\n    <entry>\n      <string>REAC_participants</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>REAC_goBiologicalProcess</string>\n      <string>1070</string>\n    </entry>\n    <entry>\n      <string>REAC_name</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'REAC_name', 'http://www.reactome.org/', 'http://www.reactome.org/cgi-bin/eventbrowser?DB=gk_current&ID=', 'A curated knowledgebase of biological pathways.', 'http://www.reactome.org/icons/R-purple.png', 23, 208093, '2012-03-08 00:23:55', '2013-10-03 21:30:23'
  UNION SELECT 10, 'UniProt KB', 'UPKB', '<obs.obr.populate.Structure>\n  <resourceId>UPKB</resourceId>\n  <contexts>\n    <entry>\n      <string>UPKB_naturalVariant</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_goAnnotationList</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_geneSymbol</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_cellularComponent</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_proteinName</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_molecularFunction</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_mutagenesis</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_biologicalProcess</string>\n      <null/>\n    </entry>\n    <entry>\n      <string>UPKB_organism</string>\n      <null/>\n    </entry>\n  </contexts>\n  <itemKeys>\n    <string>geneSymbol</string>\n    <string>goAnnotationList</string>\n    <string>proteinName</string>\n    <string>organism</string>\n    <string>naturalVariant</string>\n    <string>mutagenesis</string>\n    <string>biologicalProcess</string>\n    <string>cellularComponent</string>\n    <string>molecularFunction</string>\n  </itemKeys>\n  <weights>\n    <entry>\n      <string>UPKB_naturalVariant</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_goAnnotationList</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>UPKB_geneSymbol</string>\n      <double>1.0</double>\n    </entry>\n    <entry>\n      <string>UPKB_cellularComponent</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_proteinName</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_molecularFunction</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_mutagenesis</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_biologicalProcess</string>\n      <double>0.7</double>\n    </entry>\n    <entry>\n      <string>UPKB_organism</string>\n      <double>0.8</double>\n    </entry>\n  </weights>\n  <ontoIds>\n    <entry>\n      <string>UPKB_naturalVariant</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_goAnnotationList</string>\n      <string>1070</string>\n    </entry>\n    <entry>\n      <string>UPKB_geneSymbol</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_cellularComponent</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_proteinName</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_molecularFunction</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_mutagenesis</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_biologicalProcess</string>\n      <string>null</string>\n    </entry>\n    <entry>\n      <string>UPKB_organism</string>\n      <string>null</string>\n    </entry>\n  </ontoIds>\n</obs.obr.populate.Structure>', 'UPKB_geneSymbol', 'http://www.uniprot.org/', 'http://www.uniprot.org/uniprot/', 'The mission of UniProt is to provide the scientific community with a comprehensive, high-quality and freely accessible resource of protein sequence and functional information.', 'http://www.uniprot.org/images/logo.gif', 23, 533515, '2012-02-27 02:41:23', '2013-11-03 16:12:14'
EOS

DOCUMENTS_TEST_DATA = <<-EOS
INSERT INTO 'obr_ae_test_element'
  SELECT 1 AS 'id', 'E-MTAB-1371' AS 'local_element_id', 23 AS 'dictionary_id', 'CLIP-Seq of H. sapiens HeLa cells to investigate transcriptome-wide mapping of hnRNP C and U2AF65' AS 'ae_name', 'iCLIP experiment to assess the binding of the highly abundant nuclear RNA-binding protein hnRNP C and core splicing factor U2AF65 on a genomic scale. To investigate how both proteins compete for binding at a subset of sites, U2AF65 iCLIP experiments were performed from both HNRNPC knockdown and control HeLa cells.' AS 'ae_description', '1132/NCBITaxon:9606' AS 'ae_species', '1136/efo:EFO_0003143' AS 'ae_experiment_type'
  UNION SELECT 2, 'E-MTAB-1289', 23, 'Transcription profiling by array of spinal cord tissue from mouse with bone cancer pain', 'The aim of our study is to identify miRNAs responsible for bone-cancer pain condition and their target mRNAs. We combined mRNA profiling with Affymetrix microarray and miRNA measurement with a qRT-PCR-based technique. Then, a cross-correlation of these data highlighted miRNA-mRNA pairs that were further characterized with functional experiments.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 3, 'E-MTAB-1147', 23, 'Transcription profiling by high throughput sequencing of HNRNPC knockdown and control HeLa cells', 'We performed RNA-seq experiments on two replicate samples from each HNRNPC knockdown (KD1 and KD2) As well as from control HeLa cells. Library preparation was preformed according to mRNA Sequencing Sample Preparation Guide (Illumina, Part # 1004898 REV. D). Reagents were taken from the Illumina sample preparation kit (Illumina, CAT # RS-930-1001). Knockdown and control samples were sequenced together in one flowcell on one and two lanes, respectively.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 4, 'E-MTAB-965', 23, 'Transcription profiling by array of BT474 and SKBr3 human breast cancer cell lines following treatment with hIgG or trastuzumab', 'We found that trastuzumab (herceptin) treatment significantly decreased five miRNAs and increased three others in SKBr3 cells, whereas in BT474 cells it significantly decreased two miRNAs and increased nine. The only miRNA that shared the same change in both cell lines was miRNA-194 (miR-194), which was upregulated following trastuzumab treatment.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 5, 'E-MEXP-3704', 23, 'Transcription profiling by array of condition dependent regulation of lipid metabolism in Saccharomyces cerevisiae', 'Investigation of lipid metabolism as a function of three experimental factors: 1) aerobicity (aerobic vs anaerobic, or ''A'' vs ''O''), 2) nutrient limitation (carbon vs nitrogen limitation, or ''C'' vs ''N''), 3) temperature (30C vs 15C, or ''T'' vs ''t'') using a full factorial design.', '1132/NCBITaxon:4932', '1136/efo:EFO_0002768'
  UNION SELECT 6, 'E-MEXP-3617', 23, 'Transcription profiling by array of mouse primary hematopoietic precursor cells transformed by two different HOX genes (HOXA1 and HOXA9) and by two HOX derivatives where the homeodomain was exchanged.', 'The gene expression profile was determined in primary cells transformed by two different HOX genes (HOXA1 and HOXA9) and by two HOX derivatives where the homeodomain was exchanged.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 7, 'E-WMIT-1', 23, 'Chromatin immunoprecipitation genome wide binding site identification in yeast', 'Genome Wide Location: binding site identification', '1132/NCBITaxon:4932', '1136/efo:EFO_0002760'
  UNION SELECT 8, 'E-TIGR-135', 23, 'Transcription profiling of mouse cerebral cortex using two model strains (C57BL/6 and ApoE(-/-) on the C57BL background) upon episode hypoxia treatment', 'Study the gene expression variation at cerebral cortex using two mouse model strains (C57BL/6 and ApoE(-/-) on the C57BL background) upon episode hypoxia treatment', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 9, 'E-TIGR-120', 23, 'Transcription profiling of 4-week old Arabidopsis plants grown in soil flooded to the soil surface (root flooding) or completely submerged under 6 cm of water (submergence)', '4-week old Arabidopsis plants grown in soil were flooded to the soil surface (root flooding) or completely submerged under 6 cm of water (submergence). Samples are collected at the time specified.', '1132/NCBITaxon:3702', '1136/efo:EFO_0002768'
  UNION SELECT 10, 'E-TIGR-109', 23, 'Transcription profiling of liver and spleen RNA samples compared between A/J and C57BL/6 mice and between C57BL/6 wild type and TNF knockouts at five time points following Trypanosoma Congolense infection', 'Liver and Spleen RNA samples were compared between A/J and C57BL/6 mice and between C57BL/6 wildtype and TNF knockouts at five time points following Trypanosoma Congolense infection', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 11, 'E-TIGR-108', 23, 'Transcription profiling of liver and spleen RNA samples compared between A/J and C57BL/6 mice and between C57BL/6 wild type and TNF knockouts at five time points following Trypanosoma Congolense infection', 'Liver and Spleen RNA samples were compared between A/J and C57BL/6 mice and between C57BL/6 wildtype and TNF knockouts at five time points following Trypanosoma Congolense infection', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 12, 'E-TIGR-106', 23, 'Transcription profiling of mice fed salt water to induce hypertension to assess strain and sex specific differences', 'Some mice were fed salt water to induce hypertension and some fed normal water to see how hypertension is different among different strains, tissues, and sexes', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 13, 'E-TIGR-105', 23, 'Transcription profiling of mice fed salt water to induce hypertension to assess strain and sex specific differences', 'Some mice were fed salt water to induce hypertension and some fed normal water to see how hypertension is different among different strains, tissues, and sexes', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 14, 'E-TIGR-104', 23, 'Transcription profiling of mouse brain in male and females of three strains', 'Strain and sex variations are studied at the gene expression level. RNAs are extracted from two tissues (cerebral cortex and brain stem), three strains (DBA, A/J, C57BL/6) and both male and female are studied', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 15, 'E-TIGR-103', 23, 'Transcription profiling of mouse brain in male and females of three strains', 'Strain and sex variations are studied at the gene expression level. RNAs are extracted from two tissues (cerebral cortex and brain stem), three strains (DBA, A/J, C57BL/6) and both male and female are studied', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 16, 'E-TIGR-96', 23, 'Transcription profiling of mouse lung tissue among both sexes and three different strains(A/J, C57BL/6, DBA)', 'A comparison of lung tissue among both sexes and three different strains(A/J, C57BL/6, DBA) of mouse', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 17, 'E-TABM-86', 23, 'Transcription profiling of E. coli wild type and fis or hns transcription factor mutants in CSH50, LZ41 and LZ54 strains during different growth phases, with relaxation and hypernegative supercoiling and after norflaxacin treatment', 'Comparison of transcript profiles of E. coli CSH50 wild-type and fis or hns mutant cells to distinguish between the genes expressed either in the presence or absence of transcriptional regulators FIS and H-NS during different growth phases. Samples were taken in Mid-exponential phase (ME), Transition to Stationary (TS) and Late Stationary phase (LS).  Effect of FIS and H-NS on gene expression at relaxed and hypernegative supercoiling level was also studied using LZ41 and LZ54 strains. LZ41 and LZ54 strains contain drug-resistant alleles of different topoisomerase genes. In the LZ41 strain norfloxacin treatment strongly relaxes DNA, whereas in the LZ54 strain the same treatment generates high negative supercoiling (Khodursky et al., 1995, PNAS 92:11801-5; Ziechedrich et al, 1997, Genes Dev. 11:2580-92).', '1132/NCBITaxon:562', '1136/efo:EFO_0002768'
  UNION SELECT 18, 'E-TABM-7', 23, 'Transcription profiling of Baseline gene expression in circulating peripheral blood mononuclear cells isolated from 18 apparently healthy volunteers (10 females and 8 males)', 'Baseline gene expression in circulating peripheral blood mononuclear cells isolated from 18 apparently healthy volunteers (10 females and 8 males). The volunteers fasted overnight for 10 hours prior to collection of blood samples (but they were allowed water). 5 blood samples were collected from each volunteer at consecutive 8-day intervals.  Volunteers were included in this study provided they met the following selection criteria, that they should be Aged between 20 and 45 years of age; A non-smoker (or an ex-smoker);Not pregnant or been pregnant in the last 12 months Not diagnosed with any long-term medical condition; Or have not (in the previous 4 months) donated blood, received any inoculations and are not taking regular prescribed medicine (including HRT and oral contraception) Prior to inclusion in this study volunteers were screened for general health and had a blood sample taken for routine screening of Hb content etc. Only those volunteers who were in good general health were allowed to participate in the study. The age of all volunteers has been reported here as being 20-45 years in order to preserve the anonymity of the data, and so that the individuals involved may not be able to track down their own specific data. Please contact us if the details of volunteers'' ages are required.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 19, 'E-SNGR-9', 23, 'Transcription profiling of fission yeast cell cycle data (log normalised data from E-SNGR-2-7)', 'This experiment contains the log normalized data for ArrayExpress experiments E-SNGR-2 to E-SNGR-7', '1132/NCBITaxon:4896', '1136/efo:EFO_0002768'
  UNION SELECT 20, 'E-SNGR-8', 23, 'Transcription profiling of immortalised human mammary luminal epithelial cells and variants expressing a moderate and high level of ERBB2', 'To investigate changes in gene expression associated with ERBB2, expression profiling of immortalised human mammary luminal epithelial cells and variants expressing a moderate and high level of ERBB2 has been carried out using cDNA microarrays corresponding to approximately 6,000 unique genes/ESTs. Sixty-three (63) significantly up- or down-regulated (>2.0 fold) genes were identified and further validated by RT-PCR and Northern blot analysis as well as microarray comparisons with a spontaneously ERBB2- overexpressing breast cancer cell line and ERBB2-positive primary breast tumours. Proteins predicted to be associated ERBB2 overexpression in breast cancers were analysed together with their clinical relevance by antibody screening using a tissue array. Differentially regulated genes include those involved in cell matrix interactions including P4HA2 (proline 4-hydroxylase), galectin 1 (LGALS1) and galectin 3 (LGALS3), FN1 (fibronectin 1), and CDH3 (p-cadherin) and cell proliferation (CRIP1, IGFBP3) and transformation (S100P, S100A4). A number of genes associated with MYC signalling were also differentially expressed, including NDRG1, USF2 and the epithelial membrane proteins 1 and 3 (EMP1, EMP3). These data represent profile of the transcriptional changes associated with ERBB2-related pathways in the breast, and identify novel and potentially useful targets for prognosis and therapy.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 21, 'E-SNGR-1', 23, 'Transcription profiling of fission yeast grown in an enriched media was vs the same strain grown in minimal media', 'A wild type strain of the fission yeast Schizosaccharomyces pombe grown in an enriched media was compared to the same strain grown in minimal media.', '1132/NCBITaxon:4896', '1136/efo:EFO_0002768'
  UNION SELECT 22, 'E-SMDB-4088', 23, 'Virtual northern on human brain samples using DNA microarrays for systematic genome wide analysis of mRNA lengths', 'Background  We applied the Virtual Northern technique to human brain mRNA to systematically measure human mRNA transcript lengths on a genome-wide scale. Methodology/Principal Findings  We used separation by gel electrophoresis followed by hybridization to cDNA microarrays to measure 8,774 mRNA transcript lengths representing at least 6,238 genes at high (>90%) confidence. By comparing these transcript lengths to the Refseq and H-Invitational full-length cDNA databases, we found that nearly half of our measurements appeared to represent novel transcript variants. Comparison of length measurements determined by hybridization to different cDNAs derived from the same gene identified clones that potentially correspond to alternative transcript variants. We observed a close linear relationship between ORF and mRNA lengths in human mRNAs, identical in form to the relationship we had previously identified in yeast. Some functional classes of protein are encoded by mRNAs whose untranslated regions (UTRs) tend to be longer or shorter than average; these functional classes were similar in both human and yeast. Conclusions/Significance  Human transcript diversity is extensive and largely unannotated. Our length dataset can be used as a new criterion for judging the completeness of cDNAs and annotating mRNA sequences. Similar relationships between the lengths of the UTRs in human and yeast mRNAs and the functions of the proteins they encode suggest that UTR sequences serve an important regulatory role among eukaryotes.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 23, 'E-SMDB-3690', 23, 'Transcription profiling of Vibrio cholerae isolated from clinical samples, grown in minimal and rich media in order to validate predicted metabolic   pathways', 'Vibrio cholerae, the etiological agent of the diarrheal illness cholera, can kill an infected adult in 24 h. V.cholerae lives as an autochthonous microbe in estuaries, rivers and coastal waters. A better understanding of its metabolic pathways will assist the development of more effective treatments and will provide a deeper understanding of how this bacterium persists in natural aquatic habitats. Using the completed V.cholerae genome sequence and PathoLogic software, we created VchoCyc, a pathway-genome database that predicted 171 likely metabolic pathways in the bacterium. We report here experimental evidence supporting the computationally predicted pathways. The evidence comes from microarray gene expression studies of V.cholerae in the stools of three cholera patients [D. S. Merrell, S. M. Butler, F. Qadri, N. A. Dolganov, A. Alam, M. B. Cohen, S. B. Calderwood, G. K. Schoolnik and A. Camilli (2002) Nature, 417, 642-645.], from gene expression studies in minimal growth conditions and LB rich medium, and from clinical tests that identify V.cholerae. Expression data provide evidence supporting 92 (53%) of the 171 pathways. The clinical tests provide evidence supporting seven pathways, with six pathways supported by both methods. VchoCyc provides biologists with a useful tool for analyzing this organism''s metabolic and genomic information, which could lead to potential insights into new anti-bacterial agents. VchoCyc is available in the BioCyc database collection (http://BioCyc.org).', '1132/NCBITaxon:666', '1136/efo:EFO_0002768'
  UNION SELECT 24, 'E-SMDB-2855', 23, 'Comparative genomic hybridization of 5 hominid species, including humans', 'Given that gene duplication is a major driving force of evolutionary change and the key mechanism underlying the emergence of new genes and biological processes, this study sought to use a novel genome-wide approach to identify genes that have undergone lineage-specific duplications or contractions among several hominoid lineages. Interspecies cDNA array-based comparative genomic hybridization was used to individually compare copy number variation for 39,711 cDNAs, representing 29,619 human genes, across five hominoid species, including human. We identified 1,005 genes, either as isolated genes or in clusters positionally biased toward rearrangement-prone genomic regions, that produced relative hybridization signals unique to one or more of the hominoid lineages. Measured as a function of the evolutionary age of each lineage, genes showing copy number expansions were most pronounced in human (134) and include a number of genes thought to be involved in the structure and function of the brain. This work represents, to our knowledge, the first genome-wide gene-based survey of gene duplication across hominoid species. The genes identified here likely represent a significant majority of the major gene copy number changes that have occurred over the past 15 million years of human and great ape evolution and are likely to underlie some of the key phenotypic characteristics that distinguish these species.', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 25, 'E-SMDB-608', 23, 'ChIP-chip by array of yeast - control and mock experiments in Lieb et al 2001', 'All control or mock immunoprecipitation experiments associated with Lieb et al. Nature Genetics, August 2001, Volume 28, Issue 4', '1132/NCBITaxon:4932', '1136/efo:EFO_0002760'
  UNION SELECT 26, 'E-SMDB-605', 23, 'ChIP-chip by array of yeast using Sir2 in Lieb et al 2001', 'All Sir2 immunoprecipitation experiments associated with Lieb et al. Nature Genetics, August 2001, Volume 28, Issue 4', '1132/NCBITaxon:4932', '1136/efo:EFO_0002760'
  UNION SELECT 27, 'E-SMDB-25', 23, 'Transcription profiling of human pediatric acute lymphoblastic leukemia obtained from patients at diagnosis. All samples contain at least 77% blasts. Asparaginase LC50 values were determined for each of these samples', 'Samples of pediatric acute lymphoblastic leukemia obtained from patients at diagnosis.  All samples contain at least 77% blasts.  Asparaginase LC50 values were determined for each of these samples. (LC50 was determined on a different aliquot of the same sample.)', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 28, 'E-SMDB-23', 23, 'Transcription profiling of HIF dependent and independent effects of VHL inactivation in C. elegans', 'The von Hippel-Lindau (VHL) tumor suppressor functions as a ubiquitin ligase that mediates proteolytic inactivation of hydroxylated a subunits of hypoxia-inducible factor (HIF). Although studies of VHL defective renal carcinoma cells suggest the existence of other VHL tumor suppressor pathways, dysregulation of the HIF transcriptional cascade has extensive effects that make it difficult to distinguish whether, and to what extent, observed abnormalities in these cells represent effects on pathways distinct from HIF. Here, we report on a genetic analysis of HIF dependent and independent effects of VHL inactivation by studying gene expression patterns in C. elegans. We show tight conservation of the HIF-1/VHL-1/EGL-9 hydroxylase pathway. However, persisting differential gene expression in hif-1 versus hif-1; vhl-1 double mutant worms clearly distinguished HIF-1 independent effects of VHL-1 inactivation. Genomic clustering, predicted functional similarities, and a common pattern of dysregulation in both vhl-1 worms and a set of mutants (dpy-18, let-268, gon-1, mig-17 and unc-6), with different defects in extracellular matrix formation, suggest that dysregulation of these genes reflects a discrete HIF-1 independent function of VHL-1 that is connected with extracellular matrix function.', '1132/NCBITaxon:6239', '1136/efo:EFO_0002768'
  UNION SELECT 29, 'E-SMDB-21', 23, 'Transcription profiling of human tissues or cell line polyA+ RNA-labeled with Cy5 was hybridized against a common reference pool consisting of 11 different cell lines', 'Each tissue or cell line polyA+ RNA-labeled with Cy5 was hybridized against a common reference pool consisting of 11 different cell lines described elsewhere (Perou et al. 2000)', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 30, 'E-SMDB-18', 23, 'Transcription profiling of human serum stimulated fibroblasts', 'Serum was added to fibroblasts, and samples were taken.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 31, 'E-SMDB-15', 23, 'Transcription profiling of yeast treated with calcium, sodium', 'In Saccharomyces cerevisiae, the Ca2+/calmodulin-dependent protein phosphatase, calcineurin, is activated by specific environmental conditions, including exposure to Ca2+ and Na+, and induces gene expression by regulating the Crz1p/Tcn1p transcription factor. We used DNA microarrays to perform a comprehensive analysis of calcineurin/Crz1p-dependent gene expression following addition of Ca2+ (200 mM) or Na+ (0.8 M) to yeast. 163 genes exhibited increased expression that was reduced 50% or more by calcineurin inhibition. These calcineurin dependent genes function in signaling pathways, ion/small molecule transport, cell wall maintenance, vesicular transport, and include many open reading frames of heretofore-unknown function. Three distinct gene classes were defined: 28 genes displayed calcineurin-dependent induction in response to Ca2+ and Na+, 125 showed calcineurin-dependent expression following Ca2+ but not Na+ addition, and 10 were regulated by calcineurin in response to Na+ but not Ca2+. Analysis of crz1D cells established Crz1p as the major effecter of calcineurin-regulated gene expression in yeast. We identified the Crz1p binding site as 5-GNGGC(G/T)CA-3 by in vitro site selection. A similar sequence, 5-GAGGCTG-3, was identified as a common sequence motif in the upstream regions of calcineurin/Crz1p-dependent genes. This finding is consistent with direct regulation of these genes by Crz1p.', '1132/NCBITaxon:4932', '1136/efo:EFO_0002768'
  UNION SELECT 32, 'E-SMDB-8', 23, 'Transcription profiling of yeast treated with calcium + FK506 time series', 'Cells were grown at 30oC in YPD medium (buffered to pH 5.0 with 7.5 mM succinate) containing ET solution to a density of 0.6 X 107 cells/ml (A600 0.6). The culture was supplemented with FK506 (1.0 mg/ml final concentration) in ET at 30oC for 15 min. Cells were collected for the t=0 min sample. Next, an equal volume of YPD medium (pH 5.0) containing FK506 (1.0 mg/ml) and 0.4 M CaCl2 was added to the FK506-treated culture (final concentration 0.2 M CaCl2). Cells were collected by centrifugation at 5, 15, 30 and 60 min, frozen at 80oC and processed for RNA extraction.  For the publication this experiment was done twice, and corresponding time points were averaged.  The first 4 and last 4 arrays in the set define each experiment.', '1132/NCBITaxon:4932', '1136/efo:EFO_0002768'
  UNION SELECT 33, 'E-SMDB-6', 23, 'Transcription profiling of HPEC senescent vs immortalized cells', 'Cellular senescence forms a barrier that inhibits the acquisition of an immortal phenotype, a critical feature in tumorigenesis. The inactivation of multiple pathways that positively regulate senescence are required for immortalization. To identify these pathways in an unbiased manner, we performed DNA microarray analyses to assess the expression of 20,000 genes in human prostate epithelial cells (HPECs) passaged to senescence. These gene expression patterns were then compared with those of HPECs immortalized with the human Papillomavirus 16 E7 oncoprotein. Senescent cells display gene expression patterns that reflect their nonproliferative, differentiated phenotype and express secretory proteases and extracellular matrix components. A comparison of genes transcriptionally up-regulated in senescence to those in which expression is significantly down-regulated in immortalized HPECs identified three genes: the chemokine BRAK, DOC1, and a member of the insulin-like growth factor axis, IGFBP-3. Expression of these genes is found to be uniformly lost in human prostate cancer cell lines and xenografts, and previously, their inactivation was documented in tumor samples. Thus, these genes may function in novel pathways that regulate senescence and are inactivated during immortalization. These changes may be critical not only in allowing cells to bypass senescence in vitro but in the progression of prostate cancer in vivo.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 34, 'E-RZPD-10', 23, 'Transcription profiling of human dermal enothelial cells treated with 200ng/ml human recombinant endostatin (Pichia Pastoris)', 'Cells were treated with 200ng/ml human recombinant endostatin (Pichia Pastoris) for 6h at 37°C with 5 % CO2 and 95 % humidity.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 35, 'E-RZPD-7', 23, 'Transcription profiling of rat ovaries after 48 h of drug treatments.S1)1 IU PMSG + 1 IU hCG (this is the control)S2) 10 IU PMSG + 10 IU hCGS3) 40 IU PMSG + 30 IU hCG', 'the aim of this work is to gain insight into Ovarianhyperstimulation Syndrome.                                All samples were mRNA from rat ovaries after 48 h of  drug treatments.S1)1 IU PMSG + 1 IU hCG (this is the control)S2) 10 IU PMSG + 10 IU hCGS3) 40 IU PMSG + 30 IU hCG', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 36, 'E-RZPD-5', 23, 'Transcription profiling of rat heart (Sprague-Dawley) after subtotal nephrectomy (SNX) vs sham (sham) operation at 2 and 12 week timepoints', 'Rats (Sprague-Dawley) were allotted to subtotal nephrectomy (SNX) or sham (sham) operation. After 2 and 12 weeks the experiment was terminated. RNA was isolated from small pieces of the hearts.', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 37, 'E-RZPD-4', 23, 'Transcription profiling of U937 promonocytic cell line vs U937 promonocytic cell line, infected with wiltdype parvovirus H1 (H1_wt)', '1) Ctrl Probe: U937 promonocytic cell line. 2) U937 promonocytic cell line, infected with wiltyp parvovirus H1 (H1_wt).', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 38, 'E-RZPD-3', 23, 'Transcription profiling of rat brain response to 6-OHDA-treatment', 'influence of  6-OHDA-treatment on Rat-brain gene expression', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 39, 'E-RZPD-2', 23, 'Transcription profiling of 5 month old mouse', 'unknown', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 40, 'E-RZPD-1', 23, 'Transcription profiling of 5 month old mouse', 'unknown', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 41, 'E-MTAB-1328', 23, 'Methylome, transcriptome and miRNome profiling by array and high throughput sequencing of 89 patients with head and neck squamous cell carcinoma', 'A Cartes d''Identite des Tumeurs (CIT) project from the french Ligue Nationale Contre le Cancer (http://cit.ligue-cancer.net); Integrative study based on 89 patients with Head and Neck Squamous Cell Carcinoma (HNSCC); 89 Affymetrix HG-U133 Plus 2.0 GeneChips arrays;  88 samples on HumanCNV370-Quad GeneChips arrays; 84 samples on Illumina Human Methylation 27K arrays;Micro-RNA sequencing of  64 tumors. Please note: Characteristics[MetastasisFreeSurvivalEvent] indicates if there is a metastasis-free survival for a particular sample / patient (it is like an event-free survival variable where the event considered is a metastasis).  Characteristics[MetastasisFreeSurvivalDelay] - whenever the MetastasisFreeSurvivalEvent is 1, gives the duration of the metastasis-free survival period in months.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003737'
  UNION SELECT 42, 'E-MIMR-42', 23, 'Transcription profiling of mouse adipocytes to determine the role of RIP140 in adipocyte differentiation', 'Title: Gene Expression analysis of the role of RIP140 in adipocyte differentiation Description: RIPKO cells are mouse embryo fibroblasts (MEF) that were derived from      RIP140-knockout mice and have been established as a RIP140 null cell line,     while RIPKO-L7 and RIPKO-L16 are RIPKO cells for which the expression of     RIP140 has been reintroduced using a lentiviral expression system.       Using an established standard cocktail of hormones that includes IBMX,      insulin, dexamethasone and a PPAR gamma agonist the cells were      differentiated into adipocytes and RNA isolated at day 0 and day 10      after the addition of the cocktail.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 43, 'E-MIMR-17', 23, 'Transcription profiling of human indolent and aggressive forms of Chronic Myeloid Leukaemia (CML)', 'Title: Gene expression analysis of indolent and aggressive forms of Chronic Myeloid Leukaemia (CML).    Description: Chronic Myeloid Leukaemia presents in chronic phase (CP) and terminates in ''blast crisis''. Despite a common abnormality, the duration of CP is variable. The aim is to compare the gene expression profiles of the indolent and aggressive forms of CML. All samples were taken within 3 months of first diagnosis. Indolent patients were defined by chronic phase CML, duration minimum 7 years. Aggressive patients were defined by chronic phase, duration maximum 3 years.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 44, 'E-MIMR-7', 23, 'Transcription profiling of BXSB mice to study lupus erythematosus', 'Genetic basis of systemic lupus erythematosus in BXSB mice.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 45, 'E-MEXP-3675', 23, 'Transcription profiling by array of Candida albicans grown in different pH and media over time', 'comparison of. C.albicans grown on different pH and media over time', '1132/NCBITaxon:5476', '1136/efo:EFO_0002768'
  UNION SELECT 46, 'E-MEXP-46', 23, 'Transcription profiling of mouse B6 and A/J mouse strains', 'A cDNA variability comparison between B6 and A/J mouse strains.  This is a technical replicate flip design experiment, where each RNA will have 3 separate cDNA syntheses for a total of 6 slides.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 47, 'E-MEXP-34', 23, 'Transcription profiling time course experiment performed over 24 hours to look at the effects on gene expression of exposure to low red:far-red ratio light in Arabidopsis thaliana plants', 'This experiment was a time course performed over 24 hours to look at the effects on gene expression of exposure to low red:far-red ratio light in Arabidopsis thaliana plants. In this way genes involved in the shade avoidance response might be identified. This experiment was designed for gene identification only and containes no replicates,genes identified were verified by quantitative PCR for publication.', '1132/NCBITaxon:3702', '1136/efo:EFO_0002768'
  UNION SELECT 48, 'E-MEXP-32', 23, 'Transcription profiling of human skin biopsies from individuals with a diagnosis of systemic sclerosis with diffuse scleroderma vs biopsies from normal individiduals', 'We used DNA microarrays to characterize gene expression patterns in skin biopsies from individuals with a diagnosis of systemic sclerosis with diffuse scleroderma and compared those to the patterns of gene expression seen in biopsies from normal, unaffected individuals.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 49, 'E-MEXP-5', 23, 'Transcription profiling of mouse duodenum and liver with primary (Hfe ?/? and C282Y homozygous mice) and secondary iron overload vs 129S6/SvEvTac wild type controls', 'We determined duodenal and liver gene response patterns in mice with primary (Hfe ?/? and C282Y homozygous mice) and secondary iron overload versus 129S6/SvEvTac wild type controls.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 50, 'E-MAXD-11', 23, 'Comparative genomic hybridization of twelve strains of coccolithovirus to determine genomic content', 'The coccolithoviruses contained within the Plymouth Virus Collection (PVC) were screened for genomic content using a microarray. The diversity in genomic content was assessed for twelve strains: EhV-86, on which the microarray is based; EhV-84, EhV-88, EhV-163, EhV-V2, EhV-201, EhV-202, EhV-205, EhV-206, EhV-207, EhV-208 and EhV-209. A direct labelling method was employed to label each virus'' genomic DNA which was hybridised onto a separate microarray slides.', '1132/NCBITaxon:2903', '1136/efo:EFO_0000749'
  UNION SELECT 51, 'E-MAXD-8', 23, 'Transcription profiling of EhV-86 Infection of E. huxleyi 1516', '', '1132/NCBITaxon:2903', '1136/efo:EFO_0002768'
  UNION SELECT 52, 'E-LGCL-3', 23, 'Transcription profiling of human brain total rna against universal human reference rna (both commercially available) prepared at two different starting amounts (20 ug or 1ug', 'Commercially available human genomic microarrays from four different manufacturers were used to compare Human Brain Total RNA against Universal Human Reference RNA (both commercially available) prepared at two different starting amounts (20 µg or 1µg). For each level of RNA, 6 replicates were performed with Human Brain Total RNA labelled with Cy3, and Universal Human Reference RNA labelled with Cy5. The labelling was then reversed (dye flip) creating another 6 replicates. This meant that for each of the four manufacturers there were a total of 24 arrays. Image processing was performed with two different software packages, and data was normalized with three different strategies.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 53, 'E-LGCL-2', 23, 'Transcription profiling of human brain total RNA vs Universal Human Reference RNA on 4 different commercially available microarrays', 'Commercially available human genomic microarrays from four different manufacturers were used to compare Human Brain Total RNA against Universal Human Reference RNA (both commercially available) prepared at two different starting amounts (20 µg or 1µg). For each level of RNA, 6 replicates were performed with Human Brain Total RNA labelled with Cy3, and Universal Human Reference RNA labelled with Cy5. The labelling was then reversed (dye flip) creating another 6 replicates. This meant that for each of the four manufacturers there were a total of 24 arrays. Image processing was performed with two different software packages, and data was normalized with three different strategies.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 54, 'E-IPKG-1', 23, 'Transcription profiling of pea seed development', 'SnRK1 (sucrose-non-fermenting-1-related) protein kinases are involved in the regulation of plant metabolism controlling both gene expression and phosphorylation. The aim of the study was to investigate the role of SnRK1 in pea seed development. To study the effect of SnRK1 deficiency, transgenic pea plants were generated carrying a gene for VfSnRK1 in antisense orientation under control of seed specific vicilin promoter. Selected transgenic lines were characterized with decreased levels of PsSnRK1 mRNA and reduced up to 71% phosphorylation activity. Antisense inhibition of SnRK1 resulted in reduced seeds fresh weight, defect of pollen development. To dissect the SnRK1-antisense phenotype at the molecular level, a search for genes with differential expression patterns in transgenic plant versus wild type seeds has been performed using cDNA macroarray analysis. Radioactive labeled cDNA probes were prepared from RNA isolated from embryo of developing seeds of wild type (11, 13, 15, 17, 19, and 21 DAP) and transgenic SnRK1-antisense plant (13, 15, 17 and 19 DAP), which correspond to the transition phase of seed development, and hybridized to cDNA macroarrays.', '1132/NCBITaxon:3888', '1136/efo:EFO_0002768'
  UNION SELECT 55, 'E-GEOD-13561', 23, 'Transcriptional Control by the FATC Domain of Saccharomyces cerevisiae Tra1', 'Tra1 is a component of the Saccharomyces cerevisiae SAGA and NuA4 complexes and a member of the phosphatidylinositol 3-kinase (PI3K) related kinase family that contain a C- terminal PI3K domain followed by a ~ 35-residue FATC domain. We have characterized four alleles with single residue changes in the FATC domain. Of these tra1-L3733A had the most pronounced effects with phenotypes including temperature and cold sensitivity, and reduced growth in media containing ethanol, Calcofluor white, rapamycin, chloramphenicol and geneticin. Tra1-L3733A interacted at normal levels with components of the NuA4 and SAGA complexes, and did not significantly alter histone acetylation patterns. The tra1-L3733A allele resulted in two-fold or greater change in expression of approximately 11% of yeast genes in rich media. Of the 279 genes with increased expression, 175 were ribosomal subunits or involved in ribosomal function or biogenesis. Elevated levels of Pol I and Pol III transcripts were also observed. The phenotypes of the tra1-L3733A overlapped with but were not identical to strains containing deletions of SAGA or NuA4 components or with strains containing mutations in the PI3K domain. Our finding that the double mutant allele, tra1-SRR3413/L3733A with alterations in the PI3K and FATC domains, resulted in wild type growth, suggests a model whereby the FATC domain negatively regulates the activity of the PI3K domain. Expression of genes involved in ribosome biosynthesis, other than the ribosomal subunits themselves, returned to near normal levels in the double mutant strain.  We also characterized tra1-G3745, which contains an additional glycine residue following the normal C-terminal phenylalanine. This allele did not support viability and showed severe dominant negative effects. In contrast to what was observed for tra1-L3733A, tra1-G4745 resulted in decreased expression of genes required for ribosome biogenesis and did not interact with Esa1 or Spt7. Three biological replicate experiments including one dye-swap were performed for yeast strains CY3003(TRA1::Tn10LUK with IB150(myc9-tra1_L3733A-YCplac111)) and CY3015(TRA1::Tn10LUK with IB157(myc9-tra1_SRR3413_L3733A-YCplac111)) with reference to CY2706(TRA1::Tn10LUK with 1980(myc9-TRA1-YCplac111)).  Similarly, three biological replicate experiments including one dye-swap were performed for yeast strain CY3019(TRA1::Tn10LUK with 1259(myc-TRA1-YCplac111) and IB162(myc9-tra1_G3745-YCplac111)) with reference to CY3020(TRA1::Tn10LUK with 1259(myc-TRA1-YCplac111) and IB160(myc9-TRA1-YCplac111)).', '1132/NCBITaxon:4932', '1136/efo:EFO_0002768'
  UNION SELECT 56, 'E-FPMI-1', 23, 'Transcription profiling of ciprofloxacin induction of a susceptibility determinant in Pseudomonas aeruginosa', 'PAO1 was grown in sub-inhibitory ciprofloxacin (0.1x-, 0.3x-, 1x-MIC) until log phase. Microarrays were done on total RNA isolated from these cultures. Loop design and dye swapping were used.', '1132/NCBITaxon:287', '1136/efo:EFO_0002768'
  UNION SELECT 57, 'E-ERAD-145', 23, 'RNA-seq of human breast cancer cell lines', 'We propose to definitively characterise the somatic genetics of ER+ve, HER2-ve breast cancer through generation of comprehensive catalogues of somatic mutations in breast cancer cases by high coverage genome sequencing coupled with integrated transcriptomic and methylation analyses.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 58, 'E-ERAD-144', 23, 'RNA-seq of human Ewings Sarcoma cell lines', 'Illumina RNA-Seq will be performed on four Ewings sarcoma cell lines and two control cell lines. RNA was extracted from all the lines using a basic Trizol extraction protocol.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 59, 'E-DKFZ-1', 23, 'Transcription profiling of 74 kidney tumor samples of different histological type, differentiation grade, stage and with data on chromosomal aberrations and follow-up', 'Study comprising 74 kidney tumor samples of different histological type, differentiation grade, stage and with data on chromosomal aberrations and follow-up. Samples were hybridised against a common reference obtained from pooling different kidney tumor samples.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 60, 'E-BAIR-12', 23, 'Transcription profiling of epidydimal fat pads in mice following high fat and normal diets to study insulin resistance and obesity', 'Gene transcription profiling in Epidydimal fat pad (EPD) following High fat versus normal diet in C57/Bl6 mice.                         Characterisation of molecular mechanisms of in vivo insulin action in mouse models of experimentally induced Insulin resistance.                        Switched the diet when mice were 5 weeks old, then sampled at 8 days.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 61, 'E-BAIR-11', 23, 'Transcription profiling of muscle cells in mice at different different time points following muscle and normal diets to study insulin resistance and obesty', 'Gene transcription profiling in muscle following muscle versus normal diet in C57/Bl6 mice. Switched the diet when mice were 5 weeks old, then sampled at 8 days, 3 weeks and 15 weeks.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 62, 'E-BAIR-10', 23, 'Transcription profiling of mouse liver following high fat and normal diets to study insulin resistance and obesity', 'Gene transcription profiling in liver following high fat versus normal diet in C57/Bl6 mice.                                        Characterisation of molecular mechanisms of in vivo insulin action in mouse models of experimentally induced Insulin Resistance.                                        Switched the diet after mice are 5 weeks old, then sampled at 8 days, 3 weeks and 15 weeks.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 63, 'E-BAIR-9', 23, 'Transcription profiling of epidydimal fat pads in mice at different time points following high fat and normal diets to study insulin resistance and obesity', 'Gene transcription profiling in Epidydimal fat pad (EPD) following High fat versus normal diet in C57/Bl6 mice. The diet was switched when mice were 5 weeks old, then samples taken at 2 days, 8 days, 3 weeks and 15 weeks.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 64, 'E-BAIR-8', 23, 'Transcription profiling of mouse finally differentiated 3T3-L1 adipocytes treated with insulin (0 or 100nM)or metformin (0 or 2mM)for 2 and 12 hours', 'Finally differentiated 3T3-L1 adipocytes are treated with insulin (0 or  100nM)or metformin (0 or 2mM)for 2 and 12 hours to understand insulin  and metformin(an anti-diabetic drug commonly applied for Non-Insulin Dependent Diabetes Mellitus)action in adipose tissues.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 65, 'E-BAIR-1', 23, 'Transcription profiling of white adipose tissue in five week old Insulin Receptor Substrate-2 (IRS-2) mice', 'Mice which lack IRS-2, develop insulin resistance. Comparison between white adipose tissue from 5 week old IRS-2 KO and wild type mice. Processed data is not included for this experiment and therefore these data are not MIAME compliant', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 66, 'E-BASE-1', 23, 'Transcription profiling of human monocyte cell line treated with medicinal extract from Agaricus blazei Murill', 'The effect on gene expression of an extract from a medicinal mushroom (Agaricus blazei Murill)were compared with the effect of lipopolysaccharides (LPS). Both are potent immunomodulators. The study used a monocyte cell line.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 67, 'E-ERAD-92', 23, 'RNA-seq of Strongyloides ratti in different developmental stages', 'Strongyloides ratti is a common gastro-intestinal parasite of the rat. The adult parasites are female only, about 2mm long and live in the mucosa of the small intestine. These parasites produce eggs that pass out of the host in its faeces. In the environment infective larval stages develop either directly or after a facultative sexual free-living adult generation. Infective larvae infect hosts by skin penetration.S. ratti is the laboratory analogue of the parasite of humans, S. stercoralis. S. stercoralis is a wide-spread parasite of humans, occurring principally in the tropics and sub-tropics: some 100-200 million people are infected worldwide. Infection of immunosuppressed individuals can result in disseminated strongyloidiasis, in which worms occur throughout the body. This can be fatal unless anti-Strongyloides therapy is given. Other species of Strongyloides parasitise a wide range of vertebrates.  As part of the Strongyloides ratti genome project we are profiling the transcriptome of the parasite across its life cycle using RNA-Seq.. This data is part of a pre-publication release. For information on the proper use of pre-publication data shared by the Wellcome Trust Sanger Institute (including details of any publication moratoria), please see http://www.sanger.ac.uk/datasharing/', '1132/NCBITaxon:34506', '1136/efo:EFO_0003738'
  UNION SELECT 68, 'E-ERAD-69', 23, 'ChIP-seq of mouse mammary cell line to identify BCL11A targets', 'We have recently identified BCL11A as a key regulator of mammary stem (MaSC) and progenitor cells. Deletion of BCL11A in MaSC leads to loss of ability of those cells to engraft a cleared mammary fatpad. This biological function of BCL11A correlates with its overexpression in basal breast cancers which are thought to arise from stem and progenitor subtypes. Basal breast cancer cell lines where BCL11A is knocked down using shRNA failed to develop tumours in xenograft transplantations in mice. Therefore, BCL11A is potentially a novel target for breast cancer treatment. Microarray analysis has identified potential gene targets for BCL11A however, it is not known if these targets are directly regulated by BCL11A at the transcription level. We would like to perform CHIP-SEQ analysis on a mammary cell line using BCL11A antibody under two conditions (normal expression and overexpression of BCL11A).This data is part of a pre-publication release. For information on the proper use of pre-publication data shared by the Wellcome Trust Sanger Institute (including details of any publication moratoria), please see http://www.sanger.ac.uk/datasharing/protocol to be provided', '1132/NCBITaxon:10090', '1136/efo:EFO_0002692'
  UNION SELECT 69, 'E-ERAD-67', 23, 'RNA-seq of Onchocerca volvulus in different developmental stages', 'Onchocerca volvulus is a filarial nematode parasite of humans, causing Onchocerciasis, or River Blindness, which affects over 37 million people, mainly in Africa. It is a severely debilitating disease, which is transmitted to humans by black fly.  This project aims to undertake high-throughput sequencing of Onchocerca volvulus transcriptome for de novo assembly of transcripts. The main objective of this project is to recognize genes expressed in given life stages.. This data is part of a pre-publication release. For information on the proper use of pre-publication data shared by the Wellcome Trust Sanger Institute (including details of any publication moratoria), please see http://www.sanger.ac.uk/datasharing/', '1132/NCBITaxon:6282', '1136/efo:EFO_0003738'
  UNION SELECT 70, 'E-ERAD-56', 23, 'RNA-seq of Hymenolepis microstoma  in different developmental stages with samples from different organism parts', 'Hymenolepis spp. (H. diminuta, H. nana and H. microstoma) are rodent-hosted tapeworms (Platyhelminthes: Cestoda) that have been used as laboratory and teaching models since the 1950s, and consequently much of our understanding of the basic physiology, biochemistry and anatomy of tapeworms in general stems from research using these species. As representatives of the order Cyclophyllidea, they are closely related to species with significant medical and economic importance such as Taenia and Echinococcus spp., but unlike these may be maintained in vivo using only laboratory mice and flour beetles (n.b. Echinoccous spp. are hosted by foxes and Taenia spp. are hosted by pigs or cows). This effort brings a classical laboratory model into the genomic age, allowing researchers in silico access to its genome and expressed gene transcripts and thereby greatly expediting research directed at understanding the genetic basis of tapeworm biology. Dujardin 1845 is NHM laboratory strain since 2005, maintained using outbred conventional mice (Mus musculus) and confused flour beetles (Tribolium confusum). Seed culture gifted from laboratory strain maintained at Univ Nottingham, UK (via Prof Jerzy Behnke). Nottingham culture is 30+ years old and was seeded from a laboratory culture maintained at Univ Glasgow, UK.Common name: bile duct rodent tapeworm. Synonyms: Taenia microstoma Dujardin 1845, Vampirolepis microstoma, Rodentolepis microstoma.', '1132/NCBITaxon:85433', '1136/efo:EFO_0003738'
  UNION SELECT 71, 'E-MEXP-3723', 23, 'Transcription profiling by array of Medicago truncatula wildtype and MtRSD mutant subjected to  inoculation with Sinorhizobium meliloti strain 1021', 'Medicago wildtype and MtRSD mutant were subjected to nodulation and sample were harvested from susceptible zones of inoculated roots  0-8 days post innoculation', '1132/NCBITaxon:3880', '1136/efo:EFO_0002768'
  UNION SELECT 72, 'E-MTAB-964', 23, 'Transcription profiling by high throughput sequencing of 7 tissues of maize inbred line Zheng 58', 'RNA-sequencing of 7 tissues of maize inbred line Zheng 58: mature silk (MS), mature pollen (MP), mature ovary (MO), and 6-days old seedling (SL), immature silk (IMS), silk at 20min after pollination (20MAP), silk at 3h after pollination (3HAP)', '1132/NCBITaxon:4577', '1136/efo:EFO_0003738'
  UNION SELECT 73, 'E-ERAD-68', 23, 'RNA-seq of Brugia pahangi in different developmental stages', 'Brugia pahangi is a parasitic nematode that is closely related to B. malayi and Wuchereria bancrofti.  B. malayi and W. bancrofti are responsible for lymphatic filariasis, affecting around 120 million people in 73 countries worldwide.This project aims to undertake high-throughput sequencing of Brugia pahangi transcriptome. The objective is to use transcriptomics to support gene finding and to recognize genes expressed in given life stages.', '1132/NCBITaxon:6280', '1136/efo:EFO_0003738'
  UNION SELECT 74, 'E-MTAB-1445', 23, 'RNA-Seq of Salix suchowensis male and female buds to investigate differentially expressed genes', 'The gene expressed differently between female and male flower buds in Salix suchowensis.', '', '1136/efo:EFO_0003738'
  UNION SELECT 75, 'E-MTAB-1410', 23, 'Comprehensive gene expression profiles of NK cell neoplasms identify vorinostat as an effective drug candidate', 'Transcriptional profiling of NK cell lines (NKL & NKYS) with DMSO or vorinostat', '1132/NCBITaxon:9606', '1136/efo:EFO_0001033'
  UNION SELECT 76, 'E-MTAB-1358', 23, 'Transcription profiling by array of H. sapiens primary tumor samples related to mutation status of FGFR2, KRAS and PIK3CA to investigate expression profiles', 'The aim is to investigate expression patterns of 69 primary tumor samples related to mutation status of FGFR2, KRAS and PIK3CA, and clinico-pathological data.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 77, 'E-MTAB-1308', 23, 'Foxa2 and H2A.Z Mediate Nucleosome Depletion during Embryonic Stem Cell Differentiation - gene expression', 'We determine the mechanisms that control global nucleosome dynamics during embryonic stem (ES) cell differentiation into endoderm. Three cell types (ES, pEHP, and EHP) were assayed for RNA expression levels.', '1132/NCBITaxon:10090', '1136/efo:EFO_0001030'
  UNION SELECT 78, 'E-MEXP-3775', 23, 'Transcription profiling by array of rat preangiogenic and angiogenic tumor nodules to identify genes that were differentially expressed during the transition from the preangiogenic to the angiogenic phenotype.', 'Transcriptional profiling of preangiogenic and angiogenic tumor nodules was conducted to identify genes that were differentially expressed during the transition from the preangiogenic to the angiogenic phenotype.', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 79, 'E-MEXP-3522', 23, 'Transcription profiling by array of mice liver hepatocytes after 4-hour refeeding as compared to overnight starved controls.', 'Transcriptional and polysomal analysis of mice liver hepatocytes after 4-hour refeeding as compared to overnight starved controls.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 80, 'E-MEXP-3720', 23, 'Transcription profiling by array of human A375 melanoma cell line (ATCC, CRL-1619) to investigate the dynamic regulation and co-regulation of miRNAs and mRNAs', 'To investigate the dynamic regulation and co-regulation of miRNAs and mRNAs, a time-course experiment was performed as described before (E-MEXP-3544). Briefly, the time course experiment was carried out using the human A375 melanoma cell line (ATCC, CRL-1619).', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 81, 'E-MEXP-3350', 23, 'Transcription profiling by array of mouse bone marrow derived eosinophils mutant for miR223', 'Gene expression profiling of bone marrow derived eosinophils from miR-223 wildtype or miR-223 deficient mice', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 82, 'E-GEOD-43669', 23, 'Identification and Analysis of Red Sea Mangrove (Avicennia marina) microRNAs by High-Throughput Sequencing and their Association with Stress Responses', 'we used high-throughput Illumina Genome Analyzer IIx (GAIIx) technology to sequence the small RNA transcriptomes of the mangrove species, Avicennia marina. Based on sequence similarity or the secondary structure of precursors, we have identified 193 conserved miRNAs and 26 novel miRNAs in the small RNA transcriptome of Avicennia marina. 1 sample', '1132/NCBITaxon:82927', '1136/efo:EFO_0003737'
  UNION SELECT 83, 'E-GEOD-43653', 23, 'Expression data from endometrial carcinoma cells', 'some genes associated with cell growth, cell differentiation, meiosis, migration, and apoptosis are regulated in ECC-1 after DEHP exposure. We used microarrays to detail the global programme of gene expression underlying cellularisation and identified distinct classes of up-regulated genes during this process. ECC-1 (human endometrial cancer) cell lines were subjected to DEHP for 48 h with 50 ?M DEHP and total RNA was extracted. Quantified spectro-photometrically, and RNA quality was verified via agarose-gel electrophoresis. Gene expression changes were analyzed using oligonucleotide microarray hybridization by comparing the treated and control groups (two-fold change, P<0.05).', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 84, 'E-GEOD-43651', 23, 'Regulation of the epithelial adhesion molecule CEACAM1 is essential for palate formation.', 'Cleft palate results from a mixture of genetic and environmental factors and occurs when the bilateral palatal shelves fail to fuse. The objective of this study was to search for new genes involved in mouse palate formation. Gene expression of murine embryonic palatal tissue was analyzed at the various developmental stages before, during, and after palate fusion using GeneChip? microarrays. Ceacam1 was one of the highly up-regulated genes during and after fusion in palate formation, and this was confirmed by quantitative real-time PCR. Immunohistochemical staining showed that CEACAM1 was expressed at a very low level in palatal epithelium before fusion, but highly expressed in the midline of the palate during and after fusion. To investigate the developmental role of CEACAM1, function-blocking antibody was added to embryonic mouse palate in organ culture. Palatal fusion was inhibited by this function-blocking antibody. To investigate the subsequent  developmental role of CEACAM1, we characterized Ceacam1-deficient (Ceacam1-/-) mice. Epithelial cells persisted abnormally at the midline of the embryonic palate even on day E16.0, and palatal fusion was delayed in Ceacam1-/- mice. TGF?3 expression, apoptosis, and cell proliferation in palatal epithelium were not effected in the palate of Ceacam1-/-mice. CEACAM1 expression was down-regulated in Tgfb3-/- palate. However, exogenous TGF?3 did not induce CEACAM1 expression. These results suggest that CEACAM1 has roles in both the initiation of palate formation via epithelial cell adhesion and TGF signaling has some indirect effect on CEACAM1.  Global gene expression profiling of palatal processes before, during and after fusion of palatal shelves We used microarray to investigate the gene expression of palatal tissue during palatal development. Palatal processes were microdissected  at the stages of palatal development (before, during and after fusion) for RNA extraction and hybridization on Affymetrix microarrays.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 85, 'E-GEOD-42913', 23, 'A comprehensive gene expression analysis of resistance formation upon metronomic cyclophosphamide therapy', 'Resistance formation is one of the major hurdles in cancer therapy. Metronomic anti-angiogenic treatment of xenografted prostate cancer tumors in SCID mice with cyclophosphamide (CPA) results in the appearance of resistant tumors. To investigate the complex molecular changes occurring during resistance formation, we performed a comprehensive gene expression analysis of the resistant tumors in vivo. We observed a multitude of differentially expressed genes, e.g., PASD1, ANXA3, NTS or PLAT, when comparing resistant to in vivo passaged tumor samples. Furthermore, tumor cells from in vivo and in vitro conditions showed a significant difference in target gene expression. We assigned the differentially expressed genes to functional pathways like axon guidance, steroid biosynthesis and complement and coagulation cascades. Most of the genes were involved in anti-coagulation, indicating its possible importance. Upregulation of anti-coagulatory ANXA3 and PLAT and downregulation of PLAT inhibitor SERPINA were validated by qPCR. In contrast, coagulation factor F3 was upregulated, accompanied by the expression of an altered gene product. These findings give insights into the resistance mechanisms of metronomical CPA treatment suggesting an important role of anti-coagulation in resistance formation. 4 cell lines * 2 treatments * 4 replicates = 32 arrays. One sample (C3-T) was identified as an outlier and was omitted from further analysis; it is not included here.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 86, 'E-GEOD-33273', 23, 'Transcriptome analysis of primary human monocyte differentiation', 'In order investigate the control of genes encoding cytoskeletal motor proteins and their interaction partners in the activation program of primary monocytes, we performed whole transcriptome microarray profiling of ex vivo monocyte differentiation to activated macrophages or dendritic cells in monocytes isolated from three anonymous blood donors. Monocytes were isolated from the buffy coats of three anonymous blood donors and differentiated into activated macrophages or activated dendritic cells (DCs) ex vivo. Total RNA from monocytes, activated macrophages or DCs was isolated and submitted for whole transcriptome microarray analysis.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 87, 'E-GEOD-33272', 23, 'Transcriptome analysis of primary human CD4+ T-cell activation', 'In order investigate the control of genes encoding cytoskeletal motor proteins and their interaction partners in primary T-cells, we performed whole transcriptome microarray profiling of cell activation for memory and naïve T-cells isolated from three anonymous blood donors. CD4+CD25- naïve or memory T-cells were cultured in medium alone or stimulated ex vivo and harvested for total RNA isolation and whole transcriptome microarray analysis.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 88, 'E-GEOD-33271', 23, 'Expression of motor proteins in transgenic HeLa cell lines', 'We employed whole transcriptome microarray profiling to measure the expression of myosin and kinesin genes in multiple HeLa cell lines and confirm the expression of recombinantly-tagged BAC transgenes encoding motor genes. Normal and transfected HeLa cells grown to confluence were harvested for extraction of total RNA and microarray analysis', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 89, 'E-GEOD-22087', 23, 'Global transcriptomic profiling of nitric oxide-mediated neuronal death', 'Nitric oxide (NO) is implicated in the pathogenesis of various neuropathologies characterised by oxidative stress. NO has been reported to be involved in the exacerbation of oxidative stress by various mechanisms, including protein modification, genotoxic damage and elevated production of reactive oxygen species resulting in deregulation and disruption of cellular homeostasis. Although multiple roles for NO has been reported in neuronal death signaling, existent data fail to provide a holistic description of how nitrergic pathobiology elicits neuronal injury. Here we provide a comprehensive description of mechanisms contributing to NO-induced neuronal injury by global transcriptomic profiling. Microarray analysis was carried out using 14 GeneChip Mouse Genome 430 2.0 array (Affymetrix, Santa Clara, CA). The assignment of the arrays (GeneChip) was as follows: vehicle-treated control (n=5); NOC-18-treatment for 8, 15 and 24 hour (n=3 for each time-point).', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 90, 'E-GEOD-16035', 23, 'Global gene profiling comparison between hydrogen sulfide- and N-methyl-D-aspartate-induced neuronal deaths', 'Hydrogen sulfide (H2S), present in abundance in the mammalian brain, has recently been demonstrated to induce a dose- and time-dependent apoptotic-necrotic continuum in murine primary cortical neurons, which was successfully attenuated upon application of N-methyl-D-aspartate (NMDA) receptor antagonist. The current study focused on gaining an insight into the molecular mechanisms of H2Smediated neuronal death pertaining to NMDA receptors activation through global gene expression  comparisons. A total of 24 RNA samples were analyzed. There are 2 treatment conditions, namely 200uM sodium hydrosulfide and 200uM N-methyl-D-asparate. 3 replicates were collected for each of the selected time-points (5h, 15h and 24h), in addition to 6 replicates of shared vehicle control.  The supplementary file ''GSE16035_non-normalized_data.txt'' contains non-normalized data for Samples GSM401312-GSM401335.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 91, 'E-MTAB-1429', 23, 'Transcription profiling by array of HM1 Mouse Embryonic Stem Cells to investigate cisplatin exposure', 'mouse embyonic stem cells/ cisplatin', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 92, 'E-GEOD-43645', 23, 'Expression analysis of Mycobacterium avium subsp. paratuberculosis K-10 within macrophage cell line', 'Once entering the cell, M. avium subsp.paratuberculosis is known to survive harsh microenvironments, especially those inside activated macrophages.  To improve our understanding of M. avium subsp.paratuberculosis pathogenesis, we examined the phagosome maturation associated with transcriptional responses of M. avium subsp.paratuberculosis during macrophage infection.  Monitoring cellular markers, only live M. avium subsp.paratuberculosis bacilli were able to prevent phagosome maturation and reduce its acidification.  On the transcriptional level, over 300 of M. avium subsp.paratuberculosis genes were significantly, differentially regulated in both naïve and IFN-?-activated macrophages.  These genes include the sigma factor H (sigH) that was shown to be important during persistent infection in M. tuberculosis. Bacterial total RNA was purified from Mycobacterium avium subsp. paratuberculosis that was infected into J774A.1 cells grown with or without interferon-gamma activation at 2 or 24 h post-infection. RNA from bacteria incubated with PRMI-1640 medium for 2 h, which was the infection conditon, is used as control condition. Each condition has two biological replicates. Each hybridization represents expression levels of all 4,350 annotated genes with 20 60-mer oligonucleotides and with three technical replicates.', '1132/NCBITaxon:262316', '1136/efo:EFO_0002768'
  UNION SELECT 93, 'E-GEOD-43512', 23, 'Epigenetic conservation at gene regulatory elements revealed by non-methylated DNA profiling in seven vertebrates', 'Two-thirds of gene promoters in mammals are associated with regions of non-methylated DNA, called CpG islands (CGIs), which counteract the repressive effects of DNA methylation. In lower vertebrates, computational CGI predictions often reside away from gene promoters, suggesting a major divergence in gene promoter architecture across vertebrates. By experimentally identifying non-methylated DNA in the genomes of seven diverse vertebrates, we instead reveal that non-methylated islands (NMIs) of DNA are a central feature of vertebrate gene promoters. Furthermore, NMIs are present at orthologous genes across vast evolutionary distances, revealing a surprising level of conservation in this epigenetic feature. By profiling NMIs in different tissues and developmental stages we uncover a unifying set of features that are central to the function of NMIs in vertebrates.  Together these findings demonstrate an ancient logic for NMI usage at gene promoters and reveal an unprecedented level of epigenetic conservation across vertebrate evolution. Bio-CAP was used to identify non-methylated regions of the genome in seven diverse vertebrates (human, mouse, platypus, chicken, lizard, frog and zebrafish) across a number of tissues.', '1132/NCBITaxon:8364', '1136/efo:EFO_0002761'
  UNION SELECT 94, 'E-GEOD-43433', 23, 'Extensive changes in DNA methylation are associated with expression of mutant huntingtin', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 95, 'E-GEOD-43188', 23, 'Deep Sequencing identifies regulated small RNAs in Dugesia japonica', 'We applied Illumina sequencing to identify microRNAs (miRNAs) and piwi-interacting small RNAs (piRNAs) in Dugesia japonica. Dugesia were cut up after seven days starvation. DJ1 is the whole Dugesia body, DJ2 is the head part, DJ3 is the tail part, DJ4 is the left part and DJ5 is the right part. Total RNA was extracted by Trizol, and preserved in ethanol, stored at -80°C until further use. 5 samples examined?DJ1,DJ2,DJ3,DJ4,DJ5. SRA study (not submitted by GEO): SRP017693', '1132/NCBITaxon:6161', '1136/efo:EFO_0003737'
  UNION SELECT 96, 'E-GEOD-43179', 23, 'MicroRNA regulate immunological pathways in T-cells in immune thrombocytopenia (ITP)', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 97, 'E-GEOD-43178', 23, 'MicroRNA regulate immunological pathways in T-cells in immune thrombocytopenia (ITP) [miRNA]', 'MicroRNA are small non-coding RNA molecules that regulate gene expression. To investigate the role of microRNA in ITP, we performed genome-wide expression analyses of mRNA and microRNA in T-cells from ITP patients and controls. We identified 1,915 regulated genes and 22 regulated microRNA that differed between ITP patients and controls. Seventeen of the 22 regulated microRNA were linked to changes in target gene expression; 57 of these target genes were associated with the immune system, e.g. T-cell activation and regulation of immunoglobulin production. CXCL13 and IL-21 were two microRNA target genes significantly increased in ITP. We could demonstrate increased plasma levels of CXCL13 and others have reported increased plasma levels of IL-21 in ITP. Thus, regulated microRNA were significantly associated with both gene and protein expression of molecules in immunological pathways. We suggest that microRNA may be important regulatory molecules involved in the loss of tolerance in ITP. Microarray expression analysis of mRNA and miRNA in peripheral blood T-cell of control and ITP patients.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 98, 'E-GEOD-43177', 23, 'MicroRNA regulate immunological pathways in T-cells in immune thrombocytopenia (ITP)  [mRNA]', 'MicroRNA are small non-coding RNA molecules that regulate gene expression. To investigate the role of microRNA in ITP, we performed genome-wide expression analyses of mRNA and microRNA in T-cells from ITP patients and controls. We identified 1,915 regulated genes and 22 regulated microRNA that differed between ITP patients and controls. Seventeen of the 22 regulated microRNA were linked to changes in target gene expression; 57 of these target genes were associated with the immune system, e.g. T-cell activation and regulation of immunoglobulin production. CXCL13 and IL-21 were two microRNA target genes significantly increased in ITP. We could demonstrate increased plasma levels of CXCL13 and others have reported increased plasma levels of IL-21 in ITP. Thus, regulated microRNA were significantly associated with both gene and protein expression of molecules in immunological pathways. We suggest that microRNA may be important regulatory molecules involved in the loss of tolerance in ITP. Microarray expression analysis of mRNA and miRNA in peripheral blood T-cell of control and ITP patients.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 99, 'E-GEOD-42867', 23, 'Gene expression changes in subclones of a Burkitt lymphoma cell line with different patterns of EBV latent infection', 'In this study we have investigated the gene expression profiles of three different types of subclone all generated by single cell cloning of the same parental EBV positive Burkitt lymphoma cell line Awia-BL. These included  EBV negative clones which have lost the virus episome, EBV positive clones with a conventional Latency I form of infection and EBV positive clones with an atypical Wp-restricted form of infection. Gene expression was compared between two different EBV negative clones (both performed in biological duplicate), four different Latency I clones and four different Wp-restricted clones using HG U133 Plus 2.0 gene chip arrays.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 100, 'E-GEOD-42844', 23, 'TDP-43 Neurotoxicity in late D. melanogaster pupae', 'The aim of this study is to understand the mechanisms of TDP-43 neurotoxicity. Here, we perform a RNA-Seq analysis in  TDP-43 gain-of-fucntion (GOF) , TDP-43 loss-of-function and wild-type late pupae heads (73-90 hours APF) and perform TDP-43 GOF vs wild type and TDP-43 LOF vs wild-type differential expression analysis to show that both mechanisms presents defects in ecdysone receptor (ECR)-dependeint transcriptional program switching, and strongly deregulate expression from the neuronal microtubule associated protien Map205. RNA-seq was performed in two wild-type D.melanogaster biological replicates (Canton S, w1118 ), four biological replicates for TDP-43 (LOF) with two distinct genotypes (dTDP-43?142/Df(2R)106,dTDP-43?23/?142 ) and two TDP-43 GOF biological replicates (act5c>dTDP-43 ).', '1132/NCBITaxon:7227', '1136/efo:EFO_0003738'
  UNION SELECT 101, 'E-GEOD-42800', 23, 'Proteome reference map of Haloarcula hispanica and comparative proteomic and transcriptomic analysis of polyhydroxyalkanoate biosynthesis under genetic and environmental perturbations', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:51589', '1136/efo:EFO_0002768'
  UNION SELECT 102, 'E-GEOD-42799', 23, 'Genes differentially expressed Haloarcula hispanica strains: PHB (?phaEC mutant) vs 960 (wild type)', 'Transcriptional profiling of Haloarcula hispanica comparing PHB with 960. PHB represents the PHA synthase genes mutant (i.e., ?phaEC) which is deficient in PHBV accumulation when grown in MG medium. 960 represents the wild-type strain which can produce PHBV when grown in MG medium. Goal was to explore the PHBV biosynthesis and its impact on central metabolism in H. hispanica. Total RNA from PHB and 960 were used to generate target cDNA, and then hybridized to 8*15K Haloarcula hispanica genome array genechips, representing about 3800 genes.', '1132/NCBITaxon:51589', '1136/efo:EFO_0002768'
  UNION SELECT 103, 'E-GEOD-42798', 23, 'Haloarcula hispanica wild-type strain cultured in different media: MG vs 168', 'Transcriptional profiling of Haloarcula hispanica wild-type strain cultured in different media comparing MG medium with 168 medium. H. hispanica wild-type strain can accumulate PHBV in MG medium, while can not in 168 medium. Goal was to explore the PHA biosynthesis and its impact on central metabolism in H. hispanica. Total RNA from MG and 168 were used to generate target cDNA, and then hybridized to 8*15K Haloarcula hispanica genome array genechips, representing about 3800 genes.', '1132/NCBITaxon:51589', '1136/efo:EFO_0002768'
  UNION SELECT 104, 'E-GEOD-42698', 23, 'Transcriptional response to aneuploidy in mouse epidermis', 'Analysis of the transcriptional response to aneuploidy in mouse epidermis. In this study we measured the transcriptional response to aneuploidy by aboragting the spindle checkpoint in mouse epidermis. We found that, whereas spindle checkpoint inactivation in the epidermis is tolerated, but results in metabolic deranged cells, SAC abrogation kills bulge stem cells Mad2; K14-Cre mice were sacrificed at indicated timepoints and epidermis was separated from dermis using overnight trypsin. RNA was isolated and expression patterns were compared between K14-Cre; Mad2f/f and Cfre negative animals for all timepoints.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 105, 'E-GEOD-41688', 23, 'Different levels of canonical Wnt signaling exert distinct roles in the colonic epithelium', 'There is a gradient of ?-catenin expression along the colonic crypt axis with the highest levels at the crypt bottom. However, it remains unclear whether different levels of canonical Wnt signaling exert distinct roles in the colonic epithelium. In the present study, we first showed that the canonical Wnt signaling is active in the proliferative compartment of normal colonic crypts by separating actively proliferating progenitor cells from non-proliferating cells in the colon using transgenic mice expressing a histone H2B-green fluorescent protein (GFP) fusion protein under the control of a tetracycline responsive regulatory element. Subsequently, we investigated the dose-dependent effect of canonical Wnt activation on colonic epithelial differentiation by controlling the expression levels of stabilized ?-catenin using a doxycycline-inducible transgenic system in mice. We show that elevated levels of Wnt signaling induce the amplification of Lgr5+ cells, which is accompanied by crypt fission and a reduction in cell proliferation among progenitor cells. In contrast, lower levels of ?-catenin induction enhanced cell proliferation rates of epithelial progenitors without affecting crypt fission rates. Notably, slow-cycling cells produced by ?-catenin activation exhibit activation of Notch signaling and the treatment of ?-catenin expressing mice with a Notch inhibitor turned such slow-cycling cells into actively proliferating cells. Our results indicate that the activation of the canonical Wnt signaling pathway is sufficient for de novo crypt formation, and suggest that different levels of canonical Wnt activations, in cooperation with Notch signaling, establish a hierarchy of slower-cycling stem cells and faster-cycling progenitor cells characteristic for the colonic epithelium. To separate actively proliferating cells from non-proliferating cells in normal colon, we used transgenic mice expressing a histone H2B-GFP fusion protein under the control of a tetracycline responsive regulatory element (H2B-GFP mice). In H2B-GFP mice, all colonic crypt cells exhibited strong nuclear GFP signal 7 days after doxycycline administration. When doxycycline was withdrawn for 2 days after the labeling period, nuclear GFP signal was diluted in proliferating cells whereas non-proliferating cells retained GFP. GFPhigh non-proliferating and GFPlow proliferating epithelial cells were sorted from the isolated crypts by FACS for microarray analysis. In addition, to investigate the effects of acute Wnt activation in colon, we generated doxycycline-inducible ?-catenin mice. Colonic crypts were isolated from untreated mice and mice fed doxycycline in the drinking water (2.0 mg/ml) for 5 days and subjected to microarray analysis.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 106, 'E-GEOD-39998', 23, 'NKX2-1 occupancy in human lung adenocarcinoma cell lines', 'The NKX2-1 transcription factor, a regulator of normal lung development, is the most significantly amplified gene in human lung adenocarcinoma. To better understand how genomic alterations of NKX2-1 drive tumorigenesis, we generated an expression signature associated with NKX2-1 amplification in human lung adenocarcinoma, and analyzed DNA binding sites of NKX2-1 by genome-wide chromatin immunoprecipitation from NKX2-1-amplified human lung adenocarcinoma cell lines.  Combining these expression and cistromic analyses identified LMO3, itself encoding a transcription regulator, as a candidate direct transcriptional target of NKX2-1, in addition to consensus binding motifs including a nuclear hormone receptor signature and a Forkhead box motif in NKX2-1-bound sequences.  RNA interference analysis of NKX2-1-amplified cells compared to non-amplified cells demonstrated that LMO3 mediates cell proliferation downstream of NKX2-1; cistromic analysis that NKX2-1 may cooperate with FOXA1.  Our findings provide new insight into the transcriptional regulatory network of NKX2-1 and suggest that LMO3 is a transducer of lineage specific cell survival of NKX2-1-amplified lung adenocarcinomas. NKX2-1 ChIP-seq from three lung adenocarcinoma cell lines with amplification of NKX2-1', '1132/NCBITaxon:9606', '1136/efo:EFO_0002692'
  UNION SELECT 107, 'E-GEOD-39918', 23, 'Next-generation sequencing facilitates quantitative analysis of wild type and Bmp4f/f;Wnt1Cre tooth mesenchyme transcriptomes', 'Previous studies have suggested that Bmp4 is a key Msx1-dependent mesenchymal odontogenic signal for driving tooth morphogenesis through the bud-to-cap transition. Whereas the bud stage tooth developmental arrest in Msx1-/- mutant mice was accompanied by reduction in mesenchymal Bmp4 mRNA expression, we show that depleting functional Bmp4 mRNAs in the tooth mesenchyme, through neural crest-specific gene inactivation in Bmp4f/f;Wnt1Cre mice, caused mandibular molar developmental arrest at the bud stage but allowed maxillary molars and incisors to develop to mineralized teeth. We show that the Wnt inhibitors Dkk2 and Wif1 were much more abundantly expressed in the mandibular than maxillary molar mesenchyme in wildtype embryos and that Dkk2 expression was significantly unregulated in the tooth mesenchyme in Bmp4f/f;Wnt1Cre embryos. In addition, expression of Osr2, which encodes a zinc finger protein that antagonizes Msx1-mediated activation of odontogenic mesenchyme, is significantly upregulated in the molar mesenchyme in Bmp4f/f;Wnt1Cre embryos. Msx1 heterozygosity enhanced maxillary molar developmental defects whereas Osr2 heterozygosity rescued mandibular first molar morphogenesis in Bmp4f/f;Wnt1Cre mice. Moreover, in contrast to complete lack of supernumerary tooth initiation in Msx1-/-Osr2-/- mutant mice, Osr2-/-Bmp4f/f;Wnt1Cre compound mutant mice exhibit formation and subsequent arrest of supernumerary tooth germs that correlated with down regulation of Msx1 expression in the tooth mesenchyme. Taken together, our data indicate that, while reduction in mesenchymal Bmp4 expression alone could not account for the tooth bud arrest phenotype in Msx1-/- mutant mice, Bmp4 signaling synergizes with Msx1 and antagonizes Osr2 to activate mesenchymal odontogenic activity to drive tooth morphogenesis and sequential tooth formation. E13.5 mouse embryos tooth germs were microdissected by laser capture microdissection (LCM), and the mandibular molar and maxillary molar were separated. 3 pairs of control and mutant samples were pooled for the RNA extraction.', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 108, 'E-GEOD-39334', 23, 'DNA methylation profiling in colon cancer', 'Profiling of CpG island methylation in 19 primary colon cancer and paired normal samples Agilent 244k CpG island microarray', '1132/NCBITaxon:9606', '1136/efo:EFO_0002759'
  UNION SELECT 109, 'E-GEOD-39058', 23, 'microRNA- and mRNA-based studies in paraffin-archived human osteosarcoma specimens reveal profiles with reproducible and independent prognostic value', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 110, 'E-GEOD-39057', 23, 'mRNA profiling and clinical outcomes in human osteosarcoma (biopsy/resection pairs)', 'Osteosarcoma is a malignancy with variable outcomes that are not tightly aligned with chemotherapy response. Previous work has suggested a possible role for microRNAs (miRNAs) in determining the susceptibility of osteosarcomas to treatment but formal miRNA-based outcome models have not been reported. Formalin-fixed, paraffin-embedded (FFPE) specimens may be a unique approach for such studies in a rare malignancy. We used supervised principal components analysis and logistic regression to study survival and chemoresponse endpoints, and miRNA activity and gene set analysis algorithms to study miRNA regulatory networks using mRNA data. Our study also creates a paradigm for FFPE-based miRNA clinical biomarker research in rare tumors. 5 unique pairs of diagnostic biopsy and surgical resection specimens were profiled. Paired samples are denoted in the Sample ''characteristics'' field.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 111, 'E-GEOD-39055', 23, 'mRNA profiling and clinical outcomes in human osteosarcoma (biopsies)', 'Osteosarcoma is a malignancy with variable outcomes that are not tightly aligned with chemotherapy response. Previous work has suggested a possible role for microRNAs (miRNAs) in determining the susceptibility of osteosarcomas to treatment but formal miRNA-based outcome models have not been reported. Formalin-fixed, paraffin-embedded (FFPE) specimens may be a unique approach for such studies in a rare malignancy. We used supervised principal components analysis and logistic regression to study survival and chemoresponse endpoints, and miRNA activity and gene set analysis algorithms to study miRNA regulatory networks using mRNA data. Our study also creates a paradigm for FFPE-based miRNA clinical biomarker research in rare tumors. 37 unique diagnostic biopsy specimens were profiled. Samples with paired surgical resection specimens (not included in this submission) are denoted in the Sample ''characteristics'' field.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 112, 'E-GEOD-39052', 23, 'microRNA profiling and clinical outcomes in human osteosarcoma (biopsy/resection pairs)', 'Osteosarcoma is a malignancy with variable outcomes that are not tightly aligned with chemotherapy response. Previous work has suggested a possible role for microRNAs (miRNAs) in determining the susceptibility of osteosarcomas to treatment but formal miRNA-based outcome models have not been reported. Formalin-fixed, paraffin-embedded (FFPE) specimens may be a unique approach for such studies in a rare malignancy. We used supervised principal components analysis and logistic regression to study survival and chemoresponse endpoints, and miRNA activity and gene set analysis algorithms to study miRNA regulatory networks using mRNA data. Our study also creates a paradigm for FFPE-based miRNA clinical biomarker research in rare tumors. 26 unique pairs of biopsy and surgical resection specimens were profiled. Paired samples are denoted in the Sample ''characteristics'' field.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 113, 'E-GEOD-39040', 23, 'microRNA profiling and clinical outcomes in human osteosarcoma (biopsies)', 'Osteosarcoma is a malignancy with variable outcomes that are not tightly aligned with chemotherapy response. Previous work has suggested a possible role for microRNAs (miRNAs) in determining the susceptibility of osteosarcomas to treatment but formal miRNA-based outcome models have not been reported. Formalin-fixed, paraffin-embedded (FFPE) specimens may be a unique approach for such studies in a rare malignancy. We used supervised principal components analysis and logistic regression to study survival and chemoresponse endpoints, and miRNA activity and gene set analysis algorithms to study miRNA regulatory networks using mRNA data. Our study also creates a paradigm for FFPE-based miRNA clinical biomarker research in rare tumors. 65 unique diagnostic biopsy specimens were profiled. Samples with paired surgical resection specimens (not included in this submission) are denoted in the Sample ''characteristics'' field.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 114, 'E-GEOD-38993', 23, 'The transposable element composition of human lincRNAs reveals a role for HERVH elements to promote stem cell specific expression of lincRNAs', 'Numerous studies over the past decade have elucidated a substantial set of long intergenic noncoding RNAs (lincRNAs). It has since become clear that lincRNAs constitute an important layer of genome regulation across a wide spectrum of species. Yet, the factors governing their evolution and origins remain relatively unexplored. One possible factor that may have shaped lincRNA biology are transposable elements (TEs). Here we set out to comprehensively characterize the TE content of lincRNAs relative to genomic averages and protein coding transcripts. Our analysis of the TE composition across 9241 human lincRNAs revealed that, in sharp contrast to protein coding genes, a striking majority (83%) of lincRNAs contain a TE, and TEs comprise 42% of lincRNA transcript sequences. LincRNA TE composition varies significantly from genomic averages, being depleted of LI and Alu elements and enriched for a broad class of endogenous retroviruses (ERVs). Furthermore, specific TE families occur in biased positions and orientations within lincRNAs, particularly at their transcription start sites, suggesting a role in the origin of those lincRNAs. Finally, we find that TEs can drive gene expression regulation of lincRNAswe observed a dramatic correlation between lincRNAs containing HERVH elements and almost exclusive expression in pluripotent cells. Conversely, those lincRNAs that are devoid of TEs are more highly expressed in testis. Collectively, TEs pervade lincRNAs and have shaped lincRNA evolution and function via bestowing tissue-specific expression from donated transcriptional regulatory signals. We extracted profiled the transcriptome expression polyadenylated mRNA-Seq.  We then used these to reconstruct the transcriptome using de-novo assemblers and identify long non coding RNAs and their expression.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 115, 'E-GEOD-38746', 23, 'Effects of methylmercury on genome-wide gene expression in the liver of Atlantic cod (Gadus morhua)', 'Methylmercury (MeHg) is a highly toxic environmental pollutant. To understand the mechanisms of toxicity of the compound and possibly to discover new biomarkers for environmental monitoring, we have conducted toxicogenomics studies. We designed 126k-cod-oligonucleotide arrays and performed genome-wide gene expression assays in liver samples from juvenile cod treated with MeHg (0.5 and 2 mg/kg body weight). Microarray analysis showed MeHg differentially regulated hundreds of genes. Gene Ontology and pathway analyses of differentially regulated genes revealed that MeHg modulated mainly genes involved in immune response, oxidative stress response, tissue remodelling, and energy pathways such as lipid, carbohydrate and amino acid metabolism. The results provide insights into the mechanisms of toxicity of MeHg and provide candidate biomarkers of exposure to MeHg for further evaluation. Analyzed 5 samples from each of 3 groups: control, 0.5 mg/BW MeHg treated, and 2 mg/BW MeHg treated. Liver total RNA was reverse transcribed, Cy3-labeled, and hybridized (one-color hybridzation).', '1132/NCBITaxon:8049', '1136/efo:EFO_0002768'
  UNION SELECT 116, 'E-GEOD-38651', 23, 'Dnmt3a protects active chromosome domains against cancer-associated hypomethylation', 'The results of the study uncover conserved features of cancer methylomes and provide a mechanistic explanation for the tumor-promoting effects of Dnmt3a mutations. Whole genome methylation analysis of Mus musculus. Five samples were analyzed, one control sample containing normal healthy lung tissue, two samples containing big (WTB) or small (WTS) Dnmt3a WT tumors, two samples containing big (KOB) or small (KOS) Dnmt3a knock-out tumors', '1132/NCBITaxon:10090', '1136/efo:EFO_0002761'
  UNION SELECT 117, 'E-GEOD-37714', 23, 'Mammalian TRIM71 as repressor of mRNAs that inhibits translation and affects mRNA stability', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 118, 'E-GEOD-37713', 23, 'Expression data from HEK293 Flp-In cells constitutivly expressing FLAG-HA-tagged TRIM71 and that of the parental cell line', 'We identify mammalian TRIM71 as repressor of mRNAs that inhibits translation and affects mRNA stability. In this data set we compare the expression profile of human HEK293 Flp-In cells stably expressing TRIM71 to that of the parental cells, not expressing TRIM71 protein. Experiment was performed in triplicate.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 119, 'E-GEOD-37712', 23, 'Expression data from mouse embryonic stem cells upon TRIM71 KD and parental ctrl cells', 'We identify mammalian TRIM71 as repressor of mRNAs that inhibits translation and affects mRNA stability. In this data set we compare the expression profile of mouse ES upon Trim71 KD versus that of the parental cells. Experiment was performed in triplicate.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 120, 'E-GEOD-37040', 23, 'Expression data from human pluripotent stem cells and differentiated cells treated with pluripotent-specific inhibitors', 'The use of human pluripotent stem cells (hPSCs) in cell therapy is hindered by the tumorigenic risk from residual undifferentiated cells. Here we performed a high-throughput screen of over 52,000 small molecules, and identified 15 highly selective cytotoxic inhibitors of hPSCs (PluriSIns). Cellular and molecular analyses revealed that the most selective compound, PluriSIn #1, is a pluripotent-specific inhibitor of stearoyl-coA desaturase (SCD1), the key enzyme in the biosynthesis of monounsaturated fatty acids (MUFA). SCD1 inhibition in hPSCs induced ER stress, protein synthesis attenuation, and apoptosis of these cells, revealing that MUFA biosynthesis is crucial for their survival. PluriSIn #1 was also cytotoxic toward the ICM cells of mouse embryos, indicating that the dependence on SCD1 is inherent to the pluripotent state. Finally, application of PluriSIn #1 prevented teratoma formation from tumorigenic undifferentiated cells. Our novel method to eliminate undifferentiated cells from culture should thus increase the safety of hPSC-based treatments. Expression data from undifferentiated and differentiated human embryonic stem cells. Total RNA was isolated from undifferentiated human pluripotent stem cells grown on matrigel with mTeSR1 medium, or from early endodermal progenitor cells differentiated from human embryonic stem cells.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 121, 'E-GEOD-35710', 23, 'Human s.c adipose tissue gene expression during diet-induced weight loss', 'Obesity has considerable effects on morbidity and mortality and the prevalence of obesity has been increasing rapidly worldwide during the past two decades. Even if obesity affects the entire individual, adipose tissue play a central role in the development of obesity. Expression profiling of adipose tissue may give insights into mechanisms contributing to obesity and obesity-related disorders. 24 obese subjects (6 women and 18 men, BMI 37.6 ± 4.9) received a very low calorie diet (450 kcal/d, Cambridge diet or Modifast) for 16 weeks. On average, patients lost 27.7 kg during the diet. Microarray expression analysis (Affymetrix U133A) in subcutaneous adipose tissue was performed before diet and after 16 weeks of diet.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 122, 'E-GEOD-34845', 23, 'Gene expression analysis comparing between dormant and released leaf bud from Japanese pear ''Kosui''', 'Dormancy of buds is a critical developmental process that allows plants to survive extreme seasonal variations in climate. Dormancy transition in leaf buds of Japanese pear leading cultivar Kosui were investigated using 10 K cDNA microarray. Leaf buds collected on October (early endodormancy) and February (breaking stage) were used to monitor the changes in the transcriptome. Over 1000 genes were differentially expressed (p value < 0.05 and 2 fold change) when the early endodormant stage and breaking one were compared to each other. Functional classification revealed that the majority of genes were involved in response to abiotic/biotic stimulus and stress, signal transduction, and transcription. Among them, 76 and 22 genes were significantly (over 10 fold change) highly expressed in early endodormant stage and breaking stage, respectively, this result suggests that even in the endodormant stage, gene expression in the buds is high. Among them, several transcriptional factors were included. The quantitative real-time PCR analysis showed that at least three of their seasonal changes during endodormancy transition phases were similar to two DAM (dormancy-associated MADS box) genes isolated from Japanese pear, suggesting that they might be novel candidate genes associated to dormancy. The gene expression profiles between dormant and released leaf bud were compared.', '1132/NCBITaxon:3767', '1136/efo:EFO_0002768'
  UNION SELECT 123, 'E-GEOD-34844', 23, 'Genome-wide mRNA profiling for identification of specific body fluid markers', 'Genome-wide expression profiling of four kinds of body fluid samples (blood, saliva, semen and vaginal swab). The purpose of the present study was selection of specific mRNA markers for identification of the four body fluids. Results provide important information about gene expression level of each body fluid for forensic science. Total RNAs isolated from four kinds of body fluid samples (blood, saliva, semen and vaginal swab) obtained from Korean volunteers', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 124, 'E-GEOD-32020', 23, 'Expression data from mammary gland during pregnancy and lactation of CBA/CaH, QSi5 and Advanced Intercross Line (AIL; CBA/CaH X QSi5, the 14th generation)', 'Previously we have shown significant differences in lactation performance, mammary gland histology and expression profiles of mammary transcriptome during peak-lactation (lactation day 9; L9) between the ordinary CBA/CaH (CBA) and the superior QSi5 strains of mice. In the present study, we compared mammary gland histology between CBA and QSi5 at mid-pregnancy (pregnancy day 12; P12). We assessed lactation performance during the first 8 days of lactation of the 13th - 14th generation of the Advanced Intercross Line (AIL) (CBA X QSi5) mice. We utilized an integrative approach to analyzing mammary microarray expression profiles of CBA and QSi5 at P12 and CBA, AIL and QSi5 at L9. The inguinal mammary glands of CBA/CaH and QSi5 during mid-pregnancy (Pregnancy day 12; P12), and the glands of CBA/CaH, AIL and QSi5 during peak lactation (Lactation day 9; L9) were collected and total RNA was extracted for Affymetrix microarray (mouse genome 430 2) assay', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 125, 'E-GEOD-27916', 23, 'Human subcutaneous adipose tissue gene expression (SOS Sib-pair study, offspring cohort)', 'Obesity has considerable effects on morbidity and mortality, and the prevalence of obesity has been increasing rapidly worldwide during the past two decades. Even if obesity affects the entire individual, adipose tissue plays a central role in the development of obesity. Expression profiling of adipose tissue may give insights into the mechanisms contributing to obesity and obesity-related disorders.  The Swedish Obese Subjects (SOS) Sib-Pair Study consists of 154 nuclear families with BMI-discordant sib pairs (BMI difference more than 10 kg/m2) resulting in a study population consisting of 732 subjects. The full SOS Sib-Pair study offspring cohort consists of 425 subjects. Microarray expression analysis in subcutaneous adipose tissue was performed in 375 subjects (262 women and 113 men) of the SOS Sib-Pair offspring cohort. Microarray expression analysis in subcutaneous adipose tissue was performed in women (n=262) and men (n=113) of the SOS Sib-Pair offspring cohort.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 126, 'E-GEOD-27657', 23, 'Human subcutaneous adipose tissue and perithyroid adipose tissue gene expression', 'Different human adipose tissue depots may have functional differences. Subcutaneous human adipose tissue has been extensively studied, but less is known about other depots. Perithyroid (PT) adipose tissue contains not only white adipocytes but also brown adipocytes. The aim of this study was to compare the expression of brown adipocyte containing perithyroid adipose tissue with s.c. adipose tissue.role in the development of obesity. Expression profiling of adipose tissue may give insights into mechanisms contributing to obesity and obesity-related disorders. Expression analysis of paired biopsies from s.c and perithyriod (PT) adipose tissue from nine subjects undergoing surgery in the thyroid region.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 127, 'E-GEOD-26586', 23, 'Widespread splicing changes in human brain development and aging', 'Differences in transcript splicing are well documented among tissues and between sexes in humans and other organisms. Further, splicing changes in selected genes have been reported in human and mouse brain development. Using high-throughput RNA sequencing we characterized the transcriptome-wide splicing changes occurring over the lifespan in the human brain. We found that in two brain regions, prefrontal cortex and cerebellum, as many as 37% of all expressed protein-coding multi-exon genes undergo significant splicing changes between birth and 98 years of age. Approximately 40% of these changes take place in aging. We confirmed our findings using exon arrays and PCRs and detected identified splicing changes at the protein level. We further associated distinct patterns of age-related splicing changes with expression of the corresponding splicing factors. Thus, splicing plays an important role in shaping the human brain transcriptome in both development and aging. Human post-mortem brain samples from prefrontal cortex and cerebellum from 30 individuals with ages from birth to 100 years were collected. For each five individuals of similar age RNA, extracted from dissected tissue, was pooled resulting in 5 pooled samples. Obtained RNA was hybridized with Affymetrix Human Exon 1.0 ST arrays.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 128, 'E-GEOD-17989', 23, 'Influence of T and B lymphocytes on the antigen presentation capacities of splenic conventional dendritic cells', 'The goal of this project was to characterize DCs from lymphopenic mice, like RAG (recombination activated gene) deficient mice and to examine the influence of mature B and T cells on the antigen presenting ability of splenic cDCs. We demonstrate how cellular cross-talk can shape the character and function of cDCs. Lymphopenic conditions, where splenic cDCs have to develop and differentiate, drastically change their character and their ability to cross-present soluble antigen. In this approach we sorted out two populations (CD8?+ and CD8?-) of splenic dendritic cells (DCs) from untreated WT, and RAG2-/- C57Bl/6 mice. The age of mice was between 8-10 weeks. Further we isolated RNA and performed microarray analysis. Each DCs population was repeated twice.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 129, 'E-GEOD-15654', 23, 'Prognostic Gene-Expression Signature for Patients with Hepatitis CRelated Early-Stage Cirrhosis', 'Gene-expression profiles of hepatitis C-related, early-stage liver cirrhosis  Background & Aims: Liver cirrhosis affects 1%?2% of population and is the major risk factor of hepatocellular carcinoma (HCC). Hepatitis C cirrhosis-related HCC is the most rapidly increasing cause of cancer death in the US. Non-invasive methods have been developed to identify patients with asymptomatic, early-stage cirrhosis, increasing the burden of HCC surveillance, but biomarkers are needed to identify patients with cirrhosis who are most in need of surveillance. We investigated whether a liver-derived 186-gene signature previously associated with outcomes of patients with HCC is prognostic for patients newly diagnosed with cirrhosis but without HCC.  Methods: We performed gene expression profile analysis of formalin-fixed needle biopsies from the livers of 216 patients with hepatitis C-related early-stage (Child-Pugh class A) cirrhosis who were prospectively followed for a median of 10 years at an Italian center. We evaluated whether the 186-gene signature was associated with death, progression of cirrhosis, and development of HCC.  Results: Fifty-five (25%), 101 (47%), and 60 (28%) patients were classified as having poor-, intermediate-, and good-prognosis signatures, respectively. In multivariable Cox regression modeling, the poor-prognosis signature was significantly associated with death (P=.004), progression to advanced cirrhosis (P<.001), and development of HCC (P=.009). The 10-year rates of survival were 63%, 74%, and 85% and the annual incidences of HCC were 5.8%, 2.2%, and 1.5% for patients with poor-, intermediate-, and good-prognosis signatures, respectively.   Conclusions: A 186-gene signature used to predict outcomes of patients with HCC is also associated with outcomes of patients with hepatitis C-related early-stage cirrhosis. This signature might be used to identify patients with cirrhosis in most need of surveillance and strategies to prevent their development of HCC. 216 liver biopsy specimens', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 130, 'E-ERAD-139', 23, 'Methylation profiling of human samples using MeDIP-seq method', 'Aim: We aim to compare current (MeDIP-seq), new (Illumina Infinium 450K BeadChip) and future (PacBio) methods for whole genome DNA methylation analysis. As the interest in determination of disease methylation profiles increases, the scope, advantages and limitations of these methods requires assessment. There are key questions to answer and specific challenges to overcome. For example, how much detail/resolution is sufficient to identify regions of differential methylation and regions of biological/medical significance within a sample? How much coverage of the genome is required for accurate methylation analysis? Is it important to confirm which regions of the genome are unmethylated in addition to focusing on those that are methylated? Loss of methylation may be of equal importance within the cell since this may also contribute to disease pathogenesis. A multi-method (affinity enrichment/bisulphite-conversion based/direct sequencing of methyl-cytosine) and technology platform (Illumina HiSeq/PacBio/Illumina Infinium BeadChip) comparison will enable us to determine the strengths and weakness of each method. We propose to compare four methods using two DNA samples from the Coriell Institute for Cell Repository to assess both current and future capabilities for whole genome methylation analysis in parallel: A) MeDIP-seq using Illumina HiSeq B) Illumina Infinium HumanMethylation 450K BeadChip and C) whole genome methylation sequencing using PacBio. Existing single molecule deep bisulphite sequencing data generated previously from these same samples at the WTSI for targeted regions (30-40 genes) on the human X chromosome will be used to assess performance of each method. The methods selected for this study will generate data covering a range of resolutions from a whole genome scan to array (target defined) resolution and up to single base pair, single molecule resolution; the highest level of detail possible with methods currently available.Samples: DNA from sibling pair GM01240 (female) and GM01240 (male).Requirements: Both samples will be analysed using;A.MeDIP-seq using Illumina HiSeq (one HiSeq lane, 75bp paired end, per sample) B.Illumina Infinium HumanMethylation 450K BeadChipWe are expecting a potentially unnecessary high coverage using one HiSeq lane per sample. However, for the MeDIP procedure we do not have a multiplexing procedure in place. Our requirements for PacBio sequencing have been discussed with and will be supported by the Sequencing Technology Development group.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002761'
  UNION SELECT 131, 'E-ERAD-61', 23, 'RNA-seq of human melanoma samples', 'In this study, mutations present in a series of human melanomas (stage IV disease) will be determined, using autologous blood cells to obtain a reference genome. From each of the samples that are analyzed, tumour-infiltrating T lymphocytes have also been isolated. This offers a unique opportunity to determine which (fraction of) mutations in human cancer leads to epitopes that are recognized by T cells. The resulting information is likely to be of value to understand how T cell activating drugs exert their action.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 132, 'E-ERAD-60', 23, 'RNA-seq of human lymphoblast samples to characterise RNA structural effects in Poikiloderma syndrome', 'We will sequence the RNA of lymphoblast samples, transformed with EBV, which have poikiloderma syndrome with mutations in c16orf57. The aim of the experiment is to characterise RNA structural effects in this disease.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 133, 'E-GEOD-43363', 23, 'Stalled spliceosomes are a signal for RNAi-mediated genome defense', 'Using the yeast Cryptococcus neoformans, we describe a mechanism by which transposons are initially targeted for RNAi-mediated genome defense. We show that intron-containing mRNA precursors template siRNA synthesis. We identify a Spliceosome-Coupled And Nuclear RNAi (SCANR) complex required for siRNA synthesis and demonstrate that it physically associates with the spliceosome. We find that RNAi target transcripts are distinguished by suboptimal introns and abnormally high occupancy on spliceosomes. Functional investigations demonstrate that the stalling of mRNA precursors on spliceosomes is required for siRNA accumulation. Lariat debranching enzyme is also necessary for siRNA production, suggesting a requirement for processing of stalled splicing intermediates. We propose that recognition of mRNA precursors by the SCANR complex is in kinetic competition with splicing, thereby promoting siRNA production from transposon transcripts stalled on spliceosomes. Disparity in the strength of gene expression signals encoded by transposons versus genes offers an avenue for the evolution of genome defense. Small RNA profile of wild-type Cryptococcus neoformans was determined by deep sequencing using the Illumina SBS platform', '', '1136/efo:EFO_0003737'
  UNION SELECT 134, 'E-GEOD-42861', 23, 'Differential DNA methylation in Rheumatoid arthritis', 'Genome-wide DNA methylation level was studied to determine whether Rheumatoid arthritis patients (cases) has methylation differences comparing to normal controls in PBLs. We used Illumina HumanMethylation450 BeadChip array to determine the genome-wide DNA methylation difference in PBLs from Rheumatoid arthritis patients (cases) and normal controls Bisulphite converted DNA from the Rheumatoid arthritis patients (cases) and normal controls were hybridized to the Illumina Illumina HumanMethylation450 BeadChip arrays', '1132/NCBITaxon:9606', '1136/efo:EFO_0002759'
  UNION SELECT 135, 'E-GEOD-39521', 23, 'ChIP-seq study of inheritence of Myc in Drosophila', 'Myc is an important oncogene. It is considered as a transcription factor, but the function of Myc in normal or cancer cells have not been fully understood. In addition, Myc plays a role in cell proliferation and differentiation. It is also important for cell identity and stay on chromatin throughout the cell cycle. However, the inheritance of Myc is still a mystery. Here we study the function and inheritance of Myc in D. melanogaster by mapping the binding sites of Myc during interphase and mitosis using ChIP-seq. DNA sample of ChIP for Myc are collected from Kc cells in interphase or mitosis. Input sequences from previous study in the same cell type (GSM762848, GSM762849) are used as control.', '1132/NCBITaxon:7227', '1136/efo:EFO_0002692'
  UNION SELECT 136, 'E-GEOD-35493', 23, 'Pediatric rhabdoid tumors of kidney and brain show many differences in gene expression but share dysregulation of cell cycle and epigenetic effector genes', 'Rhabdoid tumors (RTs) are aggressive tumors of early childhood that occur most often in brain (AT/RTs) or kidney (KRTs). Regardless of location, they are characterized by loss of functional SMARCB1 protein, a component of the SWI/SNF chromatin remodeling complex. The aim of this study was to determine genes and biological process dysregulated in common to both AT/RTs and KRTs. Gene expression for AT/RTs was compared to that of other brain tumors and normal brain using microarray data from our lab. Similar analysis was performed for KRTs and other kidney tumors and normal kidney using data from GEO. Dysregulated genes common to both analyses were analyzed for functional significance. Unsupervised hierarchical clustering of RTs identified 3 major subsets: 2 comprised of AT/RTs, and 1 of KRTs. Compared to other tumors, 1187, 663 and 539 genes were dysregulated in each subset, respectively. Only 14 dysregulated genes were common to all 3 subsets. Compared to normal tissue, 5209, 4275 and 2841 genes were dysregulated in each subset, with an overlap of 610 dysregulated genes. Among these genes, processes associated with cell proliferation, MYC activation, and epigenetic dysregulation were common to all 3 RT subsets. The low overlap of dysregulated genes in AT/RTs and KRTs suggests that factors in addition to SMARCB1 loss play a role in determining subsequent gene expression. Drugs which target cell cycle or epigenetic genes may be useful in all RTs. Additionally, targeted therapies tailored to specific RT subset molecular profiles should be considered.   Molecular profiling of 20 ATRTs, 42 other pediatric CNS WHO Grade IV tumor samples and 9 pediatric normal brain samples was performed using Affymetrix U133 Plus2 GeneChips. Data were background corrected and normalized using gcRMA (as implemented in Bioconductor). ANOVA was used to identify differentially expressed genes for AT/RTs compared to other brain tumor types. Various analyses, including bioinformatics tools DAVID, Gene Set Enrichment (GSEA), and Ingenuity Pathways Analysis,  were used to identify biological processes and genes of particular interest among the  differentially expressed genes. Similar analysis of publicly available KRT and pediatric kidney tumor  and normal kidney data was also performed by us. The overlaps between the results were examined to determine genes and processes dysregulated in common to both ATRTs and KRTs.  This study includes a re-analysis of samples from other studies (AT/RT: GSE28026;  glioblastoma: GSE33331).', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 137, 'E-GEOD-32584', 23, 'Distribution of Drosophila insulator proteins during interphase and mitosis', 'Here we compare the distribution of insulator proteins during interphase and mitosis.  We performed ChIP-seq analysis on purified populations of interphase and mitotic Kc cells, using antibodies against CP190, dCTCF, BEAF, and Su(Hw). Examination of 4 different insulator proteins during interphase and mitosis', '1132/NCBITaxon:7227', '1136/efo:EFO_0002692'
  UNION SELECT 138, 'E-GEOD-43582', 23, 'Expression data from SPHINX (SPaceflight of Huvec: an INtegrated eXperiment)', 'Changes in the physical environment modulate cell responses and may lead to the impairment or even failure of tissue function as a result of mechanotransduction processes. It has been suggested that this situation occurs in some age-related diseases and some pathological conditions observed in space, such as cardiovascular deconditioning, bone loss, muscle atrophy, and impaired immune responses. All of these are associated with endothelial dysfunction but the precise mechanism is still unclear.  We used the microarray approach to obtain insights into the mechanism responsible for endothelial dysfunction by taking advantage of the challenging environment of gravitational unloading onboard the International Space Station. The effects of gravitational unloading on HUVEC gene expression were investigated by means of cDNA microarray analyses of six randomly chosen samples (three for each of the two conditions of spaceflight and 1g) using Affymetrix Gene Human 1.0 ST Arrays', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 139, 'E-GEOD-42454', 23, 'Activation of the chromatin-remodeling factor Smarca4/Brg1 cooperates with Olig2 to establish oligodendrocyte identity and control CNS myelination', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10116', '1136/efo:EFO_0003738'
  UNION SELECT 140, 'E-GEOD-42447', 23, 'Activation of the chromatin-remodeling factor Smarca4/Brg1 cooperates with Olig2 to establish oligodendrocyte identity and control CNS myelination [ChIP-seq]', 'We investigate the role of Brg1 and Olig2 during oligodendrocyte differentiation by combining gene conditional knockout and next generation sequencing technology. We generate genome-wide maps of RNA polymerase II (RPolII), Brg1 (Smarca4), Olig2 and histone modifications in primary rat oligodendrocyte precursor cells (iOLs), differentiating oligodendrocytes and mature oligodendrocytes.We found that Brg1 is intensely regulated by RPolII at the initiation of oligodedrocyte differentiation. The genomic distribution of Brg1 in differentiating oligodendrocytes is pre-directed by Olig2 in iOLs. The dynamic interaction of Brg1 and chromatin is correlate with the distinct stages of gene expression during maturation. Finally, we show that Brg1 and Olig2 localization predict critical genes controling CNS myeliantion. Our study represents the first detailed analysis of genomic landscape during the oligodendrocyte development and provides a framework for further understanding of molecular mechanisms underlying oligodendrocyte lineage progression. Genomic distribution of Brg1, Olig2, RPolII and three different histone modifications in three oligodendrocyte developemtal stages were examined using primary cells by ChIP-sequencing. Spinal cord mRNA profiles of 14-day old control and Brg1c/c;Olig1-Cre mice were generated by RNA-sequencing.', '1132/NCBITaxon:10116', '1136/efo:EFO_0002692'
  UNION SELECT 141, 'E-GEOD-42443', 23, 'Activation of the chromatin-remodeling factor Smarca4/Brg1 cooperates with Olig2 to establish oligodendrocyte identity and control CNS myelination [RNA-seq]', 'Spinal cord mRNA profiles of 14-day-old control (Brg1c/+;Olig1-Cre) and Smarca4/Brg1 conditional knockout (Brg1c/c;Olig1-Cre) mice were generated by RNA-seq. Differential expressed genes are identified using Cuffdiff and validated with qPCR. Spinal cord mRNA profiles of 14-day old control and Brg1c/c;Olig1-Cre mice were generated by RNA-sequencing.', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 142, 'E-GEOD-37842', 23, 'Generation and maintenance of hiPSCs on PCM-DM', 'Human ES cells (hESCs) and human induced pluripotent stem cells (hiPSCs) are usually generated and maintained on living feeder cells like mouse embryonic fibroblasts or on a cell-free substrate like Matrigel. For clinical applications, a quality-controlled, xenobiotic-free culture system is required to minimize risks from contaminating animal-derived pathogens and immunogens. We previously reported that the pericellular matrix of decidua-derived mesenchymal cells (PCM-DM) is an ideal human-derived substrate on which to maintain hiPSCs/hESCs. In this study, we examined whether PCM-DM could be used for the generation and long-term stable maintenance of hiPSCs. Decidua-derived mesenchymal cells (DMCs) were reprogrammed by the retroviral transduction of four factors (OCT4, SOX2, KLF4, c-MYC) and cultured on PCM-DM. The established hiPSC clones expressed alkaline phosphatase, hESC-specific genes and cell-surface markers, and differentiated into three germ layers in vitro and in vivo. At over 20 passages, the hiPSCs cultured on PCM-DM held the same cellular properties with genome integrity as those at early passages. Global gene expression analysis showed that the GDF3, FGF4, UTF1, and XIST expression levels varied during culture, and GATA6 was highly expressed under our culture conditions; however, these gene expressions did not affect the cells pluripotency. PCM-DM can be conveniently prepared from DMCs, which have a high proliferative potential. Our findings indicate that PCM-DM is a versatile and practical human-derived substrate that can be used for the feeder-cell-free generation and long-term stable maintenance of hiPSCs. Keywords: Induced pluripotent stem cells; Extracellular matrix; Decidua; Mesenchymal cells The microarray study was carried out using Affymetrix Human Genome U133 Plus 2.0 gene expression arrays. Total RNA was extracted from cells with RNeasy (Qiagen), and 100ng of total RNA was used to synthesize aRNA using the 3 IVT Express Kit, according to the manufacturers instructions (Affymetrix Inc., Santa Clara, CA). After aRNA purification, 15 ?g of aRNA was fragmented and hybridized with a pre-equilibrated GeneChip array (Human Genome U133 Plus 2.0 gene expression array, Affymetrix) at 45°C for 16 hours. The GeneChip array was then washed, stained, and scanned according to the manufacturers instructions. The gene expression data were extracted using Affymetrix Expression Console software.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 143, 'E-GEOD-43639', 23, 'Developmental Changes in the Transcriptome of Human Cerebral Cortex Tissue: Long Noncoding RNA Transcripts', 'To examine the hypothesis that lncRNA expression varies with age, potentially paralleling 25 known developmental trends in synaptogenesis, myelination, and energetics, we quantified levels of nearly 6000 lncRNAs in 36 surgically resected human neocortical samples (primarily derived from temporal cortex) spanning infancy to adulthood. One-color (Alexa 555 green) microarray experiment was performed separately on each of the 36 surgically-resected human neocortical samples (primarily derived from temporal cortex) spanning infancy to adulthood. 8 probes per gene and 5 on-chip replicates for each probe.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 144, 'E-GEOD-43636', 23, 'Expression data from mouse T cells after CLIP (Crosslinking immunoprecipitation) procedures', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 145, 'E-GEOD-43635', 23, 'Expression data from mouse T cells after CLIP (Crosslinking immunoprecipitation) procedures (mRNA)', 'Molecular networks based on genome-wide mRNA and miRNA expression in activated T cells are interated by co-regulation with miRNA and protein-coding genes during immune responses, however, accurate prediction of miRs and their targets remains a challenge. Furthermore, the specific changes in the expression of miRNAs and/or mRNAs in allogeneic activated T cells during HCT would be distinct from those responding to non-specific stimulation. Recently, Ago-CLIP has been demonstrated to provide a robust platform for identification of precise miRNA-mRNA interactions by definitively distinguishing direct from indirect miRNA-target interactions. We utilized CLIP procedure followed by screening with standard microarray platforms for miRNA and mRNA transcripts to demonstrate the miR-mRNA landscape and identify novel molecular targets for modulating T cell responses. Mixed cell culture was performed with T cells in the presence of syngeneic 57BL/6, or allogeneic BALB/c dendritic cells or cultivated with anti-CD3e and anti-CD28 mAB9 for 24 h.  Purified T cells were processde for CLIP procedures and followed by RNA isolation (TRIzol LS and miReasy Kit) and hybridization on Affymetrix microarrays and Exiqon 6th generation miRCURY LNA microRNA Arrays . We sought to obtain that the specific changes in the expression of miRNAs and/or mRNAs in allogeneic activated T cells during hematopoietic cells transplantation and that would be distinct from those responding to non-specific stimulation.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 146, 'E-GEOD-43634', 23, 'Expression data from mouse T cells after CLIP (Crosslinking immunoprecipitation) procedures (microRNA)', 'Molecular networks based on genome-wide mRNA and miRNA expression in activated T cells are integrated by co-regulation with miRNA and protein-coding genes during immune responses, however, accurate prediction of miRs and their targets remains a challenge. Furthermore, the specific changes in the expression of miRNAs and/or mRNAs in allogeneic activated T cells during HCT would be distinct from those responding to non-specific stimulation. Recently, Ago-CLIP has been demonstrated to provide a robust platform for identification of precise miRNA-mRNA interactions by definitively distinguishing direct from indirect miRNA-target interactions. We utilized CLIP procedure followed by screening with standard microarray platforms for miRNA and mRNA transcripts to demonstrate the miR-mRNA landscape and identify novel molecular targets for modulating T cell responses. Mixed cell culture was performed with T cells in the presence of syngeneic 57BL/6, or allogeneic BALB/c Dendritic cells or cultivated with anti-CD3e and anti-CD28 mAB for 24 h.  Purified T cells were processd for CLIP procedures and followed RNA isolation (miReasy Kit and TRIzol LS)  and hybridization on Exiqon 6th generation miRCURY LNA microRNA Arrays and Affymetrix microarrays. We sought to obtain that the specific changes in the expression of miRNAs and/or mRNAs from those responding to non-specific stimulation and in allogeneic activated T cells.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 147, 'E-GEOD-43630', 23, 'The gene expression profiling array of mRNAs between portal vein tumor thrombus tissues and paired tumor tissues in hapatocellular carcinoma.', 'Investigation of whole genome gene expression level changes of mRNAs in portal vein tumor thrombus tissues and paired tumor tissues in hapatocellular carcinoma. The different expression genes were further analyzed. The human mRNA microarray analysis of the 6 samples (3 portal vein tumor thrombus tissues and 3 paired tumor tissues) were completed. Total RNA from each sample was quantified using the NanoDrop ND-1000 and RNA integrity was assessed using standard denaturing agarose gel electrophoresis. Total RNA of each sample was used for labeling and array hybridization as the following steps: 1) Reverse transcription with by Invitrogen Superscript ds-cDNA synthesis kit; 2) ds-cDNA labeling with NimbleGen one-color DNA labeling kit; 3) Array hybridization using the NimbleGen Hybridization System and followed by washing with the NimbleGen wash buffer kit; 4) Array scanning using the Agilent Scanner G2505C. Scanned images (TIFF format) were then imported into NimbleScan software (version 2.5) for grid alignment and expression data analysis. Expression data were normalized through quantile normalization and the Robust Multichip Average (RMA) algorithm included in the NimbleScan software. The Probe level (*_norm_RMA.pair) files and mRNA level (*_RMA.calls) files were generated after normalization. All mRNAs level files were imported into Agilent GeneSpring GX software (version 11.5.1) for further analysis.mRNAs that at least 3 out of 6 samples have values greater than or equal to lower cut-off: 50.0 (All Targets Value) were chosen for further data analysis. Differentially expressed mRNAs were identified through Volcano Plot filtering. Pathway analysis and GO analysis were applied to determine the roles of these differentially expressed mRNAs played in these biological pathways or GO terms. Finally, Hierarchical Clustering was performed to show the distinguishable mRNAs expression pattern among samples.    .', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 148, 'E-GEOD-43621', 23, 'Quantitative analysis of miRNA changes in bone marrow stem cells under neural differentiation', 'Normal SJL mice, 6 to 8 weeks old, were used for the isolation of bone marrow stem cells (BMSC). Bone marrow cells were obtained from the femurs and tibias of euthanized mice by flushing with PBS. Cells were subjected to negative magnetic sorting using the Lineage Cell Depletion Kit. Isolation of murine bone marrow Lin-Sca-1+ cells was performed using MACS Sca-1 MultiSort Kit (fBMSC).  The differentiation of neural stem cells was induced by removing the bFGF-containing medium and resuspending cells in fresh bFGF-free medium. Microarray analysis of miRNA expression profiles was performed comparing non differentiated bone marrow stem cells (fBMSC) to bone marrow stem cells differentiated for 4 or 7 days. The data are from adult mouse stem cells isolated from bone marrow', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 149, 'E-GEOD-43618', 23, 'Identification of inflammation-related microRNAs deregulated in the plasma from intracerebral hemorrhage patients', 'We tested the hypothesis that circulating microRNAs (miRNAs) present in plasma might display a specific signature in patients with intracerebral hemorrhage (ICH). Global miRNA profiles were determined with the Agilent Human miRNA Microarray platform, 027233. ICH patients display a characteristic inflammation-related miRNA profile as compared to healthy controls. Plasma samples were collected from the following 6 subject groups: male ICH patients (n=8), female ICH patients (n=7), male healthy control (n=4), female healthy control (n=4), male ischemic stroke patients (n=8) and female ischemic stroke patients (n=8). Total RNAs isolated from 1 ml plasma were pooled for each group. A fixed volume of RNA sample was withdrawn from each pool and used for microarray detection.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 150, 'E-GEOD-43611', 23, 'Expression of M-CSF from cryopreserved teeth-derived human periodontal ligament cells with different freezing protocols', 'This study investigated which genes regarding root resorption are upregulated by cryopreservation and whether cryopreservation affects the expression of Macrophage colony stimulating factor. We manufactured the customized template which was made of genes selected regarding root resorption including osteoprotegerin (OPG), receptor activator of nuclear factor-kappa B ligand (RANKL), RANKLs cognate receptor (RANK), macrophage colony-stimulating factor (M-CSF), interleukin-1? (IL-1?), and tumor necrosis factor-alpha (TNF-?), and bone morphogenetic proteins (BMP) and analyzed gene expression. cultured human periodontal ligament cells (control) VS cryopreserved and cultured periodontal ligament cells(cryopreserved group): 3 control replicates, 3 cryopreserved replicates', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 151, 'E-GEOD-43607', 23, 'Sodium metavanadate exhibits carcinogenic tendencies in vitro in immortalized human bronchial epithelial cells', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 152, 'E-GEOD-43593', 23, 'Sodium metavanadate exhibits carcinogenic tendencies in vitro in immortalized human bronchial epithelial cells [DEG.ABC]', 'Pentavalent vanadium compounds induce intracellular changes in vitro that are consistent with those of other carcinogenic substances, including alteration of transcription factor levels, promotion of oxidative stress, inflammation, DNA damage, stimulation of mitogenic signals, and suppression of apoptosis. While there is no clear evidence that vanadium compounds cause cancer in humans, vanadium pentoxide causes lung cancer in rodents after long-term inhalation exposures and in turn IARC has categorized it as a group 2B possible human carcinogen. The goal of this study was to investigate the carcinogenicity of NaVO3 in the human immortalized bronchial epithelial cell line, Beas-2B.  Cells were treated with 10 µM NaVO3 for 5 weeks, with or without recovery time, followed by gene expression microarray analysis. In a separate experiment, cells were exposed to 1-10 µM NaVO3 for 4 weeks and then grown in soft agar to test for anchorage-independent growth. A dose-dependent increase in the number of colonies was observed. NaVO3-transformed clones could repair a wound faster than controls during the scratch test. In a gene expression microarray analysis of soft agar clones there were 2010 differentially expressed genes (DEG) (adjusted p-value ? 0.05) in NaVO3-transformed clones relative to control clones. DEG from this experiment were compared with the DEG of 5 week NaVO3 exposure with or without recovery, all with adjusted p-values <0.05, and 469 genes were altered in the same direction for transformed clones, 5 week NaVO3-treated cells, and the recovered cells; a subset of these changes were validated by QRT-PCR. The data from this study imply that chronic exposure to NaVO3 causes changes that are consistent with cellular transformation including anchorage- independent growth, enhanced migration ability, and gene expression changes that were epigenetically inherited through cell division. Beas-2B cells were chronically exposed to sodium metavanadate in growth media for 4 weeks. Then, one set of exposed cells and unexposed control cells were grown in soft agar for 3 weeks in the absence of vanadate. Transformed colonies (arising from vanadate-exposed cells) and control colonies (spontaneously grew from unexposed cells) were extracted from soft agar and expanded in the absence of vanadate, then processed for gene expression analysis . A second set of exposed cells continued treatment for a total of 5 weeks exposure. A third set of cells recovered for 1 week after the 4 week long exposure. The gene expression of 5 transformed clones, 3 control clones, 2 5 week exposures, 2 recovery, and 2 unexposed control cells, and 1 parental Beas-2B cell samples were analyzed (15 total samples) with an Affymetrix GeneChip Human Exon 1.0 ST array.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 153, 'E-GEOD-43386', 23, 'Sodium metavanadate exhibits carcinogenic tendencies in vitro in immortalized human bronchial epithelial cells [DEG.C]', 'Pentavalent vanadium compounds induce intracellular changes in vitro that are consistent with those of other carcinogenic substances, including alteration of transcription factor levels, promotion of oxidative stress, inflammation, DNA damage, stimulation of mitogenic signals, and suppression of apoptosis. While there is no clear evidence that vanadium compounds cause cancer in humans, vanadium pentoxide causes lung cancer in rodents after long-term inhalation exposures and in turn IARC has categorized it as a group 2B possible human carcinogen. The goal of this study was to investigate the carcinogenicity of NaVO3 in the human immortalized bronchial epithelial cell line, Beas-2B.  Cells were treated with 10 µM NaVO3 for 5 weeks, with or without recovery time, followed by gene expression microarray analysis. In a separate experiment, cells were exposed to 1-10 µM NaVO3 for 4 weeks and then grown in soft agar to test for anchorage-independent growth. A dose-dependent increase in the number of colonies was observed. NaVO3-transformed clones could repair a wound faster than controls during the scratch test. In a gene expression microarray analysis of soft agar clones there were 2010 differentially expressed genes (DEG) (adjusted p-value ? 0.05) in NaVO3-transformed clones relative to control clones. DEG from this experiment were compared with the DEG of 5 week NaVO3 exposure with or without recovery, all with adjusted p-values <0.05, and 469 genes were altered in the same direction for transformed clones, 5 week NaVO3-treated cells, and the recovered cells; a subset of these changes were validated by QRT-PCR. The data from this study imply that chronic exposure to NaVO3 causes changes that are consistent with cellular transformation including anchorage- independent growth, enhanced migration ability, and gene expression changes that were epigenetically inherited through cell division. Beas-2B cells were chronically exposed to sodium metavanadate in growth media for 4 weeks. Then exposed cells and unexposed control cells were grown in soft agar for 3 weeks in the absence of vanadate. Transformed colonies (arising from vanadate-exposed cells) and control colonies (spontaneously grew from unexposed cells) were extracted from soft agar and expanded in the absence of vanadate, then processed for gene expression analysis on Affymetrix GeneChip Human Exon 1.0 ST array. 5 transformed clones, 3 control clones, 1 parental Beas-2B cell sample, and acute 24 hour vanadate exposed cell samples were analyzed.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 154, 'E-GEOD-43302', 23, 'IM006A - Mouse infection with PB1-F2del: A/Vietnam/1203-CIP048_RG3/2004 (H5N1) at 10^3 PFU', 'To determine the host response (C57BL/6 mouse model) to the PB1-F2del: A/Vietnam/1203-CIP048_RG3/2004 (H5N1) virus at 10^3 PFU. Groups of 20-week-old C57BL/6 mice were infected with the PB1-F2del: A/Vietnam/1203-CIP048_RG3/2004 (H5N1) virus at 10^3 PFU or time-matched mock infected. Time points were 1, 2, 4 and 7 d.p.i. There were 5 mice/time point for infections and 3 mice/time point for time-matched mocks. Lung samples were collected for virus load, transcriptional analysis and proteomics analysis. Weight loss and animal survival were also monitored.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 155, 'E-GEOD-34213', 23, 'Gene expression data of micro RNA from skeletal muscle sample of pig DL and Pi breed', 'Total of 675 porcine miRNA candidates datasets revealed by 100% aligment homologs and review literatures publication. Probe-sets were screened, analysed for differential expression between different breeds (german landrace and pietrain) at different stages of development (63, 91 dpc and adult stage) microRNAs of skeletal muscle from german landrace and pietrain at 63, 91 dpc and adult stages were hybridized to Affymetrix GeneChip miRNA arrays.', '1132/NCBITaxon:9823', '1136/efo:EFO_0002768'
  UNION SELECT 156, 'E-GEOD-43608', 23, 'Gene expression profile of HCT116 exposed to chronic hypoxia', 'To identify genes involved in survival to prolonged hypoxia we exposed HCT116 to hypoxia for 3 days. Control cells were exposed to normoxic conditions. HCT116 colon cancer cells were serum starved and exposed to hypoxia (1%O2) or normoxia (21%O2) for 3 days.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 157, 'E-GEOD-43603', 23, 'Differential gene expression and exon usage in bone marrow from RARS patients with SF3B1 somatic mutations compared to healthy patients.', 'SF3B1, splicing factor 3b subunit 1 is a component of the RNA splicing machinery and it has been found to be highly mutated in refractory anemia with ring sideroblasts (RARS) and RARS associated with thrombocytosis (RARS-T). The aim of this study is to investigate the RNA splicing patterns in bone marrow from patients with SF3B1 somatic mutations compared to normal physiologic splicing. We compared the RNA splicing pattern as exon usage, differential gene expression, and pathway analysis with normal splicing mRNA profiles, as measured by RNA-Seq, in bone marrow derived mononuclear cells from 2 RARS patients with SF3B1 somatic mutations were compared to the mRNA profile of a normal donor.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 158, 'E-GEOD-43595', 23, 'IL-27 Inhibits HIV-1 Infection in Human Macrophages by Down-regulating Host Factor SPTBN1 during Monocyte to Macrophage Differentiation', 'The susceptibility of macrophages to HIV-1 infection is modulated during monocyte differentiation. IL-27 is an anti-HIV cytokine that also modulates monocyte activation. Here, we present new evidence that IL-27 promotes monocyte differentiation into macrophages that are non-permissive for HIV-1 infection. While IL-27 treatment does not affect expression of macrophage differentiation markers or macrophage biological functions, it confers HIV resistance by down-regulating spectrin beta non-erythrocyte 1 (SPTBN1), a required host factor for HIV-1 infection. IL-27 down-regulates SPTBN1 through a TAK-1-mediated MAPK signaling pathway. Knockdown of SPTBN1 strongly inhibits HIV-1 infection of macrophages; conversely, overexpression of SPTBN1 markedly increases HIV susceptibility of IL-27 treated macrophages. Moreover, we demonstrate that SPTBN1 associates with HIV-1 gag proteins. Collectively, our results underscore the ability of   IL-27 to protect macrophages from HIV-1 infection by down-regulating SPTBN1, thus indicating that SPTBN1 is an important host target to reduce HIV-1 replication in one major element of the viral reservoir. 2 samples with different treatments were analyzed. Genes with absolute fold change >= 5 were selected.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 159, 'E-GEOD-43588', 23, 'Reprogramming of human fibroblasts toward a cardiac fate', 'Reprogramming of mouse fibroblasts toward a myocardial cell fate by forced expression of cardiac transcription factors or microRNAs has recently been demonstrated.  The potential clinical applicability of these findings is based on the minimal regenerative potential of the adult human heart and the limited availability of human heart tissue.  An initial, but mandatory step toward clinical application of this approach is to establish conditions for conversion of adult human fibroblasts to a cardiac phenotype.  Toward this goal, we sought to determine the optimal combination of factors necessary and sufficient for direct myocardial reprogramming of human fibroblasts.  Here we show that four human cardiac transcription factors, including Gata4, Hand2, Tbx5, and myocardin, and two microRNAs, miR-1 and miR-133, activated cardiac marker expression in neonatal and adult human fibroblasts.  After maintenance in culture for 4-11 weeks, human fibroblasts reprogrammed with these proteins and microRNAs displayed sarcomere-like structures and calcium transients, and a small subset of such cells exhibited spontaneous contractility.  These phenotypic changes were accompanied by expression of a broad range of cardiac genes and suppression of non-myocyte genes.  These findings indicate that human fibroblasts can be reprogrammed to cardiac-like myocytes by forced expression of cardiac transcription factors with muscle-specific microRNAs and represent a step toward possible therapeutic application of this reprogramming approach. Human foreskin fibroblasts were transduced with 5 transcription factors and total RNA was obtained 4 weeks later.  total RNA was also obtained from human foreskin fibroblasts as a negative control and adult human heart tissue as a positive control. The expression level of genes in each sample was compared.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 160, 'E-GEOD-43585', 23, 'Expression profiles of bone marrow cells from model mice for erythropoietin-deficiency anemia (ISAM)', 'We established a novel mouse model for postnatal erythropoietin (Epo)-deficiency anaemia, designated ISAM (inherited super anemic mouse), using a transgenic complementation rescue technique. To identify Epo-regulated genes in vivo, we examined the mRNA expression profile in the bone marrow of ISAM 6 hours after recombinant human EPO (rHuEPO) administration. Erythropoietin-induced gene expression in mouse bone marrow was measured at 6 hours after rHuEPO administration (3,000 U/kg). Three Epo-treated samples were analyzed, and two PBS-treated and one untreated samples were used as a control group.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 161, 'E-GEOD-43581', 23, 'Hepatic glucose sensing is required to preserve beta-cell glucose competence', 'We assessed the impact of glucose transporter Glut2 gene inactivation in adult mouse liver (LG2KO mice). This suppressed hepatic glucose uptake but not glucose output. In the fasted state, expression of carbohydrate responsive element-binding protein (ChREBP) and its glycolytic and lipogenic target genes was abnormally elevated. Feeding, energy expenditure, and insulin sensitivity were identical in LG2KO and control mice. Glucose tolerance was normal early after Glut2 inactivation but intolerance developed at later time. This was caused by progressive impairment of glucose-stimulated insulin secretion even though beta-cell mass and insulin content remained normal. Liver transcript profiling revealed a coordinate down-regulation of cholesterol biosynthesis genes in LG2KO mice. This was associated with reduced hepatic cholesterol in fasted mice and a 30 percent reduction in bile acid production. We showed that chronic bile acids or FXR agonist treatment of primary islets increases glucose-stimulated insulin secretion, an effect not seen in islets from fxr-/- mice. Collectively, our data show that glucose sensing by the liver controls beta-cell glucose competence, through a mechanism that likely depends on bile acid production and action on beta-cells. three replicates each of ps_ctrl_refed, ps_tamox_refed, ps_ctrl_fasted, ps_tamox_fasted', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 162, 'E-GEOD-43570', 23, 'Expression data from wild type Sinorhizobium meliloti and mutants in the NSR grown under conditions of high and low nitrogen availability', 'Acquisition and utilization of nitrogen is governed by the Nitrogen Stress Response regulatory cascade.  GlnD is the primary nitrogen sensor in this system, and modifies the PII proteins to effect changes in the expression of target genes. We used microarrays to evaluate changes in gene expression due to nitrogen availability, mutation of the glnD gene, and deletion of both PII proteins. Sinorhizobium meliloti cultures were grown at 30C to mid-log phase (0.50.6 OD600) for RNA extraction and hybridization on Affymetrix microarrays.  Wild-type 1021, a strain containing a mutation in glnD, and a strain with both PII proteins deleted were grown in media containing either 0.05% ammonium chloride or 0.02% sodium glutamate as the nitrogen source.', '1132/NCBITaxon:382', '1136/efo:EFO_0002768'
  UNION SELECT 163, 'E-GEOD-43569', 23, 'Transcriptome profiling of Botrytis cinerea  conidial germination reveals upregulation of infection-related genes during the pre-penetration stage', 'Investigation of whole genome gene expression level changes in Botrytis cinerea wild type conidia during germination in comparison to the bmp1 MAP kinase deletion mutant. A 18 chip study using total RNA recovered from four different time points for the wild-type and one time point for the bmp1 deletion mutant. Each time point was analyzed with three biological replicates. Expression level of 20,885 genes from Botrytis cinerea strain B05.10 and T4 with three 60mer probes per gene was analysed.', '1132/NCBITaxon:40559', '1136/efo:EFO_0002768'
  UNION SELECT 164, 'E-GEOD-43568', 23, '8 NSCLC cell lines treated with either 50nM or 100nM of AUY for 24 hours.', 'RNA from treated cell lines are compared to RNA from matched untreated samples on the same chip to assess differential gene expression. RNA from treated cell lines are compared to RNA from matched untreated samples on the same chip to assess differential gene expression.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 165, 'E-GEOD-43567', 23, 'Baseline NSCLC cell lines compared to a reference mix consisting of 45 NSCLC cell lines', '34 NSCLC cell lines were transcriptionally profiled against a reference mix of 45 NSCLC cell lines to look subtype specific differential gene expression 34 individual cell lines were compared to a reference mix consiting of 45 NSCLC cell lines', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 166, 'E-GEOD-43566', 23, 'Biomarkers of residual disease, disseminated tumor cells, and metastases in the MMTV-PyMT breast cancer model', 'Gene expression profiling of disseminated tumor cells in lung, lung metastatses and residual tumor cells in the MMTV-PyMT breast cancer model. Profiling gene expression change between disseminated tumor cells, lung metastases and residual tumor cells from the MMTV-PyMT breast cancer model.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 167, 'E-GEOD-43563', 23, 'Expression data of  GMP and MDP from wild-type and Trib1-/- mice', 'Trib1 is critical for some myeloid cell differentiation. Therefore we used microarrays to investigate gene expression profiles in wild-type and Trib1-/- progenitor of myeloid cells. Progenitor cells, such as GMP and MDP, obtained from wild-type and Trib1-/- mice were sorted, followed by RNA extraction. Then hybridization on affymetrix microarrays was performed.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 168, 'E-GEOD-42670', 23, 'Patient specific orthotopic glioblastoma xenograft models recapitulate the histopathology and biology of human glioblastomas in situ', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 169, 'E-GEOD-42669', 23, 'Patient specific orthotopic glioblastoma xenograft models recapitulate the histopathology and biology of human glioblastomas in situ (gene expression)', 'Frequent discrepancies between preclinical and clinical results of anti-cancer agents demand a reliable translational platform that can precisely recapitulate the biology of human cancers. Another critical unmet need is the ability to predict therapeutic responses for individual patients. Toward this goal, we have established a library of orthotopic glioblastoma (GBM) xenograft models using surgical samples of GBM patients. These patient-specific GBM xenograft tumors recapitulate histopathological properties and maintain genomic characteristics of parental GBMs in situ. Furthermore, in vivo irradiation, chemotherapy, and targeted therapy of these xenograft tumors mimic the treatment response of parental GBMs. We also found that establishment of orthotopic xenograft models portends poor prognosis of GBM patients and identified the gene signatures and pathways signatures associated with the clinical aggressiveness of GBMs. Together, the patient-specific orthotopic GBM xenograft library represent the preclinically and clinically valuable patient tumors phenocopy that represents molecular and functional heterogeneity of GBMs. Gene expression profiling experiments were conducted for 58 human glioblastoma samples using Affymetrix Human Gene 1.0 ST arrays according to manufacturer''s protocol.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 170, 'E-GEOD-42616', 23, 'PRDM14 Ensures Nai?ve Pluripotency through Dual Regulation of Signaling and Epigenetic Pathways in Mouse Embryonic Stem Cells', 'Mouse embryonic stem cells (mESCs) fluctuate between a nai?ve inner cell mass (ICM)-like state and a primed epiblast-like state of pluripotency in serum, but are harnessed exclusively in a distinctive, nai?ve state of pluripotency that more faithfully captures the ICM state (the ground state) with inhibitors for mitogen-activated protein kinase (MAPK) and glycogen synthase kinase 3 pathways (2i). Understanding the mechanism ensuring the nai?ve states of pluripotency will be critical to realizing the full potential of ESCs. We show here that PRDM14, a PR domain-containing transcriptional regulator, ensures nai?ve pluripotency by a dual mechanism: Antagonizing fibroblast growth factor receptor (FGFR) signaling that is activated paradoxically by the core transcriptional circuitry for pluripotency and directs a primed state, and repressing de novo DNA methyltransferases that create a primed epiblast-like epigenome. PRDM14 exerts these functions by recruiting polycomb repressive complex 2 (PRC2) specifically to key targets and repressing their expression. ChIP-seq of PRDM14 and that of H3K27me3 and SUZ12 on Prdm14 wildtype and knockout ES cells', '1132/NCBITaxon:10090', '1136/efo:EFO_0002692'
  UNION SELECT 171, 'E-GEOD-42580', 23, 'PRDM14 Ensures a Naïve Pluripotency by a Dual Mechanism Involving Signaling and Epigenetic Pathways in Mouse Embryonic Stem Cells', 'Mouse embryonic stem cells (mESCs) fluctuate between a naïve inner cell mass (ICM)-like state and a primed epiblast-like state of pluripotency in serum, but are harnessed exclusively in a distinctive, apparently more naïve state of pluripotency (the ground state) with inhibitors for mitogen-activated protein kinase (MAPK) and glycogen synthase kinase 3 pathways (2i).  Understanding the mechanism ensuring a naïve state of pluripotency would be critical in realizing a full potential of ESCs.  We show here that PRDM14, a PR domain-containing transcriptional regulator, ensures a naïve pluripotency by a dual mechanism: Antagonizing fibroblast growth factor receptor (FGFR) signaling that is activated paradoxically by the core transcriptional circuitry for pluripotency and directs a primed state and repressing de novo DNA methyltransferases that create a primed epiblast-like epigenome.  PRDM14 exerts these functions by recruiting polycomb repressive complex 2 (PRC2) specifically to key targets and repressing their expression. Mouse Embryonic Stem Cells (mESCs) or mESC-like cells with different Prdm14 genotypes {Prdm14(+/+), Prdm14(-/-), and Prdm14(-/-) rescued with Avitag-EGFP-Prdm14 transgene [Prdm14(-/-)+AGP14]} are cultured on MEF in different medium [2i, Serum(day 2), Serum+MEK inhibitor (PD0325901) (day 2), Serum without LIF (day2)].', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 172, 'E-GEOD-42577', 23, 'Patient specific orthotopic glioblastoma xenograft models recapitulate the histopathology and biology of human glioblastomas in situ (CGH)', 'Frequent discrepancies between preclinical and clinical results of anti-cancer agents demand a reliable translational platform that can precisely recapitulate the biology of human cancers. Another critical unmet need is the ability to predict therapeutic responses for individual patients. Toward this goal, we have established a library of orthotopic glioblastoma (GBM) xenograft models using surgical samples of GBM patients. These patient-specific GBM xenograft tumors recapitulate histopathological properties and maintain genomic characteristics of parental GBMs in situ. Furthermore, in vivo irradiation, chemotherapy, and targeted therapy of these xenograft tumors mimic the treatment response of parental GBMs. We also found that establishment of orthotopic xenograft models portends poor prognosis of GBM patients and identified the gene signatures and pathways signatures associated with the clinical aggressiveness of GBMs. Together, the patient-specific orthotopic GBM xenograft library represent the preclinically and clinically valuable patient tumors phenocopy that represents molecular and functional heterogeneity of GBMs. aCGH experiments were performed for a human glioblastoma tissue (sample ID: PC-NS08-559) and the matching xenograft tumor tissue using the Agilent Human Whole Genome CGH 244K microarray according to manufacturer''s protocol (2-color).', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 173, 'E-GEOD-42261', 23, 'Notch Pathway Activation Targets AML Cell Homeostasis and Differentiation', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 174, 'E-GEOD-42260', 23, 'Notch Pathway Activation Targets AML Cell Homeostasis and Differentiation: MLL-AF9 transformed LGMP', 'To determine role of Notch signaling in AML leukemia initiating cells we used a conditional mouse knock-in model of Notch1-IC to induce Notch1-IC expression in MLL-AF9 transformed LGMP. WT and Notch1-IC+ LGMP were analyzed to determined genes controlled by Notch signaling. 12 weeks old wt lethaly irradiated mice were transplanted with 50000 cKit+ MLL-AF9-IRES-YFP infected cells from MLL-AF9 EF1 wt/wt ROSAwt/CreERT2 or MLL-AF9 EF1 wt/lsl-N1-IC ROSAwt/CreERT2 mice + 250000 support wt total bone marrow cells. 4 weeks after transplant mice were injected 2 times with tamoxifen (0.2mg/g body weight) every other day. Mice were sacrificed and analyzed 6 days after last injection. LGMP were flow purified for RNA extraction and hybridization on Affymetrix microarrays.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 175, 'E-GEOD-42259', 23, 'Notch Pathway Activation Targets AML Cell Homeostasis and Differentiation: THP1 cell line', 'Expression data from untreated or Dll4-Fc treated THP1 cell line.  We used Dll4-Fc stimulation of AML cells to study whether Notch activation has an impact on AML. We analyzed THP1 cell line in vitro treated with Dll4-Fc or vehicle control to determine genes affected by Notch activation. THP1 cell line was cultured on plate coated with 30 nM Dll4-Fc or vehicle for 48 hours prior to RNA extraction and hybridization to Human Genome U133 Plus 2.0 Affymetrix arrays.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 176, 'E-GEOD-41561', 23, 'A novel proteomic approach reveals GREB1 as an Estrogen Receptor co-factor', 'Methods for identifying protein-protein interactions have mostly been limited to tagged exogenous expression approaches. We now establish a rapid, robust and comprehensive method for finding interacting proteins using endogenous proteins from limited cell numbers. We apply this approach called Rapid IP-Mass Spectrometry of Endogenous proteins (RIME) to identify ER, FoxA1 and E2F4 interacting proteins in breast cancer cells. From small numbers of starting cells, we find a comprehensive collection of known ER, FoxA1 and E2F4 targets, plus a number of novel unexpected interactors. One of the most ER (and FoxA1) associated interactors is GREB1, an estrogen induced gene with almost no known function. We apply RIME, in parallel with ER ChIP-seq, to identify ER protein interactors and ER binding events from solid tumor xenografts, resulting in the validation of the ER-GREB1 interactions. Furthermore, we establish a method for identifying endogenous interacting proteins from solid primary breast cancer samples, whih we apply to validate ER interactions with GREB1 and additional co-factors. Mechanistically, we show that GREB1 is recruited with ER to the chromatin where it functions as an essential estrogen-mediated regulatory factor required for effective ER transcriptional activity. Our novel approach enables, for the first time, the ability for discovery and validation of protein-protein interactions in whole tissue and solid tumors, revealing significant insight into ER regulatory factors. Examination of ERGREB1 and E2F4 genomic binding patterns in cell line and xenograft tumour models', '1132/NCBITaxon:9606', '1136/efo:EFO_0002692'
  UNION SELECT 177, 'E-GEOD-41341', 23, 'Cultured Cyto and Syncytio-trophoblast samples exposed to varying degrees of hypoxia', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0002759'
  UNION SELECT 178, 'E-GEOD-41336', 23, 'Cultured Cyto and Syncytio-trophoblast samples exposed to varying degrees of hypoxia (methylation)', 'The mechanisms by which the placenta adapts to exogenous stimuli to create a stable and healthy environment for the growing fetus are not well known. Low oxygen tension and sub-optimal vitamin D levels influence placental function and both are associated with preeclampsia, a condition associated with altered development of placental trophoblast. We hypothesized that oxygen tension, vitamin D levels, or both affect villous trophoblast by modulation of gene expression through DNA methylation. To test this we used the Illumina Infinium Human Methylation 450 BeadChip array to compare the DNA methylation profile of primary cultures of human cytotrophoblasts and syncytiotrophoblasts under three oxygen tensions and three vitamin D levels. We found no effect on global DNA methylation by either treatment, but a limited set of loci became hypermethylated in cytotrophoblasts exposed for 24 hours to 1% oxygen, as compared to the same cells exposed to 8% or 20% oxygen. Vitamin D levels had no detectable effect on methylation profiles in either trophoblast type. Hypermethylation with low oxygen tension was independently confirmed by bisulfite-pyrosequencing in a subset of functionally important genes including CP, ITGA5, SOD2, XDH and ZNF2. Intriguingly, 70 out of the 147 hypoxia-associated CpGs, overlapped with CpG sites that become hypomethylated upon differentiation of cytotrophoblasts into syncytiotrophoblasts. Furthermore, the preponderance of altered sites was located at AP-1 binding sites. We suggest that AP-1 expression is triggered by hypoxia and interacts with DNA methyltransferases (DNMTs) to target methylation at specific sites in the genome, thus causing suppression of the associated genes that are responsible for differentiation of villous cytotrophoblast to syncytiotrophoblast. DNA from 2 cell types from 5 placentas exposed to 3 different conditions of hypoxia (1%,8%,20%) and treated with 3 levels of vitamin D were bisulfite converted and run on the Illumina HumanMethylation450 BeadChip', '1132/NCBITaxon:9606', '1136/efo:EFO_0002759'
  UNION SELECT 179, 'E-GEOD-41331', 23, 'Cultured Cyto and Syncytio-trophoblast samples exposed to varying degrees of hypoxia (expression)', 'The mechanisms by which the placenta adapts to exogenous stimuli to create a stable and healthy environment for the growing fetus are not well known. Low oxygen tension and sub-optimal vitamin D levels influence placental function and both are associated with preeclampsia, a condition associated with altered development of placental trophoblast. We hypothesized that oxygen tension, vitamin D levels, or both affect villous trophoblast by modulation of gene expression through DNA methylation. To test this we used the Illumina Infinium Human Methylation 450 BeadChip array to compare the DNA methylation profile of primary cultures of human cytotrophoblasts and syncytiotrophoblasts under three oxygen tensions and three vitamin D levels. We found no effect on global DNA methylation by either treatment, but a limited set of loci became hypermethylated in cytotrophoblasts exposed for 24 hours to 1% oxygen, as compared to the same cells exposed to 8% or 20% oxygen. Vitamin D levels had no detectable effect on methylation profiles in either trophoblast type. Hypermethylation with low oxygen tension was independently confirmed by bisulfite-pyrosequencing in a subset of functionally important genes including CP, ITGA5, SOD2, XDH and ZNF2. Intriguingly, 70 out of the 147 hypoxia-associated CpGs, overlapped with CpG sites that become hypomethylated upon differentiation of cytotrophoblasts into syncytiotrophoblasts. Furthermore, the preponderance of altered sites was located at AP-1 binding sites. We suggest that AP-1 expression is triggered by hypoxia and interacts with DNA methyltransferases (DNMTs) to target methylation at specific sites in the genome, thus causing suppression of the associated genes that are responsible for differentiation of villous cytotrophoblast to syncytiotrophoblast. RNA from cytotrophoblast from 2 placentas exposed to 3 different conditions of hypoxia (1%,8%,20%) and treated with 3 levels of vitamin D were run on the Illumina HT-12v4 Expression Array', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 180, 'E-GEOD-38994', 23, 'Next-Generation Transcriptome Sequencing (RNA-Seq) of Human Endobronchial Biopsies: Asthma vs Controls', 'Rationale: The cellular and molecular pathways in asthma are highly complex. Increased understanding can be obtained by unbiased transcriptomic analysis (RNA-Seq). Hypothesis and Aims: We hypothesized that the transcriptomic profile of whole human endobronchial biopsies differs between patients with asthma and controls. First, we investigated the feasibility to obtain RNA from whole endobronchial biopsies suitable for RNA-Seq. Second, we examined the difference in transcriptomic profiles between asthma and controls. Methods: This cross-sectional study compared 4 steroid-free atopic asthma patients and 5 healthy non-atopic controls.  RNA of ASM from 4 endobronchial biopsies per subject was isolated and sequenced (GS FLX+, 454/Roche). Ingenuity Pathway Analysis was used to identify gene networks. Comparison of the numbers of reads per gene in asthma and controls was based on the Poisson distribution. At the current sample size the estimated false discovery rate was 4%. Results: Yield of isolated RNA was 900-9,300ng. We identified 10,167 and 11,006 unique genes for asthma and controls, respectively. Forty-six genes were differentially expressed between asthma and controls, including pendrin, periostin, and BCL2. Ten gene networks involved in cellular morphology, movement, and development had an IPA network score ?2. Conclusion:RNA isolated from whole human endobronchial biopsies is suitable for RNA-Seq, showing different transcriptomic profiles between asthma and controls. Novel and confirmative genes were found to be linked to asthma. These results indicate that biological processes in the airways of asthma patients are differently regulated compared to healthy controls, which may be relevant for the pathogenesis and treatment of the disease. This cross-sectional transcriptomics study consisted of 2 visits. At visit 1, atopic asthma patients (n=4), and healthy non-atopic controls (n=5) were screened for eligibility to participate according to the in- and exclusion criteria. Spirometry and a methacholine bronchoprovocation test were performed. At visit 2, FEV1 reversibility was measured and 4 endobronchial biopsies per subject were collected during a bronchoscopy. Total RNA from whole endobronchial biopsies was collected. cDNA was prepared using the Ovation RNA-Seq System (NuGEN). RNA-Seq was performed using the GS FLX+ instrument (454/Roche). Sequence reads were mapped against the human genome (hg19; UCSC). Comparison of the numbers of reads per gene between asthma and healthy controls was based on the Poisson distribution.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 181, 'E-GEOD-37386', 23, 'A novel proteomic approach reveals GREB1 as an Estrogen Receptor co-factor', 'Methods for identifying protein-protein interactions have mostly been limited to tagged exogenous expression approaches. We now establish a rapid, robust and comprehensive method for finding interacting proteins using endogenous proteins from limited cell numbers. We apply this approach called Rapid IP-Mass Spectrometry of Endogenous proteins (RIME) to identify ER, FoxA1 and E2F4 interacting proteins in breast cancer cells. From small numbers of starting cells, we find a comprehensive collection of known ER, FoxA1 and E2F4 targets, plus a number of novel unexpected interactors. One of the most ER (and FoxA1) associated interactors is GREB1, an estrogen induced gene with almost no known function. We apply RIME, in parallel with ER ChIP-seq, to identify ER protein interactors and ER binding events from solid tumor xenografts, resulting in the validation of the ER-GREB1 interactions. Furthermore, we establish a method for identifying endogenous interacting proteins from solid primary breast cancer samples, whih we apply to validate ER interactions with GREB1 and additional co-factors. Mechanistically, we show that GREB1 is recruited with ER to the chromatin where it functions as an essential estrogen-mediated regulatory factor required for effective ER transcriptional activity. Our novel approach enables, for the first time, the ability for discovery and validation of protein-protein interactions in whole tissue and solid tumors, revealing significant insight into ER regulatory factors. Cells were transfected with siControl or siGREB1. The cells were treated with 10nM ICI 182780 for 24 hr to further deplete ER. 48 hr after transfection, the cells were treated with 100nM of Estrogen or vehicle for 6 hr and total RNA was collected from four biological replicates', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 182, 'E-GEOD-32868', 23, 'miRNA expression profiling in squamous cell carcinoma (SCC)', 'Despite its prevalence, the molecular basis of squamous cell carcinoma (SCC) remains poorly understood. We recently identified the developmental transcription factor Grhl3 as a potent tumor suppressor of SCC, and demonstrated that targeting of Grhl3 by a miR-21-dependent proto-oncogenic network underpins SCC in humans. Reduced levels of GRHL3 and its target gene PTEN are evident in human skin, and head and neck SCC, associated with increased expression of miR-21, which targets both tumor suppressors. Our data defines the miR-21-GRHL3-PTEN-axis as a critical tumor suppressor pathway in SCC. Total RNA was isolated from two SCC primary tissue samples and their matched normal skin controls.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 183, 'E-GEOD-32617', 23, 'MicroRNA-30c inhibits Human Breast Tumor Chemotherapy Resistance by regulating TWF1 and IL-11: MDA-MB231 DataSet', 'Chemotherapy resistance frequently drives tumor progression. However, the underlying molecular mechanisms are poorly characterized. Epithelial-to-mesenchymal transition (EMT) has been shown to correlate with therapy resistance, but the functional link and signaling pathways remain to be elucidated. We report here that miR-30c, a human breast tumor prognostic marker, plays a pivotal role in chemo-resistance by a direct targeting of the actin-transporter TWF1, which promotes EMT. An IL-6 family member, IL-11 was identified as a secondary target of TWF1 in the miR-30c signaling pathway. Expression of miR-30c inversely correlated with IL-11 expression in clinical tumors and IL-11 correlated with relapse-free survival in breast cancer patients. Identification of a novel miRNA-mediated pathway that regulates chemo-resistance in breast cancer will facilitate the development of new management strategies. MDA-MB231 30c vs. MDA-MB231 scrambled', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 184, 'E-GEOD-31466', 23, 'Centromere-Like Regions in Budding Yeast', 'Accurate chromosome segregation requires centromeres (CENs), the chromosomal sites where kinetochores form, to bridge DNA and attach to microtubules.  In contrast to most eukaryotes, Saccharomyces cerevisiae possesses sequence-defined point centromeres.  Chromatin immunoprecipitation followed by sequencing (ChIP-Seq) of four kinetochore components reveals regions of overlapping, extra-centromeric protein localization upon overproduction of the centromeric histone, Cse4 (CENP-A or CenH3). These identified sequences enhance proper plasmid and chromosome segregation, and are termed Centromere-like Regions (CLRs).  CLRs form in close proximity to S. cerevisiae CENs and share characteristics typical of point and regional centromeres.  CLR sequences are conserved among related budding yeasts, suggesting a role in vivo.  These studies provide new insights into the origin and evolution of centromeres. ChIP-Seq analysis of the kinetochore components Cse4, Mif2, Ndc10 and Ndc80 in budding yeast strains (Saccharomyces cerevisiae) with normal and elevated levels of Cse4', '1132/NCBITaxon:4932', '1136/efo:EFO_0002692'
  UNION SELECT 185, 'E-GEOD-23771', 23, 'CMV 2b immunoprecipitated small RNAs data set', 'We report that CMV2b exist in a complex of sRNAs in arabidopsis, with significant preference to repeat-associated AGO4-related 24nt siRNAs. This differential selection of siRNAs in vivo helps CMV 2b to perturb host epigenetic defense in the form of transposons activation and severeal repeat associated loci hypomethylation. Examination of CMV 2b presence in sRNAs complex in vivo', '1132/NCBITaxon:3702', '1136/efo:EFO_0003737'
  UNION SELECT 186, 'E-GEOD-22049', 23, 'MicroRNA-30c inhibits Human Breast Tumor Chemotherapy Resistance by regulating TWF1 and IL-11: Patient DataSet', 'Chemotherapy resistance frequently drives tumor progression. However, the underlying molecular mechanisms are poorly characterized. Epithelial-to-mesenchymal transition (EMT) has been shown to correlate with therapy resistance, but the functional link and signaling pathways remain to be elucidated. We report here that miR-30c, a human breast tumor prognostic marker, plays a pivotal role in chemo-resistance by a direct targeting of the actin-transporter TWF1, which promotes EMT. An IL-6 family member, IL-11 was identified as a secondary target of TWF1 in the miR-30c signaling pathway. Expression of miR-30c inversely correlated with IL-11 expression in clinical tumors and IL-11 correlated with relapse-free survival in breast cancer patients. Identification of a novel miRNA-mediated pathway that regulates chemo-resistance in breast cancer will facilitate the development of new management strategies. reference x sample', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 187, 'E-MTAB-1356', 23, 'Transcription profiling by array of H. sapiens acute myeloid leukemia mononuclear cells to investigate associations with distinct clinical and genetic features and lack KIT mutations', 'The inv(16)(p13q22)/t(16;16)(p13;q22) in acute myeloid leukemia results in multiple CBFB-MYH11 fusion transcripts, with type A being most frequent. The biologic and prognostic implications of different fusions are unclear. We analyzed CBFB-MYH11 fusion types in 208 inv(16)/t(16;16) patients with de novo disease, and compared clinical and cytogenetic features and KIT mutation status between type A (n=182; 87%) and non-type A (n=26; 13%) patients. We derived a fusion-type-associated gene- global expression profile. Gene Ontology analysis of the differentially expressed genes revealed - among others - an enrichment of up-regulated genes involved in activation of caspase activity, cell differentiation and cell cycle control in non-type A patients.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 188, 'E-MTAB-1245', 23, 'Proteomic profiling by array of tyrosine kinase activity in metastatic malignant melanoma from human tissue specimens', 'Tyrosine kinase activity profiling of metatstatic malignant melanoma and normal skin tissue samples was performed using peptide kinase arrays. All samples were run in triplicates with and withouth inhbitor PLX4032 and Sutent Malate in order to identify how tyrosine kinases responds to kinase inhibitor treatment, ex vivo.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002765'
  UNION SELECT 189, 'E-MTAB-1214', 23, 'Transcription profiling by array of house and forager bees infected with Metarhizium anisopliae', 'Transcription profiling by array of house and forager bees infected with Metarhizium anisopliae', '1132/NCBITaxon:7460', '1136/efo:EFO_0002768'
  UNION SELECT 190, 'E-MTAB-1090', 23, 'Transcription profiling by array of human mantle cell lymphoma tissue samples to identify genes involved in secondary drug resistance', 'We designed a gene profiling experiment to identify genes involved in secondary drug resistance in mantle cell lymphomas (MCL). We obtained paired tissue samples collected before treatment and after relapse or progression. Variations in gene expression between the 2 samples were estimated for 5 patients. For each gene, the mean variation was estimated for patients with a refractory primary tumor and for responders who developed secondary drug resistance. Nine genes of interest were selected on the basis of the magnitude and statistical significance of the variation of expression in responders and non-responders. BMP7 was the only one with significantly increased expression at relapse in patients who developed secondary resistance. Validation of BMP7 as a key gene involved in secondary resistance was performed using cultures of cell line. Incubation of BMP7 with MCL cell lines increased their resistance to Bortezomib and Cytarabine, while inhibition of BMP7 expression by siRNA correlated with increased cell death linked to drug application.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 191, 'E-MTAB-1049', 23, 'Gene Expression Profile Analysis of T1 and T2 Breast Cancer Tumors', 'The purpose of this study is to search for molecular signatures characterizing clinical T1 and T2 tumors and to better identify patients of high risk of relapse and/or metastasis. In total 46 samples, 29 T1 and 17 T2 infiltrating ductal carcinomas, were included.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 192, 'E-MTAB-1044', 23, 'Transcription profiling by array of CD34+ cells from the peripheral blood of 15 patients with chronic myelomonocytic leukemia and 4 healthy controls', 'Genomic studies in chronic myeloid malignancies, including myeloproliferative neoplasms (MPN), myelodysplastic syndromes (MDS) and MPN/MDS, have identified common mutations in genes encoding signaling, epigenetic, transcription and splicing factors. Analysis of 18 of these genes in a cohort of 224 chronic myelomonocytic leukemias (CMML), which is the most frequent MPN/MDS, identified at least one mutated gene in 95% of the patients, with some of the mutations affecting the disease phenotype. The number of mutated genes negatively affected progression-free and overall survival in multivariate analysis. Analysis of sorted progenitors indicated an early amplification of the CMML clone at the CD34+/CD38- stage of hematopoiesis, together with a premature granulomonocytic differentiation skewing. We interrogated the clone architecture by mutation-specific discrimination analysis of single-cell-derived colonies. The genetic classification of individual colonies allowed a designation of sub-clones and the assembly of putative evolutionary trees, indicating a linear acquisition of the studied mutations. Analysis of matched pre- and post-treatment samples demonstrated clonal persistence with limited and selective elimination of sub-clones. The disease was characterized by an amplification of multipotent and common myeloid progenitors in the CD34+ compartment, both fractions showing increased granulomonocytic differentiation at the expense of erythroid progenitors, contrasting with normal granulomonocytic progenitors. Altogether, early amplification of the leukemic clone and increased granulomonocytic differentiation of early progenitors may account for the specificity of CMML among myeloid neoplasms. This experiment correspond to the analysis of gene expression profiles in total CD34+ cells from the peripheral blood of 15 patients (Chronic Myelomonocytic Leukemia) and 4 healthy controls.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 193, 'E-MTAB-960', 23, 'microRNA profiling by array of human retinal pigment epithelial (RPE) cells to study the regulation of RPE redifferentiation', 'Retinal Pigment Epithelial (RPE) cells are located behind the retina and are critical for photoreceptor survival. Loss of RPE is associated with several pathogenic conditions such as Age Related Macular Degeneration and Retinitis Pigmentosa. RPE derived from human embryonic stem cells (hESC) offer a potential source for producing these cells for therapy. Here we report the molecular and cellular characterization of RPE differentiated from hESC. hESC derived RPE are capable of proliferation and lose their epithelial characteristics before becoming confluent and re-differentiating back into their typical pigmented, cobblestoned appearance. During the proliferative phase, they adopt a mesenchymal morphology and express mesenchymal markers. Our results demonstrate that this apparent Epithelial-Mesenchymal Transition is not regulated by the classical EMT transcription factors SNAIL and SLUG. Furthermore, it is possible to regulate RPE de-differentiation and re-differentiation by modulating the Wnt and BMP pathway respectively. These findings further our understanding of the genesis and expansion of RPE which is essential for their therapeutic use.', '1132/NCBITaxon:9606', '1136/efo:EFO_0000753'
  UNION SELECT 194, 'E-MTAB-840', 23, 'Genotyping by high throughput sequencing of four different cultivars of Lilium', 'The cDNA of 4 Lilium cultivars were sequenced in this study in order to develop SNP markers for mapping purpose. The four selected cultivars represent the four main hybrid groups in Lilium which are (Asiatic, Oriental, Longiflorum, and Trumpet). The SNP markers developed of this study will be implemented in mapping and association studies that aims to find SNP markers linked to virus resistance in Lilium. To achieve this target, SNP markers were generated from each cultivar separately, and of the four cultivars together trying to cover as much as variation in each cultivar and also to find SNP markers that can  distinguish the four cultivars. Doing so, we hope to identify different genes resistance to virus in the four hybrid groups of Lilium.', '1132/NCBITaxon:82328', '1136/efo:EFO_0002771'
  UNION SELECT 195, 'E-MTAB-696', 23, 'Transcription profiling by array of rat sympatheitic neurons that are NGF-deprived identifies molecular pathways associated with cell death', 'Developing sympathetic neurons depend on nerve growth factor (NGF) for survival and die by apoptosis after NGF withdrawal. This process requires de novo gene expression but only a small number of genes induced by NGF deprivation have been identified so far. We have used Affymetrix Exon arrays to study the pattern of expression of all known genes in sympathetic neurons deprived of NGF. We identified 415 up- and 813 down-regulated genes, including most of the genes previously known to be regulated in this system. By including a mixed lineage kinase (MLK) inhibitor, CEP-11004, in our experimental design we identified which of the genes induced after NGF withdrawal are potential targets of the MLK-JNK-c-Jun pathway. A detailed Gene Ontology and functional enrichment analysis also identified genetic pathways, such as the ER unfolded protein response, that are highly enriched and overrepresented amongst the genes expressed after NGF withdrawal whilst hierarchical cluster analysis revealed four major patterns of gene expression. Five genes not previously studied in sympathetic neurons - trb3, ddit3, txnip, ndrg1 and mxi1 - were validated by real time-PCR. The proteins encoded by these genes also increased in level after NGF withdrawal and this increase was prevented by CEP-11004, suggesting that these genes are potential targets of the MLK-JNK-c-Jun pathway. Overall, our microarray data gives a comprehensive overview of, and provides new information about, signalling pathways and transcription factors that are regulated by NGF withdrawal and identifies potential targets of the MLK-JNK-c-Jun pathway in sympathetic neurons', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 196, 'E-MEXP-3768', 23, 'Transcription profiling by array of Arabidopsis thaliana dnd1 mutant grown in control conditions', 'Transcript profiling of dnd1 mutant grown in control condition.', '1132/NCBITaxon:3702', '1136/efo:EFO_0002768'
  UNION SELECT 197, 'E-MEXP-3662', 23, 'Transcription profiling by array of human MCF7 cell lines to investigate GDNF-RET signaling as a target in aromatase inhibitor resistant breast cancers', 'Understanding the complex molecular mechanisms underlying resistance to endocrine therapy is a major challenge in the treatment of estrogen receptor-positive (ER+) breast cancers. We have previously demonstrated that glial cell line-derived neurotrophic factor (GDNF) signaling via the receptor tyrosine kinase RET promotes estrogen independent activation of ER. Here we have addressed the relevance of GDNF-RET signaling in response to aromatase inhibitor treatment and explored the efficacy of using RET inhibitors in breast cancer models of aromatase inhibitor response and resistance. A GDNF-response gene set, identified from gene expression profiling, was demonstrated to be an independent prognostic marker of poor patient outcome and, importantly, to be predictive of poor response to aromatase inhibitor treatment and development of resistance. The relevance of these findings was validated first by demonstrating an association of RET protein expression in an independent cohort of aromatase inhibitor resistant patient samples. Second, in in vitro models, GDNF-mediated RET signaling was demonstrated to enhance the survival of aromatase inhibitor resistant cells and to increase resistance in aromatase inhibitor sensitive cells. These effects could be reversed by targeting GDNF/RET signaling with the RET selective inhibitor NVP-BBT594 thus identifying GDNF-RET signaling as a potential therapeutic target, particularly in breast cancers resistant to aromatase inhibitors.MCF7 cells were E2-deprived by culturing in phenol red-free RPMI 1640 supplemented with 10% DCC for 3 days and then serum-starved overnight in the presence or absence of fulvestrant (ICI182,780) (100 nM). The following day, cells were treated with GDNF (20 ng/ml) for 0, 4, 8, 24 and 48 hours in the presence or absence of fulvestrant (ICI182,780) (100 nM).', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 198, 'E-MEXP-3657', 23, 'Transcription profiling by array of the physiological, biochemical and molecular responses to a combination of drought and chronic ozone in Medicago truncatula', 'Gene expression changes in response to drought, chronic ozone and combined drought and ozone in comparison to control plants grown under normal conditions', '1132/NCBITaxon:3880', '1136/efo:EFO_0002768'
  UNION SELECT 199, 'E-MEXP-3651', 23, 'ChIP-chip by array of 3 pairs of isogenic human mammary epithelial and fibroblast cells using anti-Methylcytosine', 'three pairs of isogenic human mammary epithelial and fibroblast cells, were used to characterized cell type specific dna methylation using using methyla cytosin immunoprecipitation', '1132/NCBITaxon:9606', '1136/efo:EFO_0002760'
  UNION SELECT 200, 'E-MEXP-3596', 23, 'MicroRNA profling by array of young and senescent human umbilical vein endothelial cells', 'miRNA transcription in senescent human umbilical vein endothelial cells (HUVEC) compared to young HUVECs.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 201, 'E-MEXP-3542', 23, 'Transcription profiling by array of human stem-like cancer cells in melanoma cell lines', 'Nine heterogeneous melanoma cell lines including D10 and WM115 were studied for cancer stem cell characteristics in vitro. D10 cell line was the only cell line expressing the cancer stem cell marker CD133. Thus, gene expression profiling on CD133+ and CD133- D10 cells was carried out using Affymetrix GeneChips Human Genome U133A 2.0.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 202, 'E-MEXP-3507', 23, 'Gene expression profile induced by Foxe1 in FRT cells', 'The gene expression profile of FRT cells stably transfected with the transcription factor Foxe1 was analyzed.', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 203, 'E-MEXP-3490', 23, 'Transcription profiling by array of mouse E14.5 Numblike (knock-out) / numb (lox) x nkx2.5-Cre embrynoic hearts to study whether Numb/Numblike are involved in inhibition of Notch activity in the developing heart', 'The ventricular wall of the heart is composed of trabeculated and compactlayers, which are separated by yet unknown processes during embryonic development. Notch signaling plays important roles in cardiac development. Mutations in Notch signaling components are related to human congenital heart diseases Two homologues, Numb and Numblike (Numbl), are expressed in mammals, which also act as inhibitors of Notch signaling Here we investigate the role of Notch2 and Numb/Numblike during myocardial trabeculation and compaction. The aim of the experiment is to analyze whether Numb/Numblike are involved in inhibition of Notch activity in the developing heart.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 204, 'E-GEOD-43598', 23, 'miR-1061 and miR-591 are involved in paclitaxel-resistance and their regulation increases the sensitivity to paclitaxel', 'MicroRNA expression in paclitaxel (PTX)-resistant SKpac sublines was compared to that of the PTX-sensitive, parental SKOV3 ovarian cancer cell line using microarray and real-time RT-PCR. The function of differentially expressed microRNAs in chemoresistant ovarian cancer was further evaluated by apoptosis, cell proliferation, and migration assays. Total RNA obtained from four different chemo-resistant subcell lines (skp 41,43,44,46) compared to parent cell line (skov3), human ovary cacinoma', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 205, 'E-GEOD-43584', 23, 'Expression analysis of systemically infected Nicotiana benthamiana infected by South African cassava mosaic virus (SACMV)', 'Micrarray analysis was used to identify gene expression changes associated with disease development  and virus movement in N.benthamina plants induced by infection with the SACMV 1-Plex , 385K array Nicotiana benthamiana (NimbleGen design name: 110121_N_benthamiana_60mer_exp) was used in this study to monitor changes in gene expression levels in SACMV- infected leaf tissue. Three biological replicates were used for infected leaf tissue and one pooled mock-inoculated sample was used as a control/reference.', '1132/NCBITaxon:4100', '1136/efo:EFO_0002768'
  UNION SELECT 206, 'E-GEOD-43564', 23, 'The Molecular Toxicity Identification Evaluation (mTIE) Approach Predicts Chemical Exposure in Daphnia magna', 'Daphnia magna is a bio-indicator organism accepted by several international water quality regulatory agencies.  Current approaches for assessment of water quality rely on acute and chronic toxicity that provide no insight into the cause of toxicity. Recently, molecular approaches, such as genome wide gene expression responses, are enabling an alternative mechanism based approach to toxicity assessment. While these genomic methods are providing important mechanistic insight into toxicity, statistically robust prediction systems that allow the identification of chemical contaminants from the molecular response to exposure are needed. Here we apply advanced machine learning approaches to develop predictive models of contaminant exposure using a D. magna gene expression dataset for 36 chemical exposures. We demonstrate here that we can discriminate between chemicals belonging to different chemical classes including endocrine disruptors, metals and industrial chemicals based on gene expression.  We also show that predictive models based on indices of whole pathway transcriptional activity can achieve comparable results while facilitating biological interpretability. D. magna were exposed to 36 Chemicals and 5 control series in quadruplicate.', '1132/NCBITaxon:35525', '1136/efo:EFO_0002768'
  UNION SELECT 207, 'E-GEOD-43559', 23, 'Environmentally Induced Transgenerational Epigenetic Reprogramming of Primordial Germ Cells and Subsequent Germline', 'A number of environmental factors (e.g. toxicants) have been shown to promote the epigenetic transgenerational inheritance of disease and phenotypic variation. Transgenerational inheritance requires the germline transmission of altered epigenetic information between generations in the absence of direct environmental exposures. The primary periods for epigenetic programming of the germline is associated with primordial germ cell development and during fetal gonadal sex determination. The current study examined the actions of an agricultural fungicide vinclozolin on gestating female (F0 generation) progeny in regards to the primordial germ cell (PGC) epigenetic reprogramming of the F3 generation (i.e. great-grandchildren). The F3 generation primordial germ cell transcriptome and epigenome (DNA methylation) was altered transgenerationally. Interestingly, the differential DNA methylation regions (DMR) and altered transcriptomes were distinct between the onset of gonadal sex determination at embryonic day 13 (E13) and after cord formation in the testis at embryonic day 16 (E16). A larger number of DMR and transcriptional alterations were observed in the E13 PGC than E16 germ cells. Observations demonstrate an altered transgenerational epigenetic reprogramming and function of the primordial germ cells and subsequent male germline is a component of vinclozolin induced epigenetic transgenerational inheritance of disease. Insights into the molecular control of germline transmitted epigenetic inheritance are provided. The combined observations demonstrate ancestral exposure of a gestating female during fetal gonadal sex determination can promote transgenerational alterations in the primordial germ cell and subsequent male germline epigenetic and transcriptional programming. This altered germline programming leads to the epigenetic transgenerational inheritance of disease and phenotypic variation. Observations support the role of the primordial germ cell programming in the molecular mechanism involved and provides insights into the molecular mechanisms that control the epigenetic transgenerational inheritance phenomena. Results suggest a cascade of epigenetic and transcriptional events during germ cell development is needed to obtain the mature germline epigenome that is then transmitted transgenerationally. RNA samples from  PGC of 2 F3-control lineage groups were compared to PGC of 2 F3-vinclozolin lineage groups for two embryonic age E13 and E16', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 208, 'E-GEOD-43548', 23, 'Increase in chromatin accessibility to MNase at sites of DNA replication', 'Faithful DNA replication is essential for normal cell division and differentiation. In eukaryotic cells, DNA replication takes place on chromatin. This poses the critical question as to how DNA replication can progress through chromatin, which is inhibitory to all DNA-dependent processes. Here, we have developed a novel genome-wide method to measure chromatin accessibility to micrococcal nuclease that is normalized for nucleosome density, NCAM (Normalized Chromatin Accessibility to MNase) assay.  This method enabled us to discover that chromatin accessibility increases specifically at and ahead of DNA replication forks in normal S phase and during replication stress.   We further found that Mec1, a key regulatory ATR-like kinase in the S-phase checkpoint, is required for both normal chromatin accessibility around replication forks and replication fork rate during replication stress. In this study we sought to analyze the chromatin structural changes that take place at sites of DNA replication. To this end, we obtained yeast cell populations synchronously undergoing DNA replication by ?-factor G1 arrest and release. In order to analyze chromatin structure at sites of DNA replication we first mapped the genomic locations undergoing DNA replication to high-resolution, strand-specific microarrays tiling chromosomes III, VI, and XII, covering ~14% of the genome. Two complementary approaches were taken: (1) we mapped fork positions by chromatin immunoprecipitation (ChIP) of a FLAG-tagged DNA polymerase 1 (Pol1), a replication fork component and (2) we mapped the sites of active DNA synthesis by generating DNA copy number profiles. We then analyzed chromatin structure at the sites of DNA replication by Micrococcal nuclease mononucleosome mapping and by generating histone H3 density maps. We further generated a Normalized Chromatin Accessibility to MNase (NCAM) signal by normalizing MNase mononucleosome signal for histone H3 density. NCAM signal represents a measure of nucleosome accessibility to MNase that is normalized for nucleosome content. These three approaches allowed us to assess potential changes in nucleosome positioning, nucleosome density, and nucleosome accessibility during DNA replication. We searched for changes in chromatin structure by comparing, during S phase, regions undergoing DNA replication to those not yet replicated, and also by comparing the same region before replication (not replicated, G1 arrested control) and during DNA replication in S phase. We typically harvested S phase cells at 30 or 60 minutes after release from G1 arrest. Experimental conditions included releasing cells into rich media at 24°C or into rich media containing 200 mM Hydroxyurea (HU). Both conditions slowed down replication fork rate and made these experiments feasible. For each strain, samples for the different experiments (Chromatin for Pol1 and H3 ChIP, in vivo MNase digestion, and DNA for DNA copy number profiles) were harvested simultaneously for each time point. Therefore, comparisons between time points for each strain must be made using samples from the same replicate experiment. Our analysis included WT cells, as well as S phase checkpoint mutants (?mec1 ?sml1 and mec1-100 ?sml1), as well as control strains (?sml1).', '1132/NCBITaxon:4932', '1136/efo:EFO_0002762'
  UNION SELECT 209, 'E-GEOD-43547', 23, 'Genome-wide analysis of hypothalamic gene expression in prenatally undernourished male rat offspring under high fat diet', 'Based on the developmental origin of health of disease hypothesis, we previously showed that prenatal 70% maternal food restriction (FR30) predisposes the offspring to development of pathologies in adulthood. In the present study, we focused on the hypothalamus gene expression profile of standard and high fat (HF)-fed FR30 adult offspring. Total RNA obtained from isolated hypothalami of control standard and high fat-fed adult offspring compared to prenatally undernourished (FR30) counterparts.', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 210, 'E-GEOD-43535', 23, 'Deregulation of microRNAs by HIV-1 Vpr protein leads to the development of neurocognitive disorders [differentiated SH-5Y cells]', 'Studies have shown that HIV-infected patients develop neurocognitive disorders characterized by neuronal dysfunction. The lack of productive infection of neurons by HIV suggests that viral and cellular proteins, with neurotoxic activities, released from HIV-1-infected target cells can cause this neuronal deregulation. The viral protein R (Vpr), a protein encoded by HIV-1, has been shown to alter the expression of various important cytokines and inflammatory proteins in infected and uninfected cells; however the mechanisms involved remain unclear. Using a human neuronal cell line, we found that Vpr can be taken up by neurons causing: (i) deregulation of calcium homeostasis, (ii) endoplasmic reticulum-calcium release, (iii) activation of the oxidative stress pathway, (iv) mitochondrial dysfunction and v- synaptic retraction. In search for the cellular factors involved, we performed microRNAs and gene array assays using human neurons (primary cultures or cell line, SH-SY5Y) that we treated with recombinant Vpr proteins. Interestingly, Vpr deregulates the levels of several microRNAs (e.g. miR-34a) and their target genes (e.g. CREB), which could lead to neuronal dysfunctions. Therefore, we conclude that Vpr plays a major role in neuronal dysfunction through deregulating microRNAs and their target genes, a phenomenon that could lead to the development of neurocognitive disorders. Human neurons SH-SY5Y were chosen to examine the impact of HIV-1 Vpr protein on gene expression', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 211, 'E-GEOD-43534', 23, 'HIV-1 Tat protein promotes neuronal dysfunction through disruption of microRNAs [differentiated SH-5Y cells]', 'Over the last decade, small noncoding RNA molecules such as microRNAs (miRNAs) have emerged as critical regulators in the expression and function of eukaryotic genomes. It has been suggested that viral infections and neurological disease outcome may also be shaped by the influence of small RNAs. This has prompted us to suggest that HIV infection alters the endogenous miRNA expression patterns, thereby contributing to neuronal deregulation and AIDS dementia. Therefore, using primary cultures and neuronal cell lines, we examined the impact of a viral protein (HIV-1 Tat) on the expression of miRNAs due to its characteristic features such as release from the infected cells and taken up by noninfected cells. Using microRNA array assay, we demonstrated that Tat deregulates the levels of several miRNAs. Interestingly, miR-34a was among the most highly induced miRNAs in Tat-treated neurons. Tat also decreases the levels of miR-34a target genes such as CREB protein as shown by real time PCR. The effect of Tat was neutralized in the presence of anti-miR-34a. Using in situ hybridization assay, we found that the levels of miR-34a increase in Tat transgenic mice when compared with the parental mice. Therefore, we conclude that deregulation of neuronal functions by HIV-1 Tat protein is miRNA-dependent. Human neurons SH-SY5Y were chosen to examine the impact of HIV-1 Tat protein on gene expression', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 212, 'E-GEOD-43524', 23, 'Microarray analysis of wildtype, Klf8gt/gt, Klf3-/- and Klf3-/- Klf8gt/gt TER119+ E13.5 fetal liver cells', 'The aim of this experiment was to investigate the regulation of gene expression by KLF3 and KLF8 in fetal erythroid cells by analyzing single and double mutant mouse models. Affymetrix microarrays were performed on RNA from TER119+ fetal liver cells from E13.5 wildtype, Klf8gt/gt, Klf3-/- and Klf3-/- Klf8gt/gt mice. Four wildtype replicates, four Klf8gt/gt replicates, three Klf3-/- replicates and four Klf3-/- Klf8gt/gt replicates of E13.5 TER119+ fetal liver cell samples, litter-matched where possible.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 213, 'E-GEOD-43509', 23, 'long noncoding RNAs expression data from GM0637 with and without DNA damage', 'GM0637 cell were treated with or without DNA damaging agent neocarzinostatin (NCS), and cells were harvested after 4 hours and 8 hours for the microarray analyses of whole-genome long noncoding RNAs. To examine how long noncoding RNAs are regulated in the DNA damage response, we assessed the genome-wide long noncoding RNA expression in GM0637 cells treated with or without DNA damage', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 214, 'E-GEOD-43496', 23, 'Gene expression of rat PSCs cultured on plastic, matrigel and collagen', 'Activated pancreatic stellate cells produce the fibrotic matrix in chronic pancreatitis and pancreatic cancer. In vitro protocols examining PSC biology have usually involved PSCs cultured on plastic, a non-physiological surface. However, PSCs cultured on physiological matrices e.g. MatrigelTM (normal basement membrane) and collagen (fibrotic pancreas), may have distinctly different behaviours compared to cells cultured on plastic. Therefore, we aimed to compare PSC gene expression after culture on plastic, MatrigelTM and collagen I. Total RNA from stellate cells in 10 cm Petri dishes was isolated by Qiagen RNeasy Mini Plus kit as per manufacturers instructions. The Agilent 2100 Bioanalyzer (Agilent Technologies Inc. Santa Clara, CA) was used for quality control of the isolated total RNA. Gene expression profiles of rat PSCs cultured on MatrigelTM, collagen I and plastic were analysed by whole rat genome microarray purchased from Affymetrix (Rat Gene 1.0 ST Array). This array was able to detect 27,342 rat genes, with approximately 26 probes on average per gene (referred to as a probe set).', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 215, 'E-GEOD-43425', 23, 'Long noncoding RNAs expression data from Atm+/+ and Atm-/- mouse embryonic fibroblasts  after DNA damage at various time point', 'Atm+/+ and Atm-/- mouse embryonic fibroblasts were treated with or without DNA damaging agent neocarzinostatin (NCS), and cells were harvested after 4 hours and 8 hours for the microarray analyses of whole-genome long noncoding RNAs. To examine how long noncoding RNAs are regulated in the DNA damage response, we assessed the genome-wide long noncoding RNA expression in Atm+/+ and Atm-/- littermate mouse embryonic fibroblasts (MEFs) treated with or without DNA damage', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 216, 'E-GEOD-42701', 23, 'Direct Conversion of Fibroblasts to Neurons by Reprogramming PTB-Regulated microRNA Circuits', 'The induction of pluripotency or trans-differentiation of one cell type to another  can be accomplished with cell lineage-specific transcription factors. Here we report that  repression of a single RNA binding protein PTB, which occurs during normal brain  development via the action of miR-124, is sufficient to induce trans-differentiation of  fibroblasts into functional neurons. Besides its traditional role in regulated splicing, we  show that PTB has a previously undocumented function in the regulation of microRNA  functions, suppressing or enhancing microRNA targeting by competitive binding on  target mRNA or altering local RNA secondary structure. A key event during neuronal  induction is the relief of PTB-mediated blockage of microRNA action on multiple  components of the REST complex, thereby de-repressing a large array of neuronal genes,  including miR-124 and multiple neuronal-specific transcription factors, in non-neuronal  cells. This converts a negative feedback loop to a positive one to elicit cellular  reprogramming to the neuronal lineage. Examination of PTB regulated  AGO2/microRNA targeting in Hela cells by CLIP-seq (two biological replicates) , paired-end RNA-seq (control and PTB knockdown) and 3end stability RNA-seq  (control and PTB knockdown)', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 217, 'E-GEOD-42576', 23, 'CBFb Stabilizes HIV Vif to Counteract APOBEC3 at the Expense of RUNX1 Target Gene Expression', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 218, 'E-GEOD-42575', 23, 'CBFb Stabilizes HIV Vif to Counteract APOBEC3 at the Expense of RUNX1 Target Gene Expression [ChIP-seq]', 'The HIV-1 accessory protein Vif hijacks a cellular Cullin-RING ubiquitin ligase, CRL5, to promote degradation of the APOBEC3 (A3) family of restriction factors. Recently, the cellular transcription cofactor CBFb was shown to form a complex with CRL5-Vif and to be essential for A3 degradation and viral infectivity. We now demonstrate that CBFb is required for assembling a well-ordered CRL5-Vif complex by inhibiting Vif oligomerization and by activating CRL5-Vif via direct interaction. The CRL5-Vif-CBFb holoenzyme forms a wellde?ned heterohexamer, indicating that Vif simultaneously hijacks CRL5 and CBFb. Heterodimers of CBFb and RUNX transcription factors contribute toward the regulation of genes, including those with immune system functions. We show that binding of Vif to CBFb is mutually exclusive with RUNX heterodimerization and impacts the expression of genes whose regulatory domains are associated with RUNX1. Our results provide a mechanism by which a pathogen with limited coding capacity uses one factor to hijack multiple host pathways. Identification of RUNX1 binding sites in the Jurkat cell line', '1132/NCBITaxon:9606', '1136/efo:EFO_0002692'
  UNION SELECT 219, 'E-GEOD-42574', 23, 'CBFb Stabilizes HIV Vif to Counteract APOBEC3 at the Expense of RUNX1 Target Gene Expression [gene expression]', 'The HIV-1 accessory protein Vif hijacks a cellular Cullin-RING ubiquitin ligase, CRL5, to promote degradation of the APOBEC3 (A3) family of restriction factors. Recently, the cellular transcription cofactor CBFb was shown to form a complex with CRL5-Vif and to be essential for A3 degradation and viral infectivity. We now demonstrate that CBFb is required for assembling a well-ordered CRL5-Vif complex by inhibiting Vif oligomerization and by activating CRL5-Vif via direct interaction. The CRL5-Vif-CBFb holoenzyme forms a wellde?ned heterohexamer, indicating that Vif simultaneously hijacks CRL5 and CBFb. Heterodimers of CBFb and RUNX transcription factors contribute toward the regulation of genes, including those with immune system functions. We show that binding of Vif to CBFb is mutually exclusive with RUNX heterodimerization and impacts the expression of genes whose regulatory domains are associated with RUNX1. Our results provide a mechanism by which a pathogen with limited coding capacity uses one factor to hijack multiple host pathways. Analysis of Vif-dependent effects on gene expression in Jurkats when cells are activated for 4 or 6 hours with PMA and PHA', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 220, 'E-GEOD-41677', 23, 'Latent enhancers unveiled by stimulation expand and adapt the available cis-regulatory repertoire (FAIRE-seq)', 'According to current models, transcription factors (TFs) activated by extracellular stimuli operate in the context of a pre-established enhancer repertoire induced and maintained by lineage-specific TFs. Here, we uncovered the existence of latent enhancers, defined as regions of the genome that in terminally differentiated cells are poorly accessible and lack the histone marks characteristic of enhancers, but readily acquire these features in response to extracellular cues. Stimulation of resting macrophages caused simultaneous binding of stimulus-activated TFs and lineage-determining TFs to these regions, enabling deposition of enhancer-specific features. Once unveiled, these enhancers did not return to a latent state even when stimulation ceased; instead, they persisted and mediated a faster and stronger response upon restimulation. We suggest that stimulus-specific expansion of the available cis-regulatory repertoire provides an epigenomic memory of the exposure to environmental agents. Formaldehyde-Assisted Isolation Of Regulatory Elements (FAIRE) followed by multiparallel sequencing was performed in untreated murine bone marrow-derived macrophages.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002692'
  UNION SELECT 221, 'E-GEOD-40739', 23, 'Genome-wide quantitative enhancer activity maps identified by STARR-seq', 'Genomic enhancers are important regulators of gene expression, but their identification is a challenge and methods depend on indirect measures of activity. We developed a method termed STARR-seq to directly and quantitatively assess enhancer activity for millions of candidates from arbitrary sources of DNA, enabling screens across entire genomes. When applied to the Drosophila genome, STARR-seq identifies thousands of cell type-specific enhancers across a broad continuum of strengths, linking differential gene expression to differences in enhancer activity and creating a genome-wide quantitative enhancer map. This map reveals the highly complex regulation of transcription, with several independent enhancers for both developmental regulators and ubiquitously expressed genes. STARR-seq can be used to identify and quantitate enhancer activity in other eukaryotes, including human. STARR-seq was performed in S2 and OSC cells with paired-end sequencing in two replicates and respective inputs. DHS-seq was done with single-end sequencing in two replicates for S2 and OSC cells. RNA-seq was performed with a strand-specific protocol using single-end sequencing in two replicates within S2 and OSC cells. STARR-seq was also performed in HeLa cells with single-end sequencing with a respective input.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 222, 'E-GEOD-38848', 23, 'Expression data from Saccharomyces cerevisiae', 'In this study we focus on two Saccharomyces cerevisiae strains with varying production of heterologous ?-amylase and we compare the metabolic fluxes and transcriptional regulation at aerobic and anaerobic conditions, in particular with the objective to identify the final electron acceptor for protein folding. We found that anaerobic conditions showed high amount of amylase productions when comparing to aerobic conditions and the genome-scale transcriptional analysis suggested that genes related to the endoplasmic reticulum (ER), lipid synthesis and stress responses were generally up-regulated at anaerobic conditions. Moreover, we proposed a model for the electron transfer from ER to the final electron acceptor, fumarate under anaerobic conditions. Three Saccharomyces cerevisiae strains with varied amylase productions were selected at early glucose phase in batch fermentations for RNA extraction and hybridization on Affymetrix microarrays. Biological triplicates were applied, and strains with empty plasmid (no amylase productions) were used as control strain.', '1132/NCBITaxon:4932', '1136/efo:EFO_0002768'
  UNION SELECT 223, 'E-GEOD-38379', 23, 'Latent enhancers unveiled by stimulation expand and adapt the available cis-regulatory repertoire', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 224, 'E-GEOD-38377', 23, 'Latent enhancers unveiled by stimulation expand and adapt the available cis-regulatory repertoire (ChIP-seq)', 'According to current models, transcription factors (TFs) activated by extracellular stimuli operate in the context of a pre-established enhancer repertoire induced and maintained by lineage-specific TFs. Here, we uncovered the existence of latent enhancers, defined as regions of the genome that in terminally differentiated cells are poorly accessible and lack the histone marks characteristic of enhancers, but readily acquire these features in response to extracellular cues. Stimulation of resting macrophages caused simultaneous binding of stimulus-activated TFs and lineage-determining TFs to these regions, enabling deposition of enhancer-specific features. Once unveiled, these enhancers did not return to a latent state even when stimulation ceased; instead, they persisted and mediated a faster and stronger response upon restimulation. We suggest that stimulus-specific expansion of the available cis-regulatory repertoire provides an epigenomic memory of the exposure to environmental agents. Chromatin immunoprecipitations of H3 lysine 4 mono-methylated, H3 lysine 27 acetylated, H3 lysine 4 tri-methylated, the transcription factor PU.1 and total RNA polymerase II followed by multiparallel sequencing performed in murine bone marrow-derived macrophages (BMDMs). Experiments were carried out in untreated cells as well as in cells treated for 4hrs (lipopolysaccharide (LPS), IFNg, IL4, TNFa, TGFb1, IL1b, MALP2, CpG) and 24hrs (LPS).', '1132/NCBITaxon:10090', '1136/efo:EFO_0002692'
  UNION SELECT 225, 'E-GEOD-38371', 23, 'Latent enhancers unveiled by stimulation expand and adapt the available cis-regulatory repertoire (RNA-seq)', 'According to current models, transcription factors (TFs) activated by extracellular stimuli operate in the context of a pre-established enhancer repertoire induced and maintained by lineage-specific TFs. Here, we uncovered the existence of latent enhancers, defined as regions of the genome that in terminally differentiated cells are poorly accessible and lack the histone marks characteristic of enhancers, but readily acquire these features in response to extracellular cues. Stimulation of resting macrophages caused simultaneous binding of stimulus-activated TFs and lineage-determining TFs to these regions, enabling deposition of enhancer-specific features. Once unveiled, these enhancers did not return to a latent state even when stimulation ceased; instead, they persisted and mediated a faster and stronger response upon restimulation. We suggest that stimulus-specific expansion of the available cis-regulatory repertoire provides an epigenomic memory of the exposure to environmental agents. Poly(A) fraction of the total mRNA of resting and stimulated murine bone marrow derived macrophages was extracted and subjected to by multiparallel sequencing. Experiments carried out in untreated cells as well as in cells treated for 4hrs (IFNg, IL4, TNFa, TGFb1, IL1b, MALP2, CpG)', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 226, 'E-GEOD-34132', 23, 'Mitochondrial-nuclear DNA interactions contribute to the regulation of nuclear transcript levels as part of the inter-organelle communication system', 'Nuclear and mitochondrial organelles must maintain a communication system. Loci on the mitochondrial genome were recently reported to interact with nuclear loci. To determine whether this is part of a DNA based communication system we used genome conformation capture to map the global network of DNA-DNA interactions between the mitochondrial and nuclear genomes (Mito-nDNA) in Saccharomyces cerevisiae cells grown under three different metabolic conditions. The interactions that form between mitochondrial and nuclear loci are dependent on the metabolic state of the yeast. Moreover, the frequency of specific mitochondrial - nuclear interactions (i.e. COX1-MSY1 and Q0182-RSM7) showed significant reductions in the absence of mitochondrial encoded reverse transcriptase machinery. Furthermore, these reductions correlated with increases in the transcript levels of the nuclear loci (MSY1 and RSM7). We propose that these interactions represent an inter-organelle DNA mediated communication system and that reverse transcription of mitochondrial RNA plays a role in this process.  This SuperSeries is composed of the SubSeries listed below. Refer to individual Series.', '1132/NCBITaxon:4932', ''
  UNION SELECT 227, 'E-GEOD-33074', 23, 'Inter- and intra- chromosomal interactions change on a global scale according to the metabolic regime employed by Saccharomyces cerevisiae', 'Nuclear and mitochondrial organelles must maintain a communication system. Loci on the mitochondrial genome were recently reported to interact with nuclear loci. To determine whether this is part of a DNA based communication system we used genome conformation capture to map the global network of DNA-DNA interactions between the mitochondrial and nuclear genomes (Mito-nDNA) in Saccharomyces cerevisiae cells grown under three different metabolic conditions. The interactions that form between mitochondrial and nuclear loci are dependent on the metabolic state of the yeast. Moreover, the frequency of specific mitochondrial - nuclear interactions (i.e. COX1-MSY1 and Q0182-RSM7) showed significant reductions in the absence of mitochondrial encoded reverse transcriptase machinery. Furthermore, these reductions correlated with increases in the transcript levels of the nuclear loci (MSY1 and RSM7). We propose that these interactions represent an inter-organelle DNA mediated communication system and that reverse transcription of mitochondrial RNA plays a role in this process. Genome Conformation Capture (GCC) has been performed on exponentially growing Saccharomyces cerevisiae cultures in glycerol lactate or galactose media. Paired end sequencing on an Illumina Genome Analyser was performed before the sequences were analysed by the propieatry software Topography 1.19. Inter- and intra- chromosomal interactions were mapped onto the S. cerevisiae S288 genome scaffold.', '1132/NCBITaxon:4932', ''
  UNION SELECT 228, 'E-GEOD-33073', 23, 'The Saccharomyces cerevisiae genome is highly connected by inter- and intra- chromosomal interactions', 'Nuclear and mitochondrial organelles must maintain a communication system. Loci on the mitochondrial genome were recently reported to interact with nuclear loci. To determine whether this is part of a DNA based communication system we used genome conformation capture to map the global network of DNA-DNA interactions between the mitochondrial and nuclear genomes (Mito-nDNA) in Saccharomyces cerevisiae cells grown under three different metabolic conditions. The interactions that form between mitochondrial and nuclear loci are dependent on the metabolic state of the yeast. Moreover, the frequency of specific mitochondrial - nuclear interactions (i.e. COX1-MSY1 and Q0182-RSM7) showed significant reductions in the absence of mitochondrial encoded reverse transcriptase machinery. Furthermore, these reductions correlated with increases in the transcript levels of the nuclear loci (MSY1 and RSM7). We propose that these interactions represent an inter-organelle DNA mediated communication system and that reverse transcription of mitochondrial RNA plays a role in this process. Genome Conformation Capture (GCC) has been performed on exponentially growing Saccharomyces cerevisiae cultures in glucose containing media. Paired end sequencing on an Illumina Genome Analyser was performed before the sequences were analysed by the propieatry software Topography 1.19. Inter- and intra- chromosomal interactions were mapped onto the S. cerevisiae S288 genome scaffold.', '1132/NCBITaxon:4932', ''
  UNION SELECT 229, 'E-GEOD-31073', 23, 'Transcriptome profiling of low temperature-treated cassava apical shoots reveals dynamic responses of a tropical plant to cold stress', 'Background: Cassava is an important tropical root crop adapted to a wide range of environmental stimuli such as drought and acid soils. Nevertheless, it is an extremely cold-sensitive tropical species. Thus far, there is limited information about gene regulation and signaling pathways related to the cold stress response in cassava. The development of microarray technology has accelerated the study of global transcription profiling under certain conditions.  Results: A 60-mer oligonucleotide 4X44K Agilent microarray representing 20,840 genes was used to perform transcriptome profiling in cassava apical shoots subjected to cold at 7°C for 0 h, 4 h, and 9 h. A total of 508 transcripts were identified as early cold-responsive genes in which 319 sequences had descriptions when they were aligned with Arabidopsis proteins. Gene ontology (GO) annotation analysis identified many interesting categories including Response to abiotic and biotic stimulus, Response to stress, Transcription factor activity, and Chloroplast. Various stress-associated genes comprising signal transduction components (e.g., MAP kinase 4), transcription factors (TFs; e.g., RAP2.11), and ROS scavenging enzymes (e.g., catalase 2), as well as photosynthesis-related genes (e.g., PSAL), were found. Seventeen major TF families were also identified as being involved in the early response to cold stress (e.g., AP2-EREBP). Meanwhile, KEGG pathway analysis uncovered many important pathways, including Plant hormone signal transduction, Starch and sucrose metabolism, and Plant-pathogen interaction. Furthermore, the expression changes of 18 genes under cold and other abiotic stresses conditions were validated by real-time RT-PCR. As a response to cold stress in cassava, an increase in the ROS scavenging enzyme activities of catalase and superoxide dismutases and the accumulation of total soluble sugars were also detected. Importantly, most of the tested stress-responsive genes were primarily expressed in mature leaves, stem cambia, and fibrous roots rather than apical buds and young leaves.  Conclusions: The dynamic expression changes reflect the integrative controlling and transcriptome regulation of the networks in the early cold stress response of cassava. The biological processes involved in the signal perception and physiological response might shed light on the molecular mechanisms related to cold tolerance in tropical plants and provide useful candidate genes for genetic improvement. Apical shoots subjected to cold at 7°C for 0 h, 4 h, and 9 h were collected for RNA extractions from three independent healthy 3-month-old cassava (cultivar TMS60444) plants in the greenhouse.', '1132/NCBITaxon:3983', '1136/efo:EFO_0002768'
  UNION SELECT 230, 'E-GEOD-29767', 23, 'Gene expression profiles in the hippocampi and prefrontal cortex of Df16(A)+/- mice at embryonic day 17, postnatal day 6, and adult stages', 'Df16(A)+/- mice line is a model of human 22q11 microdeletion syndrome. We conducted an unbiased evaluation of the transcriptional difference in the prefrontal cortex and hippocampus areas between mutant and wild type animals at two early developmental stages (embryonic day 17 and postnatal day 6). These mice were generated by chromosomal engineering and carry a microdeltion of ~1.3Mb in the mouse locus syntenic to the human 22q11.1 The reasoning behind this expression profiling is that consistent alterations in transcriptional programs reflect either downstream (immediate or remote) effects of the deficiency or reactive (compensatory) changes, and can thus point to affected biological processes and molecular functions. Df(16)A+/- mice line is a model of human 22q11 microdeletion syndrome and Dp(16)B mice line is a model of human 22q11 microduplication. We conducted an unbiased evaluation of the transcriptional difference in the prefrontal cortex and hippocampus areas between the mice with microdeletion, microduplication and heterogygous balanced normal copy number of the mouse syntenic locus. Mutant mice and their wild type littermate from two development stages (embryonic day 17 and postnatal day 6) were selected. Hipocampal and prefrontal cortex tissues were dissected, total RNA were extracted, processed and hybridized on Affymetrix microarrays. We sought to obtain temporal gene expression difference of expression profiles between mutant and wild mice. Adult male mice of microdeletion, microduplication were used and mice with heterozygous balanced copy number as reference. Hipocampal and prefrontal cortex tissues were dissected, total RNA were extracted, processed and hybridized on Affymetrix microarrays. we attempted to distinguish primary versus secondary gene targets of the 22q11.2 microdeletion by looking for genes whose expression changes in negative correlation as a result of genomic losses or gains in this locus.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 231, 'E-GEOD-26751', 23, 'Gene expression of cerebellar Purkinje cells and granule cell layer', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 232, 'E-GEOD-26750', 23, 'Gene expression of cerebellar granule cell layer', 'We performed gene-expression analysis of mouse cerebellar granule cell layer as compared to that of Purkinje cells. DNA microarray analysis detected genes in cerebellar granule cell layer, most of which are classified into functional molecule categories. Our comparative analysis between Purkinje cells and the granule cell layer showed that the characteristic expression pattern in Purkinje cells was particularly represented by the neural communication system components. Pukinje cells and granule cell layer of the mouse cerebellum were collected by laser microdissection for RNA extraction and hybridization on Affymetrix microarrays.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 233, 'E-GEOD-26749', 23, 'Gene expression of cerebellar Purkinje cells', 'We performed gene-expression analysis of mouse Purkinje cells as a model single-type neuron. DNA microarray analysis detected at least 7,055 genes in Purkinje cells, most of which are classified into functional molecule categories. Our comparative analysis between Purkinje cells and the granule cell layer showed that the characteristic expression pattern in Purkinje cells was particularly represented by the neural communication system components. Pukinje cells and granule cell layer of the mouse cerebellum were collected by laser microdissection for RNA extraction and hybridization on Affymetrix microarrays.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 234, 'E-MTAB-1121', 23, 'Comparative genomic hybridization of two human CMML samples to study the potential deletion of RUNX1, FLI1 or other transcriptional or epigenetic regulator involved in megacaryopoiesis', 'Characterization of genetic alterations in monocoyte lineage (CD14+ cells) of two CMML (Chronicle Myelo-Monocytic Leukemia) patients. Previously we have shown that RUNX1 downregulates MYH10 expression in megacaryocyte lineage leading in MYH10 overexpression in platelets of patients wiht RUNX1 genetic alterations. MYH10 overexpression could be thus useful as a rapid test of RUNX1 alterations. In these two CMML patients, MYH10 is overexpressed in their platelets but no RUNX1 mutations were detected. The CGH array could evidence not only the large RUNX1 deletion (not detected by sequencing) but also potential deletions in other regulators of MYH10 which are present in the same regulator complex (such FLI1).', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 235, 'E-MEXP-3807', 23, 'Transcription profiling by array of intestinal tissues from mice receiving bone marrow transplantation with or without T-cell stimulation', 'Mice received TBI and allogeneic bone marrow transplantation. One group without and one group with T cells. On day 14 after transplantation mice were sacrificed and the intestinal tract was isolated. RNA was isolated from the intestinal tract of the two experimental groups.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 236, 'E-MEXP-3679', 23, 'Transcription profiling by array of Rattus norvegicus cardiomyocytes to investigate the effects of PD184352 and BI-D1870 on changes in expression induced by endothelin-1', 'Cardiomyocytes were exposed to PD184352, BI-D1870 or endothelin-1 alone, or to endothelin-1 in the presence of each inhibitor', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 237, 'E-MEXP-3678', 23, 'Transcription profiling by array of Rattus norvegicus cardiomyocytes to investigate the effects of 30 or 60 min phenylephrine exposure on gene expression', 'Rat neonatal cardiomyocytes were exposed to 100 uM phenylephrine for 0, 30 or 60 mins', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 238, 'E-MEXP-3518', 23, 'Transcription profiling by array of mouse liver hepatocytes to study the effect of deletion of S6 kinases 1 and 2.', 'Transcriptional analysis of concequent lack of rpS6 kinase  activity in mice livers hepatocytes due to deletion of S6 kinases 1 and 2 (S6K1  and S6K2).', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 239, 'E-MEXP-3071', 23, 'Transcription profiling by array of human endothelial progenitor and mesenchymal stem cells', 'Comparison of EPC (Endothelial progenitor) versus MSC (Mensenchymal stem) at the transcriptomics level', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 240, 'E-GEOD-43553', 23, 'Microarray-based gene expression profiling in patients with cryopyrin-associated periodic syndromes defines a disease-related signature and IL-1-responsive transcripts.', 'To analyse gene expression patterns and to define a specific gene expression signature in patients with the severe end of the spectrum of cryopyrin-associated periodic syndromes (CAPS). The molecular consequences of interleukin 1 inhibition were examined by comparing gene expression patterns in 16 CAPS patients before and after treatment with anakinra. Many DEG include transcripts related to the regulation of innate and adaptive immune responses, oxidative stress, cell death, cell adhesion and motility. A set of gene expression-based models comprising the CAPS-specific gene expression signature correctly classified all 17 samples from an independent dataset. This classifier also correctly identified 15 of 16 post-anakinra CAPS samples despite the fact that these CAPS patients were in clinical remission. We identified a gene expression signature that clearly distinguished CAPS patients from controls. A number of DEG were in common with other systemic inflammatory diseases such as systemic onset juvenile idiopathic arthritis. The CAPS-specific gene expression classifiers also suggest incomplete suppression of inflammation at low doses of anakinra. We collected peripheral blood mononuclear cells from 23 CAPS patients with active disease and from 14 healthy children. Transcripts that passed stringent filtering criteria (p values ? false discovery rate 1%) were considered as differentially expressed genes (DEG). A set of DEG was validated by quantitative reverse transcription PCR and functional studies with primary cells from CAPS patients and healthy controls. We used 17 CAPS and 43 non-CAPS patient samples to create a set of gene expression models that differentiates CAPS patients from controls and from patients with other autoinflammatory conditions.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 241, 'E-GEOD-43517', 23, 'Transcriptome analysis on whole kidney of Usp2 knockout mice', 'Ubiquitylation plays an important role in the control of Na+ homeostasis by the kidney. It is well established that the epithelial Na+ channel ENaC is regulated by the ubiquitin-protein ligase NEDD4-2, limiting ENaC cell surface expression and activity. Ubiquitylation can be reversed by the action of deubiquitylating enzymes (DUBs). One such DUB, USP2-45, was identified previously as an aldosterone-induced protein in the kidney, and is also a circadian output gene. In heterologous expression systems USP2-45 binds to ENaC, deubiquitylates it and enhances channel density and activity at the cell surface. Because the role of USP2-45 in renal Na+ transport had not been studied in vivo, we investigated here the effect of Usp2 gene inactivation in this process. We demonstrate first that the USP2-45 protein has a rhythmic expression with a peak at ZT12. Usp2-KO  mice did not show any differences to wild-type littermates with respect to the diurnal control of Na+ or K+ urinary excretion and plasma levels neither on standard diet, nor after acute and chronic changes to low and high Na+ diets, respectively. Moreover, they had similar aldosterone levels either at low or high Na+ diet. Blood pressure measurements using telemetry did not reveal variations as compared to control mice. Usp2-KO did neither display alternations in ENaC or Na+,Cl--cotransporter (NCC) expression, nor were there any changes in regulatory protein levels, as evidenced by immunoblotting and transcriptome analysis. We conclude that USP2-45 is not crucial for the regulation of Na+ balance or maintenance of blood pressure in vivo. We compared gene expression in kidneys from Usp2 Knock-Out versus Wild-Type. Six pairs of male littermates Usp2-KO and WT were combined in two pools per condition (3 animals per pool) keeping the littermate pairing between pools.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 242, 'E-GEOD-43516', 23, 'Gene expression changes under cyclic mechanical stretch in rat retinal Muller cells', 'The retina is often subjected to tractional forces in a variety of conditions, for instance, pathological myopia, proliferative vitreoretinopathy. As the predominant glial element in the sensory retina, Muller cells are responsible for the homeostatic and metabolic support of retinal neurons and active players in virtually all forms of retinal injury and disease. Besides, Muller cells span the entire retinal thickness, extending from the inner to the outer limiting membranes, with cell bodies located in the inner nuclear layer and lateral processes expanding into the plexiform layers of the tissue. Because of this unique morphology, Muller cells can sense even minute changes in the retinal structure because of the mechanical stretching of their long processes or side branches. Thus, its reasonable to infer that Muller cells also participate in ocular diseases when the retina is overstretched. In this study, we aim to investigate the whole genome regulation of Muller cells under mechanical stretching, which may help in excluding possible molecular mechanisms that would account for many retinal diseases in which the retina is often subjected to mechanical forces. We used microarrays to identify patterns of gene expression changes induced by cyclic mechanical stretching in Muller cells. Rat Muller cells were seeded onto flexible bottom culture plates and subjected to a cyclic stretching regimen of 15% equibiaxial stretching for 1 and 24 h.Muller cells cultured under the same conditions but with no applied mechanical strain were considered as the unstretched control. At each time points (1 and 24 h), three totally independent experiments (3 stretched samples and 3 control samples) were conducted. Muller cells were selected for RNA extraction and hybridization on Affymetrix microarrays. Stretch (S); Control (C)', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 243, 'E-GEOD-43508', 23, 'Comparison of gene expression signature between nascent Lhx6-GFP+ cells from mouse embryonic brain and differentiated Lhx6-GFP+ cells from ES cells (as well as comparing those of Lhx6-GFP+ and Lhx6-GFP- cells from ES cells)', 'There was a remarkable similarity in the molecular properties of the MGE-GFP+ and ES-GFP+ cells. In particular, genes that are important for medial ganglionic eminence (MGE) and cortical interneurons development are both high in expression in both MGE-Lhx6-GFP+ and ES-Lhx6-GFP+ cells (compared to ES-Lhx6-GFP- cells). To investigate how closely ES cells-derived Lhx6-GFP+ cells resembled authentic Lhx6+ MGE cells, and  to define the molecular properties of the Lhx6-GFP+ and Lhx6-GFP- cells from differentiated ES cells, we compared their gene expression profiles. We used FACS to purify GFP+ cells from the E12.5 MGE of Lhx6-GFP transgenic mice. ES-Lhx6-GFP+ cells and ES-Lhx6-GFP- cells (both from D12 EB aggregates) were also isolated by fluorescent activated cell sorting (FACS) and all of the RNA samples were subjected to RNA expression microarray analyses.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 244, 'E-GEOD-43506', 23, 'Sequencing mRNA from cryo-sliced Drosophila embryos to determine genome-wide spatial patterns of gene expression', 'We sequenced mRNA from transverse slices of 3 wild-type and 3 high bcd copy-number embryos at the blastoderm stage to determine genome-wide patterns of gene expression. mRNA from transverse sections of single D. melanogaster embryos was sequenced', '1132/NCBITaxon:7227', '1136/efo:EFO_0003738'
  UNION SELECT 245, 'E-GEOD-43503', 23, 'Expression of genes in the spleen of WT and MUC1.Tg mice immunized with dendritic cells loaded with MUC1 peptide', 'Goal of study: To determine differentially expressed genes with potential roles in tolerance and immunity in MUC1-immunized WT and MUC1.Tg mice WT (C57Bl/6) and MUC1.Tg mice were immunized (i.v.) with dendritic cells loaded with sythetic, human MUC1 peptide. At 24h and 72h splenic RNA from vaccination groups (n=3) was pooled and microarray performed.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 246, 'E-GEOD-43498', 23, 'Differentially expressed genes in H2AX knockdown cells undergoing apoptosis', 'H2AX has been characterized as a novel tumor suppressor protein. Difficiency of H2AX will result in apoptotic inhibition of cancer cells. However, how H2AX epigenetically regulates apoptosis of cancer cells is still unclear. To reveal the genes expression regulated by H2AX and involved in apoptosis, the microarray profiling analysis was employed to identify differentially expressed genes in H2AX knockdown lung cancer cells and control ones after apoptotic induction. H2AX was knockdown by miRNA interfering system in human lung cancer originated cell A549 and the stable cell line named P1, while the control stable cell line named C. Genes with greater than 1.2-fold change and P-value ?0.05 were identified as differentially expressed genes between C and P1 cells both with apoptosis induction. The two groups including control (C) and H2AX knockdown lung cancer cells (P1) were harvested 48 h after VP-16 treatment. Three independent experiments were performed for each group.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 247, 'E-GEOD-43497', 23, 'Effector cell signatures in peripheral blood after nasal allergen challenge: biomarkers of the allergic response', 'This study investigated temporal transcriptomic changes in response to nasal allergen challenge of titrated timothy grass pollen. This is an open single-center observational study conducted outside the pollen season. Twelve participants with seasonal allergic rhinitis underwent a control (diluent) challenge followed by nasal allergen challenge after an interval of 14 days. On each challenge day, nasal challenge with control or titrated timothy grass pollen (Aquagen, phleum pratense; ALK) was administered. Peripheral blood was collected before nasal challenge (baseline) and at 3, 6 and 24 hours following challenge. RNA was extracted from whole blood and CD4 cells for microarray experiment using Affymetrix Human Gene 1.0 ST arrays.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 248, 'E-GEOD-40126', 23, 'Transcriptional profile Lactobacillus plantarum WCFS1 : p-coumaric acid (p-CA) treated cells', 'Whole genome transcriptional profiling was used to characterize the response of Lactobacillus plantarum WCFS1 human isolate during challenge with p-coumaric acid (p-CA). Twelve independent experiments were performed and mixed at random in groups of four for total of three RNA samples. The transcriptional profile reveals a massive induction of genes involved in stress resistance and detoxification-related functions and a global shutdown of growth-associated processes. Biological triplicates: untreated cells: Control vs. p-coumaric treated cells: Experimental', '1132/NCBITaxon:1590', '1136/efo:EFO_0002768'
  UNION SELECT 249, 'E-GEOD-39630', 23, 'Transcriptional Analysis of lsm1a lsm1b Arabidopsis mutant.', 'Characterization of the lsm1a lsm1b transcriptional profile. LSM1 protein is involved in RNA decay through decapping facilitation. The performed array help us to understand the transcription level alterations produced by the absence of LSM1. One-condition experiment, Col-0 vs. lsm1a lsm1b plants. Biological replicates: 3 control replicates.', '1132/NCBITaxon:3702', '1136/efo:EFO_0002768'
  UNION SELECT 250, 'E-GEOD-39617', 23, 'Transcriptome of lsm8-1 mutant using whole genome tiling arrays', 'Characterization of gene expression and intron retention in lsm8-1 mutants. Pooled 2 week old rosettes of Col-0 (3 biological replicates) and lsm8-1 mutants (3 biological replicates).', '1132/NCBITaxon:3702', '1136/efo:EFO_0002769'
  UNION SELECT 251, 'E-GEOD-38230', 23, 'SCC of the vulva', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 252, 'E-GEOD-38229', 23, 'aCGH was used to identify the overall genomic imbalances of SCC of the vulva in 14 patients', 'The imbalances scored by arrayCGH mapped to different chromosomes with losses being more common than gains. Frequent losses of large chromosomal segments were scored from 3p and 8p whereas same-sized gains were frequent from 3q and 8q. This is the first study of vulvar tumors using arrayCGH, and some frequent imbalances could be defined precisely. 14 patients we analyzed without replicates against a commercial common reference.', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 253, 'E-GEOD-38228', 23, 'Expression analysis of 13 squamous cell carcinomas arising in the vulva', 'The genome of 14 vulvar SCC was analyzed by aCGH and microarray to identify common imbalances present in the tumors as well as which genes were deregulated. Overall there was a good concordance between the imbalances scored by aCGH and the level of gene expression found by microarray, i.e., the genes located in gained regions were overexpressed while those located in lost regions were found down-regulated. The whole-gene expression profile of 14 SCC of the vulva was compared to 5 normal vulvar samples to identify genes that were deregulated in the tumors genome. Vulvar hyperplasia 03-48 not further analyzed and not included in the normalized data but included in the non-normalized data', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 254, 'E-GEOD-38045', 23, 'The global gene expression pattern between human fibroblasts and human neural stem cells', 'Recent advances in direct reprogramming using cell type-specific transcription factors provide an unprecedented opportunity for rapid generation of desired human cell types from easily accessible tissues. However, due to the diversity of conversion factors that facilitate the process, an arduous screening step is inevitable to find the appropriate combination(s). Here, we show that under chemically defined conditions minimal pluripotency factors are sufficient to directly reprogram human fibroblasts into stably self-renewing neural progenitor/stem cells (NSCs), but without passing through a pluripotent intermediate stage. These NSCs can be expanded and propagated in vitro without losing their potential to differentiate into various neuronal subtypes and glia. Our direct reprogramming strategy represents a simple and advanced paradigm of direct conversion that will provide an unlimited source of human neural cells for cell therapy, disease modeling, and drug screening.  We used microarray to compare the global gene expression pattern between human fibroblasts and human neural epitheliums from human ESCs or directly from fibroblasts. We cultured cells and harvested them and then extracted total RNA for microarray.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 255, 'E-MTAB-1186', 23, 'Comparative genomic hybridization by array of human peripheral T-cell lymphoma clinical samples which occurred in thyroid gland to study their genomic aberration profiles', 'Genomic aberration profiles of peripheral T-cell lymphoma, not otherwise specified of thyroid gland (clinical samples)', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 256, 'E-MTAB-1093', 23, 'Transcription profiling by array of Arabidopsis thaliana Ler and uvr8-2 genotypes exposed to solar ultraviolet radiation', 'The experiment was performed to search for genes regulated by the UV RESISTANT LOCUS (UVR8) under solar ultraviolet radiation (UV).', '1132/NCBITaxon:3702', '1136/efo:EFO_0002768'
  UNION SELECT 257, 'E-MTAB-951', 23, 'Genome and transcriptome study of the response to irinotecan and bevacizumab in recurrent glioblastomas (CIT - Homo sapiens - BEVA)', 'A Cartes d''Identite des Tumeurs (CIT) project from the French National League Against Cancer (http://cit.ligue-cancer.net )  25 glioblastoma multiforme tumors  hybridized on Illumina SNP and Affymetrix gene expression arrays. Project leader : François DUCRAY (francois.ducray@chu-lyon.fr). CIT Analysis : Julien LAFFAIRE (laffairej@ligue-cancer.net). Note: PFS : progression-free survival, OS: Overall Survival,BCNU : Carmustine (chemotherapy agent). RESPONDER: if the patient has shown or not shown a response to the treatment (Bevacizumab (Avastin) plus Irinotecan). Progression during : If the disease has progressed (cancer relapse or patient''s death);  otherwise (patient is alive without relapse).', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 258, 'E-MEXP-3763', 23, 'Transcription profiling by array of mice exposed to arsenic and iron by drinking water', 'Five-week-old male mice (Mus musculus, ICR) were exposed to single As, single Fe and combined As and Fe for 90 days. A total of 40 mice were randomly assigned to control and three metal-treated groups. Nine mice were applied in every group. For control group, mice were fed with pure water. For three metal-treated groups, mice were feed with 3mg/L As, 5mg/L Fe and 3mg/L As + 5mg/L Fe, respectively. After exposure, mice were anaesthetized under isoflurane followed by exsanguination. Livers were removed, and hepatic RNA for each mouse was immediately extract. The gene expression profiles for control and three metal-treated groups were determined by the GeneChip Mouse Gene 1.0 ST arrays. Biological meanings of DEGs were analyzed by pathway analysis. DEGs were mapped to different biological pathways according to Kyoto encyclopedia of genes and genomes (KEGG) pathway database (http://www.genome.ad.jp/kegg/ pathway.html) using SBC Analysis System of Shanghai Biotechnology Corporation (http://sas.ebioservice.com/).', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 259, 'E-GEOD-43502', 23, 'Identification of Prognosis-Relevant Subgroups in Patients with Chemoresistant Triple Negative Breast Cancer', 'Triple-negative breast cancer (TNBC) patients with residual disease after neoadjuvant chemotherapy generally have worse outcome; however, some patients with residual tumor after neoadjuvant chemotherapy do not relapse. We hypothesize that there are subgroups of chemoresistant TNBC patients with different prognosis. In this study, 25 chemoresistant samples from 47 neoadjuvant chemotherapy-treated TNBC (The Methodist Hospital) are chosen for study We used gene expression data of TNBC patients with residual disease and different prognosis to molecularly define the clinically relevant subgroups, and developed a 7-gene prognostic signature for chemoresistant TNBCs', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 260, 'E-GEOD-43494', 23, 'Noncoding RNA expression profiling of cisplatin-resistant cells derived from the A549 lung cell line', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 261, 'E-GEOD-43493', 23, 'Noncoding RNA expression profiling of cisplatin-resistant cells derived from the A549 lung cell line (lncRNA and mRNA)', 'To determine the signaling networks that are dysregulated in cisplatin-resistant non-small cell lung cancer, noncoding RNA expression data were obtained from, and compared between, the lung adenocarcinoma cell line, A549, and its cisplatin-resistant derivative, A549/CDDP. Noncoding RNA expression data from a cisplatin-sensitive lung adenocarcinoma cancer cell line (A549) were collected and compared to noncoding RNA expression data from a cisplatin-resistant cell line (A549/CDDP). 3 independent experiments were completed for both the sensitive and resistant cell lines.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 262, 'E-GEOD-43492', 23, 'The effects of a novel agrochemical in alleviating salinity stress', 'Yield losses as a result of abiotic stress factors present a significant challenge for the future of global food production. While breeding technologies provide potential to combat negative stress-mediated outcomes over time, interventions which act to prime plant tolerance to stress, via the use of phytohormone-based elicitors for example, could act as a valuable tool for crop protection. However, the translation of fundamental biology into functioning solution is often constrained by knowledge-gaps. Photosynthetic and transcriptomic responses were characterised in young tomato (Solanum lycopersicum) seedlings in response to pre-treatment with a new plant health activator technology, Alethea, followed by a subsequent 100 mM salinity stress. Salinity treatment led to a maximal 47% reduction in net photosynthetic rate 8 d following NaCl treatment. In Alethea pre-treated seedlings, sensitivity to salinity stress was markedly reduced during the experimental period. Microarray analysis of leaf transcriptional responses showed that while salinity stress and Alethea individually impacted on largely non-overlapping, distinct groups of genes, Alethea pre-treatment substantially modified the response to salinity. Alethea affected the expression of genes related to biotic stress, ethylene signalling, cell wall synthesis, redox signalling and photosynthetic processes.  Since Alethea had clear effects on photosynthesis/chloroplastic function at the physiological and molecular levels, we also investigated the ability of Alethea to protect plants against methyl viologen, a potent generator of oxidative stress in chloroplasts. Alethea pre-treatment produced dramatic reductions in visible foliar necrosis caused by methyl viologen compared with non-primed controls. Tomato leaves from four groups of treated plants were harvested at a single point following treatment for RNA extraction and hybridization on Affymetrix microarrays. Plants came from two initial groups, pre-treated either with water (control) or the Alethea product. Equal numbers of plants from within these groups were subsequently treated either with water (control) or salinity (100 mM NaCl), generating 4 main experimental groups in a 2x2 factorial design. Samples were collected from 3 biological replicate experiments, and each sample included leaves pooled from 3 individual plants from each treatment group.', '1132/NCBITaxon:4081', '1136/efo:EFO_0002768'
  UNION SELECT 263, 'E-GEOD-43487', 23, 'Mammary tumors', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 264, 'E-GEOD-43486', 23, 'miRNA expression analysis in AKXD RI strain mammary tumors', 'Transcriptional profiling of miRNA levels in mammary tumors from 18 [PyMT x AKXD]F1 sublines. The PyMT strain was FVB/N-TgN(MMTV-PyVT)634Mul. Mammary tumor total RNA from mice representing one of 18 AKXD RI strains were pooled to represent each strain and expression profiled using a custom miRNA microarray.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 265, 'E-GEOD-43485', 23, 'Expression profiling in miR-290 6dt1 mammary tumors', 'mRNA expression data from mammary tumors extracted 30 days after orthotopic injection of miR-290-expressing and negative control 6dt1 cells into female FVB/N mice. At 6 weeks of age, FVB/N female mice were orthotopically injected with 1.0x10E5 6dt1 cells, either expressing miR-290 or a negative control RNA construct. 4 negative control tumors and 3 miR-290 tumors were analyzed.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 266, 'E-GEOD-43484', 23, 'Comparison study between Uremic patient with Healthy control', 'Patients with chronic kidney disease (CKD) have significantly increased morbidity and mortality resulting from infections and cardiovascular diseases. Since monocytes play an essential role in host immunity, this study was directed to explore the gene expression profile and to identify differences in activated pathways in monocytes relevant to the pathophysiology of atherosclerosis and increased susceptibility to infections. Monocytes from CKD patients (stages 4 and 5, eGFR <20 ml/min x 1.73 m2) and healthy donors were collected from peripheral blood. Microarray gene expression profile was done and data were interpreted by Genespring software and Panther tools website. Western blot was done to validate the pathway members. The results demonstrated that 487 and 258 genes were differentially up- and down regulated respectively in the patient group. Pathways involved in the inflammatory response were highly expressed and the Wnt/?- catenin signaling pathway was the most significant pathway expressed in the patient group (P=0.02). Since this pathway has been attributed to a variety of inflammatory manifestations, the current finding may contribute to dysfunctional monocytes in CKD patients. To develop strategies to interfere with this pathway may improve host immunity and prevent cardiovascular complications in CKD patients. Six gene expression Affymetrix chip were used. Three for each group', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768';

INSERT INTO 'obr_ae_test_element'
  SELECT 267 AS 'id', 'E-GEOD-43482' AS 'local_element_id', 23 AS 'dictionary_id', 'Cytosolic DNA stimulated gene regulation' AS 'ae_name', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series' AS 'ae_description', '1132/NCBITaxon:10090' AS 'ae_species', '1136/efo:EFO_0002768' AS 'ae_experiment_type'
  UNION SELECT 268, 'E-GEOD-43481', 23, 'Gene expression profiling of dsDNA90 or ssDNA90 stimulated Mouse Embryonic Fibroblasts', 'Gene expression analysis of wild type, STING knock-out and STAT1 knock-out Mouse Embryonic Fibroblasts (MEFs) stimulated with 90-mer dsDNA or 90-mer ssDNA. Genes whose expression that are affected by cytosolic DNA in a STING dependent manner will be identified and signaling pathways regulated by STING will be elucidated. Primary MEFs were mock treated or transfected with dsDNA90 or ssDNA90 for 3 hours. Total RNA was then extracted for array analysis.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 269, 'E-GEOD-43480', 23, 'Gene expression profiling of human telomerase fibroblasts stimulated with dsDNA90', 'Gene expression analysis of dsDNA90 stimulated human telomerase fibroblasts (hTERT-BJ1) after STING siRNA treatment. Genes whose expression that are affected by cytosolic DNA in a STING dependent manner will be identified and signaling pathways regulated by STING will be elucidated. hTERT-BJ1 cells were transfected with non-specific or STING siRNA for 72 hours followed by dsDNA90 stimulation for 3 hours. Total RNA was extracted for array analysis.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 270, 'E-GEOD-43479', 23, 'Gene expression profiling of ?34.5 deleted HSV1 infected Mouse Embryonic Fibroblasts', 'Gene expression analysis of wild type and STING knock-out Mouse Embryonic Fibroblasts (MEFs) infected with ?34.5 deleted HSV1. Genes whose expression that are affected by HSV1 in a STING dependent manner will be identified and signaling pathways regulated by STING will be elucidated. Primary MEFs were mock treated or infected with ?34.5 deleted HSV1 at M.O.I. 1. Total RNA was extracted 3 hours post infection for array analysis.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 271, 'E-GEOD-43476', 23, 'Genomic Reprogramming Errors Do Not Accumulate with Serial Recloning in the Mouse', 'To date, the success rate of serial animal cloning has decreased with increasing iterations and recloning has failed in all species after a few generations. This has suggested that recloning might be impossible because of the accumulation of lethal genetic or epigenetic abnormalities. Here, we carried out repeated recloning in the mouse using our somatic cell nuclear transfer method combined with a histone deacetylase inhibitor. The success rate of recloning did not decrease even after 25 iterations and more than 500 viable offspring have been obtained from a single original donor. Although the genomic reprogrammability of somatic cell nuclei did not improve, reprogramming errors or clone-specific abnormalities did not accumulate with recloning. Our results provide the first evidence that iterative recloning is possible, suggesting that animals might be able to be recloned indefinitely. Brain and liver samples from 4 somatic cell clone (CC), 4 somatic cell clones of 20 generations (G20) and 4 normal controls (ICSI) neonate.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 272, 'E-GEOD-43472', 23, 'Regional Differences in gene expression and promoter usage in aged human brains', 'Cap analysis of gene expression (CAGE) and massive parallel sequuencing were used to profile the promoterome of aged human brains from five regions, namely: caudate, frontal cortex, hippocampus, putamen and temporal cortex. 25 RNA  libraries from post-mortem brain tissue (five caudate, five frontal, 5 hippocampus, 5 putamen, five temporal RNA libraries from seven individuals) were processed using CAGE protocol and CAGE tags derived from the 25 libraries were sequenced with Illumina.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 273, 'E-GEOD-43471', 23, 'Gene expression changes in adipose tissue with diet- and/or exercise-induced weight loss', 'An ancillary study within a randomized trial of diet, exercise, or combined diet+exercise vs. control among overweight/obese postmenopausal women. Subcutaneous adipose-tissue biopsies were performed at baseline and after 6 months and changes in adipose-tissue gene expression were determined by microarray with an emphasis on pre-specified candidate pathways, as well as by unsupervised clustering. Analyses were conducted first by randomization group, and then by degree of weight change at 6-months in all women combined. Total RNA was obtained from subcutaneous adipose tissue biopsies at baseline and 6 months. A total of 47 women were biopsied and one replicate participant was included for a total of 96 samples.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 274, 'E-GEOD-43469', 23, 'Real-time quantitative PCR analysis of 88 microRNAs involved in human cell differentiation and development in involved skin of systemic scleroderma (SSc), localized scleroderma (LSc) and Keloid.', 'Skin specimens were derived from involved skin of 3 systemic scleroderma (SSc), 3 localized scleroderma (LSc) and 3 keloid patients. These skin samples and 3 control skins were collected and fixed in formaldehyde immediately after resections. The microRNA (miRNA) isolation from human skin tissue was performed using miRNeasy FFPE kit (Qiagen). For PCR array, miRNAs were reverse-transcribed into first strand cDNA using RT2 miRNA First Strand Kit (SABiosciences). A mixture of equal amounts of miRNAs from 3 normal skins, 3 SSc, 3 LSc or 3 keloid were prepared, and miRNA expression profile in each disease in vivo was evaluated using RT2 Profiler PCR Array. The cDNA was mixed with RT2 SYBR Green/ROX qPCR Master Mix and the mixture was added into 96-well RT2 miRNA PCR Array that includes primer pairs for 88 human miRNAs (SABiosciences). Skin specimens were derived from involved skin of 3 systemic scleroderma (SSc), 3 localized scleroderma (LSc) and 3 keloid patients. These skin samples and 3 control skins were collected and fixed in formaldehyde immediately after resections. Control donors were each matched with diseases for age, sex, and biopsy site.', '1132/NCBITaxon:9606', ''
  UNION SELECT 275, 'E-GEOD-43468', 23, 'Hepatic Leukemia Factor Reproduces Circadian Resistance To Cell Death', 'Here we examine the regulation of cell death by hepatic leukemia factor (HLF), which is an output regulator of circadian rhythms and is aberrantly expressed in human cancers, using an ectopic expression strategy in JB6 mouse epidermal cells and human keratinocytes.  Ectopic HLF expression inhibited cell death in both JB6 cells and human keratinocytes, as induced by serum-starvation, tumor necrosis factor alpha and ionizing radiation. Microarray analysis indicates that HLF regulates a complex multi-gene transcriptional program encompassing upregulation of anti-apoptotic genes, downregulation of pro-apoptotic genes, and many additional changes that are consistent with an anti-death program.  Collectively, our results demonstrate that ectopic expression of HLF, an established transcription factor that cycles with circadian rhythms, can recapitulate many features associated with circadian-dependent physiological variation. Murine progenitor cells, JB6 cell line, were stably transformed with an HLF retroviral expression system or vector only control.  Confirmation of HLF ectopic expression was performed with western blot analysis.  Samples were collected from randomly cycling, preconfluent cells maintained in normal growth medium.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 276, 'E-GEOD-43464', 23, 'Gene Bionetworks that Regulate Ovarian Primordial Follicle Assembly', 'Primordial follicle assembly is the process by which ovarian primordial follicles are formed.  During follicle assembly oocyte nests break down and a layer of pre-granulosa cells surrounds individual oocytes to form primordial follicles.  The pool of primordial follicles formed is the source of oocytes for ovulation during a females reproductive life.  Complex networks of cellular signaling and gene expression are essential for any biological process.  A systems biology experimental approach provides a global view of these gene relationships in a particular developmental process.  The current study utilized a systems approach to detect all genes that are differentially expressed in response to seven different growth factor and hormone treatments known to influence primordial follicle assembly in a neonatal rat ovary culture system.  One novel growth factor, basic fibroblast growth factor (FGF2), was experimentally determined to inhibit follicle assembly.  The different growth factor and hormone treatments were all found to affect the same physiological pathways, but each treatment affected a unique set of differentially expressed genes (signature gene set).  A gene bionetwork analysis identified gene modules of coordinately expressed interconnected genes and it was found that different gene modules appear to accomplish distinct tasks during primordial follicle assembly.  Unique gene networks were identified for a number of the modules and signature gene sets. Predictions of physiological pathways important to follicle assembly were validated using ovary culture experiments in which ERK1/2 (MAPK1) activity was increased. A number of the highly interconnected genes in these gene networks have previously been linked to primary ovarian insufficiency (POI) and polycystic ovarian disease syndrome (PCOS). Observations have identified novel factors and gene networks that regulate primordial follicle assembly. This systems approach has helped elucidate the molecular control of primordial follicle assembly and provided potential therapeutic targets for the treatment of ovarian disease. We used microarrays to determine genes expressed differentially between control and P0 ovaries  treated with 7 growth factors: AMH, CTGF, estradiol (E2), Activin-a, FGF2, progesterone (P4), and TNFa. RNA samples from 7 control samples  are compared to 3 growth factor treated ovary samples  for each AMH (human Anti-Mülerian hormone), CTGF (connective tissue growth factor ),  estradiol (E2), TNF (tumor necrosis factor ), FGF2 (fibroblast growth factor 2), Inhba (inhibin, beta A), and 2 samples for progesterone (P4).', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 277, 'E-GEOD-43447', 23, 'Evaluation of subsequent passaging on whole genome expression', 'Heart diseases are as most leading causes of morbidity and mortality in the world. One of the promising approaches to cure these diseases is cell therapy using endogenous cardiac precursor cells resident in heart tissues. These cells are isolated from heart biopsies, cultured and passaged to reach to the desirable cell count necessary for transplantation. The process of cell propagation and subsequent passages may cause gene expression alteration unsuitable and dangerous for cell therapy; so in our study, we analyzed whole genome expression using microarray technology to compare expressed transcripts in 4 passages 6, 9, 12 and 15. There were 4 different passage samples (6, 9, 12, and 15) each stage with three replicates. Replicates for passage 6 were determied as control group and the data was analyzed compaired to this passage.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 278, 'E-GEOD-43381', 23, 'Expression profiling across mouse epithelial tissues', 'To characterize genes, pathways, and transcriptional regulators enriched in the mouse cornea, we compared the expression profiles of whole mouse cornea, bladder, esophagus, lung, proximal small intestine, skin, stomach, and trachea. Total RNA was purified from esophagus, lung, proximal small intestine, skin, stomach, and trachea. Data from previous studies in the lab (cornea, GSE43155; bladder, GSE15770; and E18.5 skin, GSE15772) were utilized in the comparison.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 279, 'E-GEOD-43249', 23, 'Noncoding RNA expression profiling of cisplatin-resistant cells derived from the A549 lung cell line (miRNA)', 'To determine the signaling networks that are dysregulated in cisplatin-resistant non-small cell lung cancer, noncoding RNA expression data were obtained from, and compared between, the lung adenocarcinoma cell line, A549, and its cisplatin-resistant derivative, A549/CDDP. Noncoding RNA expression data from a cisplatin-sensitive lung adenocarcinoma cancer cell line (A549) were collected and compared to noncoding RNA expression data from a cisplatin-resistant cell line (A549/CDDP). 3 independent experiments were completed for both the sensitive and resistant cell lines.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 280, 'E-GEOD-43220', 23, 'Functional genomic and phenotypic responses to desiccation in natural populations of a desert drosophilid.', 'Investigation of whole genome transcription expression level changes in Drosophila mojavensis wild-type populations (1. Punta Onah:PO; 2. Organ Pipe National Manument:OPNM; 3. Punta Prieta:PP; & 4. San Quintin:SQ). The experiment was designed to investigate effects of desiccation exposure (0, 9 & 18 hr) and host plant (diet) on transcriptome. A total of  95 hybridizations were performed in this entire experiment. We used 135K 12-plex NimbleGen arrays. Total RNA was recovered from each sample listed below. The experimental design consisted a total of four populations (1. Punta Onah:PO; 2. Organ Pipe National Manument:OPNM; 3. Punta Prieta:PP; & 4. San Quintin:SQ),  two host diets (Agria and Organ pipe) and three desiccation treatments (0, 9 and 18 hours). Each chip measures the expression level of 15528 transcripts. Four to 5 replicates were used for each type (R-1, R-2, R-3 etc.)', '1132/NCBITaxon:7230', '1136/efo:EFO_0002768'
  UNION SELECT 281, 'E-GEOD-42936', 23, 'Erf chromatin binding-sites in E13.5 mouse embryo fibroblasts', 'We show that reduced dosage of ERF, which encodes an inhibitory ETS transcription factor directly bound by ERK1/2 , causes complex craniosynostosis (premature fusion of the cranial sutures) in humans and mice. Features of this newly recognized clinical disorder include multiple suture synostosis, craniofacial dysmorphism, Chiari malformation and language delay. Mice with functional Erf reduced to ~30% of normal exhibit postnatal multisuture synostosis; by contrast, embryonic calvarial development appears mildly delayed. Using chromatin immunoprecipitation in mouse embryonic fibroblasts and high-throughput sequencing, we find that ERF binds preferentially to distal regulatory elements containing RUNX or AP1 motifs. This work identifies ERF as a novel regulator of osteogenic stimulation by RAS-ERK signaling, potentially by competing with activating ETS factors in multifactor transcriptional complexes. Examination of Erf binding site in E13.5 mouse embryo fibroblasts, growing in the presence or absence of serum for 4 hours', '1132/NCBITaxon:10090', '1136/efo:EFO_0002692'
  UNION SELECT 282, 'E-GEOD-42883', 23, 'Gene expression data following Neurogenin 1 expression in mouse embryonic stem cells', 'To study the early gene expression events involved in neurogenesis, we expressed the bHLH neural determination factor neurogenin 1 in mouse embryonic stem cells. We used microarray to examine the program of gene expression at 24, 48, 72 hours of transgene expression. Mouse embryonic stem cells were engineered to express neurogenin 1 using a tet-sensitive expression system. RNAs from uninduced and induced cells were isolated at 24, 48, and 72 hours of differentiation and subjected to microarray analysis.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 283, 'E-GEOD-42655', 23, 'Importance of Pol? for damage-induced cohesion reveals differential regulation of cohesion establishment at the break site and genome-wide', 'Correct chromosome segregation requires that sister chromatids are held together by the protein complex cohesin, from S phase until anaphase. This S phase established cohesion is, together with DSB recruitment of cohesin and formation of damage induced (DI) cohesion, also important for repair of DSBs. Eco1 is a common essential factor for S phase and DI-cohesion. The fission yeast Eco1ortholog, Eso1, is important both for S phase cohesion and for bypass of UV induced lesions, and is expressed as a fusion protein with Pol?. The cohesion function has been attributed solely to Eso1 and the lesion bypass function to the Pol? part of the protein. As we found the interaction between the two proteins intriguing we decided to look for a functional connection also in budding yeast. Indeed, despite being dispensable for S phase cohesion, budding yeast Pol? is required for formation of DI genome-wide cohesion. However, Pol? deficient cells are DSB repair competent, revealing differential regulation of DI-cohesion at the break and genome-wide. This finding challenges the importance of DI genome-wide cohesion for DSB repair, and based on our findings we suggest that S phase cohesion is not sufficient for correct chromosome segregation in the presence of DNA damage. Whole Genome binding of G2 expressed Scc1 in the presence and absence of Rad30.', '1132/NCBITaxon:4932', '1136/efo:EFO_0002692'
  UNION SELECT 284, 'E-GEOD-42551', 23, 'Gene expression in mouse primary pre-B cells transduced with Ikaros', 'Ikaros family DNA binding proteins are critical regulators of B cell development. To identify Ikaros-regulated genes in primary pre-B cells we performed gene expression microarrays. We used retroviral gene transfer to express Ikaros proteins. Total RNA from 3 biological replicates of primary pre-B cells transduced with wild type Ikaros (HA-Ikaros-IRES-GFP) and control vector (IRES-GFP) was isolated 48h after infection.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 285, 'E-GEOD-42462', 23, 'Genome-wide identification of Ikaros binding sites in primary pre-B cells', 'Ikaros DNA binding proteins are important regulators of haematopoiesis and genetic deletion of Ikaros results in severe developmental disturbances, including delayed thymocyte differentiation and an early and complete block in B cell development. Although Ikaros ChIP-seq data are available for mouse thymocytes and human haematopoietic progenitors, it has not been achieved in B cell progenitors. The goal of this study was to identify Ikaros binding sites in pre-B cells to define Ikaros target genes which could explain the essential role of Ikaros proteins in B cell differentiation. We carried out chromatin immunoprecipitation and high throughput sequencing (ChIP-seq) with antibodies to the C-terminus of endogenous Ikaros in primary pre-B cells isolated from mouse fetal livers.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002692'
  UNION SELECT 286, 'E-GEOD-40147', 23, 'Gene expression analysis of human microRNA-150 expression in PL21 and HL60 cells', 'Analysis of gene expression changes associated with lentiviral vector mediated expression of  pre-microRNA-150 expression in the AML cell lines PL21 and HL60 cells. To identify genes, direct putative miRNA-150 targets, biological and molecular functions and pathways that distinguish cells that express miR-150 from those that do not. Total RNA isolated from HL60 or PL21 cells transduced with pre-microRNA-150 or empty vector control and assayed by gene expresssion analysis 20 days post transduction. Two replicates were performed for each condition (miR-150 and control) in each of two cell lines (HL60 and PL21).', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 287, 'E-GEOD-40116', 23, 'mRNA profiling of glucose-repressed 14-3-3 and hdac yeast mutants', 'Previous results suggest that Bmh might inhibit the activity of the transcription factor Adr1 after binding to Adr1-dependent promoters. In a strain lacking the two major histone deacetylases, Hda1 and Rpd3 (hdac?), Adr1 is bound to its target promoters recruiting what appears to be an inactive RNA ploymerase II preinitiation complex (PIC). To determine whether Bmh activity inhibits this inactive PIC and the generality of this effect on glucose-repressed gene expression, the mRNA profiles of wild type, bmh mutant, hdac mutant, and bmh hdac mutant cells grown in high glucose medium were compared. Total RNAs were purified from triplicate cultures of W303-1A (wild type), YLL1087 (bmh1-ts bmh2?), CTY-TY44 (hda1? rpd3?) and KBY3 (bmh1-ts bmh2? hda1? rpd3?) cells exponentially growing in YP broth containing 5% glucose and then profiled using Affymetrix Yeast 2.0 arrays.', '1132/NCBITaxon:4932', '1136/efo:EFO_0002768'
  UNION SELECT 288, 'E-GEOD-39842', 23, 'Hypoxia induces myocardial regeneration in zebrafish', 'Conditional expression of dominant-negative HIF1a in zebrafish cardiomyocytes severely inhibits heart regeneration. To understand more about the mechanism, we performed microarray analysis of wildtype regenerating zebrafish and dnHIF1a regenerating zebrafish to determine which genes are regulated by hypoxia/HIF1a. We amputated both wildtype and dnHIF1a zebrafish and allowed them to regenerate for 7 days, then compared their transcriptomic profile with unamputated wildtype and dnHIF1a zebrafish, respectively.', '1132/NCBITaxon:7955', '1136/efo:EFO_0002768'
  UNION SELECT 289, 'E-GEOD-38897', 23, 'Bacillus subtilis Rnase Y activity in vivo analysed by tiling microarrays', 'RNase Y is an essential endoribonuclease affecting global mRNA stability in Bacillus subtilis. We used high-resolution tiling arrays to analyze the effect of RNase Y depletion on RNA abundance covering the entire genome. The data confirm that this endoribonuclease plays a key role in initiating the decay of a large number of mRNAs as well as non coding RNAs. Comparison of the data with that of two other recent studies revealed very significant differences. About two thirds of the mRNAs upregulated following RNase Y depletion were different when compared to either one of these studies and only about 10% of were in common in all three studies.Our data confirmed already known RNase Y substrates and due to the precision and reproducibility of the profiles allows an exceptionally detailed view of the turnover of hundreds of new RNA substrates. A four arrays study of the transcriptome profiles of the B. subtilis RNase Y depletion strain SSB447 (Shahbabian et al., 2009) in which the gene rny is under the control of an IPTG inducible promoter. Rnase Y depleted cells (IPTG-) are compared to the transcript profiles of non depleted cells (IPTG+). As the rny gene is essential, depletion can only be partial. Here we carefully selected the time point for taking the sample in order to minimize growth rate effects and to obtain an highly reproducible data set.', '1132/NCBITaxon:224308', '1136/efo:EFO_0002769'
  UNION SELECT 290, 'E-GEOD-38200', 23, 'Ikaros target genes in the mouse pre-B cell line B3', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 291, 'E-GEOD-38169', 23, 'Genome-wide identification of Ikaros binding sites in pre-B cells', 'Ikaros DNA binding proteins are important regulators of haematopoiesis and genetic deletion of Ikaros results in severe developmental disturbances, including delayed thymocyte differentiation and an early and complete block in B cell development. Although Ikaros ChIP-seq data are available for mouse thymocytes and human haematopoietic progenitors, it has not been achieved in B cell progenitors. The goal of this study was to identify Ikaros binding sites in pre-B cells to define Ikaros target genes which could explain the essential role of Ikaros proteins in B cell differentiation. We carried out chromatin immunoprecipitation and high throughput sequencing (ChIP-seq) with antibodies to the C-terminus of endogenous Ikaros in B3 cells and with anti-haemagglutinin (HA) in B3 cells transduced with epitope-tagged HA-Ikaros.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002692'
  UNION SELECT 292, 'E-GEOD-38110', 23, 'Gene expression in mouse pre-B cells transduced with Ikaros.', 'Ikaros family DNA binding proteins are critical regulators of B cell development. To identify Ikaros-regulated genes in pre-B cells we performed gene expression studies at enhanced temporal resolution. We used retroviral gene transfer to express Ikaros proteins in the pre-B cell line B3. Total RNA from 2 biological replicates of B3 cells transduced with wild type Ikaros (HA-Ikaros-IRES-GFP) and DNA binding-deficient Ikaros mutant 159A (HA-159A Ikaros-IRES-GFP) was isolated 48h after infection. To increase the temporal resolution of Ikaros-regulated gene expression we transduced B3 cells with inducible Ikaros (HA-Ikaros-ERt2-IRES-GFP) and isolated  total RNA from 3 biological replicates after 2 h and 6 h of exposure to 4-hydroxytamoxifen. Vector transduced B3 cells (ERt2-IRES-GFP)  treated with 4-hydroxytamoxifen were used as controls.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 293, 'E-GEOD-27237', 23, 'The Genetic Basis of Hypodiploid Acute Lymphoblastic Leukemia', 'The genetic basis of hypodiploid acute lymphoblastic leukemia (ALL), characterized by aneuploidy and poor outcome, is unknown. Here, using complementary genome-wide profiling approaches, we show that hypodiploid ALL comprises two major subtypes that differ in the severity of aneuploidy, transcriptional profile and submicroscopic genetic alterations. Near haploid cases with 24-31 chromosomes frequently harbor alterations targeting receptor tyrosine kinase- and Ras signaling (71%) and IKZF3 (AIOLOS; 13%). In contrast, low hypodiploid ALL cases with 32-39 chromosomes are characterized by TP53 alterations (88%), almost half of which are present in non-tumor cells, and have alterations of IKZF2 (HELIOS; 53%) and RB1 (41%). Both near haploid and low hypodiploid tumors exhibit activation of Ras and PI3K signaling pathways, and are sensitive to PI3K inhibition, indicating that these drugs should be explored as a new therapeutic strategy for this frequently lethal form of leukemia. Gene expression profiling was performed on 117 single diagnosis tumor samples and one relapse sample [SJHYPO117]. No control or reference samples were included.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 294, 'E-GEOD-42912', 23, 'Nurr1 maintains fiber integrity and nuclear-encoded mitochondrial gene expression in dopamine neurons', 'Nurr1 (Nr4a2, nuclear receptor subfamily 4 group A member 2) is needed for the development of ventral midbrain dopaminergic neurons, and has been associated with Parkinson''s disease. We used mice where the Nurr1 gene is ablated by tamoxifen treatment selectively in dopaminergic neurons. As a control, we used tamoxifen-treated mice where Nurr1 is not ablated. By laser microdissection of neurons selected by their TH1 (Th1l, TH1-like homolog) gene expression, we selected dopaminergic neurons for RNA extraction and high-throughput mRNA sequencing, in order to identify genes regulated by Nurr1. We found the main functional category of Nurr1-regulated genes are the nuclear-encoded mitochondrial genes. Dopaminergic neurons with or without Nurr1 knocked out. TH-positive neurons were laser capture microdissected from cryostat coronal sections of the midbrain.', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 295, 'E-GEOD-42880', 23, 'MeCP2 binds to 5hmC enriched within active genes and accessible chromatin in the nervous system', 'The high level of 5-hydroxymethylcytosine (5hmC) present in neuronal genomes suggests  that mechanisms interpreting 5hmC in the central nervous system (CNS) may differ from those present  in embryonic stem cells. Here we present quantitative, genome-wide analysis of 5hmC, 5-  methylcytosine (5mC) and gene expression in differentiated CNS cell types in vivo. We report that  5hmC is enriched in active genes, and that surprisingly strong depletion of 5mC is observed over these  regions. The contribution of these epigenetic marks to gene expression depends critically on cell type.  We identify methyl-CpG binding protein 2 (MeCP2) as the major 5hmC binding protein in the brain,  and demonstrate that MeCP2 binds 5hmC and 5mC containing DNA with similar high affinities. The  Rett Syndromecausing mutation R133C preferentially inhibits 5hmC binding. These findings support a  model in which 5hmC and MeCP2 constitute a cell specific epigenetic mechanism for regulation of  chromatin structure and gene expression. Isolation and sorting was performed as described in (Kriaucionis and  Heintz, 2009). Briefly, cerebella were dissected as described above and homogenized in homogenization  medium (0.25 M sucrose, 150 mM KCl, 5 mM MgCl2, 20 mM Tricine pH 7.8, 0.15 mM spermine,  0.5 mM spermidine, EDTA-free protease inhibitor cocktail (Roche) using loose (A) and tight (B) glassglass  dounce.Homogenate was supplemented with 50% iodixanol (OptiPrep, Axis-Shield,  Scotland), 150 mM KCl, 5 mM MgCl2, 20 mM Tricine pH 7.8; and laid on a 29% iodixanol cushion. GC nuclei from WT and Mecp2-null mice were sorted by selecting a specific FSC/SSC  population identified from GC EGFP+ mice. We observed that this FSC/SSC gated population in GC EGFP+  animals resulted >92% of the sorted nuclei being EGFP positive. Thus, the gate selected allows us  to enrich the sample from 65-70% to 92-96% on GC cells. 5hmC was pulled down as described (Song et al., 2010). After  purification DNA was amplified as described in TruSeq DNA Sample kit. 1-0.5 ?g of DNA was used for each experiment.  Sonicated DNA was end-repaired following by ligation to Illumina paired end sequencing adapters (Illumina,  PE? 102? 1003).), following  by amplification with Illumina primers. Input samples were produced for  each cell types in both procedures  5hmC enriched were then sequenced using Illumina platform obtaining more than 50 x10-6, 36-  bp single-end reads per sample. Reads were aligned to mm9 mouse genome assembly using Bowtie v0.12.7  (-m1 --best).  Two biological replicas were done for each of the cell  types Single animals per replica were euthanized  with CO2 and cerebella were dissected. 4 cerebella was immediately homogenized in ice-cold  homogenization buffer (10 mM HEPES [pH 7.4], 150 mM KCl, 5 mM MgCl2, 0.5 mM dithiothreitol (DTT), 100  ?g/ml cycloheximide, Complete-EDTA-free protease inhibitors (Roche) and RNasin RNase inhibitors  (Promega, Madison, WI) using a Teflon-glass homogenizer. Homogenates were centrifuged for 10 minutes at  2,000 × g, 4 °C, to pellet cell debris, and incubated with 1% IGEPAL CA-630 (NP-40, Sigma) and 30 mM  DHPC (Avanti Polar Lipids, Alabaster, AL) for 5 min. Lysates were centrifuged for 15 minutes at 20,000 × g to  pellet insoluble material. At this point, 30 ?L of supernatant was saved as input sample. Two custom made  mouse anti-GFP (clones 19C8 and 19F7; see Heiman et al., 2008) antibodies were captured on Dynal  magnetic beads (Invitrogen Corporation, Carlsbad, CA) coupled with protein L (Pierce, Rockford, IL) . The  homogenate was incubated with antibody coupled beads at 4°C with end-over-end rotation for approximately  16 hours. Beads were subsequently collected on a magnetic rack, washed three times with high-salt wash  buffer (10 mM HEPES [pH 7.4], 350 mM KCl, 5 mM MgCl2, 1% NP-40, 0.5mM DTT, 100 ?g/ml cycloheximide)  and RNA was released and purified using Rneasy Micro Kit (Qiagen, Valencia, CA) with in-column DNase  digestion. RNA quantity and quality were determined with a Nanodrop 1000 spectrophotometer (Wilmington,  DE) and Agilent 2100 Bioanalyzer using RNA 6000 Pico Chip. 4 biological replicas was produced per each cell  type. 10 ng of total RNA from cerebellum were converted to cDNA using the NuGEN Ovation RNA-Seq  (NuGEN,San Carlos,CA,USA) following manufacturers instructions. The Single Primer Isothermal  Amplification (SPIA) method used in this protocol allows the amplification of RNA target in double stranded  cDNA under standardized conditions that markedly deplete rRNA without preselecting mRNA. cDNA obtained  was quality scored by RNA 6000 PicoChip for Agilent 2100 Bioanalyzer (Agilent, Santa Clara, CA, USA). 1 ?g  of cDNA per sample was sonicated using Covaris-S2 system (Covaris Inc., Woburn, MA, USA), cDNA  fragments of 200 bp were end-repaired and adapters were ligated for HiSeq 2000 (Ilumina Inc., San Diego,  CA, USA) technology using TruSeq DNA Sample kit (Illumina) and following manufacturer`s instructions.  Quality of libraries was assesed using HT DNA High Sensitivity Chip (Agilent) for 2100 Bioanalyzer. Single animals per replica were euthanized  with CO2 and cerebella were dissected. 4 cerebella was immediately homogenized in ice-cold  homogenization buffer (10 mM HEPES [pH 7.4], 150 mM KCl, 5 mM MgCl2, 0.5 mM dithiothreitol (DTT), 100  ?g/ml cycloheximide, Complete-EDTA-free protease inhibitors (Roche) and RNasin RNase inhibitors  (Promega, Madison, WI) using a Teflon-glass homogenizer. Homogenates were centrifuged for 10 minutes at  2,000 × g, 4 °C, to pellet cell debris, and incubated with 1% IGEPAL CA-630 (NP-40, Sigma) and 30 mM  DHPC (Avanti Polar Lipids, Alabaster, AL) for 5 min. Lysates were centrifuged for 15 minutes at 20,000 × g to  pellet insoluble material. At this point, 30 ?L of supernatant was saved as input sample. Two custom made  mouse anti-GFP (clones 19C8 and 19F7; see Heiman et al., 2008) antibodies were captured on Dynal  magnetic beads (Invitrogen Corporation, Carlsbad, CA) coupled with protein L (Pierce, Rockford, IL) . The  homogenate was incubated with antibody coupled beads at 4°C with end-over-end rotation for approximately  16 hours. Beads were subsequently collected on a magnetic rack, washed three times with high-salt wash  buffer (10 mM HEPES [pH 7.4], 350 mM KCl, 5 mM MgCl2, 1% NP-40, 0.5mM DTT, 100 ?g/ml cycloheximide)  and RNA was released and purified using Rneasy Micro Kit (Qiagen, Valencia, CA) with in-column DNase  digestion. RNA quantity and quality were determined with a Nanodrop 1000 spectrophotometer (Wilmington,  DE) and Agilent 2100 Bioanalyzer using RNA 6000 Pico Chip. 4 biological replicas was produced per each cell  type. 10 ng of total RNA per IP or input sample were converted to cDNA using the NuGEN Ovation RNA-Seq  (NuGEN,San Carlos,CA,USA) following manufacturers instructions. The Single Primer Isothermal  Amplification (SPIA) method used in this protocol allows the amplification of RNA target in double stranded  cDNA under standardized conditions that markedly deplete rRNA without preselecting mRNA. cDNA obtained  was quality scored by RNA 6000 PicoChip for Agilent 2100 Bioanalyzer (Agilent, Santa Clara, CA, USA). 1 ?g  of cDNA per sample was sonicated using Covaris-S2 system (Covaris Inc., Woburn, MA, USA), cDNA  fragments of 200 bp were end-repaired and adapters were ligated for HiSeq 2000 (Ilumina Inc., San Diego,  CA, USA) technology using TruSeq DNA Sample kit (Illumina) and following manufacturer`s instructions.  Quality of libraries was assesed using HT DNA High Sensitivity Chip (Agilent) for 2100 Bioanalyzer. Single animals per replica were euthanized  with CO2 and cerebella were dissected. 4 cerebella was immediately homogenized in ice-cold  homogenization buffer (10 mM HEPES [pH 7.4], 150 mM KCl, 5 mM MgCl2, 0.5 mM dithiothreitol (DTT), 100  ?g/ml cycloheximide, Complete-EDTA-free protease inhibitors (Roche) and RNasin RNase inhibitors  (Promega, Madison, WI) using a Teflon-glass homogenizer. Homogenates were centrifuged for 10 minutes at  2,000 × g, 4 °C, to pellet cell debris, and incubated with 1% IGEPAL CA-630 (NP-40, Sigma) and 30 mM  DHPC (Avanti Polar Lipids, Alabaster, AL) for 5 min. Lysates were centrifuged for 15 minutes at 20,000 × g to  pellet insoluble material. At this point, 30 ?L of supernatant was saved as input sample. Two custom made  mouse anti-GFP (clones 19C8 and 19F7; see Heiman et al., 2008) antibodies were captured on Dynal  magnetic beads (Invitrogen Corporation, Carlsbad, CA) coupled with protein L (Pierce, Rockford, IL) . The  homogenate was incubated with antibody coupled beads at 4°C with end-over-end rotation for approximately  16 hours. Beads were subsequently collected on a magnetic rack, washed three times with high-salt wash  buffer (10 mM HEPES [pH 7.4], 350 mM KCl, 5 mM MgCl2, 1% NP-40, 0.5mM DTT, 100 ?g/ml cycloheximide)  and RNA was released and purified using Rneasy Micro Kit (Qiagen, Valencia, CA) wit', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 296, 'E-GEOD-42865', 23, 'Genome-wide DNA methylation profiling of Progeria and Werner syndrome patients', 'DNA methylation gradiently changes with age and is likely to be involved in aging-related processes resulting in phenotype changes and increased susceptibility to certain diseases. The Hutchinson-Gilford Progeria Syndrome (HGP) and Werner Syndrome are two premature aging diseases showing features of common aging. Mutations in LMNA and WRN genes were associated to disease onset; however for a subset of patients the underlying causative mechanisms remain elusive. We aimed to evaluate the role of epigenetic alteration on premature aging diseases by performing genome-wide DNA methylation profiling of HGP and WS patients. DNA was quantified by Quant-iT PicoGreen dsDNA Reagent (Invitrogen) and the integrity was analyzed in a 1.3% agarose gel. Bisulfite conversion of 600 ng of each sample was perform according to the manufacturer''s recommendation for Illumina Infinium Assay. Effective bisulphite conversion was checked for three controls that were converted simultaneously with the samples. 4 ul of bisulfite converted DNA were used to hybridize on Infinium HumanMethylation 450 BeadChip, following Illumina Infinium HD Methylation protocol. Chip analysis was performed using Illumina HiScan SQ fluorescent scanner. The intensities of the images are extracted using GenomeStudio (2010.3) Methylation module (1.8.5) software. Methylation score of each CpG is represented as beta value. Naive B-cells and peripheral blood mononuclear cells were analyzed to correct for the epigenetic effects of the Epstein-Barr virus immortalization (lymphoblastoid cell lines) and cell composition of the samples, respectively.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002759'
  UNION SELECT 297, 'E-GEOD-41054', 23, 'Agilent data from Arabidopsis thaliana enriched guard cell tissue and leafs.', 'To identify genes of the guard cell transcriptome of Arabidopsis thaliana enriched guard cell samples were compared with total leaf tissue. Genes of the abscisic acid and humidity response of Arabidopsis thaliana guard cells were identified by treatment with ABA-Spray and low humidity. total samples analysed are 24: 4 biological independent replicates of: total leaf (COL-0) vs. enriched guard cells (COL-0); ABA-sprayed enriched guard cells (gl1-1) vs. control-sprayed enriched guard cells (gl1-1); low humidity (20%rh) treated enriched guard cells (COL-0) vs. high humidity (80%) treated enriched guard cells (COL-0)', '1132/NCBITaxon:3702', '1136/efo:EFO_0002768'
  UNION SELECT 298, 'E-GEOD-40841', 23, 'Safety and efficacy of CYT387, a JAK1 and JAK2 inhibitor, in myelofibrosis', '17 patients with myelofibrosis contributed paired research samples; the time points were pre-study and 12 weeks after commencing study treatment with CYT387.Gene expression analysis was investigated to study the effects of drug treatment. Additional analysis invesitigated differences in expression patterns in groups of anemia responders and non-responders following drug treatment. 34 samples representing 17 patients. Each patient submitted 1 pre-treatment sample and 1 post 12 weeks of treatment.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 299, 'E-GEOD-39509', 23, 'Lineage analysis of basal epithelial cells reveals their unexpected plasticity and supports a cell of origin model for prostate cancer heterogeneity', 'We used RNA-seq to compare the expression profiles of mouse prostate tumors originated from epithelial basal cells to those originated from luminal cells. We next generated expression signatures for both basal and luminal origin tumors by comparison of tumor samples to their respective controls. By comparing luminal to basal signatures we identified a prognostic molecular signature for prostate cancer patient survival. Pten was deleted in prostate basal or luminal cells to induce tumor formation and tumor expression profiles were analyzed by RNA-seq.', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 300, 'E-GEOD-39150', 23, 'Transcriptome sequencing of Chinese tree shrew, Tupaia belangeri chinensis', 'Deep sequencing of mRNA from Chinese tree shrew; Chinese tree shrew (Tupaia belangeri chinensis) is placed in Order Scandentia and embraces many unique features for a good experimental animal model. Currently, there are many attempts to employ tree shrew to establish model for a variety of human disorders such as social stress, myopia, HCV and HBV infection, and hepatocellular carcinoma .We present here a publicly available annotated genome sequence for Chinese tree shrew. Phylogenomic analysis of tree shrew and other mammalians highly supported its close affinity to primates. Characterization of key factors and signaling pathways of the nervous and immune systems in tree shrews showed that this animal had common and unique features, and had essential genetic basis for being a promising model for biomedical researches. Analysis of ploy(A)+ RNA of different specimens:kidney, pancreas, heart, liver, brain, testis and ovary form Chinese tree shrew', '1132/NCBITaxon:246437', '1136/efo:EFO_0003738'
  UNION SELECT 301, 'E-GEOD-38463', 23, 'Gene expression profiling of mouse B cell progenitors', 'Gene expression profiling was performed of Pax5 wild type bone marrow subsets from common lymphoid progenitors through to Hardy stage F cells. These cells were obtained by flow sorting of bone marrow. Gene expression profiling was performed on 41 samples of RNA extracted from flow sorted B cell precursor populations from common lymphoid progenitor (CLP) through to Hardy stage F. Note that several of the genes expressed in normal mouse B cell progenitors are also involved by deletion/mutation in hypodiploid ALL. Related hypodiploid ALL studies: Gene expression data: GEO GSE27237 SNP & Sequencing data: dbGaP phs000341.v1.p1', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 302, 'E-GEOD-35896', 23, 'Gene expression data from 62 colorectal cancers', 'We stratified colorectal tumor samples using a new unsupervised, iterative method based on non-negative matrix factorization (NMF). The resulting five subtypes exhibited activation of specific signaling pathways, and significant differences in microsatellite status and tumor location. We could also align three CRC cell lines panels to these subtypes. mRNA from 62 colorectal tumor samples was hybridized to Affymetrix HGU133 Plus 2.0 expression arrays. Log2 gene expression values were calculated using RMA.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 303, 'E-GEOD-35288', 23, 'Transcriptional profiling of the Arabidopsis abscission mutant hae hsl2 by RNA-Seq', 'The receptor-like protein kinases encoded by HAESA (HAE) and HAESA-LIKE 2 (HSL2) are essential for floral organ abscission in Arabidopsis thaliana and the double hae hsl2 mutant fails to abscise. Expression of HAE and HSL2 is specific to Abscission Zone (AZ) cells and is higher in stage 15 flowers than in earlier developmental stages. By stage 16 floral organs have begun to abscise, suggesting that HAE HSL2 are most active in stage 15 flowers. Samples were enriched for AZ RNA by isolating RNA from flower receptacles, the region from the base of the flower to slightly above the base of attachment of the sepals, petals, and stamen. RNA-seq was then used to analyze and compare the transcriptomes of wild type and hae-3 hsl2-3 mutants. 2034 genes were differentially expressed with a False Discovery Rate adjusted p < 0.05, of which 349 genes 2 fold or greater change. Of these 349, 277 were lower in the mutant and 72 were higher. Differentially expressed genes with lower expression were enriched for hydrolytic enzymes, cell-wall modifying enzymes, and defense related genes. This suggests that HAE HSL2 signaling regulates gene expression of enzymes necessary for abscission. 6 samples were sequenced, 3 biological replicates of Col-0 wild type and 3 biological replicates of the hae-3 hsl2-3 double mutant. Samples were barcoded and all 6 samples multiplexed and sequenced on 3 lanes, each lane on a separate flow cell, of an Illumina HiSeq 2000.', '1132/NCBITaxon:3702', '1136/efo:EFO_0003738'
  UNION SELECT 304, 'E-GEOD-23435', 23, 'Gene expression data from AKT induced salivary gland tumors', 'Rapamycin treatment of AKT induced salivary gland tumors Normal tissue (parotid and submandibular), untreated tumor tissue, and tumor tissue treated for 3/14 days', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 305, 'E-GEOD-32934', 23, 'Gene expression profiling of primary canine insulinomas and their metastases', 'Insulinomas (INS) are rare pancreatic endocrine tumours (PETs), arising from tumoral beta-cells from the pancreatic islets of Langerhans. Canine INS are malignant in >95% of cases. This study aims to provide further insights on molecular pathways involved in INS pathogenesis and to identify prognostic genes. Canine specific cDNA microarray representing 20,313 genes was used to analyze expression profiles in a panel of ten primary INS and ten INS metastases. Eighty-four genes were found to be significantly down-regulated in the group of metastases in comparison to the group of primary INS. From the microarray analysis it also followed that primary INS, based on their gene expression profiles, can be divided into two groups, which differ in their malignancy potential. It was also demonstrated that canine INS have both endocrine and exocrine cell features. Furthermore, the present study has revealed pancreatic lipase and chymotrypsinogen B1 as candidate genes that can be followed up in prospective studies as prognostic markers for INS. Our findings add to the current knowledge of INS pathogenesis and tumor progression, which can be used for future therapeutic and prognostic strategies, for both dogs and humans. From 1993 to 2009, ten histologically confirmed primary insulinomas (INS) and their accompanying metastases were obtained from dogs that were presented at the Department of Clinical Sciences of Companion Animals of the Faculty of Veterinary Medicine of Utrecht University. Tumors were resected by experienced surgeons, and fragments (5-15 mm diameter) for RNA extraction were carefully removed from the central portion of the tumors to avoid contamination with normal adjacent tissue. Tissue specimens collected for RNA isolation were snap-frozen in liquid nitrogen immediately after surgical removal and stored at  70oC. Various clinical and pathological parameters were evaluated retrospectively for the dogs included in this study. Half of both the primary insulinoma samples, as well as half of the metastasis samples was labelled with Cy3, while the other half of both groups was labelled with Cy5. A common reference cRNA pool consisting of equal quantity of cRNA from all 20 tumor samples was used to hybridize against each sample on the array.', '1132/NCBITaxon:9615', '1136/efo:EFO_0002768'
  UNION SELECT 306, 'E-GEOD-43466', 23, 'Reconstruction of the Mycobacterium tuberculosis Regulatory Network and Deconstruction of the Hypoxic Response', 'Expression data from hypoxic time course and transcription factor over expression experiments. (Abstract from paper will be appended after publication) Hypoxic time course experiments were done with a minimum of 3 replicates. TFOE are matched to the ChIP-seq experiment done simultaneously. This submission consists of the the gene expression component only.', '1132/NCBITaxon:83332', '1136/efo:EFO_0002768'
  UNION SELECT 307, 'E-GEOD-43465', 23, 'Effect of dietary vitamin B6 on white adipose tissue', 'Transcriptional profiling of mouse white adipose tissues The objective of this study is to explore the relationship between vitamin B6 activity and chronic inflammation of white adipose tissue in mice fed a high-fat diet. Twenty-four CD-1 mice were divided into two groups (n = 12) and fed either a 1 mg pyridoxine (PN) HCl /kg diet or a 35 mg PN HCl /kg diet for 8 weeks. We isolated total RNA from epididymal white adipose tissue of each group and compared gene expression profiles by DNA microarray data analysis. Noriyuki Yanaka', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 308, 'E-GEOD-43452', 23, 'Gene profile of glioblastoma cells treated with y15 and temozolomide', 'The gene expression profiles were identified in glioblastoma cells treated with FAK inhibitor Y15, temozolomide alone or with combination of Y15 and Temozolomide DBTRG and U87 were treated with FAK inhibitor Y15 at 10 microM for 24 h; U87 cells were treated with Temozolomide 100 microM for 24 h and Y15+temozolomide at the same dose as each agent alone', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 309, 'E-GEOD-43446', 23, 'Influenza peptide reactivity within a cohort of age-stratified individuals pre and post vaccination with the 2008/2009 trivalent influenza vaccine', 'The aim of the study was to test the hypothesis that peptide array reactivity will reveal patterns that can be associated with poor or effective neutralization response to seasonal influenza vaccine We diluted patient sera 1:80 and incubated dilutions on a streptavidin-coated glass surface that was printed with biotinylated peptides derived from seasonal strains of influenza hemagglutinin included in the 2008/2009 trivalent influenza vaccine In this study, serum from 76 study participants, subsetted into young (?30 years old) and elderly (?60 years old) categories were profiled for IgG/IgM antibody reactivity to influenza hemagglutinin peptudes. Using multivariate linear regression and correcting for multiple hypothesis testing by cross-validation, ee found that patterns of peptide reactivity associated with a poor or effective neutralization response to H1N1 and B strains of influenza.', '1132/NCBITaxon:9606', ''
  UNION SELECT 310, 'E-GEOD-43445', 23, 'MicroRNA profiling in primary and metastatic HCC cell lines using PLC8024 and MHCC97H derived cell lines', 'Goal for this study is to identified miRNA involved in metastasis development using PLC8024 and MHCC97H derived cell lines. PLC8024 derived cell lines, PLC-PT (Primary Tumor) and PLC-LM (Lung Metastasis), and MHCC97H derived cell lines, MHCC97H-PT and MHCC97H-LM were compared using the Ncode miRNA microarray platform. The experiment were repeat twice with dye-swap.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 311, 'E-GEOD-43441', 23, 'Expression data from laser captured gastric neck cells and zymogenic cell from wild type and MIST1 knockout mice', 'MIST1 is a bHLH transcription factor that is necessary for the maturation of gastric zymogenic cells as they differentiate from their precursor mucous neck cells. In this experiment, mucous neck cells and zymogenic cells of normal, adult C57BL/6 and MIST1 knockout mice were laser-capture microdissected in order to determine MIST1-dependent, zymogenic cell specific gene expression. Stomachs were excised immediately following sacrifice, quickly flushed with room-temperature PBS, inflated by duodenal injection of OCT compound, frozen in Cytocool II, and cut into serial 7-?m-thick cryosections, which were mounted on Superfrost slides, fixed in 70% EtOH, rehydrated in nuclease-free water, and then incubated in Alexa Fluor 488-conjugated Griffonia simplicifolia GS-II (diluted 1:500 in nuclease-free water) for 15 min. Sections were washed in nuclease-free water and dehydrated in graded ethanol followed by xylene. ZCs were identified as corpus cells that were basal to GS-II labeling and which did not show the dark silhouettes and characteristic shape of parietal cells following xylene dehydration. Four wild-type mice and 5 Mist1?/? mice were used for dissection (PixCell II LCM apparatus [7.5-?m spot diameter] and CapSure HS LCM caps) to generate two caps per mouse. RNA was purified by PicoPure kit, and RNA integrity was confirmed by an Agilent 2100 Bioanalyzer. All RNA from each cap was treated with DNase I and then reverse transcribed using the SuperScript III (Invitrogen) standard protocol (most cDNA syntheses started with 10 ng of total RNA). Biotinylated cRNA probes were hybridized to the GeneChips. Gene chip arrays used in these experiments were Affymetrix Mouse Gene 1.0ST arrays. Chip quality control and GeneChip to GeneChip comparisons to generate expression profiles were performed using dChip.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 312, 'E-GEOD-42544', 23, 'Pseudomonas syringae B728a responses to environmental stresses and growth in and on bean leaves', 'Some strains of the foliar pathogen Pseudomonas syringae are adapted for growth and survival on leaf surfaces and in the leaf interior.  Global transcriptome profiling was used to evaluate if these two habitats offer distinct environments for bacteria and thus present distinct driving forces for adaptation.  Further transcriptome profiling was performed to understand the various environmental conditions that P. syringae cells encounter during their association with plants.   RNA was collected from P. syringae pv. syringae strain B728a cells that were exposed to seven treatments.  The treatments included five in vitro treatments, namely exposing cells to a basal medium, sodium chloride to confer an osmotic stress, hydrogen peroxide to confer an apoplastic growth, iron limitation, nitrogen limitation.  They also included two in planta treatments, namely recovering cells from epiphytic sites after surface inoculation and 72 h of growth on bean (Phaseolus vulgaris L.) leaves and recovering cells from apoplastic sites after infiltration and 48 h of growth in bean leaves.  The results suggested that B728a cells experience vastly different environments when growing on the surface versus the interior of leaves and identified distinct traits that are likely used for persistence and growth in these environments. RNA was collected from B728a cells that were exposed to seven treatments, each with two biological replicates in each of three laboratories.  Experimental methods were standardized across the three laboratories.  For the in vitro treatments, exponential cells from two independent cultures were each exposed to the five treatments.  For each treatment, the cells originating from the two cultures were pooled and the RNA was extracted; this was repeated in its entirety and the two RNA pools were combined.  The resulting RNA representing four independent cultures served as a single biological replicate.  Two biological replicates for each treatment were generated in this manner in each of three separate laboratories. Epiphytic B728a populations were established following spray inoculation onto bean leaves and subsequent incubation; the epiphytic cells recovered from 400 to 600 leaves inoculated at one time served as a biological replicate, and the procedure was repeated to provide two biological replicates.  Due to the availability of facilities, all of the epiphytic treatments were conducted at as single laboratory, with cultures provided from each of the other laboratories, and the cell pellets were returned to those laboratories for RNA extraction and analysis. Apoplastic B728a populations were established following vacuum infiltration into bean leaves and subsequent incubation; the apoplastic cells recovered from 40 to 80 leaves inoculated at one time served as a biological replicate, and two biological replicates were generated at each of the three laboratories. The RNA from all treatments was submitted to Roche Nimblegen, Inc where it was labeled and hybridized to an ORF-based microarray that included 5,071 ORFs and 61 putative sRNAs, with each ORF represented by 14 60-mer nucleotide probes.  The fluorescence intensity for each probe was measured and subjected to robust multiarray averaging, which included adjustment for the background intensity, log2 transformation, quantile normalization and median polishing, and a robust estimated mean value was determined for each ORF and putative sRNA on the array.', '1132/NCBITaxon:205918', '1136/efo:EFO_0002768'
  UNION SELECT 313, 'E-GEOD-41193', 23, 'High fidelity patient-derived xenografts for accelerating prostate cancer discovery and drug development', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 314, 'E-GEOD-41192', 23, 'High fidelity patient-derived xenografts for accelerating prostate cancer discovery and drug development (expression)', 'Prostate cancer discovery and translational research are hampered by a lack of preclinical models which accurately reproduce the biological heterogeneity observed in patients. Accordingly, we have established a bank of transplantable patient-derived prostate tumor xenograft lines, using subrenal capsule grafting of human tumor tissue into immuno-deficient mice. This panel includes the first lines generated from primary prostate cancer tissue, and also new lines from metastatic tissue. Critically, the lines retained salient features of the original patient tumors, including histopathology, clinical marker expression, chromosomal aberration and gene expression profiles. Furthermore, they span major histopathological and molecular subtypes of prostate cancer, capturing diverse inter- and intra-tumoral heterogeneity. Host castration led to the development of castrate-resistant tumors, including the first model of complete neuroendocrine transdifferentiation. This publicly-available resource provides novel tools to advance mechanistic understanding of disease progression and response to therapy, and delivers clinically-relevant model systems for evaluation of preclinical drug efficacy. 3 primary tumors and 22 xenograft tumors', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 315, 'E-GEOD-41188', 23, 'High fidelity patient-derived xenografts for accelerating prostate cancer discovery and drug development (aCGH)', 'Prostate cancer discovery and translational research are hampered by a lack of preclinical models which accurately reproduce the biological heterogeneity observed in patients. Accordingly, we have established a bank of transplantable patient-derived prostate tumor xenograft lines, using subrenal capsule grafting of human tumor tissue into immuno-deficient mice. This panel includes the first lines generated from primary prostate cancer tissue, and also new lines from metastatic tissue. Critically, the lines retained salient features of the original patient tumors, including histopathology, clinical marker expression, chromosomal aberration and gene expression profiles. Furthermore, they span major histopathological and molecular subtypes of prostate cancer, capturing diverse inter- and intra-tumoral heterogeneity. Host castration led to the development of castrate-resistant tumors, including the first model of complete neuroendocrine transdifferentiation. This publicly-available resource provides novel tools to advance mechanistic understanding of disease progression and response to therapy, and delivers clinically-relevant model systems for evaluation of preclinical drug efficacy. 3 primary tumors and 21 xenograft tumors', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 316, 'E-GEOD-14995', 23, 'JAG1 is an Invasion and Metastasis promoting genes of Lung cancer', 'Background: JAG-1 is a ligand of Notch signaling and can regulate cell differentiation and proliferation in cancers. Recent studies indicated that JAG1 is a gene associated with cancer progression. Therefore, we investigated the role of JAG1 in lung cancer progression. Methods: The expression of JAG1 was manipulated by overexpression or RNA silencing in several human lung cell lines. The effect of JAG1 on tumorigenesis and invasion was assessed by the cell anchorage-independent growth, cell proliferation, cell migration and invasion assays in vitro as well as metastasis in vivo. The potential downstream genes of JAG1 were identified by oligonucleotide microarrays and quantitative reverse transcription¡Vpolymerase chain reaction (RT-PCR). We further measured JAG1 expression in lung cancer specimens by RT-PCR. Correlation between JAG1 expression and overall survival of lung cancer patients was determined by using the log-rank test and multivariable Cox proportional hazards regression analysis. All statistical tests were two-sided. Results: JAG1 enhanced anchorage-independent growth, cell migration, invasion in the lower invasive cells, CL1-0. JAG1 also increased the capability of migration and invasion in the other two lung cancer cell lines (A549 and NCI-H226). The silencing of JAG1 inhibited migration and invasion activities of the higher invasive cells, CL1-5, by siRNA technology. The invasion-promoting activity of JAG1 was also demonstrated in vivo by using a mouse metastasis model. By microarray analysis, we found that the expression of heat shock 70kDa protein 2 (HSPA2) was activated by JAG1 overexpression and eliminated by JAG1 silencing. Moreover, lung cancer patients with high JAG1 expressing tumors had shorter overall survival than those with low-expressing tumors. Conclusion: JAG1 might be an oncogene which promotes colonogenesis and metastasis, and high JAG1 expression is associated with shorten survival in lung cancer. In this investigation, we used a lung cancer invasion cell model to identify the genes involved in cancer progression. JAG1 is a potential oncogene whose expression is correlated to the survival of patients with breast, prostate and liver cancers. However, the role of JAG1 in lung caner progression has not been reported, particularly in metastasis. Here, JAG1 was ectopically expressed in lower invasive lung cancer cell line its impact on colonogenesis, migration and invasiveness was assessed. The underlying mechanism was explored by JAG1-expressed transfectants and microarrays and the clinical relevance was evaluated by quantitative RT-PCR.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 317, 'E-MTAB-981', 23, 'Transcription profiling by array of Nab2 depletion effect on yeast mRNA', 'Transcription profiling was performed on the time course of Nab2 protein depletion in THC-Nab2 strain at 0h, 7h, 12h in biological triplicates to access its affect on the composition and structure of mRNA in budding yeast. For the protein depletion experiment THC-NAB2 (THJ collection Y2484, THC-NAB2:KAN tTA:URA, Open Biosystems) strain was used, where expression of the NAB2 gene is under control of a tetracyclin-repressible promoter.', '1132/NCBITaxon:4932', '1136/efo:EFO_0002768'
  UNION SELECT 318, 'E-GEOD-43415', 23, 'Estrogen Receptor alpha ChIP-Seq in mouse mammary gland', 'Estrogen Receptor is a key transcriptional regulator in mammary gland development and breast cancer. In this study, we have mapped the Estrogen Receptor chromatin binding patterns in healthy mouse mammary gland A minimum of 6 pairs of mouse mammary gland pads from mice at 5-6 weeks of age were excised and Estrogen Receptor ChIp-seq was performed.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002692'
  UNION SELECT 319, 'E-GEOD-43413', 23, 'Expression data from the telencephalon of wild-type and rSey2/rSey2 rats', 'Pax6 is one of the important transcription factors involved in regional specification and neurogenesis in the developing cortex. To identify candidate target genes of Pax6, we performed transcriptome analyses of wild-type (WT) and Pax6 homozygous mutant rats (rSey2/rSey2) telencephalons at E11.5 within a day of onset of Pax6 expression. In our transcriptome analyses, down-regulated genes in the rSey2/rSey2 rat exhibited larger fold changes, whereas up-regulated genes had relatively small fold changes. Total RNA was prepared using  from 13 telencephalon dissected from E11.5 WT or rSey2/rSey2 rats embryos. Experiments using 13 telencephaon were repeated twice in each genotype.', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 320, 'E-GEOD-43407', 23, 'Dosage dependent tumor suppression by histone deacetylase 1 and 2 by regulation of Myc collaborating genes and p53 function', 'comparative genome hybridisation of Hdac1/2 cKO lymphomas and matched normal tissue Histone deacetylases (HDACs) are epigenetic erasers of lysine-acetyl marks. Inhibition of HDACs using small molecule inhibitors (HDACi) is a potential strategy in the treatment of various diseases and is approved for treating hematological malignancies. Harnessing the therapeutic potential of HDACi requires knowledge of HDAC-function in vivo. Here, we generated a thymocyte-specific gradient of HDAC-activity using compound conditional knockout mice for Hdac1 and Hdac2. Unexpectedly, gradual loss of HDAC-activity engendered a dosage dependent accumulation of immature thymocytes and correlated with the incidence and latency of monoclonal lymphoblastic thymic lymphomas. Strikingly, complete ablation of Hdac1 and Hdac2 abrogated lymphomagenesis due to a block in early thymic development. Genomic, biochemical and functional analyses of pre-leukemic thymocytes and tumors revealed a critical role for Hdac1/Hdac2-governed HDAC-activity in regulating a p53-dependent barrier to constrain Myc-overexpressing thymocytes from progressing into lymphomas by regulating Myc-collaborating genes. One Myc-collaborating and p53-suppressing gene, Jdp2, was derepressed in an Hdac1/2-dependent manner and critical for the survival of Jdp2-overexpressing lymphoma cells.  Although reduced HDAC-activity facilitates oncogenic transformation in normal cells, resulting tumor cells remain highly dependent on HDAC-activity, indicating that a critical level of Hdac1 and Hdac2 governed HDAC-activity is required for tumor maintenance. genomic DNA from LckCre+;Hdac1/2 cKO lymphomas and matched normal genomic DNA was hybridized onto a Nimblegen whole genome array', '1132/NCBITaxon:10090', '1136/efo:EFO_0000749'
  UNION SELECT 321, 'E-GEOD-43403', 23, 'Molecular and functional features demarcate embryonic and induced pluripotent stem cells in rabbits', 'Little is known about the molecular and functional features of pluripotent stem cells (PSCs) in rabbits. To address this, we derived and characterized two types of rabbit PSCs from the same breed of New Zealand White rabbits: four lines of embryonic stem cells (rbESCs), and three lines of induced PSCs (rbiPSCs) that resulted from reprogramming adult skin fibroblasts. All lines were dependent on fibroblast growth factor 2. All rbESC lines exhibited molecular and functional properties typically associated with primed pluripotency. The rbESC lines also exhibited a cell cycle with a prolonged G1 phase and a DNA damage checkpoint prior to entry into the S phase, which are two features typically associated with the somatic cell cycle. In contrast, the rbiPSC lines exhibited some characteristics of naïve pluripotency as defined in rodents, including resistance to single cell dissociation by trypsin, robust activity of the distal enhancer of the mouse Oct4 gene, and expression of naïve-specific genes. According to gene expression profiles, rbiPSCs were closer to the rabbit inner cell mass than rbESCs. We propose that rbiPSCs self-renew in an intermediate state between naïve and primed pluripotency, which represents a key step toward the generation of bona fide naïve PSC lines in rabbits The patterns of stage specific embryonic antigen (SSEA) expression were also assessed between cell lines. All two rbiPSC lines exhibited heterogeneous expression of SSEA1 and SSEA4 that leads to obtain 3 different cell sub-populations (SSEA1+, SSEA1+/4+, SSEA1-/4-). All four rbESC lines expressed only SSEA1 and not SSEA4. Then, for this category of cell lines, we used 3 different cell sub-populations (SSEA1+, SSEA1-, and total which contained all cell types). Three independent experiments were performed for each line and for each cell sub-population.', '1132/NCBITaxon:9986', '1136/efo:EFO_0002768'
  UNION SELECT 322, 'E-GEOD-43291', 23, 'Genome wide binding sites of the Arabidopsis B3 domain protein FUSCA3', 'FUSCA3 (FUS3) is a B3 domain transcription factor that is a member of the LEAFY COTYLEDON (LEC) group of genes. The LEC genes encode proteins that also include LEC2, a B3 domain factor related to FUS3, and LEC1, a CCAAT box binding factor. LEC1, LEC2 and FUS3 are essential for plant embryo development. We report ChIP-chip experiments using the Affymetrix tiling array to globally map binding sites for FUS3. Fangfang Wang and Sharyn E. Perry (2013) Plant Physiology preview FUSCA3 was expressed by the native promoter and included a C-terminal 10x-c-myc tag. This transgene was able to rescue the fus3-3 mutant. A second transgene (35S:AGL15) was present to allow establishment of stable somatic embryo cultures. ChIP was performed using anti-c-myc antibody (Myc-Tag (9B11) Mouse mAb; Cell Signaling 2276S) on tissue expressing FUS3-myc (three biological replicates) and on three controls of tissue expressing FUS3 (no epitope tag).', '1132/NCBITaxon:3702', '1136/efo:EFO_0002762'
  UNION SELECT 323, 'E-GEOD-41986', 23, 'Immune and tumor cell gene expression from node positive and node negative breast cancers'' primary tumor, lymph nodes and peripheral blood', 'Analysis of purified immune and breast tumor cells from three major compartments where cancer and immune cells interact: primary tumor, tumor draining lymph nodes (tumor invaded or tumor free), and peripheral blood.  The results suggests that node-positive patients immune regulation and functionality is down-regulated compared to node-negative patients. CD45+ Immune and ESA+ tumor cells were purified from breast cancer patients'' primary tumor, tumor-draining lymph node, and peripheral blood (ficoll) and placed onto Agilent microarrays using the dye-swap method. A universal human reference was used as a reference for the patient samples.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 324, 'E-GEOD-41383', 23, 'Effect of infection with Aggregatibacter actinomycetemcomitans or Porphyromonas gingivalis on primary murine bone marrow-derived dendritic cells', 'To determine the biological mechanisms underlying a dampened immune response to Porphyromonas gingivalis, as compared to Aggregatibacter actinomycetemcomitans challenge, we infected primary BMDCs with either pathogen or left uninfected Total RNA from uninfected BMDCs compared to BMDCs infected with either  Aggregatibacter actinomycetemcomitans or Porphyromonas gingivalis', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 325, 'E-GEOD-39901', 23, 'Comprehensive analysis of silencing mutants reveals complex regulation of the Arabidopsis methylome.', 'Cytosine methylation is involved in various biological processes such as silencing of transposable elements (TEs) and imprinting.  Multiple pathways regulate DNA methylation in a sequence specific manner. What factors regulate DNA methylation at a given site in the genome largely remains elusive. Here by generating single nucleotide resolution maps of DNA methylation we have surveyed the effect of mutations in a comprehensive list of genes involved in gene silencing in Arabidopsis. We find that DNA methylation is site-specifically regulated by different factors. Furthermore, we have identified novel regulators of DNA methylation. We have generated a comprehensive resource to further understanding the control of DNA methylation patterning. Whole genome methylation maps of 86 silencing mutants involved in gene silencing and chromatin modifications were generated using BS-seq (Cokus et al., Nature 2008 ).', '1132/NCBITaxon:3702', '1136/efo:EFO_0002761'
  UNION SELECT 326, 'E-GEOD-39273', 23, 'Effect of transgenic human IL8 on gene expression in mouse colon cancer tumors', 'mRNA expression in  colon cancer tumores of CBA/C57BL/6 transgenic mice bearing a BAC which contains the human IL8 gene and promoter was compared to that of CBA/C57BL/6 controls. A. CBA/C57BL/6 mice controls (n=3) B. human IL8 transgenic CBA/C57BL/6 mice (n=3).', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 327, 'E-GEOD-36311', 23, 'High-resolution genomic profiling of marginal zone B-cell lymphomas and variants', 'We studied marginal zone B-cell lymphomas of the gastrointestinal tract including seven small cell MALT lymphomas, eight composite lymphomas and thirteen large cell variants using SNP-array profiling. We found an increase of genomic complexity with lymphoma progression, and could identify gains of prominent (proto)oncogenes REL, ETS1, PTPN1, PTEN and KRAS that were associated exclusively with the large cell presentation of MALT lymphoma. Losses of ADAM3A and SCAPER, as well as gains of SIRPB1 occur during progression from small cell to large cell lymphoma. In two case studies, we examined lymphoma progression comparing two different regions of the same tumor and material acquired at two different time points from another lymphoma. Our analyses reveal genomic heterogeneity in both cases, supporting the theory of an oligoclonal tumor evolution model for MALT lymphomas and its variants. Affymetrix SNP arrays were performed according to the manufacturer''s directions on DNA extracted from cryopreserved tumor samples. Copy number and loss of heterozygosity analysis of Affymetrix GenomeWide SNP 6.0 arrays was performed for 7 small cell MALT lymphomas, 8 large cell areas of Composite lymphomas and 13 large cell variants. Two independant hybridizations of one of the small cell MALT lymphoma and one of the large cell areas of Composite lymphoma Samples were performed.', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 328, 'E-GEOD-35309', 23, 'Gene expression signatures for Normal-looking, fibrotic foci and hyperplastic (adjacent to fibrotic foci) area in IPF lungs', 'To further understand the pathologic microenvironment in IPF, we have employed whole genome microarray expression profiling as a discovery platform to identify genes with the potential to distinguish normal and IPF lung in normal-looking, fibrotic foci and hyperplastic areas of IPF lung.   Four IPF lungs were dissected into normal-looking, fibrotic foci and hyperplastic areas by Laser-Capture-Microdissection. Gene expression analysis showed that 638 significantly different genes were identified that clearly distinguished the different IPF microenvironments . Among them, MMP19 was revealed as one of the most significantly up-regulated genes that distinguished normal looking epithelial cells (N) to hyperplastic epithelial cells, MMP19 up-regulation in IPF lungs was verified by immunohistochemical (IHC), qRT-PCR and Western-blot. IPF lungs are heterogeneity complex, which comprise normal looking area, fibrotic foci and hyperplastic area. In this study we separated the normal, fibrotic foci and hyperplastic area by LCM and employed Agilent whole genome gene expression microarray profiling to identify genes with the potential to distinguish the unique microenironment of IPF', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 329, 'E-GEOD-34994', 23, 'Gene expression signatures for MMP19 full length cDNA transfection in A549 cells.', 'To further understand the role of MMP19 in IPF, particularly in genotype and phenotype changes, we have employed whole genome microarray expression profiling as a discovery platform to identify the target genes of MMP19 in epithelial cells. Gene expression analysis showed that 8358 genes were differentially expressed at p-value 0.05 level and 748 genes were significantly changed with MMP19 expression, which covers matrix protien and cell proliferation, motality and phenotype changes (EMT). Among them, PTGS2 was revealed as one of the most significantly up-regulated genes. And PTGS2 up-regulation in IPF lungs was verified by immunohistochemical (IHC), qRT-PCR and Western-blot as well. A549 is a typical epithelial cell line from ATCC which are widely used in the lung cell biology studies in vitro. To define the role of MMP19 in epithelial cells, we employed Agilent whole genome gene expression microarray profiling to identify genotype alteration with overexpression of MMP19 to find the target genes of MMP19, further to explore the function of MMP19 in the unique microenironment of IPF', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 330, 'E-GEOD-34643', 23, 'Circulating microRNAs to predict neurological outcome after sudden cardiac arrest', 'Studies of miRNA profiling to predict outcome in cardiac arrest patients treated with therapeutic hypothermia (TH) Venous blood was collected in citrated tubes prior discharge. Plasma was harvested by centrifugation and stored at -80°C until assayed. Identical volumes of plasma from the 14 patients of a group (cerebral performance category (CPC): 1-2 or CPC: 3-5) were pooled to reach a final volume of 400µL for each group. The two pools were processed conjointly. Total RNA was extracted using miRVana isolation kit (Applied Biosystems), dephosphorylated and labeled using miRNA Complete Labeling and hybridization kit (Agilent). Hybridization was performed on miRNA Human Microarray Release 12.0 slides (Agilent). 4 arrays per pool were hybridized. Scanning was achieved with the Genepix 4000B Scanner (Molecular Devices, Sunnyvale, USA). Raw data were acquired with the Genepix Pro software (Molecular Devices).', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 331, 'E-GEOD-33341', 23, 'Gene Expression-Based Classifiers Identify Staphylococcus aureus Infection in Mice and Humans', 'Staphylococcus aureus causes a spectrum of human infection. Diagnostic delays and uncertainty lead to treatment delays and inappropriate antibiotic use. A growing literature suggests the hosts inflammatory response to the pathogen represents a potential tool to improve upon current diagnostics. The hypothesis of this study is that the host responds differently to S. aureus than to E. coli infection in a quantifiable way, providing a new diagnostic avenue. This study uses Bayesian sparse factor modeling and penalized binary regression to define peripheral blood gene-expression classifiers of murine and human S. aureus infection. The murine-derived classifier distinguished S. aureus infection from healthy controls and Escherichia coli-infected mice across a range of conditions (mouse and bacterial strain, time post infection) and was validated in outbred mice (AUC>0.97). A S. aureus classifier derived from a cohort of 95 human subjects distinguished S. aureus blood stream infection (BSI) from healthy subjects (AUC 0.99) and E. coli BSI (AUC 0.82). Murine and human responses to S. aureus infection share common biological pathways, allowing the murine model to classify S. aureus BSI in humans (AUC 0.84). Both murine and human S. aureus classifiers were validated in an independent human cohort (AUC 0.95 and 0.94, respectively).  The approach described here lends insight into the conserved and disparate pathways utilized by mice and humans in response to these infections.  Furthermore, this study advances our understanding of S. aureus infection; the host response to it; and identifies new diagnostic and therapeutic avenues. To create a host gene expression-based classifier for S. aureus infection, mice from a variety of experimental conditions were utilized.  Seven different strains of inbred mice (n=187 total) were challenged with four different S. aureus strains via intraperitoneal inoculation and sacrificed at various time points as described in Methods.  The comparator group for model derivation included 50 A/J or C57BL/6J mice inoculated with E. coli (O18:K1:H7) as well as 54 uninfected mice. Next, the murine S. aureus classifier was externally validated in outbred CD-1 mice with S. aureus infection (Sanger 476 or USA300), E. coli infection (O18:K1:H7), or uninfected controls (10 animals per condition).   Method: All experiments were performed on mice 6-8 weeks old.  For the murine S. aureus predictor, seven inbred mouse strains (3 mice/strain: 129S1/SvImJ, A/J, AKR/J, BALB/cByJ, C57BL/6J, C3H/HeJ, and NOD/LtJ) were IP inoculated with 107 CFU/g of S. aureus Sanger476, euthanized at 2h after injection, and bled. This was repeated using four different S. aureus strains (USA100, USA300, MW2, and Sanger476) in A/J mice (n=3 per S. aureus strain). For time series experiments, both A/J and C57BL/6J mouse strains were IP inoculated with S. aureus Sanger476 as above, and sacrificed at 2, 4, 6, and 12h after injection (n=5 per time point).', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 332, 'E-GEOD-32175', 23, 'Development of Transcriptomic Biomarker Signature in Human Saliva to Detect Lung Cancer', 'Affymetrix HG U133 Plus 2.0 Array (Affymetrix, Santa Clara, CA) was used to profile transcriptomes and discover altered gene expression in saliva supernatant. Salivary transcriptomic biomarker discovery was performed on 10 lung cancer patients and 10 matched controls. Seven messenger RNA biomarkers were discovered and pre-validated This study consisted of two phases, including a discovery phase, followed by a pre-validation phase. 10 lung cancer samples and 10 matched control samples were chosen for the biomarker discovery phase. The transscriptomic approach profiled the saliva supernatant samples from 10 lung cancer patients and 10 healthy control subjects using the Affymetrix HG U133 Plus 2.0 Array (Affymetrix, Santa Clara, CA). Biomarkers identified from the microarray study were first verified using the discovery sample set (10 lung cancer and 10 healthy control).', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 333, 'E-GEOD-31030', 23, 'Targeting microRNAs for asthma treatment', 'Profiling miRNA expression in cells that directly contribute to human disease pathogenesis is likely to aid the discovery of novel drug targets and biomarkers. However, tissue heterogeneity and the limited amount of human diseased tissue available for research purposes present fundamental difficulties that often constrain the scope and potential of such studies. We established a flow cytometry-based method for isolating pure populations of pathogenic T cells from bronchial biopsy samples of asthma patients, and optimized a high-throughput nano-scale qRT-PCR method capable of accurately measuring 96 miRNAs in as little as 100 cells. Comparison of circulating and airway T cells from healthy and asthmatic subjects revealed asthma-associated and tissue-specific miRNA expression patterns. These results establish the feasibility and utility of investigating miRNA expression in small populations of cells involved in asthma pathogenesis, and set a precedent for application of our nano-scale approach in other human diseases. We analyzed the concordance in results obtained by nano-scale qPCR and miRNA microarrays. RNA extracted from human Th2 cells was used for parallel profiling by both nano-scale PCR and microarray method. Fifty nanograms (ng) of RNA was used for the microarray method and cDNA from 1 ng (~1000 cell equivalent) of RNA, pre-amplified by 18 cycle PCR reaction, was used for miRNA detection by the nano-scale qPCR method (G.Seumois et al. in submission). Out of the 92 miRNAs assayed, 51 were detected by nano-scale qPCR. Of these, 45 were detected by microarray analysis, including the 32 miRNAs with the strongest signal intensities on the nano-scale qPCR platform. One sample; one color design. This is a single array from a larger normalized set of data investigating targeting microRNAs for asthma treatment that is to published separately.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 334, 'E-GEOD-29448', 23, 'Klebsiella pneumoniae: WT vs. Hfq mutant', 'Transcriptional profiling of K. pneumoniae comparing the wild type strain (S5) with a hfq deletion mutant (H0201). Goal was to determine the effects of Hfq absence on global K. pneumoniae gene expression. WT vs. Hfq mutant. Eight replicates for each ORF presented in the 44K array.', '1132/NCBITaxon:573', '1136/efo:EFO_0002768'
  UNION SELECT 335, 'E-GEOD-29365', 23, 'Whole genome scanning of lung tumors and mucosal biopsies on Affymetrix NspI-250K platform', 'Matched normal lymph node, tumors and corresponding mucosal biopsies from 4 lung cancer patients were scrutinized on Affymetrix 250K-NspI array platform. Frequently deleted regions genes were identified based on the LOH call as per manufacturer''s guidelines and normalization. Frequently deleted genes in the tumors and mucosa were identified though this approach Affymetrix 250K-NspI- SNP arrays were performed according to the manufacturer''s directions on DNA extracted from lung cancer and corresponding mucosal biopsy specimens. LOH analysis was done to identify genes that are frequently deleted.', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 336, 'E-GEOD-28731', 23, 'Effects of Hsp72 on astrocyte activation following stroke in the mouse', 'Astrogliosis is a hallmark of the response to brain ischemia, comprised of changes in gene expression and morphology. Hsp72 protects from cerebral ischemia, and although several mechanisms of protection have been investigated, effects on astrocyte activation are unknown.  To identify potential mechanisms of protection, gene expression was assessed in mice subjected to middle cerebral artery (MCAO) or sham surgery, of either wildtype (WT) or Hsp72-overexpressing (Hsp72Tg) mice. After stroke, both genotypes exhibited genes related to cell death, stress response, and immune response.  Furthermore, genes indicative of astrocyte activation, including cytoskeletal proteins and cytokines, were upregulated.  To measure astrocyte activation after stroke, detailed histological and morphological analyses were performed in the cortical penumbra after stroke using unbiased stereology.  Consistent with other reports, we observed a marked and persistent increase in glial fibrillary acidic protein (GFAP ) as soon as 3 hours after MCAO.  In contrast, vimentin immunoreactivity appeared 12-24 hours after stroke, peaked at 72 hours, and returned to baseline after 30 days.  Surprisingly, no change in overall astrocyte number was observed based on glutamine synthetase (GS) immunoreactivity.  To determine if Hsp72Tg mice exhibited altered astrocyte activation compared to WT controls, morphological evaluation by fractal analysis was used.  Overexpression of Hsp72 reduced astrocyte cell area, arbor area, and to a lesser extent fractal dimension, 72 hours following stroke.  In conclusion, in vivo overexpression of Hsp72 alters gene expression following stroke, including genes involved in astrocyte activation, and decreases astrocyte activation acutely following MCAO.  Thus, modulation of astrogliosis may be a neuroprotective mechanism exerted by Hsp72 after ischemia. A total of 10 samples were analyzed, with 5 of each genotype, wildtype (WT) and Hsp72-overexpressing (Hsp72Tg) mice.  Of the 5 in each group, 3 received middle cerebral artery occlusion (MCAO) and 2 received a sham surgery.  The sham samples serve as the controls for the MCAO samples in each genotype.  All samples were taken from the ischemic or control hemisphere 24 hours after surgery.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 337, 'E-GEOD-27302', 23, 'Expression data from mouse colon tissue response  to T cell transfer at week 0, 2, 4 and 6', 'Temporal geneome profiling of T cell transfer colitis model T cells critically regulate clinical inflammatory bowel diseases and T cell dependent experimental colitis models have gained prominent favor as useful models to identify potential pathogenic mechanisms. The naïve CD4+CD45Rbhigh cell transfer model into recombinase activating gene-1 deficient (RAG-/-) mice induces both colitis and small bowel inflammation reflecting Crohn''s disease with unclear pathogenic mechanisms. Given the pathological similarities between the T cell transfer model of colitis and clinical disease, we sought to identify changes in whole genome expression profiles over time during the delelopment of colitis. Male RAG-1-/- C57BL/6 mice were injected with a half million CD4+CD45RBhigh T cells from healthy wild type C57BL/6 male mice. The colon tissue from individual cohorts were collected at weeks 0, 2, 4 and 6. Total RNA were extracted from the colon tissue and detected by Affymetrix GeneChip Mouse Genome 430 2.0 Array.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 338, 'E-GEOD-25609', 23, 'Plasma microRNA profiling by microarray elucidates several candidate biomarkers for colorectal cancer screening', 'MicroRNAs (miRNAs) are small non-coding RNA molecules which function as negative gene regulators. The tissue expression profile of miRNAs shows great promise as a novel biomarker for diagnosis of cancer and other diseases. In addition, some recent reports have demonstrated that are present in human serum and plasma which could make them an ideal non-invasive biomarker for diagnosis of cancer. The aim of this study is to analyze the value/efficacy of the expression profile of plasma miRNAs in differentiating between patients with advanced adenomas and CRC and healthy individuals. The microRNA profiling study comprises serum plasmas from 20 Control, 21 colorectal cancer,20 advanced adenomas.The study also include some samples from patients after treatment.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 339, 'E-GEOD-13214', 23, 'Evaluation of gene expression profile in postmortem brain with Alzheimer´s disease-type neuropathological changes', 'Unravel the mechanisms underlying brain aging and Alzheimer´s disease (AD) has been difficult because of complexity of the networks that drive these aging-related changes. Analysis of the gene expression in the brain is a valuable tool to study the function of the brain under normal and pathological conditions. Gene microarray technology allows massively parallel analysis of most genes expressed in a tissue, and therefore is an important research tool that potentially can provide the investigative power needed to address the complexity of brain aging and neurodegenerative processes. One of the reasons that account for the resistance of AD pathogenesis to analysis is that clinically normal subjects may exhibit considerable AD pathology, blurring criteria for distinguishing subjects with normal aging or AD. Here, we analyzed hippocampal and cortex frontal gene expression from 32 subjects separated in individuals presenting, 1) both pathologic and clinical AD (definitive AD); 2) AD pathology and normal clinic (pathologic AD); 3) cognitive impairment, without AD pathology (others dementias); and 4) no cognitive impairment, without AD pathology (normal individuals). Our results show that based on gene expression profile these individuals we could verify similarity between the definitive AD group and the group that only had AD-type pathology (pathologic AD). Specimens of hippocampus and cortex frontal used in this study were obtained at autopsy from 32 subjects. Neuropathology, specifically NFT and NP, was assessed in accordance to the Braak staging and CERAD scores, respectively. Based on pathologic and clinic criteria, subjects were categorized into four groups: 1) nine subjects with AD neuropathologic (Braak = IV / V / VI and CERAD ? 0) presenting cognitive impairment (CDR ? 1), termed definitive AD (dAD); 2) five subjects with AD neuropathologic (Braak = IV / V / VI and CERAD ? 0) and without cognitive impairment (CDR = 0), termed pathologic AD (pAD); 3) nine subjects without AD neuropathologic (Braak = 0 / I / II and CERAD ? C) presenting cognitive impairment (CDR ? 1), termed other dementias (OD); 4) - nine subjects without AD neuropathologic (Braak = 0 / I / II and CERAD ? C) and without cognitive impairment (CDR = 0), termed normal (N).  RNA isolation and Amplification. From total RNA, a two-round linear amplification procedure (T7-based protocol) was carried out for all samples and for a pool of RNAs obtained from 15 distinct human cell lines used as reference. Labeled cDNA was generated in a reverse transcriptase reaction using amplified RNA (aRNA). Equal amounts of test and reference cDNA reverse color Cy-labeled aRNA targets were competitively hybridized against the cDNA probes in a customized cDNA platform with 4,608 ORESTES representing human genes. Dye-swap was performed for each sample as control for dye bias and used as replicate.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 340, 'E-MTAB-1436', 23, 'Transcription profling by array of liver tissues from an ex-vivo swine model of Acute Respiratory Distress Syndrome (ARDS) before and after endotoxin perfusion', 'Acute Respiratory Distress Syndrome (ARDS) is a very common clinical entity and a major cause of morbidity and mortality in the critical care setting. Historically, ARDS has been associated with mortality ranging from 40% to 60% . In the US alone,200,000 new cases of ARDS occur every year. ARDS can be associated with different clinical disorders affecting the lungs. A novel ex-vivo swine model developed by our group permits measurements of lung pathophysiology and simultaneous collection of lung and liver tissue for histologic and molecular comparisons during the early phase of the response to endotoxemia.  In this preparation, endotoxemia causes a liver-dependent inflammatory response and severe lung injury and dysfunction.  We chose swine, as an experimental animal because, like humans, they are especially sensitive to endotoxin and the pathophysiology of the response appears to be similar to that in humans. Gene expression was evaluated in swine liver before and after endotoxin treatment.', '1132/NCBITaxon:9823', '1136/efo:EFO_0002768'
  UNION SELECT 341, 'E-MTAB-1231', 23, 'MicroRNA profiling by array of human Centenarians to investigate the expression of microRNAs', 'microRNA analysis', '1132/NCBITaxon:9606', '1136/efo:EFO_0000753'
  UNION SELECT 342, 'E-MTAB-1217', 23, 'Transcription profiling by array of human olfactory neurosphere derived (ONS) cells to investigate a patient-specific olfactory stem cell disease model for ataxia-telangiectasia.', 'Nasal biopsies were collected from control and A-T patients prior to establishing olfactory neurosphere derived (ONS) cells. These cells were phenotyped using stem cell markers, a series of A-T cellular characteristics  determined and cells differentiated into neuronal cells.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 343, 'E-MTAB-744', 23, 'ChIP-Seq of human IMR-90 and MCF 10A cell lines by using H3K36me3 antibody', 'Chromatin immunoprecipitation analysis of  RNAP density and associated histone mark H3K36me3 at introns and exons.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002692'
  UNION SELECT 344, 'E-MTAB-743', 23, 'RNA-Seq of human MCF 10A cell line to investigate genome-wide mRNA expression', 'RNA sequencing analysis was performed to determine genome-wide mRNA expression in MCF10A cells', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 345, 'E-MTAB-742', 23, 'RNA-Seq of human H1, IMR-90 and MCF 10A cell lines', 'Global nuclear run-on sequencing analysis was performed to determine transcription elongation rates across the genome of three human cell lines.', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 346, 'E-MEXP-3800', 23, 'Transcription profiling by array of differential utilization of carbon sources in Escherichia coli based on previous nutritional conditions', 'Model topology is divided into two compartments, cell programming and performance testing. The cell programming compartment is split into history and pre-treatment. History ( History I or H1: E.coli grown for 18hrs in LB flask, transferred to fresh LB flask after that. History II or H2: E.coli  grown for 18hrs in LB flask, transferred to fresh LB flask for 45 min. From this flask, 0.1O.D./ml transferred to rich medium and grown for 4 hours. From this,0.1O.D./ml transferred to fresh rich medium. History III or H3: E.coli  grown for 18hrs in LB flask, transferred to fresh LB flask for 45 min. From this flask, 0.1O.D./ml transferred to starvation medium and grown for 4 hours. From this,0.1O.D./ml transferred to fresh starvation medium. Pre-treatment (Pre-treatment 1(T1):  2.5g glucose/litre  5mM NH4Cl. Pre-treatment 2 (T2): 2.5g glucose/litre 0.25mM NH4Cl. Pre-treatment 3 (T3): - 0.25g glucose/litre  5mmM NH4Cl. Pre-treatment 4 (T4): 0.25g glucose/litre 0.25mM NH4Cl). Each pre-treatment given for 2.5 hours.The culture nomenclature indicates the adaptive path followed, for example, H1T1 indicates the culture has encountered history I (H1) and then transferred to pre-treatment 1 (T1).RNA was extracted for selected combinations. Performance testing : Performance testing describes the type of analysis done which is the growth pattern study onto three substrates, glucose, succinate and pyruvate. This performance testing revealed specific history-pretreatment combinations to be better suited for growth on certain substrate and some not suited for growth. The samples were harvested for RNA isolation at peak growth points and named worst_glucose, best_glucose,worst_succinate, best_succinate, worst_pyruvate and best_pyruvate according to the growth shown after testing all 12 history-pretreatment combinations. The differences in physiology were studied in details using microarray analysis of 13 samples including 3 history samples, 4 pre-treatment samples and 6 samples at performance testing level. RNA extraction was done using Qiagen RNeasy minikit (Germany). Standard Affymetrix protocol was followed for hybridization on Affymetrix E. coli Genome 2.0 Array.', '1132/NCBITaxon:562', '1136/efo:EFO_0002768'
  UNION SELECT 347, 'E-GEOD-43410', 23, 'Effects of cocaine and withdrawal on the mouse nucleus accumbens transcriptome.', 'Genetic association studies, pharmacological investigations and analysis of mice-lacking individual genes have made it clear that Cocaine administration and Withdrawal have a profound impact on multiple neurotransmitter systems. The GABAergic medium spiny neurons of the nucleus accumbens (NAc) exhibit changes in the expression of genes encoding receptors for glutamate and in the signaling pathways triggered by dopamine binding to G-protein-coupled dopamine receptors. Deep sequence analysis provides a sensitive, quantitative and global analysis of the effects of Cocaine on the NAc transcriptome. RNA prepared from the NAc of adult male mice receiving daily injections of Saline or Cocaine, or Cocaine followed by a period of Withdrawal, was used for high-throughput sequence analysis. Changes were validated by quantitative polymerase chain reaction or Western blot. On the basis of pathway analysis, a preponderance of the genes affected by Cocaine and Withdrawal was involved in the cadherin, heterotrimeric G-protein and Wnt signaling pathways. Distinct subsets of cadherins and protocadherins exhibited a sustained increase or decrease in expression. Sustained down-regulation of several heterotrimeric G-protein ?- and ?-subunits was observed. In addition to altered expression of receptors for small molecule neurotransmitters, neuropeptides and endocannabinoids, changes in the expression of plasma membrane transporters and vesicular neurotransmitter transporters were also observed. The effects of chronic Cocaine and Withdrawal on the expression of genes essential to cholinergic, glutamatergic, GABAergic, peptidergic and endocannabinoid signaling are as profound as their effects on dopaminergic transmission. Simultaneous targeting of multiple Withdrawal-specific changes in gene expression may facilitate development of new therapeutic approaches that are better able to prevent relapse. High-throughput sequence analysis of RNA prepared from the nucleus accumbens of adult male mice receiving daily injections of Saline or Cocaine, or Cocaine followed by a period of Withdrawal. Nucleus accumbens libraries were sequenced in nine lanes (three technical replicates per sample) on an Illumina GAIIx using a 37-cycle paired-end sequencing protocol. Replicates were analyzed for intra-sample disparity and read data from all three lanes were then merged into one composite data file per sample; intra-sample coefficient of determination, R2 ? 0.98. The composite file was used for subsequent analyses.', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 348, 'E-GEOD-43377', 23, 'Mouse Diversity Array SNP Genotyping of B10.BALBChr16 and BALB.B10Chr16 Reciprocal Consomic Strain Mice', 'To identify novel mechanisms regulating allogeneic hematopoietic cell engraftment, we previously used a forward genetic approach and described identification, in mice, of the Bmgr5 bone marrow (BM) engraftment quantitative trait locus (QTL).  This QTL confers dominant and large allele effects for engraftment susceptibility.  It was localized to chromosome 16 by classical quantitative genetic techniques in a segregating backcross bred from susceptible BALB.K and resistant B10.BR mice.  We now report verification of the Bmgr5 QTL using reciprocal chromosome 16 consomic strains.  The BM engraftment phenotype in these consomic mice shows that Bmgr5 susceptibility alleles are not only sufficient but also indispensable for conferring permissiveness for allogeneic BM engraftment.  Using panels of congenic mice, we resolved the Bmgr5 QTL into two separate subloci, termed Bmgr5a and Bmgr5b, each conferring permissiveness for the engraftment phenotype and both fine mapped to an interval amenable to positional cloning.  Candidate Bmgr5 genes were then prioritized using whole exome DNA sequencing and microarray gene expression profiling.  Further studies are needed to elucidate the genetic interaction between Bmgr5a and Bmgr5b and identify causative genes and underlying gene variants.  This may lead to new approaches for overcoming the problem of graft rejection in clinical hematopoietic cell transplantation. B10.BALBChr16 and BALB.B10Chr16 consomic strain mice were constructed in our laboratory for validation of bone marrow engraftment and graft-vs-host diseaes QTLs.  The parental strains for consomic line construction were B10.BR (B10.BR-H2k H2-T18a/SgSnJJrep) and BALB.K (C.C3-H2k/LilMcdJ).  The JAX Mouse Diveristy 620K SNP Array was used to verify adequate removal of residual background heterozygosity.  Liver DNA was delivered to JAX Mouse Diversity Genotyping Array Service (Jackson Laboratory) for the SNP Array genotyping.', '1132/NCBITaxon:10090', '1136/efo:EFO_0000749'
  UNION SELECT 349, 'E-GEOD-43370', 23, 'Global transcriptome analysis of eukaryotic genes affected by Gromwell extract', 'BACKGROUND: Lithospermum erythrorhizon, popularly known as Gromwell, is a perennial herb belonging to the family of Boraginaceae and its extract is known to have diverse pharmacological, cosmetic, and nutritional values for human. Nevertheless the biological influence of Gromwell extract on general physiology of eukaryotic cells remains unknown. In this study, we for the first time performed a global transcriptome analysis by using DNA microarray analysis to identify genes affected by the addition of Gromwell extract (GE) by using a eukaryotic microorganism, basidiomycete fungus Cryptococcus neoformans, as a model system. RESULTS: The transcriptome analysis revealed that a total of 1932 genes are differentially regulated in a statistically significant manner and expressions of 715 genes are up- (315 genes) or down-regulated (457 genes) more than twofold upon GE treatment. In response to GE treatment, genes involved in signal transduction genes are immediately regulated and the evolutionarily conserved set of genes involved in the core cellular functions, including DNA replication, RNA transcription/processing, and protein translation/processing, are generally upregulated. In contrast, a number of genes involved in carbohydrate metabolism and transport, inorganic ion transport and metabolism, post-translational modification/protein turnover/chaperone functions, and signal transduction are downregulated by the GE treatment. Among the GE-responsive genes that are also evolutionarily conserved in the human genome, we confirmed expression patterns of YSA1, TPO2, FET5 (CFO1), and PZF1 by Northern blot analysis. Based on the functional characterization of some GE-responsive genes through phenotypic analysis of gene knockout mutants, we found that GE treatment may promote cellular tolerance against a variety of environmental stresses in eukaryotes. CONCLUSIONS: A significant portion of the Cryptococcus genome is affected in expression levels by treatment of GE, implying that the general physiology of eukaryotic cells is significantly affected by the GE treatment. There are more than 95% of genome homology between JEC21 and H99. Therefore 6 slides of JEC21 (Cryptococcus neoformans var. neoformans serotype D) 70-mer oligo are used in this analysis, 3 biological replicate experiments are performed, total RNAs are extracted from H99 (H99 Wild type strain (Cryptococcus neoformans var. grubii serotype A)) with 6.4 mg/ml Gromwell extract in 0, 10, 30, 60 min. We use the mixed all of total RNAs from this experiment as a control RNA. We use Cy3 as Sample dye and Cy5 as a control dye.', '1132/NCBITaxon:235443', '1136/efo:EFO_0002768'
  UNION SELECT 350, 'E-GEOD-43193', 23, 'dsGFP as exogenous control on RNAi assays and its effects on gene expression in Apis mellifera', 'Double stranded RNAs are used to induce gene silencing in functional studies. In Apis mellifera green fluorescent protein (GFP) dsRNA (dsGFP) has been used as an exogenous control as its sequence has no homology in honeybee genome. However, some undesirable effects are observed after dsGFP treatment. A microarray approach comparing gene expression differences between untreated and dsGFP treated groups, containing honeybees workers in two different developmental stages, pre-pupae and light-brown eyed pupae, were used to test the reliability of dsGFP as a control for RNAi experiments. According to these microarrays results dsGFP can be used as a control in RNAi assays as long as the affected genes are taken into account in the analysis. Analysis used loop-design to compare dsGFP treated and noon-treated workers in pre-pupae and light-brown-eyed pupae stages.', '1132/NCBITaxon:7460', '1136/efo:EFO_0002768'
  UNION SELECT 351, 'E-GEOD-42882', 23, 'DNA methylation in the malignant transformation of meningiomas', 'Genome wide DNA methylation profiling of malignant, atypical and benign meningiomas samples. The Illumina infinium HumanMethylation450 beadchip kit was used, which assayed 450,000 potential methylation sites. Bisulphite converted DNA from the 23 samples were hybridised to the Illumina Infinium 450 Human Methylation Beadchip.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002759'
  UNION SELECT 352, 'E-GEOD-41004', 23, 'Effects of dsRNA-GFP as exogenous control on honey bee RNAi assays: gene expression changes in workers', 'A green fluorescence protein (GFP)-derived dsRNA (dsRNA-GFP) has been used as an exogenous control for Apis mellifera RNAi assays by multiple research groups. Its sequence does not share any significant homology with any known honey bee genes. Although dsRNA-GFP is not expected to trigger an RNAi response in treated bees, undesirable effects on gene expression, pupal pigmentation or developmental timing have been routinely observed. To better understand the multiple molecular and phenotypic effects of dsRNA-GFP in honey bees and to evaluate its use as a control for RNAi studies, we examined the impact of dsRNA-GFP on global gene expression patterns in developing workers. We found that dsRNA-GFP causes large-scale changes in gene expression associated with multiple biological processes. Furthermore, dsRNA-GFP exposure tended to preferentially decrease, rather than increase, expression of genes compared to controls. Gene expression differences were analyzed using a dye-swap design. Second instar worker larvae received 0.5 ?g of dsRNA-GFP. Non-treated and dsRNA-GFP treated 7 day-old workers (W7d) were sampled.', '1132/NCBITaxon:7460', '1136/efo:EFO_0002768'
  UNION SELECT 353, 'E-GEOD-40722', 23, '5''-Serial Analysis of Gene Expression of the transcriptomes of Coprinopsis cinerea during fruiting body initiation', 'Large scale assessment of the transcriptomes of the vegetative mycelium and primordium will facilitate the generation of a more comprehensive picture of the fruiting process in basidiomycetes. We coupled 5''-Serial Analysis of Gene Expression (5''-SAGE) to high-throughput pyrosequencing from 454 Life Sciences to analyze the transcriptomes and identify up-regulated genes (URGs) among vegetative mycelium (Myc) and stage 1 primordium (S1-Pri) of Coprinopsis  cinerea during fruiting body development. We evaluated the expression of >3,000 genes in the two respective growth stages and demonstrated that almost one-third of these genes were preferentially expressed in either stage, which implicated a significant transcriptomic switch during fruiting body initiation. We annotated >34,000 and >45,000 transcription start sites (TSSs) in the transcriptomes of Myc and S1-Pri respectively. We identified a wealth of potential URGs related to early fruiting events, although their functions and roles are not exactly known. This study serves to advance our understanding of the molecular mechanisms of fruiting body development in the model mushroom C. cinerea. 5''-SAGE analysis of the transcriptomes of vegetative mycelium and primordium of C. cinerea by 454 GS20 high-throughput pyrosequencer', '1132/NCBITaxon:5346', '1136/efo:EFO_0003738'
  UNION SELECT 354, 'E-GEOD-40583', 23, 'FAT1 knockdown in immortalized astrocyte and glioma cell lines', 'The FAT1 gene was knocked down using 2 independent siRNAs, in immortalized human astrocytes and U87 and U251 glioma cell lines.  A non-targeted scramble siRNA was used as a control. Scramble, FAT1 siRNA#1 and FAT1 siRNA#2 were transfected into each cell line, in duplicate, and RNA analyzed using the Affymetrix U133A 2.0 platform.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 355, 'E-GEOD-40366', 23, 'Transcriptional analysis of PBMCs reveals sex specific changes in the aging of the immune system', 'Aging and sex have a strong influence on the functional capacity of the immune system. In general, the immune response in females is stronger than that in males, but there is little information about the effect of aging on this difference. To address this question, we performed a transcriptomic analysis of peripheral blood mononuclear cells derived from nonagenarians and young controls. We found 337 and 269 genes to be differentially expressed (p<0.05, fold change >1.5 or <-1.5) in nonagenarian females and males, respectively; 177 of these were changed in both sexes. An analysis of the affected signaling pathways revealed a clear sex bias: the number of significantly changed pathways was 43 in females and 40 in males; 23 were shared. These data show that the effects of aging on the immune system are significantly different in males and females. Our study population consisted of 146 nonagenarians (103 females, 43 males) and 30 young controls (19-30 years of age, 21 females, 9 males). In our study, we analyzed the gene expression difference between nonagenarian and control women as well as between nonagenarian and control males and then compared these results.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 356, 'E-GEOD-40110', 23, 'Saccharomyces cerevisiae 3'' poly(A) site mapping', 'The use of alternative polyadenylation sites is common and affects the post-transcriptional fate of mRNA, including its stability, localization, and translation. Here we present a method for genome-wide and strand-specific mapping of poly(A) sites and quantification of RNA levels at unprecedented efficiency by using an on-cluster dark T-fill procedure on the Illumina sequencing platform. Our method outperforms former protocols in quality and throughput, and reveals new insights into polyadenylation in Saccharomyces cerevisiae. Experimental benchmark of five different protocols (3tfill, bpmI, internal, rnaseq and yoon) for genome-wide identification of polyadenylation sites in Saccharomyces cerevisiae and transcript quantification. RNA was extracted from WT cells grown in glucose (ypd) or galactose (ypgal) as carbon source. The same RNA was used for 3 independent library constructions (technical replicates, rep).', '1132/NCBITaxon:4932', '1136/efo:EFO_0003738'
  UNION SELECT 357, 'E-GEOD-40031', 23, 'Retrograde bilin signaling enables Chlamydomonas greening and phototrophic survival', 'Here we show that the phytochrome-less chlorophyte Chlamydomonas reinhardtii retains a functional pathway to synthesize the linear tetrapyrrole (bilin) precursor of the phytochrome chromophore. Reverse genetic, metabolic inactivation and bilin rescue experiments establish that this pathway is needed for heme iron acquisition and for the diurnal transition to phototrophic growth. RNA-Seq measurements reveal a bilin-dependent signaling network that is necessary for the heterotrophic to phototrophic transition. These results imply the presence of a novel bilin sensor pathway that may be widely distributed amongst oxygenic photosynthetic organisms. We isolated RNA from heterotrophic suspension cultures of 4A+ WT and the hmox1 mutant grown in the presence or absence of 0.1 mM BV IX? before and after transfer to low light.', '1132/NCBITaxon:3055', '1136/efo:EFO_0003738'
  UNION SELECT 358, 'E-GEOD-39976', 23, 'Determination of molecular markers for BRCA1 and BRCA2 heterozygosity using gene expression profiling', 'Approximately 5% of all breast cancers can be attributed to an inherited mutation in one of two cancer susceptibility genes, BRCA1 and BRCA2. We searched for genes that have the potential to distinguish healthy BRCA1 and BRCA2 mutation carriers from non-carriers based on differences in expression profiling. Using expression microarrays we compared gene expression of irradiated lymphocytes from BRCA1 and BRCA2 mutation carriers versus control non-carriers. We identified 137 probe sets in BRCA1 carriers and 1345 in BRCA2 carriers with differential gene expression. Gene Ontology analysis revealed that most of these genes relate to regulation pathways of DNA repair processes, cell cycle regulation and apoptosis. Real-time PCR was performed on the 36 genes which were most prominently differentially expressed in the microarray assay; 21 genes were shown to be significantly differentially expressed in BRCA1 or BRCA2 mutation carriers  as compared to controls (p<0.05). Based on a validation study with 40 mutation carriers and 17 non-carriers, a multiplex model that included six or more coincidental genes of 18 selected genes was constructed in order to predict the risk of carrying a mutation. The results using this model showed sensitivity 95% and specificity 88%.   In summary, our study provides insight into the biological effect of heterozygous mutations in BRCA1 and BRCA2 genes in response to ionizing irradiation induced DNA damage. We also suggest a set of 18 genes that can be used as a prediction and screening tool for BRCA1 or BRCA2 mutational carriers by using easily obtained lymphocytes. Using expression microarrays we compared gene expression of irradiated lymphocytes from BRCA1 and BRCA2 mutation carriers versus control non-carriers Fresh blood samples were obtained from 9 BRCA1 and  8 BRCA2 mutation carriers and  9 mutation-negative women. Lymphocytes were collected from fresh blood samples, and RNA was extracted one hour after ?-irradiation', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 359, 'E-GEOD-39889', 23, 'Neutrophil gene expression in response to Mycobacterium abscessus', 'We used microarrays to describe the gene expression in response to two isogenic morphotypes of M. abscessus (Mab) and to Staphylococcus aureus. Human neutrophils from healthy donors were exposed to rough Mab (ATCC 19977T), smooth Mab (ATCC 19977T) and S. aureus (CF clinical strain) for two hours; control cells were exposed to saline.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 360, 'E-GEOD-39793', 23, 'Copy number profiling of 92 human lung tumors on Affymetrix 100K SNP arrays', 'Copy number profiling of 92 human lung tumors on Affymetrix 100K SNP arrays was conducted in order to assess the interaction of common genomic alterations with response to targeted anti-cancer therapeutics. Class 1 phosphatidylinositol 3'' kinase (PI3K) plays a major role in cell proliferation and survival in a wide variety of human cancers. Here we investigate biomarker strategies for PI3K pathway inhibitors in non-small-cell lung cancer (NSCLC).   Molecular profiling of NSCLC tumor samples showed that copy number gains in PIK3CA and total loss of PTEN protein were common in squamous cell carcinoma samples, whereas LKB1 loss and mutations in KRAS and EGFR were common in adenocarcinomas. A panel of NSCLC cell lines characterized for alterations in the PI3K pathway was screened with PI3K and dual PI3K/mTOR inhibitors to assess the preclinical predictive value of candidate biomarkers.   Cell lines harboring pathway alterations (RTK activation, PI3K mutation or amplification, PTEN loss) were exquisitely sensitive to the PI3K inhibitor GDC-0941. A dual PI3K/mTOR inhibitor had broader activity across the cell line panel and in tumor xenografts. The combination of GDC-0941 with paclitaxel, erlotinib, or a MEK inhibitor had greater effects on cell viability than PI3K inhibition alone.    CONCLUSIONS: Candidate biomarkers for PI3K inhibitors have predictive value in preclinical models and show histology-specific alterations in primary tumors, suggesting that distinct biomarker strategies may be required in squamous compared with non-squamous NSCLC patient populations. Lung tumors were profiled on Affymetrix GeneChip Mapping 100K Set Arrays Tumor samples were profiled for copy number without any treatment of the tumor.', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 361, 'E-GEOD-38972', 23, 'Wee1 Inhibition sensitizes prostate cancer cells to Hsp90 inhibitor to activate the intrinsic apoptotic pathway', 'Heat shock protein 90 (Hsp90) is an essential evolutionarily conserved molecular chaperone in eukaryotes. Cancer cells rely on Hsp90 to chaperone activated oncoproteins, and its involvement in numerous signaling pathways makes it an attractive target for drug development. Surprisingly, however, the impact of Hsp90 inhibitors on cancer cells is most commonly cytostatic, and efforts to enhance the anti-tumor activity of Hsp90 inhibitors in the clinic remain a significant challenge. In this study, we show that dual inhibition of Wee1 tyrosine kinase and Hsp90 causes prostate cancer cells to undergo apoptosis. Gene-expression profiling revealed that induction of the intrinsic apoptotic pathway by this drug combination coincided with transcriptional down-regulation of Survivin and Wee1, an outcome not seen in cells treated separately with either agent. At the translational level, expression of these two proteins as well as activated Akt was completely abrogated. Similar results were obtained in prostate cancer xenografts. These data establish a novel therapeutic strategy to enhance the efficacy of Hsp90 inhibitors in prostate cancer, and they provide a mechanistic rationale for stimulating the pro-apoptotic activity of Hsp90 inhibitors. In order to explore the mechanism underlying the enhanced cell death caused by Wee1 inhibitorII and 17-AAG combination, we performed microarray analysis using PC3 cells treated with Wee1 inhibitorII alone, 17-AAG alone, or the two drugs in combination. There are 12 samples in total. There are three experimental replicate. Samples 1, 5 and 9 are control (C)  (untreated PC3- prostate cancer cells). Samples 2, 6, and 10 are cells treated with Wee1 inhibitor II (W). Samples 3, 7, and 11 are treated with 17-AAG (A), (an Hsp90 inhibitor). Samples 4, 8, and 12 are treated with both Wee1 inhibitorII and 17-AAG (WA). Samples 5 was removed from our analysis due to weak signal.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 362, 'E-GEOD-38168', 23, 'Identification of transcripts predominantly expressed in cultured chicken primordial germ cells and embryonic fibroblasts.', 'The present investigation was to identify the signaling and metabolic pathways of expressed genes by microarray comparison between Primordial Germ Cells (PGCs) and their somatic counterpart, chicken embryonic fibroblasts (CEFs). We identified a total of 2,605 expressed transcripts. Among these, 1,197 were predominantly expressed in PGCs, and 1,408 were predominantly expressed in CEFs. Cell culture and microarray data generation were performed in triplicate. Blood PGCs from E2.5 embryos (N=20 for each replication) and CEFs from E6.5 embryos (N=5 for each replication) were cultured in appropriate culture medium. Total RNA was extracted from cultured PGCs (1,500,000 cells for each replication) and cultured CEFs (6,000,000 cells for each replication) with a Qiagen RNeasy kit. About 5 µg of total RNA from each replication was used for labeling. Probe synthesis from total RNA samples, hybridization, detection, and scanning were performed according to standard protocols from Affymetrix.', '1132/NCBITaxon:9031', '1136/efo:EFO_0002768'
  UNION SELECT 363, 'E-GEOD-36893', 23, 'Redox regulation of auxin and strigolactone signaling in the control of Arabidopsis thaliana root growth', 'Reduced glutathione (GSH) is required for cell cycle initiation and auxin-regulated root meristem development. Transcriptome profiling of the roots and shoots of the root meristemless 1 (rml1) mutant, which has about 3% of the wild type GSH, revealed a divergent auxin and strigolactone response linked to the arrest of the cell cycle. Plants of the rml1 mutant and Columbia-0 ecotype were harvested and separated into roots and shoots, then RNA extraction and Affymetrix Agronomics Tiling Array were performed.', '1132/NCBITaxon:3702', '1136/efo:EFO_0002768'
  UNION SELECT 364, 'E-GEOD-36808', 23, 'Transcriptoma comparison of mantle cell lymphoma of c-myc-3''RR/Cdk4R24C mice and c-myc-3''RR/p53+/- mice', 'Mantle cell lymphoma (MCL) is a B cell malignancy characterized by a monoclonal proliferation of lymphocytes with co-expression of CD5, CD43 but not CD23. We have developed two murine models of MCL-like lymphoma.  Breeding Cdk4R24C mice (a knock-in strain that express a Cdk4 protein resistant to inhibition by p16INK4a and other INK4 family members) with c-myc-3RR transgenic mice (prone to develop aggressive Burkitt lymphoma-like lymphoma) leads in c-myc/Cdk4R24C mice to development of clonal blastoid MCL-like lymphoma. Breeding p53+/- mice with c-myc-3RR transgenic mice lead to the development of several mature B cell lymphomas including MCL. In this study we compare MCL transcriptomas of c-myc-3''RR/Cdk4R24C mice and c-myc-3''RR/p53+/- mice. B splenocytes from 2 c-myc/Cdk4R24C lymphoma mice and 2  c-myc-3''RR/p53+/- mice were investigated', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 365, 'E-GEOD-35843', 23, 'Local response of rice roots upon root knot and migratory nematode infection revealed by massive parallel sequencing ', 'In this study  a comparison was made between the local transcriptional changes at two time points upon root knot (Meloidogyne graminicola) and migratory nematode (Hirschmanniella oryzae) infection in rice. Using mRNA-Seq we have characterized specific and general responses of the root challenged with these endoparastic root nematodes with very different modes of action. Root knot nematodes induce major developmental reprogramming of the root tip, where they force the cortical cells to form multinucleate giant cells, resulting in gall-development. Our results show that root knot nematodes force the plant to produce and transfer nutrients, like sugars and amino acids, to this tissue. Migratory nematodes, on the other hand, induce the expression of proteins involved in plant death and oxidative stress, and obstruct the normal metabolic activity of the root. While migratory nematode infection also causes upregulation of biotic stress-related genes early in the infection, the root knot nematodes seem to actively suppress the local defence  of the plant root. This is exemplified by a downregulation of genes involved in the salicylic acid and ethylene pathways. Interestingly, hormone pathways usually involved in plant development, were strongly induced (auxin and gibberellin) or repressed (cytokinin) in the galls. In addition, thousands of novel transcriptionally active regions as well as highly expressed nematode transcripts were detected in the infected root tissues. These results uncover previously unrecognized nematode-specific expression profiles and provide an interesting starting point to study the physiological function of many yet unannotated transcripts potentially targeted by these nematodes. 2 or 3 biological replicates of nematode infected roots and root tips and their respective controls were sampled at two time points (1 biological replicate contains pooled tissue from 6 plants)', '1132/NCBITaxon:39947', '1136/efo:EFO_0003738'
  UNION SELECT 366, 'E-GEOD-35179', 23, 'Gene expression profiles of monocytes derived from chronic lymphocytic leukemia patients and normal controls', 'We collected monocytes from peripheral blood of 5 chronic lymphocytic leukemia (CLL) patients and 5 healthy donors and we performed gene expression analysis by microarray. Comparison of gene expression profiles (GEPs) between CLL-derived and normal monocytes was used to discover molecular abnormalities in this nonmalignant immune cellular population in leukemia-bearing patients. Although analysed cells were not part of the malignant clone, in unsupervised hierarchical clustering analysis, GEPs of normal monocytes were clearly distinguishable from those of monocytes obtained from CLL patients. Supervised analysis identified 65 genes significantly up-regulated and 48 genes down-regulated in CLL monocytes compared with monocytes from normal controls (FC=2, p<0.05). Modification of gene expression profile would imply impairment of phagocytosis and production of immunosuppressive mediators in CLL-derived monocytes. The alterations described in our study further contribute to characterize the complexity of factors potentially involved in acquired immune deficiency of CLL patients. Large-scale gene expression profiling (GEP) was performed on total RNA extracted from purified CD14+ monocytes (RNeasy Mini kit Plus, QIAGEN, Valencia, CA, USA) isolated from 5 individual CLL patients and 5 healthy controls by hybridization on 4X44K Whole Human Genome Microarray (Agilent Technologies, Palo alto, CA). Fluorescence data were analysed with Feature Extraction Software v.10.5 (Agilent Technologies) an QC Chart tool v.1.3. Agglomerative two-dimensional clustering analysis and supervised analyses based on t-test were performed using Gene Spring GX (Agilent) software. Genes were defined as differentially expressed between groups at a significant level of p<0.05 and with a fold change cut off ± 2 in all the pair wise comparisons.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 367, 'E-GEOD-35051', 23, 'Imatinib/Rapamycin Gradient Encoding, mRNA abundance', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 368, 'E-GEOD-35040', 23, 'Imatinib/Rapamycin mRNA abundance', 'Measurement of mRNA abundance from the following cells lines (red) versus universal mouse reference RNA (green). Wildtype v-Abl transformed pre-B cells were treated for 12 hours with 2.5 uM imatinib mesylate, 10ng/mL rapamycin or nothing. Compound Based Treatment: wildtype v-Abl transformed pre-B cells were treated with imatinib mesylate (IMA), rapamycin (RAP) or nothing (NONE) compound_treatment_design', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 369, 'E-GEOD-35039', 23, 'Imatinib/Rapamycin Gradient Encoding', 'The purpose of this study was to compare changes in translation (using Gradient Encoding, described below) to changes in mRNA abundance. Lysates of wildtype v-Abl transformed pre-B cells harvested before and after 12 hours of treatment either with 2.5 uM imatinib, a v-Abl kinase inhibitor, or 10ng/mL (10.9 nM) rapamycin, an mTOR inhibitor, were fractionated by sedimentation through linear sucrose gradients.  Gradient fractions were encoded such that the mRNA from successive fractions was labeled with increasing ratios of Cy5 to Cy3.  mRNAs derived from fractions in the lighter portion of the gradient therefore have a lower Cy5 to Cy3 ratio, whereas those deeper in the gradient have a higher Cy5 to Cy3 ratio.  The ratio of Cy5 to Cy3 for each mRNA therefore reflects its average position within the gradient. We thus encoded the sedimentation rate of each mRNA across the entire gradient.  The resulting ratios were quantitatively measured for each mRNA species by hybridization to DNA microarrays, and related to the 260 nm absorbance peaks representing different numbers of ribosomes bound per mRNA Compound Based Treatment: wildtype v-Abl transformed pre-B cells were treated with imatinib mesylate (IMA), rapamycin (RAP) or nothing (NONE) compound_treatment_design', '1132/NCBITaxon:10090', ''
  UNION SELECT 370, 'E-GEOD-35000', 23, 'Interaction effects of knockdown of PML and TNFalpha treatment in HUVEC cells', 'To study the interaction effects between promyelocytic leukemia (PML) gene knockdown and tumor necrosis factor-alpha (TNFalpha) signaling in human umbilical vein endothelial cells (HUVECs), we transfected control or two independent PML siRNAs into HUVEC cells without or with 20 ng/mL TNFalpha treatment for 20 h. The total RNA was extracted for gene expression microarray analyses. The experiment is a 3x2 two-factor design. One factor is siRNA and it has three levels (siCtrl, siPML1, siPML2). siCtrl is RISC-inducing non-targeting control siRNA. siPML1 and siPML2 designate two independent siRNAs against the PML gene. Two independent siRNAs were used to eliminate the off-target effects of siRNA. The other factor is TNFalpha treatment (20 ng/mL for 20 h) and it has two levels: Untreated (U) or Treated (T). Each sample has technical duplicates.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 371, 'E-GEOD-34585', 23, 'Three acyltransferases and a nitrogen responsive regulator are implicated in nitrogen starvation-induced triacylglycerol accumulation in Chlamydomonas', 'Three acyltransferases, DGAT1, DGTT1, and PDAT1, are induced by nitrogen starvation and are likely to have a role in TAG accumulation based on their patterns of expression. Each gene also shows increased mRNA abundance in other TAG-accumulating conditions (-S, -P, -Zn, -Fe). Insertional mutants, pdat1-1 and pdat1-2, accumulate 25% less TAG compared to the parent strain, D66+, which demonstrates the relevance of the trans-acylation pathway in Chlamydomonas. The biochemical functions of DGTT1 and PDAT1 were validated by rescue of oleic acid sensitivity and restoration of TAG accumulation in a yeast strain lacking all acyltransferase activity.  Time course analysis suggest than an SBP domain transcription factor protein, whose mRNA increases precede that of other genes like DGAT1, is a candidate regulator of the N-deficiency response. An insertional mutant, nrr1-1, accumulates only 50% of the TAG compared to the parental strain in N-starvation conditions and is unaffected by other nutrient stresses, suggesting the operation of multiple signaling pathways leading to stress-induced TAG accumulation. Sampling of Chlamydomonas 2137 cultivated in TAP, N-Free TAP or TAP with varying amounts of N.', '1132/NCBITaxon:3055', '1136/efo:EFO_0003738'
  UNION SELECT 372, 'E-GEOD-33958', 23, 'Pig Longissimus muscle in Basque and Large White pigs reared in 3 different rearing systems', 'Transcriptional profiling of pig skeletal muscle in 2 breeds [(Large white , LW, conventional) and (Basque, B, local, indigeneous)] and 3 rearing systems [(Conventional, C), (Alternative, A) and (Extensive, E)]. 5 groups (BC,BA,BE,LWC and LWA) of 10 samples (i.e. biological replicates).  One replicate per array.', '1132/NCBITaxon:9823', '1136/efo:EFO_0002768'
  UNION SELECT 373, 'E-GEOD-33957', 23, 'Gene expression in Longissimus lumborum and Semimembranosus skeletal muscles in pig using the GenmascqChip (custom 15K Agilent microarray)', 'Transcriptional profiling between two different skeletal muscles (Longissimus and Semimembranosus) in pig The 90 pigs (females and barrows) used in gene expression analyses are part of a larger F2 population of 1,000 animals, setup within the framework of a QTL detection program. These animals were produced as a second generation intercross between two commercial sire lines (F016, Pietrain type line, and FH019, synthetic line from Hampshire, Duroc and Large-White founders, FRANCE-HYBRIDES, St Jean de Braye, France).', '1132/NCBITaxon:9823', '1136/efo:EFO_0002768'
  UNION SELECT 374, 'E-GEOD-26600', 23, 'Cycad Genotoxin Methylazoxymethanol (MAM) Modulates Cellular Pathways Involved in Cancer and Neurodegenerative Disease', 'Methylazoxymethanol (MAM), the genotoxic metabolite of the cycad azoxyglucoside cycasin, induces genetic alterations in bacteria, yeast, plants, insects and mammalian cells, but adult nerve cells are thought to be unaffected. We show that the brains of young adult mice treated with a single systemic dose of MAM display DNA damage (O6-methylguanine lesions) that peaks at 48 hours and decline to near-normal levels at 7 days post-treatment.  By contrast, at this time, MAM-treated mice lacking the gene encoding the DNA repair enzyme O6-methylguanine DNA methyltransferase (MGMT), showed persistent O6-methylguanine DNA damage. The DNA damage was linked to cell-signaling pathways that are perturbed in cancer and neurodegenerative disease.  These data are consistent with the established carcinogenic and developmental neurotoxic properties of MAM in rodents, and they support the proposal that cancer and neurodegeneration share common signal transduction pathways. They also strengthen the hypothesis that early life exposure to the MAM glucoside cycasin has an etiological association with a declining, prototypical neurodegenerative disease seen in Guam, Japan, and New Guinea populations that formerly used the neurotoxic cycad plant for medicine and/or food.  Exposure to environmental genotoxins may have relevance to the etiology of related tauopathies, notably, Alzheimers disease, as well as cancer. Mouse brains from eleven-week-old male C57BL6 wild-type and Mgmt-/- mice were exposed to either MAM or Vehicle for 6 hr, 24 hr, 48 hr, 168 hr. Five or six mice per group for a total of 94 mice.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 375, 'E-MTAB-1428', 23, 'Transcription profiling by array of 550 children from families recruited through a proband with atopic dermatitis', 'We have created a global map of the effects of polymorphism on gene expression in 550 children from families recruited through a proband with atopic dermatitis. We genotyped >300,000 SNPs and identified expression quantitative trait loci from measurements of 16,487 probes representing 15,576 genes in Epstein-Barr virus¨Ctransformed lymphoblastoid cell lines. We executed genome-wide association scans for these traits.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 376, 'E-MTAB-1425', 23, 'Transcription profiling by array of human lymphoblastoid cell lines derived from 400 children from families recruited through a proband with asthma', 'We have created a global map of the effects of polymorphism on gene expression in 400 children from families recruited through a proband with asthma. We genotyped 408,273 SNPs and identified expression quantitative trait loci from measurements of 54,675 transcripts representing 20,599 genes in Epstein-Barr virus¡§Ctransformed lymphoblastoid cell lines. We executed genome-wide association scans for these traits.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 377, 'E-MEXP-3509', 23, 'Transcription profiling by array of two genotypes of wheat to study their response to Fusarium head blight', 'Fusarium Head Blight (FHB) is a disease of wheat and other cereal crops, where, among other species, Fusarium graminearum infects the wheat inflorescence. Microarrays were used to observe differential gene expression in FHB-challenged spikes of the two European winter wheat genotypes Dream (moderately resistant) and Lynx (susceptible). Plants were either inoculated with the Fusarium graminearum strain IFA 65 (IFA Tulln) (500 macroconidia/floret) or were as control plants mock treated with desalted water. The inocula were injected into four spikelets at early anthesis and spikelets were later on collected at 32 and 72 h after inoculation. Four plants were sampled per genotype/treatment/sampling date. Total RNA was extracted from collected spikelets, and microarray analysis was performed using the Affymetrix Wheat GeneChip.', '1132/NCBITaxon:4565', '1136/efo:EFO_0002768'
  UNION SELECT 378, 'E-GEOD-43373', 23, 'Molecular and cellular response profiles induced by the TLR4 agonist-based adjuvant Glucopyranosyl Lipid A', 'Toll-like receptor (TLR)4 agonists are known potent immunostimulatory compounds. These compounds can be formulated as part of novel adjuvants to enhance vaccine medicated immune responses. However, the contribution of the formulation to the innate in vivo activity of TLR4 agonist compounds is not well understood. Molecular expression profiles and cellular responses in a mouse model were used to compare for GLA-SE, GLA, Alum, SE, PBS. This study was to evaluate the mechanism of actions of different adjuvants and to identify potential biomarkers for clinical evaluation. Muscle, lymphnode, and blood tissues from Balb/c mice were examined after treatments with GLA-SE, GLA, SE and Alum.  A time course study at 0, 6h, day 1, day 2, and day 4 were carried out. ACF Study No.  078-10-007 and Study No.  078-11-008', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 379, 'E-GEOD-43351', 23, 'Transcriptional Profiling of Mouse Uterus at Pre-implantation Stage under VEGF Repression', 'In this study, a genetically repressed VEGF mouse model was used to analyze uterus transcriptome at G2.5 (gestation day 2.5) by Solexa/ Illuminas digital gene expression (DGE) system. 831 uterus-specific and 2398 VEGF regulated genes were identified. Gene ontology analysis indicated that genes actively involved in uterus development were members of collagen biosynthesis, cell proliferation and cell apoptosis. Uterus-specific genes were enriched in activities of phosphatidyl inositol phosphate kinase, histone H3-K36 demethylation and protein acetylation. Among VEGF regulated genes, up-regulated genes were associated with RNA polymerase III activity while down-regulated genes were strongly associated with muscle development.Comparable numbers of antisense RNA transcripts were also identified. Expression levels of the antisense RNAs were found tightly correlated with their sense expression levels, an indication of possibly non-specific transcripts generated around the active promoters and enhancers. The antisense RNAs with exceptionally high or low expression levels and the antisense RNAs under VEGF regulation were also identified. These transcripts may be important candidates in regulation of uterus development. This study provides a global survey on genes and antisense transcripts regulated by VEGF in the pre-implantation stage. Results will contribute to the further studies of candidate genes and pathways in regulating implantation process and related diseases. Uteri mRNA profiles of G2.5 VEGF-normal (Dox+) and VEGF-repression (Dox-) were generated by Solexa/ Illuminas digital gene expression (DGE) system.', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 380, 'E-GEOD-43339', 23, 'Gene expression profiling of murine liver NK cells', 'The NK cell pool is composed of distinct NK cell subsets with divergent phenotypic and functional features. In order to determine whether DX5- and DX5+ NK cells from murine livers represent different NK cell subsets, DX5- and DX5+ liver NK cells of adult mice were respectively sorted for gene expression analysis using the Affymetrix GeneChip Mouse Genome 430 2.0 arrays. DX5-NK1.1+CD3- cells and DX5+NK1.1+CD3- cells were sorted from the liver of naive B6 wild-type mice. RNA of each sample was then extracted and hybridized on Affymetrix microarrays to detail differences between DX5- and DX5+ liver NK cells in gene expression.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 381, 'E-GEOD-43333', 23, 'Cell of origin determines clinically relevant subtypes of MLL-rearranged AML', 'MLL-fusion proteins can induce acute myeloid leukemias (AML) from either hematopoietic stem cells (HSC) or granulocyte macrophage progenitors (GMP), but it remains unclear if the cell of origin influences the biology of the resultant leukemia. MLL-AF9 transduced single HSC or GMP could be continuously replated, but HSC-derived clones were more likely than GMP-derived clones to initiate AML in mice. Leukemia stem cells derived from either HSC or GMP had a similar immunophenotype consistent with a maturing myeloid cell (LGMP). Gene expression analyses demonstrated that LGMP inherited gene expression programs from the cell of origin including high-level Evi-1 expression in HSC derived LGMP. The gene expression signature of LGMP derived from HSC was enriched in poor prognosis human MLL-rearranged AML in three independent data sets. Moreover, global 5-mC levels were elevated in HSC-derived leukemias as compared to GMP-derived leukemias. This mirrored a difference seen in 5-mC between MLL-rearranged human leukemias that are either EVI1-positive or EVI1-negative. Finally, HSC derived leukemias were more resistant to chemotherapy than GMP-derived leukemias. These data demonstrate that the cell of origin influences the gene expression profile, the epigenetic state, and the drug response in AML, and that these differences can account for clinical heterogeneity within a molecularly defined group of leukemias. Differential DNA methylation between of LSC isolated from murine HSC and GMP derived AMLs', '1132/NCBITaxon:10090', '1136/efo:EFO_0002761'
  UNION SELECT 382, 'E-GEOD-42952', 23, 'Molecular markers associated with outcome and metastasis in human pancreatic cancer', 'Pancreatic ductal adenocarcinoma (PDAC) is a heterogeneous cancer in which differences in survival rates might be related to a variety in gene expression profiles. Although the molecular biology of PDAC begins to be revealed, genes or pathways that specifically drive tumour progression or metastasis are not well understood. Therefore, we performed microarray analyses on whole-tumour samples of 2 human PDAC subpopulations with similar clinicopathological features, but extremely distinct survival rates after potentially curative surgery, i.e., good outcome (OS and DFS?>?50?months) versus bad outcome (OS?<?19?months and DFS?<?7?months). Additionally, liver- and peritoneal metastases were analysed and compared to primary cancer tissue. The integrin and ephrin receptor families were upregulated in all PDAC samples, irrespective of outcome, supporting an important role of the interaction between pancreatic cancer cells and the surrounding desmoplastic reaction in tumorigenesis and cancer progression. Moreover, some components, such as ITGB1 and EPHA2, were upregulated in PDAC samples with a poor outcome, Additionally, overexpression of the non-canonical Wnt/?-catenin pathway and EMT genes in PDAC samples with bad versus good outcome suggests their contribution to the invasiveness of pancreatic cancer, with ?-catenin being also highly upregulated in metastatic tissue. Thus, we conclude that components of the integrin and ephrin pathways and EMT-related genes might serve as molecular markers in pancreatic cancer as their expression seems to be related with prognosis. Microarray analysis was performed on ''Good'' and ''Bad'' patient samples (samples with similar pathological characteristics were chosen based on the definition of the 2 diverse survival outcome groups and the required RIN values above 7.1), on surrounding non-tumoural pancreatic control samples, liver metastasis (LM) and peritoneal metastasis (PM). Comparisons between good vs. control, bad vs. control, good vs. bad, and primary pancreatic cancer versus metastasis were performed.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 383, 'E-GEOD-42890', 23, 'Epididymal Adipose Tissue Profiling In HMDP', 'Identify genes in the epididymal adipose tissue whose expression is under genetic regulation in the hybrid mouse diversity panel.  The hybrid mouse diversity panel is comprised of classical inbred and recombinant inbred wild type mice.  The RMA values of genes were used for genome wide association as described in Bennett et al Genome Research 2010.  These data are used to identify candidate genes at loci associated with obesity and dietary responsiveness. GWAS for expression of epididymal adipose tissue in inbred strains', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 384, 'E-GEOD-42140', 23, 'Gene expression in signaling subsets of AML blasts induced by G-CSF', 'While the response to G-CSF is a significant predictor of AML outcome, the nature of cells responding differentially to this and other cytokines remains unclear. Based on gene expression signatures observed in G-CSF Responsive Cells (RC) and non-G-CSF Responsive Cells (NRC) a cohort of AML samples were segregated into two types. In Type1 both cell subsets displayed an undifferentiated phenotype.  In the Type2 AML the RC showed a very similar undifferentiated progenitor-like phenotype as in Type1 RC, whereas the NRC exhibited a more differentiated hematopoietic phenotype. The frequency of RC per sample was a determinant of whether a patient belonged to Type1 or Type2. Type2 RC had higher expression of CD321, higher colony formation capacity and greater responsiveness to cytokines secreted by NRC. Further analysis of clinical mRNA databases revealed that CD321 was a prognostic indicator for AML and a potential marker of clonogenic activity associated with this AML cancer lineage. Cryopreserved PBMC from 11 AML patinets were stimulated with G-CSF. Following stimulation cells were fixed, permeabilized and stained for selected surface markers (CD3, CD33, CD45) and and phospho-Stat3/5. After staining subsets positive and negative for phospo-Stat3/5 were sorted out for further microarray analysis. T-cells were sorted from each sample to control for the RNA quality by comparisons to RNA prepared from live T-cells sorted form healthy donors.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 385, 'E-GEOD-40205', 23, 'Lactate influences the gene expression profile of human mesenchymal stem cells (hMSC) in a dose dependent manner', 'Wounds, especially non healing wounds are characterised by elevated tissue lactate concentrations. Lactate is known for being able to stimulate collagen synthesis and vessel growth. Lately it has been shown that lactate, in vivo, plays an important role in homing of stem cells. With this work we aimed to show the influence of lactate on the gene expressionprofil of human mesenchymal stem cells (hMSC).  hMSCs were obtained from bone marrow and characterised with fluorescence-activated cell sorting (FACS) analysis. Subsequently the hMSCs were treated with either 0, 5, 10  and 15 mM lactate (pH 7,4) for 24 hours. RNA Isolation from stimulated hMSCs and controls was performed. The Microarray analysis was performed using Affymetrix HuGene 1.0 ST Gene Chip. Selected targets were subsequently analysed using quantitative real time PCR (RTq-PCR). We were able to show that lactate in moderate concentrations of 5 respectively 10 mM leads to an anti-imflammatory, anti-apoptotic but growth and proliferation promoting gene expression after 24 h. In contrast, high latate concentrations of 15 mM leads to the opposed effect, namely promoting inflammation and apoptosis. Hypoxia induced genes did not not show any significant regulation. Contrary to expectation, we were not able to show any significant regulation of glycolysis associate candiadtes. We were able to show that lactate alters gene expression but does not change the cell phenotype, which might be helpful for further investigations of new treatment strategies for chronic non-healing wounds as well as tumor-therapy and neuronal plasticity. Timecourse experiment with hMSCs treated with 5, 10, 15 mM lactic acid for 1, 3, 7 days.  hMSCs without lactate served as controls *** CEL files lost due to hard disk crash', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 386, 'E-GEOD-39270', 23, 'Corticosterone regulation of mRNAs and and corresponding RU486 antagonism in astrocytes in vitro', 'Analysis of mRNA gene expression changes mediated by natural glucocorticoid ligand corticosterone in mouse cortical primary astrocyte cultures; also, parallel analysis of RU486 (mifepristone) antagonism of corticosterone-mediated mRNA changes. Total RNA obtained from astrocyte cultures subjected to 2, 4, or 6 hours of (1) 100nM corticosterone alone, (2) 500nM RU486 alone, or (3) 100nM corticosterone + 500nM RU486 exposure compared to vehicle control astrocyte cultures.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 387, 'E-GEOD-37653', 23, 'Whole-transcriptome profiling analysis of first trimester placental villi in Indian women', 'Whole-transcriptome profiles of individual human placental villi samples from twenty-five (25) Indian women with normal pregnancies during 6- to 8-weeks of gestation were examined using human whole genome expression arrays (NimbleGen 135K). The present study focused on the whole-transcriptome profiling using NimbleGen135K (070925_HG18_exp__12X135K) human whole genome expression arrays of individual human placental villi samples obtained from twenty-five (25) proven-fertile women bearing normal pregnancies voluntarily terminated between 6- and 8-weeks of gestation. Gestational age was estimated from menstrual history, physical and ultrasonographic evaluation. No case of complicated pregnancy from infection, and other significant fetal and maternal clinical indications was included. These twenty five (25) samples include biological replicates of 6, 7 and 8 weeks placental villi samples.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 388, 'E-GEOD-36696', 23, 'Comparative transcriptome analysis of transporters, phytohormone and lipid metabolism pathways in response to arsenic stress in rice (Oryza Sativa L.)', 'Arsenic (As) is a carcinogenic metalloid that is a contaminant widely polluting rice paddy soils around the world. In order to gain better insight into molecular mechanism of rice exposed to As(III) stress, we used next-generation sequencing technology to acquire global transcriptome alteration and miRNA regulation in rice upon As(III) treatments.   Our results suggest time course and As(III)-dosing treatments were devised. Cluster analyses show that root and shoot samples were differentially grouped. For roots, sub-clusters were more distinct in the dosage course whereas for shoots they were most recognizable for the time course treatments.   Other than the significantly regulated gene expression in the heavy metal-responsive sulfur and glutathione metabolism pathways, the expression of genes related to heavy metal transportation, jasmonate biosynthesis and signaling pathways, lipid metabolism and gene transcription were sharply regulated, indicating that rice allocates energy and resources from growth to stress response under As(III) stress. In addition to the detection of previously identified stress-related miRNAs, we further discovered 36 new As(III)-responsive miRNAs.  These results expand our understanding of As(III) stress mechanism to the As(III)-responsive mRNA and miRNA transcriptomes, which provide a foundation for subsequent functional research. 10 mRNA samples examined   10 miRNA samples examined', '1132/NCBITaxon:39947', '1136/efo:EFO_0003737'
  UNION SELECT 389, 'E-GEOD-34977', 23, 'Gene expression profile in mouse tumor RIF-1 cell line after knockdown of Bre1a (Rnf20) or Bre1b (Rnf40).', 'Mammalian Bre1 complexes (BRE1A/B (RNF20/40) in humans and Bre1a/b (Rnf20/40) in mice) function similarly to their yeast homolog Bre1 as ubiquitin ligases in monoubiquitination of histone H2B. This ubiquitination facilitates methylation of histone H3 at K4 and K79, and accounts for the roles of Bre1 and its homologs in transcriptional regulation. Recent studies by others suggested that Bre1 acts as a tumor suppressor, augmenting expression of select tumor suppressor genes and suppressing select oncogenes. In this study we present an additional mechanism of tumor suppression by Bre1 through maintenance of genomic stability.  We track the evolution of genomic instability in Bre1-deficient cells from replication-associated double-strand breaks (DSBs) to specific genomic rearrangements that explain a rapid increase in DNA content and trigger breakage-fusion-bridge cycles. We show that aberrant RNA-DNA structures (R-loops) constitute a significant source of DSBs in Bre1-deficient cells. Combined with a previously reported defect in homologous recombination, generation of R-loops is a likely initiator of replication stress and genomic instability in Bre1-deficient cells. We propose that genomic instability triggered by Bre1 deficiency may be an important early step that precedes acquisition of an invasive phenotype, as we find decreased levels of BRE1A/B and dimethylated H3K79 in testicular seminoma and in the premalignant lesion in situ carcinoma. A cellular modification design type is where a modification of the transcriptome, proteome (not genome) is made, for example RNAi, antibody targeting. Knock down by RNA interference: gene knocked down in RIF-1 cell line cellular_modification_design', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 390, 'E-GEOD-34729', 23, 'Gene expression changes induced by overexpression of EVI1 in Lin- hematopoietic cells [Lin]', 'The transcription factor Evi1 is essential for the formation and maintenance of hematopoietic stem cells, and induces clonal dominance with malignant progression upon constitutive activation by chromosomal rearrangements or transgene integration events. To understand the immediate and adaptive response of primary murine hematopoietic cells to the transcriptional upregulation of Evi1, we developed an inducible lentiviral vector system with a robust expression switch. We found that Evi1 delays differentiation and promotes survival in myeloid culture conditions, orchestrating a battery of genes involved in stemness (Aldh1a1, Ly6a [Sca1], Abca1, Epcam, among others). Importantly, Evi1 suppresses Cyclins and Cyclin-dependent kinases (Cdk), while it upregulates Cdk inhibitors, inducing quiescence in various proliferation-inducing cytokine conditions and operating in a strictly dose-dependent manner. Hematopoietic cells with persisting Evi1-induction tend to adopt a relatively low expression level. We thus classify Evi1 as a dormancy-inducing oncogene, likely requiring epigenetic and genetic compensation for cell expansion and malignant progression. Lin- Rosa26rtTA cells were isolated, transduced in S3F11 cytokines, induced the next day with DOX [1 ?g/ml] and  20 hours later sorted for negative/low or highly EGFP expressing cells, from which total RNA was extraced and subjected to Microarray Analysis', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 391, 'E-GEOD-33217', 23, 'Expression profiles of water deficit stress (WDS) regulated genes in rose petals', 'Water deficit stress (WDS) is a crucial factor that causes the inhibition of petal expansion and abnormal flower opening in rose. The regulatory mechanisms of petal expansion by WDS at transcriptional level were investigated by analysis expression profiles under WDS. Analysis used total RNA samples of petals taken from flowers treated by WDS comparison to those from control flowers. Transcriptome dynamics during treatment time responsive to WDS.', '1132/NCBITaxon:128735', '1136/efo:EFO_0002768'
  UNION SELECT 392, 'E-GEOD-33106', 23, 'Expression data from livers in  wildtype and Sox17+/-mice at 17dpc', 'The onset of the liver inflamentation in the Sox17+/- embryos. The expression of Cxcl10, Cxcl2, Cxcl1 and stress-induced heat shock protein Hspa1a and Hspa1b were elevated in Sox17+/- livers at 17dpc, as compared with the wildtype livers. Total RNAs from liver tissues of wildtype and Sox17+/- mice at 17dpc were subjected to microarray analyses. Two biological replicates of each genotype were analyzed.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 393, 'E-ERAD-129', 23, 'RNA-seq of Gorilla, chimpanzee and bonobo', 'RNA-Seq from lymphoblastoid cell lines for Gorilla, chimpanzee and bonobo species. This RNA-Seq data has been described in the following article: Scally et al., Nature 2012;483;7388;169-75, DOI: 10.1038/nature10842, and its further analysis can be freely submitted for publication. For information on the proper use of data shared by the Wellcome Trust Sanger Institute (including information on acknowledgement), please see http://www.sanger.ac.uk/datasharing/>', '1132/NCBITaxon:9598', '1136/efo:EFO_0003738'
  UNION SELECT 394, 'E-MTAB-1433', 23, 'Comparative genomic hybridisation of gDNA from HDAC1-/-; HDAC2+/- diseased mouse thymocytes and sample matched wild-type tail', 'Histone deacetylase 1 and 2 (HDAC1/2) regulate chromatin structure as the catalytic core of the Sin3A, NuRD and CoREST co-repressor complexes.  To better understand the key pathways regulated by HDAC1/2 in the adaptive immune system and inform their exploitation as drug targets, we have generated mice with a T cell specific deletion and performed comparative genomic hybridisation using genomic DNA from HDAC1-/-; HDAC2+/- diseased thymocytes and sample-matched, wild-type tail. A data set from a related transcription-profiling study comparing HDAC1-/-; HDAC2+/- mouse thymus tissue against thymus tissues from wild-type counterparts has also been deposited at ArrayExpress under accession E-MTAB-1432: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1432', '1132/NCBITaxon:10090', '1136/efo:EFO_0000749'
  UNION SELECT 395, 'E-MTAB-1432', 23, 'Transcription profiling by array of wildtype and mutant  HDAC1-/-;HDAC2+/- in 2-3 week old TCRb/CD5 low Mouse T cells', 'Histone deacetylase 1 and 2 (HDAC1/2) regulate chromatin structure as the catalytic core of the Sin3A, NuRD and CoREST co-repressor complexes. To better understand the key pathways regulated by HDAC1/2 in the adaptive immune system and inform their exploitation as drug targets, we have generated mice with a T cell specific deletion. We then performed transcriptiona profiling on thymus tissues harbouring such T-cell specific deletion and normal thymus tissues from wild-type mice.  A data set from a related comparative genomic hybridisation experiment comparing HDAC1-/-; HDAC2+/- diseased thymocytes and sample-matched wild-type tail tissues has also been deposited at ArrayExpress under accession E-MTAB-1433: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1433', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 396, 'E-MTAB-1059', 23, 'Transcription profiling by array of total RNA and newly transcribed RNA from yeast mutants in response to osmotic stress', 'Dynamic transcriptome analysis of osmotic stress response. Unperturbed RNA synthesis and decay rates can be obtained via metabolic RNA labeling and kinetic modeling (Cleary et al, 2005; Kenzelmann et al, 2007; Dolken et al, 2008; Friedel and Dolken, 2009; Miller et al, 2009). The nucleoside analog 4-thiouridine (4sU) is taken up by eukaryotic cells and incorporated into mRNA during Pol II transcription (Melvin et al, 1978). The thiol-labeled newly transcribed RNA can then be isolated by affinity chromatography (Kenzelmann et al, 2007) or by biotinylation and purification with streptavidin-coated magnetic beads (Cleary et al, 2005; Dolken et al, 2008).', '1132/NCBITaxon:4932', '1136/efo:EFO_0002768'
  UNION SELECT 397, 'E-MTAB-388', 23, 'Gene expression profiling of neuroendocrine primary tumors: effect of Octreotide treatment', 'Gene expression profiles of primary neuroendocrine tumors treated or not with 1µM Octreotide.We investigated 5 human neuroendocrine cell lines, CNDT2.5 and KRJ1, established from ileum NETs, QGP1 by a pancreatic NET, NCI H720 and NCI H727 from bronchopulmonary NETs. CNDT2.5 cell wereconstantly treated with 1µM octreotide for 10 and 16 months. We isolated total RNA (Ambion, PARIS Kit) from 5 WT cell lines and CNDT2.5 treated with octreotide (Santostatin, Novartis). Total RNA was hybridized onto the Affymetrix Human Gene 1.0 ST Array by Affymetrix Uppsala Platform, UU. SE (Uppsala, Sweden). We first wanted to verify whether the different cell lines may become reliable models to study neuroendocrine signaling pathways. The main objective of this project aimed at understanding the mechanisms by which octreotide (Sandostatin, Novartis) alter cellular growth and differentiation of CNDT2.5 cells. We therefore focused on intermediate (10 months) and long stimulation (16 months) events triggered by sandostatin, which lead variation of CNDT.2.5 cells gene expression to identify potential pivotal genes involved in the development of drug resistance in neuroendocrine cells.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 398, 'E-MEXP-3799', 23, 'Transcription profiling by array of Escherichia coli subjected to two different types of nutritional scenarios to investigate nutritional condition dependant differential tolerance to hydrogen peroxide stress', 'Escherichia coli culture was subjected to two different types of nutritional scenarios, abundant carbon/ nitrogen sources and scarce carbon/nitrogen medium. Study revealed that scarce medium adapted culture were more tolerant to hydrogen peroxide than abundant medium.', '1132/NCBITaxon:562', '1136/efo:EFO_0002768'
  UNION SELECT 399, 'E-MEXP-3752', 23, 'Transcription profiling by array of human acute myeloid luekemia cell lines NB4 and NB4.437r to study their sensitivity-resistance to the atypical retinoids ST1926 and CD437', 'The two cell lines NB4 and NB4.437r, respectively sensitive and resistant to the cytotoxic action of the atypical retinoid ST1926 and CD437 were used to unveil the mechanism of action of these drugs.We compared their transcriptional profiles in conditions of exponential growth in basal medium by microarray gene expressio analysis.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 400, 'E-MEXP-3714', 23, 'Transcription profiling by array of genes regulated by ahk2 ahk3 in response to cold in Arabidopsis thaliana', 'The experiment was designed to enable comparison of ahk2 ahk3 double mutant in response to cold compared with wild types.', '1132/NCBITaxon:3702', '1136/efo:EFO_0002768'
  UNION SELECT 401, 'E-GEOD-43325', 23, 'Atypical Enteropathogenic Escherichia coli Virulence Regulation by QseC Quorum Sensing Adrenergic Sensor', 'Atypical EPEC (aEPEC) strains are part of group of pathogens capable of forming the Attaching and Effacing (A/E) lesion. This lesion is characterized by intimate adherence of bacteria to enterocytes, and microvilli destruction. The genes responsible to cause that lesion are located in a pathogenicity island called Locus of Enterocyte Effacement (LEE). Transcription of LEE genes is subjected to various levels of regulation, including quorum sensing through autoinducer 3 (AI-3) system. AI-3 is an aromatic compound with similar characteristics to the epinephrine and norepinephrine hormones. This similarity allows bacteria to use these hormones and AI-3 to perform cell  to  cell signaling processes and bacteria - host communication processes in order to modulate its virulence. AI-3, epinephrine and norepinephrine are detected by a sensor kinase named quorum sensing E.coli regulator (QseC). In order to investigate the role of QseC and epinephrine in atypical EPEC O55:H7 virulence, we constructed a QseC mutant of this strain and performed transcription and phenotypic analyses in the presence or absence of epinephrine. We have reported here, for the first time, the quorum sensing QseC regulation of virulence genes in atypical EPEC. Our results shown that QseC is a global regulator of gene expression in aEPEC and positively regulates flagellar genes, LEE and non-LEE encoded factors. We also have shown that the presence of epinephrine could be sensed by other receptor that acts as negative regulator of LEE4 and LEE5 genes. Comparison of transcriptional regulation of enteropathogenic E. coli serotype O55:H7 wild type and the qseC mutant in the absence or presence of epinephrine signal to identify the regulated targets', '1132/NCBITaxon:244320', '1136/efo:EFO_0002768'
  UNION SELECT 402, 'E-GEOD-43324', 23, 'Brain expression data from fetal mouse brains prenatally exposed to ethanol', 'Moderate alcohol exposure during pregnancy can result in brain gene expression changes in resulting offspring.  We have developed a mouse model of FASD that involves moderate ethanol exposure in mid-gestation (trimester 2 equivalent) achieved by injections of ethanol.  We have previously shown that this model results in phenotypes relevant to FASD.  Since ethanol is known to directly affect the expression of genes in the developing brain leading to abnormal cell death, changes to cell proliferation, migration, and differentiation, and potential changes to epigenetic patterning, we hypothesize that there will be gene expression changes immediately following acute ethanol exposure in the fetal brain. We used a microarray experiment and focused on the genes identified to evaluate the genome-wide alterations to the fetal brain transcriptome caused by prenatal ethanol exposure. To generate samples, female C57BL/6J mice were given ethanol injections (2.5g/kg of ethanol in saline) twice on gestational days 14 and 16 to produce acute ethanol exposure effects.  Control females were injected with the same volume of saline.  Dams were sacrified on gestational day 16, following ethanol exposure, and whole brains from fetuses were then extracted.  RNA was isolated from brain tissue and samples from three mice were pooled to reduce litter effects and the pooled samples were hybridized on Affymetrix arrays (2 control and 2 ethanol chips, total n=12 mice).', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 403, 'E-GEOD-43315', 23, 'Discovery of a splicing regulator required for cell cycle progression', 'Toxoplasma gondii temperature sensitive mutant 12-109C6 conditionally arrests in the G1 phase due to a single point mutation in a novel protein containing a single RNA-recognition-motif (TgRRM1).  The resulting tyrosine to asparagine amino acid change in TgRRM1 causes severe temperature instability that generates an effective null phenotype for this protein when the mutant is shifted to the restrictive temperature.  Orthologs of TgRRM1 are widely conserved in diverse eukaryote lineages, and the human counterpart (RBM42) can functionally replace the missing Toxoplasma factor. The interaction of TgRRM1 with factors of the tri-SNP complex (U4/U6 & U5 snRNPs) indicate this factor may be required to assemble an active spliceosome.  Thus, the TgRRM1 family of proteins is an unrecognized and evolutionarily conserved class of splicing regulators. We describe transcriptome of the temperature sensitive mutant 12-109C6. Transcriptome studies demonstrated that gene expression is largely downregulated in the mutant at the restrictive temperature 40oC and is accompanied by a severe defect in splicing that affects both cell cycle and constitutively expressed mRNAs. RNA samples were isolated in duplicate from mutant 12-109C6 parasites grown for 24h at permissive (34oC) and 6h and 24h at non-permissive temperatures (40oC). Samples were hybridized to the Toxoplasma gondii Affymetrix microarray (ToxoGeneChip: http://ancillary.toxodb.org/docs/Array-Tutorial.html). Hybridization data was preprocessed with Robust Multi-array Average (RMA) and normalized using per chip and per gene median polishing and analyzed using the software package GeneSpring GX (Agilent Technologies, Santa Clara CA).', '1132/NCBITaxon:5811', '1136/efo:EFO_0002768'
  UNION SELECT 404, 'E-GEOD-43310', 23, 'Gene expression profile in spleens of mice 24 hours after total body irradiation.', 'A major challenge in bone allogeneic marrow (BM) transplantation is overcoming engraftment resistance to avoid the clinical problem of graft rejection.  Identifying genes and pathways that regulate BM engraftment may reveal molecular targets for overcoming these engraftment barriers.  Previously, we developed a mouse model of BM transplantation that utilizes recipient conditioning with non-myeloablative total body irradiation followed by transplantation of allogeneic BM cells.  We defined TBI doses that lead to graft rejection, that are permissive for engraftment, and mouse strain variation with regards to the permissive TBI dose.  We now report gene expression analysis, using the Agilent Mouse 8x60K v3 expression arrays, in spleen of mice conditioned with TBI doses for evaluation in correlation to the expected engraftment phenotype.  The spleens of mice given TBI demonstrated extensive differential gene expression, compared with un-irradiated controls, at both multiple testing-corrected P < .05 and fold change greater than or equal to  2 levels. Functional analysis revealed significant enrichment for up-regulation of canonical pathways involved in inflammation, even when treated with TBI doses not permissive for engraftment.  Consistent with this the most significantly associated upstream regulators included lipopolysaccharide, tumor necrosis factor, and a toll-like receptor.  Unique to engrafting mice, however, were enrichment for a down-regulated canonical pathway related to B-cell development.  Results from this analysis have identified canonical pathways and upstream regulators of BM engraftment in mice and may lead to new approaches to overcome the problem of graft rejection complicating clinical BM transplantation. Each individual sample was pooled from 3 mice and performed in triplicate for each experimental group (TBI 300 cGy, 400 cGy, or 500 cGy) and for each strain mice (BALB.K or B10.BR)  for statistical analysis.  Additional arrays were perfromed for TBI 800 cGy (BALB.K) and TBI 900 cGy (B10.BR) for anlaysis of gene expressive after myeloablative condition.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 405, 'E-GEOD-41777', 23, 'Identification of DNA-replication domains in single S-phase cells: part 3 multi-cell controls', 'DNA-replication is a key process in life and can lead to disease when disturbed. Cell-type specific early and late replication domains have been discovered throughout genomes by analysis of DNA from populations of cells. However, cell to cell differences and the association of these differences with other cellular processes remain largely elusive. Here we demonstrate for the first time that consecutive domains of early and late DNA-replication can be detected in single S-phase cells using array comparative genomic hybridization, providing proof-of-concept for a novel tool to investigate DNA-replication genome wide at the single-cell level. Furthermore, methods to profile the genome of a single cell for DNA-copy number aberrations are revolutionizing both basic genome research and clinical genetic diagnosis. It is thus important to apprehend not only technical but also biological reasons for false positive copy number detection. None of the current single-cell copy number calling methods distinguishes between a cell in G1-, S- or G2/M-phase of the cell cycle and mostly use cells isolated randomly from populations. We demonstrate that the oscillating pattern between early and late replicating loci instigates significantly more false-positive DNA copy-number calls in a diploid cell in S-phase cells than in G1- or G2/M-phase, depending on the specific aCGH-signal normalization method used. We propose a work-flow to detect single cells in S-phase and to correct for DNA-replication bias before copy number profiling. DNA was extracted from many cells from 1 S-phase, 1 G1-phase and 1 G2/Mphase enriched cell population from an EBV-transformed lymphoblastoid cell line from a male carrying a 750kb amplification on the short arm of chromosome 4. These test samples were hybridized comparatively to commercial male reference DNA.', '1132/NCBITaxon:9606', ''
  UNION SELECT 406, 'E-GEOD-41776', 23, 'Identification of DNA-replication domains in single S-phase cells: part 4 whole genome amplified single-cell DNA', 'DNA-replication is a key process in life and can lead to disease when disturbed. Cell-type specific early and late replication domains have been discovered throughout genomes by analysis of DNA from populations of cells. However, cell to cell differences and the association of these differences with other cellular processes remain largely elusive. Here we demonstrate for the first time that consecutive domains of early and late DNA-replication can be detected in single S-phase cells using array comparative genomic hybridization, providing proof-of-concept for a novel tool to investigate DNA-replication genome wide at the single-cell level. Furthermore, methods to profile the genome of a single cell for DNA-copy number aberrations are revolutionizing both basic genome research and clinical genetic diagnosis. It is thus important to apprehend not only technical but also biological reasons for false positive copy number detection. None of the current single-cell copy number calling methods distinguishes between a cell in G1-, S- or G2/M-phase of the cell cycle and mostly use cells isolated randomly from populations. We demonstrate that the oscillating pattern between early and late replicating loci instigates significantly more false-positive DNA copy-number calls in a diploid cell in S-phase cells than in G1- or G2/M-phase, depending on the specific aCGH-signal normalization method used. We propose a work-flow to detect single cells in S-phase and to correct for DNA-replication bias before copy number profiling. The genome of 7 S-phase, 2 M-phase and 2 G-phase single cells was amplified using the Sureplex amplification system. These test samples were hybridized comparatively to commercial male reference DNA.', '1132/NCBITaxon:9606', ''
  UNION SELECT 407, 'E-GEOD-41616', 23, 'Identification of Novel Molecular Markers through Transcriptomic Analysis in Human Fetal and Adult Corneal Endothelial Cells', 'Corneal endothelium is composed of a monolayer of corneal endothelial cells (CECs) in the inner layer of cornea, which is essential for maintaining corneal transparency. In order to better characterize CECs in different developmental stages, we profiled mRNA transcriptomes in human fetal and adult corneal endothelium with the goal to identify novel molecular markers in these cells. By comparing CECs with 12 other types of tissues, we identified 245 and 284 signature genes that are highly expressed in fetal and adult CECs, respectively. Functionally, these genes are characteristic of CECs, involving in cell adhesion, proteoglycan and sulfur metabolic process. Importantly, several of these genes are disease target genes in hereditary corneal dystrophies, consistent with their functional significance in CEC physiology. By comparing fetal and adult CECs, we also identified stage-specific markers associated with CEC maturation, such as the activation of the Wnt pathway genes in fetal, but not in adult CECs. Lastly, by immunohistochemistry of ocular tissues, we further confirmed the unique protein expression patterns for Wnt5a, S100A4, S100A6, and IER3, the four novel markers for either fetal or adult CECs.  The identification of a new panel of molecular markers for fetal and mature CECs would be very useful for characterizing and quality controlling CECs through ex vivo expansion or stem cell differentiation for cell replacement therapy. mRNA profile between adult and fetal CECs by high-throughput sequencing', '1132/NCBITaxon:9606', '1136/efo:EFO_0003738'
  UNION SELECT 408, 'E-GEOD-41257', 23, 'Transcriptional changes in intraepithelial CD4 lymphocytes', 'We investigated transcriptional changes in CD4CD8aa and CD4 intraepthelial lymphocytes. TCR?? thymocytes differentiate to either CD8?? cytotoxic T lymphocytes or CD4 T helper cells. This functional dichotomy is controlled by key transcription factors, including the T helper master regulator, ThPOK, which suppresses the cytolytic-program in MHC class II restricted CD4 thymocytes. ThPOK continues to repress CD8-lineage genes in mature CD4 T cells, even as they differentiate to T helper effector subsets. Surprisingly, we show here that the T helper-fate is not fixed and that mature antigen-stimulated CD4 T cells can switch off Thpok expression and reactivate CD8-lineage genes. This unexpected plasticity results in the post-thymic termination of the T helper program and the functional differentiation of distinct MHC class II restricted CD4 cytotoxic T lymphocytes. Intraepithelial_CD4_CD8a neg vs CD8a pos.  Two sample set of CD4CD8aa and CD4 intraepthelial lymphocytes (IEL) from small intestine of RAG knockout mice ( 8 weeks after transfer of naive CD4 cells, adoptive tranfer model of colitis), were prepared via  cell sorting, and RNA was prepared by TRIZol (Invitrogen, USA) . Data were analyzed in GeneSpring GX10.  For microarray analysis, RNA was labeled and hybridized to GeneChip Mouse Genome 430 2.0 arrays according to the Affymetrix protocols. Data were analyzed in GeneSpring GX10.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 409, 'E-GEOD-39301', 23, 'Genome-wide analysis of Xbp1s effect on gene expression in liver by Doxycycline induced TRE-Xbp1s overexpression in mouse', 'Analysis of Xbp1s overexpression in liver after 24 hr indution or 48 hr induction in LIXs mouse. As a control, WT mice after 2 hr refeed and 24 hr fast without refeed are used for analysis of postprandial gene expression. This microarray study was used to screen for target genes activated by Xbp1s in liver. Results provide important information for the role of Xbp1s during postprandial. Total RNA obtained from liver tissues from mice on Dox200 chow diet for 24 hr or 48 hr. Mice were fasted for 6 hr before sacrifice for liver.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 410, 'E-GEOD-39174', 23, 'EH001: Lmx1a is an activator of Rgs4 and Gbr10 and is responsible for the correct specification of rostral and medial mdDA neurons', 'The LIM homeodomain transcription factor Lmx1a is a very potential inducer of stem cells towards dopaminergic neurons. Despite several studies on the function of this gene, the exact in vivo role of Lmx1a in mesodiencephalic dopamine (mdDA) neuronal specification is still not understood. To analyze the genes functioning downstream of Lmx1a, we performed expression microarray analysis of LMX1A overexpressing MN9D dopaminergic cells. Several interesting regulated genes were identified, based on their regulation in other, previously generated expression arrays, and their expression pattern in the developing mdDA neuronal field. Post analysis through in vivo expression analysis in Lmx1a mouse mutant (drJ/drJ) embryos demonstrated a clear decrease in expression of the genes Grb10 and Rgs4, in and adjacent to the rostral and dorsal mdDA neuronal field and within the Lmx1a expression domain. Interestingly, the DA marker Vmat2 was significantly up-regulated as a consequence of increased LMX1A dose, and subsequent analysis on Lmx1a mutant E14.5 and adult tissue revealed a significant decrease in Vmat2 expression in mdDA neurons. Taken together, microarray analysis of an LMX1A overexpression cell system resulted in the identification of novel downstream targets of Lmx1A in mdDA neurons: Grb10, Rgs4 and Vmat2. RNA was isolated from MN9D cells. Each experimental sample consisted of a RNA pool derived from 3 separate 10-cm dishes containing Lmx1a overexpressing MN9D cells (transfected with pcDNA3.1(-)-Lmx1a). microarray analysis was performed in triplicate, each experimental sample was hybridized to the same reference pool of RNA derived from 9 10-cm dishes containing control MN9D cells (transfected with empty pcDNA3.1(-)). On each of three microarray samples, dye swap was performed to correct for dye effects.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 411, 'E-GEOD-39104', 23, 'Gene expression changes induced by overexpression of EVI1 in Lin- hematopoietic cells', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 412, 'E-GEOD-39103', 23, 'Gene expression changes induced by overexpression of EVI1 in Lin- hematopoietic cells [EVI1_ST]', 'The transcription factor Evi1 is essential for the formation and maintenance of hematopoietic stem cells, and induces clonal dominance with malignant progression upon constitutive activation by chromosomal rearrangements or transgene integration events. To understand the immediate and adaptive response of primary murine hematopoietic cells to the transcriptional upregulation of Evi1, we developed an inducible lentiviral vector system with a robust expression switch. We found that Evi1 delays differentiation and promotes survival in myeloid culture conditions, orchestrating a battery of genes involved in stemness (Aldh1a1, Ly6a [Sca1], Abca1, Epcam, among others). Importantly, Evi1 suppresses Cyclins and Cyclin-dependent kinases (Cdk), while it upregulates Cdk inhibitors, inducing quiescence in various proliferation-inducing cytokine conditions and operating in a strictly dose-dependent manner. Hematopoietic cells with persisting Evi1-induction tend to adopt a relatively low expression level. We thus classify Evi1 as a dormancy-inducing oncogene, likely requiring epigenetic and genetic compensation for cell expansion and malignant progression. Lin- Rosa26rtTA cells were isolated, transduced either with pRRL.PPT.Tet.Evi1.IRES.EGFP.pre or with the pRRL.PPT.Tet.EGFP.pre control vector in S3F11 cytokines, induced the next day with DOX [1 ?g/ml] and 20 hours later sorted for EGFP expressing cells, from which total RNA was extraced and subjected to Microarray Analysis.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 413, 'E-GEOD-38761', 23, 'Identification of DNA-replication domains in single S-phase cells', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', ''
  UNION SELECT 414, 'E-GEOD-38760', 23, 'Identification of DNA-replication domains in single S-phase cells: part 3 whole genome amplified single-cell DNA', 'DNA-replication is a key process in life and can lead to disease when disturbed. Cell-type specific early and late replication domains have been discovered throughout genomes by analysis of DNA from populations of cells. However, cell to cell differences and the association of these differences with other cellular processes remain largely elusive. Here we demonstrate for the first time that consecutive domains of early and late DNA-replication can be detected in single S-phase cells using array comparative genomic hybridization, providing proof-of-concept for a novel tool to investigate DNA-replication genome wide at the single-cell level. Furthermore, methods to profile the genome of a single cell for DNA-copy number aberrations are revolutionizing both basic genome research and clinical genetic diagnosis. It is thus important to apprehend not only technical but also biological reasons for false positive copy number detection. None of the current single-cell copy number calling methods distinguishes between a cell in G1-, S- or G2/M-phase of the cell cycle and mostly use cells isolated randomly from populations. We demonstrate that the oscillating pattern between early and late replicating loci instigates significantly more false-positive DNA copy-number calls in a diploid cell in S-phase cells than in G1- or G2/M-phase, depending on the specific aCGH-signal normalization method used. We propose a work-flow to detect single cells in S-phase and to correct for DNA-replication bias before copy number profiling. The genome of 1 S-phase, 2 M-phase and 1 G-phase single cells was amplified using the Sureplex amplification system. These test samples were hybridized comparatively to commercial male reference DNA,', '1132/NCBITaxon:9606', ''
  UNION SELECT 415, 'E-GEOD-38759', 23, 'Identification of DNA-replication domains in single S-phase cells: part 2 whole genome amplified single-cell', 'DNA-replication is a key process in life and can lead to disease when disturbed. Cell-type specific early and late replication domains have been discovered throughout genomes by analysis of DNA from populations of cells. However, cell to cell differences and the association of these differences with other cellular processes remain largely elusive. Here we demonstrate for the first time that consecutive domains of early and late DNA-replication can be detected in single S-phase cells using array comparative genomic hybridization, providing proof-of-concept for a novel tool to investigate DNA-replication genome wide at the single-cell level. Furthermore, methods to profile the genome of a single cell for DNA-copy number aberrations are revolutionizing both basic genome research and clinical genetic diagnosis. It is thus important to apprehend not only technical but also biological reasons for false positive copy number detection. None of the current single-cell copy number calling methods distinguishes between a cell in G1-, S- or G2/M-phase of the cell cycle and mostly use cells isolated randomly from populations. We demonstrate that the oscillating pattern between early and late replicating loci instigates significantly more false-positive DNA copy-number calls in a diploid cell in S-phase cells than in G1- or G2/M-phase, depending on the specific aCGH-signal normalization method used. We propose a work-flow to detect single cells in S-phase and to correct for DNA-replication bias before copy number profiling. The genome of 2 S-phase, 1 M-phase and 1 G-phase single cells were amplified using the Sureplex amplification system. These test samples were hybridized comparatively to commercial male reference DNA,', '1132/NCBITaxon:9606', ''
  UNION SELECT 416, 'E-GEOD-38758', 23, 'Identification of DNA-replication domains in single S-phase cells: part 1 whole genome amplified single-cell DNA', 'DNA-replication is a key process in life and can lead to disease when disturbed. Cell-type specific early and late replication domains have been discovered throughout genomes by analysis of DNA from populations of cells. However, cell to cell differences and the association of these differences with other cellular processes remain largely elusive. Here we demonstrate for the first time that consecutive domains of early and late DNA-replication can be detected in single S-phase cells using array comparative genomic hybridization, providing proof-of-concept for a novel tool to investigate DNA-replication genome wide at the single-cell level. Furthermore, methods to profile the genome of a single cell for DNA-copy number aberrations are revolutionizing both basic genome research and clinical genetic diagnosis. It is thus important to apprehend not only technical but also biological reasons for false positive copy number detection. None of the current single-cell copy number calling methods distinguishes between a cell in G1-, S- or G2/M-phase of the cell cycle and mostly use cells isolated randomly from populations. We demonstrate that the oscillating pattern between early and late replicating loci instigates significantly more false-positive DNA copy-number calls in a diploid cell in S-phase cells than in G1- or G2/M-phase, depending on the specific aCGH-signal normalization method used. We propose a work-flow to detect single cells in S-phase and to correct for DNA-replication bias before copy number profiling. The genome of 4 S-phase, 3 M-phase and 4 G-phase single cells was amplified using the Sureplex amplification system. These test samples were hybridized comparatively to commercial male reference DNA,', '1132/NCBITaxon:9606', ''
  UNION SELECT 417, 'E-GEOD-38757', 23, 'Identification of DNA-replication domains in single S-phase cells: part 2 multi-cell controls', 'DNA-replication is a key process in life and can lead to disease when disturbed. Cell-type specific early and late replication domains have been discovered throughout genomes by analysis of DNA from populations of cells. However, cell to cell differences and the association of these differences with other cellular processes remain largely elusive. Here we demonstrate for the first time that consecutive domains of early and late DNA-replication can be detected in single S-phase cells using array comparative genomic hybridization, providing proof-of-concept for a novel tool to investigate DNA-replication genome wide at the single-cell level. Furthermore, methods to profile the genome of a single cell for DNA-copy number aberrations are revolutionizing both basic genome research and clinical genetic diagnosis. It is thus important to apprehend not only technical but also biological reasons for false positive copy number detection. None of the current single-cell copy number calling methods distinguishes between a cell in G1-, S- or G2/M-phase of the cell cycle and mostly use cells isolated randomly from populations. We demonstrate that the oscillating pattern between early and late replicating loci instigates significantly more false-positive DNA copy-number calls in a diploid cell in S-phase cells than in G1- or G2/M-phase, depending on the specific aCGH-signal normalization method used. We propose a work-flow to detect single cells in S-phase and to correct for DNA-replication bias before copy number profiling. DNA was extracted from many cells from 1 S-phase, 1 G1-phase and 1 G2/Mphase enriched cell population from an EBV-transformed lymphoblastoid cell line from a male carrying a 750kb amplification on the short arm of chromosome 4. These test samples were hybridized comparatively to commercial male reference DNA.', '1132/NCBITaxon:9606', ''
  UNION SELECT 418, 'E-GEOD-38756', 23, 'Identification of DNA-replication domains in single S-phase cells: part 1 multi-cell controls', 'DNA-replication is a key process in life and can lead to disease when disturbed. Cell-type specific early and late replication domains have been discovered throughout genomes by analysis of DNA from populations of cells. However, cell to cell differences and the association of these differences with other cellular processes remain largely elusive. Here we demonstrate for the first time that consecutive domains of early and late DNA-replication can be detected in single S-phase cells using array comparative genomic hybridization, providing proof-of-concept for a novel tool to investigate DNA-replication genome wide at the single-cell level. Furthermore, methods to profile the genome of a single cell for DNA-copy number aberrations are revolutionizing both basic genome research and clinical genetic diagnosis. It is thus important to apprehend not only technical but also biological reasons for false positive copy number detection. None of the current single-cell copy number calling methods distinguishes between a cell in G1-, S- or G2/M-phase of the cell cycle and mostly use cells isolated randomly from populations. We demonstrate that the oscillating pattern between early and late replicating loci instigates significantly more false-positive DNA copy-number calls in a diploid cell in S-phase cells than in G1- or G2/M-phase, depending on the specific aCGH-signal normalization method used. We propose a work-flow to detect single cells in S-phase and to correct for DNA-replication bias before copy number profiling. DNA was extracted from many cells from 1 S-phase, 1 G1-phase and 1 G2/Mphase enriched cell population for two female normal control EBV-transformed lymphoblastoid cell lines. These test samples were hybridized comparatively to commercial male reference DNA.', '1132/NCBITaxon:9606', ''
  UNION SELECT 419, 'E-GEOD-37136', 23, 'Gene expression profiling of enforced HOXA1 expression in melanoma cell line', 'We recently reported an oncogenomics-guided screening approach designed to identify genetic drivers of early stage melanoma metastasis, and in this study we functionally validate the top-scoring candidate, homeobox transcription factor A1 (HOXA1), by demonstrating HOXA1s robust effects on melanoma cell invasion, metastasis and tumorigenicity.  Transcriptome and pathway profiling analyses of cells expressing HOXA1 reveal up-regulation of factors involved in diverse cytokine pathways that include the TGF? signaling axis, which we further demonstrate to be required for HOXA1-mediated cell invasion.  Transcriptome profiling also informed HOXA1s ability to potently down-regulate expression of microphthalmia-associated transcription factor (MITF) and other genes required for melanocyte differentiation, suggesting a mechanism by which HOXA1 expression de-differentiates cells into a pro-invasive precursor cell state concomitant with TGF? activation.  Our analysis of publicly available datasets indicate that the HOXA1-induced gene signature successfully categorizes melanoma specimens based on their metastatic potential and, importantly, is capable of stratifying melanoma patient risk for metastasis based on expression in primary tumors. The HOXA1-induced transcription analysis was conducted using RNAs extracted from WM115 cells transduced with either control or HOXA1, followed by hybridization of labeled cDNA onto Affymetrix GeneChips (Human Genome U133Plus2.0).', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 420, 'E-GEOD-31189', 23, 'Molecular Biomarker Signature for Bladder Cancer Detection', 'In this study we applied differential gene expression analysis to exfoliated human urothelia obtained from patients of known bladder disease status. Selected targets from the microarray data were validated in an independent set of samples using a quantitative PCR approach. Total RNA was extracted from 52 samples of human urothelial cancer cells (C) and from 40 samples of human non-cancer (NC) urothelial cells. Fragmented, biotinylated cRNA was hybridized to Affymetrix Human Genome U133 Plus 2.0 microarrays.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 421, 'E-GEOD-28471', 23, 'miR-29c Targets Tropomyosin-1? and Is Down-regulated in Rat Remnant Kidneys and IgA Nephropathy Patients with Interstitial Fibrosis', 'MicroRNAs (miRNAs) are small regulatory RNA molecules that modulate the activity of specific mRNA targets and play important roles in a wide range of physiologic and pathologic processes.  We hypothesized that miRNAs might be involved in the progression of CKD.  In our previous studies we found chronic renal damages developed progressively in rats with 5/6 nephrectomy.  L-mimosine(L-Mim) intervention from wk 5 to wk 12 improved renal function and resulted in additional accumulation of HIF-1 ? and -2 ? at wk 12.  In the current study we found miR-29c was up-regulated in the L-Mim treated group compared with the control using Agilent miRNA microarrays.  Of the microRNAs and proteins that exhibited reciprocal changes in expression following the L-Mim treatment, miR-29c and tropomyosin 1? (TPM1), which is involved in stress fiber function, met the sequence criteria for microRNA-target interaction,  were later confirmed by 3''-untranslated region reporter analysis.  TGF?1 treatment (3 ng/ml, 24 hours) decreased miR-29c expression and up-regulated protein expression of TPM1 in human renal epithelial cells. Overexpression of miR-29c significantly attenuated TGF-?1 induced increase in TPM1 in vitro. Moreover,  intrarenal expression of miR-29c was decreased in IgAN patients with moderate to severe tubulointerstital fibrosis (TIF), compared with IgAN patients without TIF, and intrarenal protein expression of TMP1 was significantly increased in IgAN patients with TIF.  The results suggest that intrarenal expression of miR-29c was down-regulated while its predicted target, TPM1 was up-regulated in the progression of CKD.  Short term stabilizing of HIF up-regulates miR-29c and attenuates CKD in the remnant kidney model. Four weeks after 5/6 nephrectomy, rats were treated with intraperitoneal injections of vehicle or L-mimosine (L-Mim, Calbiochem), a prolyl 4-hydroxylase inhibitor (PHD), at a dosage of 50 mg/kg every other day.  At the end of wk 12 after 5/6 nephrectomy, all rats (n=4, for each group) were sacrificed and blood samples were collected via cardiac puncture.  Renal tissue were harvested, one piece of which was fixed in neutral formalin and then embedded in paraffin.  The remaining kidney tissue was dissected in ice-cold PBS to, remove medulla, and then snap-frozen in liquid nitrogen before transferring to storage at -80°C until further analysis.', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 422, 'E-ERAD-130', 23, 'RNA-seq of Xenopus tropicalis rax mutants and wild type siblings', 'RNA was isolated from Xenopus tropicalis rax mutants and wild type siblings.  This was used to generate mutant and wild type RNA libraries for solexa sequencing.  The sequencing data will be compared to isolate changes that may be caused by loss of rax activity. The RNA samples were extracted with Trizol, then were DNAse treated following the Invitrogen DNAse I protocol and re-precipitated with ethanol. This data is part of a pre-publication release. For information on the proper use of pre-publication data shared by the Wellcome Trust Sanger Institute (including details of any publication moratoria), please see http://www.sanger.ac.uk/datasharing/', '', '1136/efo:EFO_0003738'
  UNION SELECT 423, 'E-GEOD-43307', 23, 'Feed restriction, but not L-carnitine infusion, affects the liver transcriptome with an evident and inhibition of energy production and sterol synthesis and increase of gluconeogenesis  in mid-lactating dairy cows', 'Feed restriction and L-carnitine infusion are known to affect the liver metabolism of dairy cows. In the present experiment the effects on liver transcriptome of feed restriction and L-carnitine abomasal infusion and the interaction of the two in mid-lactation Holstein dairy cows was assessed. Data clearly indicated a lack of transcriptomics effect by L-carnitine but a strong effect due to feed restriction. The functional analysis identified a overall reduction of cholesterol synthesis and oxidative phosphorylation and data suggested an increase flux toward gluconeogenesis and fatty acid oxidation. The liver biopsy was performed after 14 days of treatment in 8 Holstein dairy cows in a 2 x 2 factorial arrangment with 5 days washout between treatments. A dye-swap reference design (reference =  mixture of RNA from several bovine tissues) was used.', '1132/NCBITaxon:9913', '1136/efo:EFO_0002768'
  UNION SELECT 424, 'E-GEOD-43282', 23, 'Assessing global transcriptomic changes in response to South African cassava mosaic virus [ZA-99] infection in susceptible Arabidopsis thaliana', 'In susceptible plant hosts, co-evolution has favoured viral strategies to evade host defenses and utilize resources to their own benefit. The degree of manipulation of host gene expression is dependent on host-virus specificity and certain abiotic factors. In order to gain insight into global transcriptomic changes for a geminivirus pathosystem, South African cassava mosaic virus [ZA:99] (SACMV-ZA:99]) and Arabidopsis thaliana, 4 x 44K Agilent microarrays were adopted. After normalization, a 2-fold change filtering of data (p<0.05) identified 1,820 differentially expressed genes in apical leaf tissue. A significant increase in differential gene expression over time (451 genes at 14 dpi, 742 genes at 24 dpi, and 1011 genes at 36 dpi) was observed. This increase in expression, correlated with an increase in SACMV accumulation as virus copies were 5-fold higher at 24 dpi and 6-fold higher at 36 dpi than at 14 dpi (1.1x104 virus copies present at 14 dpi, 5.7x104 copies at 24 dpi, and 6.3x104 copies at 36 dpi). Many 2-fold genes were primarily involved in stress and defense responses, phytohormone signalling pathways, cellular transport, cell-cycle regulation, transcription, oxidation-reduction, and other metabolic processes. Forty-one genes (2.3%) were shown to be continuously expressed across the infection period, indicating that the majority of genes were transient and unique to a particular time point. Plant signalling networks were disrupted and manipulated by SACMV-[ZA:99] in order to affect homeostasis and antagonize hosts defense responses. At the same time, an adaptive response was initiated to reprogramme metabolism and divert energy from growth-related processes to defense, all leading to disruption of normal biological host processes. Comparisons between SACMV-[ZA:99] with plant-infecting RNA and DNA viruses revealed similarities and differences in expression patterns among viruses, showing either general defense or virus-specific responses. Within the Geminiviridae family in particular, similarities in cell-cycle regulation and gene expression patterns correlated between SACMV-[ZA:99] and Cabbage leaf curl virus (CaLCuV) but differences were also evident. For instance, CaLCuV showed antagonistic interactions between Salicyclic Acid (SA) and Jasmonic Acid (JA) pathways, whereas SACMV displayed synergism. Differences in gene induction, repression and outcome between the two geminiviruses clearly demonstrated host-specific interactions with SACMV-[ZA:99] leading to infection. To our knowledge this is the first geminivirus study identifying differentially expressed transcripts across 3 time points A three time-point (14, 24, and 36 dpi) study was carried out to identify differentially expressed genes in SACMV-[ZA:99] infected Arabidopsis leaf cells using a direct comparison design against mock-inoculated controls. Three biological replicates and 1 technical replicate for both SACMV-[ZA:99]-infected and mock-inoculated controls were conducted at each time point.', '1132/NCBITaxon:3702', '1136/efo:EFO_0002768'
  UNION SELECT 425, 'E-GEOD-43208', 23, 'Niche-dependent expression profiles of intratumoral heterogenous ovarian clear cell carcinoma cells', 'Exploring the expression profile of ovarian clear cell carcinoma cancer cell subpopulations- derived tumors grown within a murine and a human cellular tissues. RNA samples were obtained from laser micro-dissected from two cancer cell subpopulations (CCSPs C12 and C13)- derived tumors that developed within a murine (intratumoral) and a human (intra-teratoma) and from  in vitro grown cells;  a total of 9 RNA samples of CCSP''s C12 and 9 RNA samples of CCSP''s C13 .', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 426, 'E-GEOD-41937', 23, 'Dual Functions of TAF7L in Adipocyte Differentiation', 'The diverse transcriptional mechanisms governing cellular differentiation and development of mammalian tissue remains poorly understood. Here we report that TAF7L, a paralogue of TFIID subunit TAF7, is enriched in adipocytes and mouse white fat tissue (WAT). Depletion of TAF7L reduced adipocyte-specific gene expression and compromised adipocyte differentiation as well as WAT development. Ectopic expression of TAF7L in myoblasts reprograms these muscle precursors into adipocytes upon induction. Genome-wide mRNA-seq expression profiling and ChIP-seq binding studies confirmed that TAF7L is required for activating adipocyte-specific genes via a dual mechanism wherein it interacts with PPAR? at enhancers and TBP/Pol II at core promoters. In vitro binding studies confirmed that TAF7L forms complexes with both TBP and PPAR?. These findings suggest that TAF7L plays an integral role in adipocyte gene expression by targeting enhancers as a cofactor for PPAR? and promoters as a component of the core transcriptional machinery. Genome-wide mapping of TAF7L and additional factors, and mRNA-seq expression profiling prior to and following mouse adipocyte differentiation.', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 427, 'E-GEOD-40767', 23, 'Genome-wide mapping of FOXM1 binding reveals co-binding with estrogen receptor alpha in breast cancer cells', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 428, 'E-GEOD-40766', 23, 'Genome-wide mapping of FOXM1 binding reveals co-binding with oestrogen receptor alpha in breast cancer cells (expression)', 'The forkhead transcription factor FOXM1 is a key regulator of the cell cycle and is overexpressed in cancer. Increased levels of FOXM1 are associated with both poor prognosis and oestrogen receptor (ERalpha) status in primary breast cancer. In this study, we map FOXM1 binding genome wide in both ERalpha-positive (MCF-7) and -negative (MDA-MB-231) breast cancer cells. We identify a common set of FOXM1 binding events at cell cycle-regulating genes, but in addition, in MCF-7 cells we find a high level of concordance with ERalpha-binding regions. FOXM1 binding at these co-binding sites is dependent on ERalpha binding, as depletion of ER protein levels reduced FOXM1 binding. FOXM1 interacts directly with both ERalpha co-activator CARM1 and is required for H3 arginine methylation at the ERalpha complex. Inhibition of FOXM1 activity with the ligand thiostrepton resulted in decreased FOXM1 binding at cca. 1400 sites genome wide and reduced expression of genes correlated with poor prognosis in ERalpha-positive tumour samples. These data demonstrate a novel role for the forkhead protein FOXM1 as an ERalpha cofactor and provide insight into the role of FOXM1 in ERalpha-positive breast cancer. MCF-7 cells were treated with either thiostrepton or DMSO for 6 hr. Each treatment was carried out in replicates of 6.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 429, 'E-GEOD-40762', 23, 'Genome-wide mapping of FOXM1 binding reveals co-binding with oestrogen receptor alpha in breast cancer cells (ChIP-seq)', 'The forkhead transcription factor FOXM1 is a key regulator of the cell cycle and is overexpressed in cancer. Increased levels of FOXM1 are associated with both poor prognosis and oestrogen receptor (ERalpha) status in primary breast cancer. In this study, we map FOXM1 binding genome wide in both ERalpha-positive (MCF-7) and -negative (MDA-MB-231) breast cancer cells. We identify a common set of FOXM1 binding events at cell cycle-regulating genes, but in addition, in MCF-7 cells we find a high level of concordance with ERalpha-binding regions. FOXM1 binding at these co-binding sites is dependent on ERalpha binding, as depletion of ER protein levels reduced FOXM1 binding. FOXM1 interacts directly with both ERalpha co-activator CARM1 and is required for H3 arginine methylation at the ERalpha complex. Inhibition of FOXM1 activity with the ligand thiostrepton resulted in decreased FOXM1 binding at cca. 1400 sites genome wide and reduced expression of genes correlated with poor prognosis in ERalpha-positive tumour samples. These data demonstrate a novel role for the forkhead protein FOXM1 as an ERalpha cofactor and provide insight into the role of FOXM1 in ERalpha-positive breast cancer. The FOXM1-binding sites were mapped by ChIP-Seq in MCF-7 and MDA-MB-231 cells. Cells were treated either with thiostrepton, a FOXM1 inhibitor, or with DMSO (as control). Four replicates were performed in MCF7 cells and two replicates in MDA-MB-231 cells.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002692'
  UNION SELECT 430, 'E-GEOD-38995', 23, 'Juvenile postnatally overfed mice heart microarray anaylsis', 'C57BL/6 mice underwent early postnatal overfeeding by litter size reduction (3 pups/dam, instead of 10 pups/dam, in postatally normally fed mice). These overfeeding conditions lead to early and permanent modifications of body weight and to an increase in body fat mass in the adulthood. When these postnatally overfed mice become adult, they develop several cardio-metabolic alterations (mild hypertension, impairment of heart contractility, remodeling, fibrosis, hyperinsulinemia and hyperleptinemia) but also increases in markers of cardiac and plasma oxidative stress. Moreover, we observed a higher sensitivity of the heart to injuries induced by ischemia reperfusion, such as an increase in necrosis. We now aim to establish if these late-coming modifications might be induced in a very early state of postnatal development, such as a postnatal programming situation. To achieve this objective, we are now comparing heart gene expression between 24 days old postnatally overfed and normally fed mice. Comparison of heart gene expression between 24 days old postnatally overfed and normally fed mice.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 431, 'E-GEOD-38573', 23, 'Expression data from cerebrum in a spontaneous mutant mouse, laggard.', 'We found a new spontaneous mutant mouse, laggard, characterized by general weakness in movements and retardation in growth. We used microarray to identify genes whose expression levels are affected downstream from the mutation. RNA was extracted from the cerebrum of each mutant mouse or its littermate, performing hybridization to Affymetrix DNA chip with cRNA probe. The profiles were compared comprehensively between them to investigate the molecular pathology in the mutant mouse.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 432, 'E-GEOD-38261', 23, 'Distinct Transcriptional Signatures of Bone Marrow-Derived C57BL/6 and DBA/2 Dendritic Leucocytes Hosting Live Leishmania amazonensis Amastigotes', 'To determine the modulation of gene expression of  C57BL/6 and DBA/2 BMDLs in the presence of living intracellular Leishmania amazonensis amastigotes A genome-wide transcriptional analysis was performed by comparing the gene expression profiles of control DLs and live amastigote-hosting DLs from both mouse strains. Dendritic Leucocytes were generated in vitro from bone marrow progenitors (C57Bl/6 and DBA/2 mice). Leishmania amazonensis amastigotes were purified from mouse cutaneous lesions and were added to DL cultures. After 24h, and following a sorting procedure, only BMDls housing living amastigotes were selected for total RNA extraction. Three Biological replicates per condition were run.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 433, 'E-GEOD-38212', 23, 'Transcriptomics studies in Eugenia uniflora', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:119951', '1136/efo:EFO_0003737'
  UNION SELECT 434, 'E-GEOD-38211', 23, 'Identification of miRNAs from Eugenia uniflora by high-throughput sequencing and bioinformatics analysis [small RNA-seq]', 'miRNAs are small non-coding regulatory RNAs that play important functions in the regulation of gene expression at the post-transcriptional level by targeting mRNAs for degradation or by inhibiting protein translation. Eugenia uniflora is a plant native to tropical America with pharmacological and ecological importance without previous studies about its gene expression and regulation. To date, there is not miRNAs reported in species of Myrtaceae. A small RNA library was constructed to identify miRNAs in Eugenia uniflora. Solexa technology was used to perform high throughput sequencing of the library and the data obtained was analysed using bioinformatics tools. From 14,489,131 clean reads, we obtained 1,852,722 small RNAs representing 45 known miRNA families that have been identified in other plant species. Further analysis using contigs assembled from Illumina mRNA sequencing of leaves from the same individual allowed the prediction of secondary structures of 25 known and 17 novel miRNAs. Potential targets were predicted for the most abundant mature miRNAs in the identified pre-miRNAs based on sequence homology. This study provide the first large scale identification of miRNAs and their potential targets of a species from Myrtaceae without previous genomic sequence resources. Our study provides more information about the evolutionary conservation of the regulatory network of miRNAs in plants and highlights the miRNAs species-specific. microRNA profiles in 1 leaf library of Eugenia uniflora by deep sequencing (Illumina HiSeq2000)', '1132/NCBITaxon:119951', '1136/efo:EFO_0003737'
  UNION SELECT 435, 'E-GEOD-38210', 23, 'Identification of miRNAs from Eugenia uniflora by high-throughput sequencing and bioinformatics analysis [mRNA-seq]', 'miRNAs are small non-coding regulatory RNAs that play important functions in the regulation of gene expression at the post-transcriptional level by targeting mRNAs for degradation or by inhibiting protein translation. Eugenia uniflora is a plant native to tropical America with pharmacological and ecological importance without previous studies about its gene expression and regulation. To date, there is not miRNAs reported in species of Myrtaceae. A small RNA library was constructed to identify miRNAs in Eugenia uniflora. Solexa technology was used to perform high throughput sequencing of the library and the data obtained was analysed using bioinformatics tools. From 14,489,131 clean reads, we obtained 1,852,722 small RNAs representing 45 known miRNA families that have been identified in other plant species. Further analysis using contigs assembled from Illumina mRNA sequencing of leaves from the same individual allowed the prediction of secondary structures of 25 known and 17 novel miRNAs. Potential targets were predicted for the most abundant mature miRNAs in the identified pre-miRNAs based on sequence homology. This study provide the first large scale identification of miRNAs and their potential targets of a species from Myrtaceae without previous genomic sequence resources. Our study provides more information about the evolutionary conservation of the regulatory network of miRNAs in plants and highlights the miRNAs species-specific. mRNA profiles in 1 leaf library of Eugenia uniflora by deep sequencing (Illumina HiSeq2000)', '1132/NCBITaxon:119951', '1136/efo:EFO_0003738'
  UNION SELECT 436, 'E-GEOD-34119', 23, 'Aberrant microRNA expression in radiation-induced rat mammary cancer: the potential role of miR-194 overexpression in cancer cell proliferation.', 'Aberrant expression of microRNAs (miRNAs) is frequently associated with a variety of cancers, including breast cancer. We and others have demonstrated that radiation-induced rat mammary cancer exhibits a characteristic gene expression profile and a random increase in aberrant DNA copy number; however, the role of aberrant miRNA expression is unclear. We performed a microarray analysis of frozen samples of eight mammary cancers induced by gamma-irradiation (2 Gy), eight spontaneous mammary cancers, and seven normal mammary samples. We found that a small set of miRNAs was characteristically overexpressed in radiation-induced cancer. Quantitative RT-PCR analysis confirmed that miR-135b, miR-192, miR-194, and miR-211 were significantly upregulated in radiation-induced mammary cancer compared with spontaneous cancer and normal mammary tissue. The expression of miR-192 and miR-194 also was upregulated in human breast cancer cell lines compared with non-cancer cells. Manipulation of the miR-194 expression level using a synthetic inhibiting RNA produced a small but significant suppression of cell proliferation and upregulation in the expression of several genes that are suggested to act as tumor suppressors in MCF-7 and T47D breast cancer cells. Thus, the induction of rat mammary cancer by radiation involves aberrant expression of miRNAs, which may favor cell proliferation. We performed miRNA microarray analysis on mammary carcinomas in Sprague-Dawley rat to identify radiation-specific miRNA expression patterns compared with spontaneous mammary carcinomas and normal mammary tissues.', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 437, 'E-ERAD-59', 23, 'RNA-seq of mouse to investigate the immune response to Schistosomiasis', 'Schistosomiasis is a severe debilitating illness that represents one of the most important parasitic diseases in tropical areas. It affects over 200 million people across 74 countries. The complete genome sequence of Schistosoma mansoni was published in 2009, but immune response against this parasite is not well characterized. We will study the overall immune response, antibody and TCR repertoire of immunologically nave mice, mice infected with S.mansoni and mice challenged with a secondary schistosome infection.We plan to use RNA-seq to characterise the systemic immune response of mouse to Schistosoma infection, including both whole-transcriptome sequencing for a global picture of the immune response, and to sequence the  lymphocyte antigen receptor genes to get a detailed picture of the adaptive immune response.  B and T lymphocytes are the most important cells in this response, as they are responsible for recognising immune targets through the specificities of their antigen receptors. Antibodies and TCR are formed by recombination among gene segments (V, D and J), sequence diversification at the junctions, and somatic hypermutation throughout the genes. Two complementary sequencing strategies will allow us to investigate these two aspects of the immune response. RNA-seq data should give novel insight into the global response, as well as acting as a control for the PCR of lymphocyte receptor genes, by allowing us to quantify and correct for any bias in the PCR. rtPCR of lymphocyte receptor mRNA (initially immunoglobulins, but later we hope to investigate the T-cell receptor also) followed by 454 sequencing should allow us to characterise the antigen receptor repertoire and its response to schistosome infection.  A number of different pools of redundant PCR primers have been described for the amplification of lymphocyte receptor genes, including one known to work in BALB/c mice (Nat. Biotech. 28:965-969). Muros laboratory in Salamanca is performing the infections and RNA extraction, and has provided phenotypic data and immunological measurements that we hope to relate to the expression data we will generate. Multiple replicates are needed to quantify the variation in immune response observed in animals with different intensity of infection. For the total RNA sequencing, samples will be multiplexed on illumina lanes. It is difficult to statistically design experiments of this kind without preliminary data on the effect size (change in expression between infected and uninfected mice), but initial inspection of RNAseq experiments of spleen in uninfected mice shows very high expression of many immune-related genes. 1/16 plate of 454 sequencing will give about 37,500 reads per sample, which will give us a 98% chance of detecting any transcript present at over 0.01% frequency in the repertoire of any sample. This is clearly a small sampling of the estimated 109 antibody-producing B cells in a mouse, and we do not expect to be able to characterise the entire repertoire, but to detect shared antibody responses between infections. The power of this approach depends on how polarised the immunoglobulin repertoire becomes in response to infection. The only published experimental evidence that seems to give us any handle on to what extent shared antibody molecules are expressed in different individuals (Nat. Biotech. 28:965-969) finds particular immunoglobulin sequences at frequencies of between 1% and 10% of the repertoire in mice immunised with a defined antigens, present in multiple mice. Evidence about the nave state comes from an analysis of the (admittedly rather simpler zebrafish repertoire), where  (Science 324:807-810) a relatively small number of reads capture the vast majority of VDJ diversity.', '1132/NCBITaxon:10090', '1136/efo:EFO_0003738'
  UNION SELECT 438, 'E-ERAD-52', 23, 'RNA-seq of different organism parts from different reptile species', 'High-throughput sequencing of RNA from secretory tissues of reptiles.Tissues were dissected from freshly-euthanised individuals and either snap-frozenin liquid nitrogen or stored short-term in RNAlater (Ambion). Total RNA was extracted from approximately 30mg of tissue using the Qiagen RNeasy Mini kit according to manufacturer instructions, with on-column DNase treatment. Typically, single whole scent glands and venom/salivary glands were used in the relevant extractions and skin samples were taken from the dorsal side at approximately mid-body level.This data is part of a pre-publication release. For information on the proper use of pre-publication data shared by the Wellcome Trust Sanger Institute (including details of any publication moratoria), please see http://www.sanger.ac.uk/datasharing/', '1132/NCBITaxon:51751', '1136/efo:EFO_0003738'
  UNION SELECT 439, 'E-ERAD-48', 23, 'Methylation profiling of mouse cells to investigate DNA methylation reprogramming in PGCs', 'This study focuses on epigenetic reprogramming in the mouse germ line: DNA methylation marks, including those of imprinted genes, are thought to be erased between E11.5 and E13.5 in primordial germ cells (PGCs), which are the direct progenitors of sperm or oocyte. DNA methylation patterns are then re-established during the de novo methylation phase in the male germ line several days later (around E15.5) and in the female germ line after birth during adult life. Epigenetic reprogramming in PGCs is poorly understood mainly because of the technical challenges that arise from very low cell numbers in the embryo. The aim of this study is to create genome-wide maps of DNA methylation patterns of male and female PGCs at crucial time points during epigenetic reprogramming and to investigate the changes in those profiles on a single-gene level. We have already successfully prepared and sequenced the first set of BS-Seq libraries of PGCs with Illumina''s GAIIx platform. From this, we gained significant insight into the global DNA methylation levels and methylation patterns over multy-copy-loci such as repeat elements. In order further enhance our analysis and finish this project for publication, it is crucial to increase the genomic coverage of our datasets. This will also allow us to link the BS-Seq data with other MeDIP-Seq and RNA-Seq datasets that have already been created in this study.Protocol: Input DNA was sonicated using a Bioruptor UCD-200 (Diagenode) to a final size distribution of 300bp  1000bp. End-repair and A-tailing were performed with the NEBNext DNA Sample Prep Master Mix Set 1. Illuminas Early Access Methylation Adaptor Oligo Kit was used for the adapter ligation. The adapter-ligated DNA was treated with sodium-bisulfite using the Imprint DNA Modification Kit from Sigma-Aldrich according to the manufacturers instructions for the two-step protocol. After the clean up, the bisulfite-treated DNA was amplified using PfuTurbo Cx Hotstart DNA Polymerase from Agilent with 18 cycles of amplification. The entire PCR reaction was run on a 1.5% agarose TBE gel and size selection was performed for DNA fragments between 200bp  250bp. DNA from the excised gel piece was purified with the Qiagen Gel Extraction Kit.This data is part of a pre-publication release. For information on the proper use of pre-publication data shared by the Wellcome Trust Sanger Institute (including details of any publication moratoria), please see http://www.sanger.ac.uk/datasharing/', '1132/NCBITaxon:10090', '1136/efo:EFO_0002761'
  UNION SELECT 440, 'E-GEOD-41691', 23, 'Early limb bud transcriptome analysis between anterior and posterior domain', 'Shh signal mediated by Gli family of transcription factors regulates digit growth and patterning in early limb development. Shh expression in the posterior margin of the limb bud defines the zone of polarizing activity. However, much less is know about downstream targets that mediate Shh signal functions. In this dataset, we include the expression data obtained from dissected anterior and posterior halves of mouse limb bud respectively. These data are used to obtain 889 transcripts that were upregulated 1.3 fold or more in the posterior limb bud, and 1189 transcripts that were enriched in the anterior limb bud at 1.3 fold or more. Two samples were analyzed. We generate pairwise comparisons between anterior and posterior limb tissues. Genes with a fold-change ?1.3 were selected.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 441, 'E-GEOD-41232', 23, 'Adaptive evolution of Corynebacterium glutamicum resistant to oxidative stress and its global gene expression profiling', 'Corynebacterium glutamicum was adapted in a chemostat for 1,900 h with gradually increasing H2O2 stress to understand the oxidative stress response of an industrial host. After 411 generations of adaptation, C. glutamicum developed the ability to grow under stress of 10 mM H2O2, whereas the wild-type did not. The adapted strain also showed increased stress resistance to diamide and menadione, SDS, Tween 20, HCl, NaOH, and ampicillin. A total of 1,180 genes in the RNA-seq transcriptome analysis of the adapted strain were up-regulated twice or higher (corresponding to 38.6 % of the genome), and 126 genes were down-regulated half or less (4.1 % of genome) under 10 mM H2O2-stress conditions compared with those of the wild-type under a no stress condition. Especially the aromatic compound-degrading gene clusters (vanRABK, pcaJIRFLO, and benABCDRKE) were more than threefold up-regulated. Plausible reasons for the H2O2-stress tolerance of the adapted strain are discussed as well as the potential strategy for development of oxidative stress-tolerant strain. Comparison of global gene expression profiling using RNA-seq data of wild-type strain under normal condition (WT_MCGC) vs. 10 mM H2O2-adapted strain under oxidative stress condition Global gene expression profiling_WT vs. H2O2 adapted C. glutamicum strains', '1132/NCBITaxon:196627', '1136/efo:EFO_0003738'
  UNION SELECT 442, 'E-GEOD-39323', 23, 'Ogt chromatin recruitment is mediated by TET proteins in mouse ES cells', 'This SuperSeries is composed of the SubSeries listed below. Refer to individual Series', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 443, 'E-GEOD-39321', 23, 'Ogt chromatin recruitment is mediated by TET proteins in mouse ES cells [expression array]', 'O-linked N-acetylglucosamine (O-GlcNAc ) transferase (OGT) activity is essential for embryonic stem (ES) cell viability and mouse development. OGT is present in both cytoplasm and nucleus of different cell types and mediates serine or threonine glycosylation. The Ogt gene locus resides on the X-chromosome and its activity is required for the viability of male ES cells. Using Ogt conditional knock out (KO) ES cells it was shown the failure of establishing stable KO ES clones further suggesting that Ogt activity is required for ES cell self-renewal and pluripotency. For understanding these changes, we performed global gene expression upon silencing of Ogt mediated by esiRNA in mouse Embryonic Stem Cells. Total RNA was extracted from E14 ES cells (derived from Ola129 mice) transiently transfected with Ogt specific and control esiRNA oligos 48 hrs. post-transfection.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 444, 'E-GEOD-39159', 23, 'Skeletal muscle gene expression data from Down syndrome mice', 'Persons with Down syndrome (DS) exhibit low muscle strength that significantly impairs their physical functioning. The Ts65Dn mouse model of DS also exhibits muscle weakness in vivo and may serve as a useful model to examine potential factors responsible for DS-associated muscle dysfunction. Therefore, the purpose of this experiment was to directly assess skeletal muscle function in the Ts65Dn mouse and to reveal potential mechanisms of DS-associated muscle weakness. Soleus muscles were harvested from anesthetized male Ts65Dn and wild-type (WT) colony controls.  In vitro muscle contractile experiments revealed normal force generation of unfatigued Ts65Dn soleus, but a 12% reduction in force was observed in Ts65Dn muscle during recovery following fatiguing contractions compared to WT muscle (p<0.05). Oxidative stress may contribute to DS-related pathologies, including muscle weakness, which may be the result of overexpression of chromosome 21 genes (e.g., copper-zinc superoxide dismutase (SOD1)). SOD1 expression was 25% higher (p<0.05) in Ts65Dn soleus compared to WT muscle but levels of other antioxidant proteins were unchanged. Lipid peroxidation (4-hydroxynoneal) was unaltered in Ts65Dn muscle although protein carbonyls were 20% greater compared to muscle of WT animals (p<0.05). Cytochrome c oxidase expression was reduced 22% in Ts65Dn muscle, suggesting a limitation in mitochondrial function may contribute to post-fatigue muscle weakness. Microarray analysis of Ts65Dn soleus revealed alteration of numerous cellular pathways including: proteolysis, glucose and fat metabolism, neuromuscular transmission, and ATP biosynthesis. In summary, the Ts65Dn mouse displays evidence of muscle dysfunction, and the potential role of mitochondria and oxidative stress warrants further investigation. We used microarrays to gain insight into potential mechanisms of DS-associated skeletal muscle weakness. Soleus muscles were obtained from four male Ts65Dn (DS) mice and four male colony control mice. Animals were apparently healthy and approximately five months old. The muscles were harvested in the basal state under anesthesia induced by sodium pentobarbital at approximately the same time of the day.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 445, 'E-GEOD-39154', 23, 'Ogt chromatin recruitment is mediated by TET proteins in mouse ES cells [ChIP-Seq]', 'These data include the genome wide location analysis of Ogt by ChIP of Ogt in mouse ES cells. Precipitation of formaldehyde cross-linked chromatin prepared from mouse ES cell using specific antibody against OGT and IgG as control.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002692'
  UNION SELECT 446, 'E-GEOD-34923', 23, 'In vivo nucleosome occupancy in yeast (MNase-seq)', 'Open chromatin provides access to a wide spectrum of DNA binding proteins for DNA metabolism processes such as transcription, repair, recombination, and replication. In this regard, open chromatin profiling has been widely used to identify the location of regulatory regions, including promoters, enhancers, insulators, silencers, replication origins, and recombination hotspots. Regulatory DNA elements are made accessible by nucleosome-depeleted states. Thus, nucleosome remodelling and modification should be intimately coupled with open chromatin formation and regulation. However, our knowledge of nucleosome regulation is largely limited to promoter regions, which comprise only a subset of all regulatory loci in the genome. In order to examine nucleosome patterns in open chromatin regions, we performed micrococcal nuclease (MNase) sequencing for a laboratory strain of yeast.  Nucleosome occupancy profiled by Micrococcal nuclease (MNase) digestion', '1132/NCBITaxon:4932', '1136/efo:EFO_0002692'
  UNION SELECT 447, 'E-GEOD-32909', 23, 'Human transcriptome analysis of acute responses to glucose ingestion reveals a role of leukocytes in hyperglycemia induced inflammation.', 'Objective was to examine acute gene expression responses to physiologic oral glucose ingestion in human circulating leukocytes. Microarray study of human circulating leukocytes sampled before, 1 hour after and 2 hours after glucose ingestion was performed. The present study demonstrated 36 genes which showed acute gene expression change in human leukocytes within 1 hour after glucose ingestion and suggest that leukocytes participate in the inflammatory process induced by acute hyperglycemia. Microarray study of human circulating leukocytes sampled before, 1 hour after and 2 hours after glucose ingestion', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 448, 'E-GEOD-32422', 23, 'Expression data from the dorsal and lateral lobes of the prostates of TRAMP mice treated with OSU-CG5', 'Cells undergoing malignant transformation often shift their cellular metabolism from primarily oxidative phosphorylation to aerobic glycolysis (the Warburg effect). Energy restriction-mimetic agents (ERMAs), such as 2-deoxyglucose and resveratrol, that target this shift in cellular metabolism have been effective in inhibiting cancer cell growth in vitro, and xenograft tumor growth in vivo. We recently developed a novel ERMA, OSU-CG5, that exhibits higher in vivo activity than previously described ERMAs. In this study, we investigated the effect of OSU-CG5 on global gene expression in the dorsal and lateral lobes of the prostate of transgenic adenocarcinoma of the mouse prostate (TRAMP) mice, and on its ability to suppress lesion progression in these mice. Intact male TRAMP mice were randomized into 2 groups and treated for 4-weeks (from age 6 to 10 weeks) with a vehicle or 100 mg/kg/day OSU-CG5 via oral gavage. At the end of the study, the urogenital tracts were collected and prostates microdissected. RNA was extracted from the dorsal and lateral lobes of the prostates from 3 mice per group, and Affymetrix microarrays were performed.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 449, 'E-MTAB-930', 23, 'ApoE study total (20 and 40 wks)', 'Local heart irradiation of 2, 8 or 16 Gy to ApoE-/- mice and follow up tp 20 and 40 weeks. Th goal of this study is to understand microvascular alterations of radiation-induced cardiactoxicity.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 450, 'E-GEOD-43301', 23, 'IM005 - Mouse infection with PB2-627E: A/Vietnam/1203-CIP048_RG2/2004 (H5N1)', 'To determine the host response (C57BL/6 mouse model) to the PB2-627E: A/Vietnam/1203-CIP048_RG2/2004 (H5N1) virus at 10^4 PFU. Groups of 20-week-old female C57BL/6 mice were infected with the PB2-627E: A/Vietnam/1203-CIP048_RG2/2004 (H5N1) virus at 10^4 PFU or time-matched mock infected. Time points were 1, 2, 4 and 7 d.p.i. There were 5 mice/time point for infections and 3 mice/time point for time-matched mocks. Lung samples were collected for virus load, transcriptional analysis and proteomics analysis. Weight loss and animal survival were also monitored.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 451, 'E-GEOD-43300', 23, '1,25-Dihydroxyvitamin D Promotes Negative Feedback Regulation of Toll-Like Receptor Signaling via Targeting MicroRNA-155-SOCS1 in Macrophages', 'The negative feedback mechanism is essential to maintain effective immunity and tissue homeostasis. 1,25-dihydroxyvitamin D (1,25(OH)2D3) modulates innate immune response, but the mechanism remains poorly understood. Here we report that vitamin D receptor (VDR) signaling attenuates Toll-like receptor-mediated inflammation by enhancing the negative feedback inhibition. VDR inactivation leads to a hyperinflammatory response in mice and macrophage cultures when challenged with lipopolysaccharide (LPS) due to miR-155 overproduction that excessively suppresses SOCS1, a key regulator that enhances the negative feedback loop. Deletion of miR-155 attenuates vitamin D suppression of LPS-induced inflammation, confirming that 1,25(OH)2D3 stimulates SOCS1 by down-regulating miR-155. 1,25(OH)2D3 down-regulates bic transcription by inhibiting NF-kappaB activation, which is mediated by a kappaB cis-DNA element located within the first intron of the bic gene. Together these data identify a novel regulatory mechanism for vitamin D to control innate immunity. MicroRNA arrays. Total RNAs were extracted from RAW264.7 cells (mouse macrophage line) treated with LPS (100ng/ml) in the presence or absence of 1,25(OH)2D3 (20 nM) overnight. MicroRNA profiling was performed using the miRCURY LNA microRNA Arrays (Exiqon, Vedvaek,Denmark) according to the manufacture''¹s standard protocols. The arrays were scanned with the GenePix 4000B scanner using the manufacturer''s recommended settings. The raw data was extracted using GenePix Pro and imported into GeneSpring GX10 for analyses.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 452, 'E-GEOD-43290', 23, 'Expression data from meningiomas and normal meninges', 'Correlate the gene expression profiles with the most relevant patterns of chromosome abnormalities (cytogenetic subgroups of meningiomas) and the gene expression profiles could help to explain the differences in clinical behaviour of meningiomas. The impact of tumor cytogenetics on the gene expression analyzed in meningiomas. Here, we analyzed the gene expression profiles (GEP)  of 47 tumors and correlated them with the most clinical relevant cytogenetic subgroups of meningiomas, including diploid tumors, isolated -22/22q-, del(1p36) alone and complex karyotypes associated with del(1p36) and/or -14q. In addition, 4 samples containing purified RNA extracted from normal meningeal cells (BioChain Institute, Hayward, CA and US Biological, Swampscott, MA, USA) were also processed.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 453, 'E-GEOD-43287', 23, 'Expression data from P0 forebrain of wild-type and Elavl3/Elavl4 double knockout mice.', 'Neuronal Elavl-like (nElavl) RNA binding proteins have three paralogs (Elavl2, Elavl3 and Elavl4) in the mouse genome. This family of RNABPs have been implicated in a variety of post-transcriptional RNA processing mechanisms, including regulation of mRNA stability, alternative splicing, and translational regulation. In this study, using mouse exon arrays, we identify significant differences (p<0.01) in 119 transcripts between wild-type and Elavl3/Elavl4 double knockout forebrain tissue at postnatal day 0. A total of 10 samples were analyzed.  These samples consisted of 5 littermate pairs of wild-type and Elavl3/Elavl4 double knockout mice.', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 454, 'E-GEOD-43204', 23, 'ICL0012 - Host response in human Calu-3 cells to infection with NS1trunc124: A/Vietnam/1203-CIP048_RG4/2004 (H5N1) or WT: A/Vietnam/1203/2004 (H5N1)', 'Differential expression was determined in Calu-3 cells between mock infected and infection with NS1trunc124: A/Vietnam/1203-CIP048_RG4/2004 (H5N1) or WT: A/Vietnam/1203/2004 (H5N1) at different times post infection. Purpose: To obtain samples for transcriptional analysis in triplicate using the VN1203 pathogenicity mutant NS1-trunc124.  Overview of Experiment: . Time Points: 0, 3, 7, 12, 18 and 24 hrs post infection. . Done in triplicate. . Triplicates are defined as 3 different wells, plated at the same time using the same cell stock for all replicates. . Time-matched mocks were done in triplicate from the same cell stock as the rest of the samples. . Culture medium (the same as what the virus stock is in) was used for the mock infections.  Calu-3 cells were infected with NS1trunc124: A/Vietnam/1203-CIP048_RG4/2004 (H5N1) or mock infected and samples were collected at 0, 3, 7, 12, 18 and 24 hpi. Calu-3 cells were infected with WT: A/Vietnam/1203/2004 (H5N1) and samples were collected at 7 and 24 hpi. There are 3 mock (2 included here) and 3 infected replicates for each time point. Expression profiles were determined.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 455, 'E-GEOD-43203', 23, 'ICL0011 - Host response in human Calu-3 cells to infection with A/Vietnam/1203-CIP048_RG3/2004 (H5N1) (PB1-F2 deletion), A/Vietnam/1203-CIP048_RG3/2004 (H5N1) (PB2-627E mutant) or WT: A/Vietnam/1203/2004 (H5N1)', 'Differential expression was determined in Calu-3 cells between mock infected and infection with one of 3 Influenza viruses (wild-type VN1203, VN1203 mutant PB1-F2del, VN1203 mutant PB2-627E) at different times post infection. Purpose: To obtain samples for transcriptional analysis in triplicate using the VN1203 pathogenicity mutants PB1-F2del and PB2-627E.  Overview of Experiment: . Time Points: 0, 3, 7, 12, 18 and 24 hrs post infection. . There are two time points for wild type VN1203. . Done in triplicate. . Triplicates are defined as 3 different wells, plated at the same time using the same cell stock for all replicates. . Time-matched mocks were done in triplicate from the same cell stock as the rest of the samples. . Culture medium (the same as what the virus stock is in) was used for the mock infections.  Calu-3 cells were infected with A/Vietnam/1203-CIP048_RG3/2004 (H5N1) (PB1-F2 deletion), A/Vietnam/1203-CIP048_RG3/2004 (H5N1) (PB2-627E mutant) or mock infected and samples were collected at 0, 3, 7, 12, 18 and 24 hpi. Calu-3 cells were infected with WT: A/Vietnam/1203/2004 (H5N1) and samples were collected at 7 and 24 hpi. There are 3 mock and 3 infected replicates for each time point. Expression profiles were determined.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 456, 'E-GEOD-42908', 23, 'Direct reprogramming of human astrocytes into neural stem cells and neurons (MeDIP)', 'Ectopic expression of the reprogramming factors OCT4, SOX2, or NANOG into human astrocytes in specific cytokine/culture conditions activated the neural stem gene program and induced generation of cells expressing neural stem/precursor markers (ASTRO-NSC). To evaluate the epigenetic changes associated with this reprogramming, we analyzed the DNA methylation patterns of Astro-NSC relative to untransfected astrocytes. We compared three human AstroNANOG-NSC clones to the astrocytes from which they were derived using NimbleGen 3x720K CpG Island Plus RefSeq Promoter Arrays', '1132/NCBITaxon:9606', '1136/efo:EFO_0002759'
  UNION SELECT 457, 'E-GEOD-36596', 23, 'Functional Comparison of Chronological and In Vitro Aging: Differential Role of the Cytoskeleton and Mitochondria in Mesenchymal Stromal Cells', 'Mesenchymal stromal cells (MSCs) are of high relevance for the regeneration of mesenchymal tissues such as bone and cartilage. The promising role of MSCs in cell-based therapies and tissue engineering appears to be limited due to a decline of their regenerative potential with increasing donor age, their limited availability in human tissues and the need of in vitro expansion prior to treatment. We therefore aimed to determine to which degree in vitro aging and chronological aging may be similar processes or if in vitro culture-related changes at the cellular and molecular level are at least altered as a function of donor age. For that purpose we established MSCs cultures from young (yMSCs) and aged (aMSCs) rats that were cultured for more than 100 passages. These long-term MSCs cultures were non-tumorigenic and exhibited similar surface marker patterns as primary MSCs of passage 2. During in vitro expansion, but not during chronological aging, MSCs progressively lose their progenitor characteristics, e.g., complete loss of osteogenic differentiation potential, diminished adipogenic differentiation, altered cell morphology and increased susceptibility towards senescence. Transcriptome analysis revealed that long-term in vitro MSCs cultivation leads to down-regulation of genes involved in cell differentiation, focal adhesion organization, cytoskeleton turnover and mitochondria function. Accordingly, functional analysis demonstrated altered mitochondrial morphology, decreased antioxidant capacities and elevated ROS levels in long-term cultivated yMSCs as well as aMSCs. Notably, only the MSC migration potential and their antioxidative capacity were altered by in vitro as well as chronological aging. Based on specific differences observed between the impact of chronological and in vitro MSC aging we conclude that both are distinct processes. Total mRNA obtained from MSCs from 12 month and three week old Sprague Dawley rats of passage 2, 30 and 100. Total RNA was isolated using Trizol® (Invitrogen) reagent as describes previously and purified using Qiagen RNeasy® mini kit (Qiagen, Germany, www.qiagen.com) according to manufacturers instruction. Illumina® BeadChip hybridization: Biotinylated cRNA was produced from 500ng total RNA using llumina® TotalPrep RNA amplification kit (Invitrogen). Illumina® RatRef-12 Expression BeadChips hybridization, washing, Cy3 streptavidin staining, and scanning were performed using Illumina® BeadStation 500 platform.', '1132/NCBITaxon:10116', '1136/efo:EFO_0002768'
  UNION SELECT 458, 'E-GEOD-33116', 23, 'Breast cancer and liver tissue biopsies used to develop and validate an index for liver contamination in metastatic breast cancer', 'PURPOSE: To develop an index capable of detecting and quantifying the extent of liver RNA contamination in liver biopsies of metastatic breast cancer METHODS: We developed a microarray-based gene expression liver index by comparing the expression levels of genes in liver tissue biopsies and in primary breast cancer biopsies.  The predictive performance of the index was then evaluated in defined mixtures of liver and breast cancer RNA.  A calibration curve was established to allow estimation of the liver RNA content in unknown metastatic breast cancer samples. A set of liver biopsies and primary breast cancer biopsies were used to identify genes that were expressed significantly higher in liver tissue.  These genes were used in a liver index, which was validated in a new set of liver and primary breast cancer biopsies.  A set of liver/breast cancer RNA mixtures was used to establish a calibration curve to allow quantification of liver contamination in metastatic breast cancer samples.  The prediction error of the calibration curve was assessed in a new set of liver/breast cancer RNA mixtures and applied to estimate the liver content in a set of unknown metastatic breast cancer biopsies.', '1132/NCBITaxon:9606', '1136/efo:EFO_0002768'
  UNION SELECT 459, 'E-MTAB-1435', 23, 'ChIP-seq of CENP-A in S. pombe to investigate localisation of specific nucleosomes in the genome', 'ChIP-seq of CENP-A in S. pombe', '1132/NCBITaxon:4896', '1136/efo:EFO_0002692'
  UNION SELECT 460, 'E-MTAB-1404', 23, 'Transcription profiling by array of mouse pancreatic islets from mice with pancreatic conditional Ring1b homozygous knock-out vs littermate controls', 'To gain insights into how pancreatic cells are programmed in vivo, we profiled RNA expression in pancreatic islets of pancreatic Ring1b conditional KO mice (conditional using a pancreas specfic Cre; Pdx1-Cre) and their littermate controls. Data from an accompanying Ring1b ChIP-on-chip experiment has also been deposited at ArrayExpress: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1402', '1132/NCBITaxon:10090', '1136/efo:EFO_0002768'
  UNION SELECT 461, 'E-MTAB-1402', 23, 'ChIP-chip by tiling array of mouse pancreatic islets and embryonic stem cells to study how pancreatic cells are programmed in vivo', 'To gain insights into how pancreatic cells are programmed in vivo, we profiled Ring1b in embryonic stem cells and pancreatic islets. Data from an accompanying Ring1b knockout transcriptional profiling experiment has also been deposited at ArrayExpress: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1404', '1132/NCBITaxon:10090', '1136/efo:EFO_0002762'
  UNION SELECT 462, 'E-GEOD-43272', 23, 'MCL1 is deregulated in subgroups of diffuse large B-cell lymphoma', 'MCL1 is an anti-apoptotic member of the BCL2 family that is deregulated in various solid and hematological malignancies. However, its role in the molecular pathogenesis of diffuse large B-cell lymphoma (DLBCL) is unclear. We analyzed gene expression profiling data from 350 DLBCL patient samples and detected that activated B-cell-like (ABC) DLBCLs express MCL1 at significantly higher levels compared to germinal center B-cell-like (GCB) DLBCL patient samples (p=2.7 x 10(-10)) [PMID 23257783]. Immunohistochemistry confirmed high MCL1 protein expression predominantly in ABC DLBCL in an independent patient cohort (n=249; p=0.001). To elucidate molecular mechanisms leading to aberrant MCL1 expression, we analyzed array comparative genomic hybridization (aCGH) data of 203 DLBCL samples [GSE11318] and identified recurrent chromosomal gains/amplifications of the MCL1 locus that occurred in 26% of ABC DLBCLs. In addition, aberrant STAT3 signaling contributed to high MCL1 expression in this subtype. Knockdown of MCL1 as well as treatment with the BH3-mimetic obatoclax induced apoptotic cell death in MCL1 positive DLBCL cell lines. In summary, MCL1 is deregulated in a significant fraction of ABC DLBCLs and contributes to therapy resistance. These data suggest that specific inhibition of MCL1 might be utilized therapeutically in a subset of DLBCLs. This GEO dataset is comprised of aCGH measurements for DLBCL cell lines, which are used in the above-mentioned paper. Cell lines were measured against the DNA of a healthy male donor who in turn was measured against a pool of healthy DNAs to correct for individual CNVs of the donor.', '1132/NCBITaxon:9606', '1136/efo:EFO_0000749'
  UNION SELECT 463, 'E-GEOD-43271', 23, 'Comparison of S. boulardii cells in H2O vs. S. boulardii cells in H2O+HCl', 'We sought to better understand the mechanism of acid-induced cell death in Saccharomyces boulardii--a probiotic yeast routinely used to prevent and treat gastrointestinal disorders. To do this we generated microarray gene expression profiles of S. boulardii cells cultured in an acidic environment. Two samples were analyzed. One replicate per array.', '1132/NCBITaxon:252598', '1136/efo:EFO_0002768'
  UNION SELECT 464, 'E-GEOD-43255', 23, 'Susceptibilities of Atlantic salmon to Piscirickettsia salmonis', 'The aquaculture industry has confronted severe economic losses due to infectious diseases in the last years. Piscirickettsiosis or Salmonid Rickettsial Septicaemia (SRS) is the bacterial disease caused by Piscirickettsia salmonis. This Gram-negative, non-motile, cellular pathogen has the ability to infect, survive, replicate, and propagate in salmonid monocytes/macrophages generating a systemic infection characterized by the colonization of several organs including kidney, liver, spleen, intestine, brain, ovary and gills. In this study, we attempted to determine whether global gene expression differences can be detected in different genetic groups of Atlantic salmon as a result of Piscirickettsia salmonis infection. Moreover, we sought to characterize the fish transcriptional response in order to reveal the mechanisms that might confer resistance in Atlantic salmon to an infection with Piscirickettsia salmonis. In doing so, after challenging with Piscirickettsia salmonis, we selected the families with the highest (HS) and the lowest (LS) recorded susceptibility for gene expression analysis using 32K cGRASP microarrays. Our results revealed in LS families expression changes are linked to iron depletion, as well as, low contents of iron in kidney cells and low bacterial load, indicated that the iron-withholding strategy of innate immunity is part of the mechanism of resistance against Piscirickettsia salmonis. This information contributes to elucidate the underlying mechanisms of resistance to Piscirickettsia salmonis infection in Atlantic salmon and to identify new candidate genes for selective breeding programmes. Forty full-sibling families of Atlantic salmon (Salmo salar) were infected by intraperitoneal injection with 0.2 mL Piscirickettsia salmonis (PS889, isolated from Oncorhynchus kisutch, 1×104 PFU/mL). After forty days, the fishes were harvested and the cumulative mortality (dead fish / total fish) for each family was calculated. For the second challenge, the six families with the highest cumulative mortality levels were considered of relatively high susceptibility (HS) and the six families with the lowest cumulative mortality levels were considered of relatively low susceptibility (LS) to the infection. Five control and five infected fish from three HS and three LS families were analyzed. For each HS and LS family, pools of RNA from control and infected fish were prepared separately and were reverse transcribed. Four slides were for each used family hybridized including two dye-swaped slides. Labeled samples were hybridized on a 32K cDNA microarray, developed at the Consortium for Genomics Research on All Salmonids Project (cGRASP), GEO accession number: GPL8904.', '1132/NCBITaxon:8030', '1136/efo:EFO_0002768';
EOS
